<!DOCTYPE html>
<html lang="en" dir="ltr" class="client-nojs">
<head>
<meta charset="UTF-8" />
<title>Gene Disease Database - Wikipedia, the free encyclopedia</title>
<script>document.documentElement.className = document.documentElement.className.replace( /(^|\s)client-nojs(\s|$)/, "$1client-js$2" );</script>
<script>(window.RLQ = window.RLQ || []).push(function () {
mw.config.set({"wgCanonicalNamespace":"","wgCanonicalSpecialPageName":false,"wgNamespaceNumber":0,"wgPageName":"Gene_Disease_Database","wgTitle":"Gene Disease Database","wgCurRevisionId":705536691,"wgRevisionId":705536691,"wgArticleId":44248347,"wgIsArticle":true,"wgIsRedirect":false,"wgAction":"view","wgUserName":null,"wgUserGroups":["*"],"wgCategories":["Use dmy dates from November 2014","Biological databases","Articles which use infobox templates with no data rows","Articles with hCards","Biology","Bioinformatics"],"wgBreakFrames":false,"wgPageContentLanguage":"en","wgPageContentModel":"wikitext","wgSeparatorTransformTable":["",""],"wgDigitTransformTable":["",""],"wgDefaultDateFormat":"dmy","wgMonthNames":["","January","February","March","April","May","June","July","August","September","October","November","December"],"wgMonthNamesShort":["","Jan","Feb","Mar","Apr","May","Jun","Jul","Aug","Sep","Oct","Nov","Dec"],"wgRelevantPageName":"Gene_Disease_Database","wgRelevantArticleId":44248347,"wgIsProbablyEditable":true,"wgRestrictionEdit":[],"wgRestrictionMove":[],"wgWikiEditorEnabledModules":{"toolbar":true,"dialogs":true,"preview":false,"publish":false},"wgBetaFeaturesFeatures":[],"wgMediaViewerOnClick":true,"wgMediaViewerEnabledByDefault":true,"wgVisualEditor":{"pageLanguageCode":"en","pageLanguageDir":"ltr","usePageImages":true,"usePageDescriptions":true},"wikilove-recipient":"","wikilove-anon":0,"wgGatherShouldShowTutorial":true,"wgGatherEnableSample":0,"wgGatherPageImageThumbnail":"//upload.wikimedia.org/wikipedia/en/thumb/3/38/UPlogo1.png/100px-UPlogo1.png","wgRelatedArticles":null,"wgRelatedArticlesUseCirrusSearch":true,"wgRelatedArticlesOnlyUseCirrusSearch":false,"wgULSAcceptLanguageList":["en-us","en"],"wgULSCurrentAutonym":"English","wgFlaggedRevsParams":{"tags":{"status":{"levels":1,"quality":2,"pristine":3}}},"wgStableRevisionId":null,"wgCategoryTreePageCategoryOptions":"{\"mode\":0,\"hideprefix\":20,\"showcount\":true,\"namespaces\":false}","wgNoticeProject":"wikipedia","wgCentralNoticeCategoriesUsingLegacy":["Fundraising","fundraising"],"wgCentralAuthMobileDomain":false,"wgWikibaseItemId":"Q18392279","wgVisualEditorToolbarScrollOffset":0}); /* @nomin */mw.loader.implement("user.options",function($,jQuery){mw.user.options.set({"variant":"en"});});mw.loader.implement("user.tokens",function ( $, jQuery ) {
mw.user.tokens.set({"editToken":"+\\","patrolToken":"+\\","watchToken":"+\\","csrfToken":"+\\"}); /* @nomin */ ;

});mw.loader.load(["mediawiki.page.startup","mediawiki.legacy.wikibits","ext.centralauth.centralautologin","ext.gadget.WatchlistBase","ext.gadget.WatchlistGreenIndicators","mmv.head","ext.visualEditor.desktopArticleTarget.init","ext.uls.init","ext.uls.interface","ext.quicksurveys.init","mw.MediaWikiPlayer.loader","mw.PopUpMediaTransform","ext.centralNotice.bannerController","skins.vector.js"]);
} );</script>
<link rel="stylesheet" href="/w/load.php?debug=false&amp;lang=en&amp;modules=ext.cite.styles%7Cext.gadget.DRN-wizard%2CReferenceTooltips%2CWatchlistBase%2CWatchlistGreenIndicators%2Ccharinsert%2Cfeatured-articles-links%2CrefToolbar%2Cswitcher%2Cteahouse%7Cext.math.desktop.styles%7Cext.math.scripts%2Cstyles%7Cext.tmh.thumbnail.styles%7Cext.uls.nojs%7Cext.visualEditor.desktopArticleTarget.noscript%7Cext.wikimediaBadges%7Cmediawiki.legacy.commonPrint%2Cshared%7Cmediawiki.raggett%2CsectionAnchor%7Cmediawiki.skinning.interface%7Cskins.vector.styles%7Cwikibase.client.init&amp;only=styles&amp;skin=vector" />
<meta name="ResourceLoaderDynamicStyles" content="" />
<link rel="stylesheet" href="/w/load.php?debug=false&amp;lang=en&amp;modules=site&amp;only=styles&amp;skin=vector" />
<script async="" src="/w/load.php?debug=false&amp;lang=en&amp;modules=startup&amp;only=scripts&amp;skin=vector"></script>
<meta name="generator" content="MediaWiki 1.27.0-wmf.13" />
<link rel="alternate" href="android-app://org.wikipedia/http/en.m.wikipedia.org/wiki/Gene_Disease_Database" />
<link rel="alternate" type="application/x-wiki" title="Edit this page" href="/w/index.php?title=Gene_Disease_Database&amp;action=edit" />
<link rel="edit" title="Edit this page" href="/w/index.php?title=Gene_Disease_Database&amp;action=edit" />
<link rel="apple-touch-icon" href="/static/apple-touch/wikipedia.png" />
<link rel="shortcut icon" href="/static/favicon/wikipedia.ico" />
<link rel="search" type="application/opensearchdescription+xml" href="/w/opensearch_desc.php" title="Wikipedia (en)" />
<link rel="EditURI" type="application/rsd+xml" href="//en.wikipedia.org/w/api.php?action=rsd" />
<link rel="copyright" href="//creativecommons.org/licenses/by-sa/3.0/" />
<link rel="canonical" href="https://en.wikipedia.org/wiki/Gene_Disease_Database" />
<link rel="dns-prefetch" href="//meta.wikimedia.org" />
<!--[if lt IE 7]><style type="text/css">body{behavior:url("/static/1.27.0-wmf.13/skins/Vector/csshover.min.htc")}</style><![endif]-->
</head>
<body class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Gene_Disease_Database skin-vector action-view">
		<div id="mw-page-base" class="noprint"></div>
		<div id="mw-head-base" class="noprint"></div>
		<div id="content" class="mw-body" role="main">
			<a id="top"></a>

							<div id="siteNotice"><!-- CentralNotice --></div>
						<div class="mw-indicators">
</div>
			<h1 id="firstHeading" class="firstHeading" lang="en">Gene Disease Database</h1>
									<div id="bodyContent" class="mw-body-content">
									<div id="siteSub">From Wikipedia, the free encyclopedia</div>
								<div id="contentSub"></div>
												<div id="jump-to-nav" class="mw-jump">
					Jump to:					<a href="#mw-head">navigation</a>, 					<a href="#p-search">search</a>
				</div>
				<div id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><div role="note" class="hatnote">For Bioinformatics Databases, see <a href="/wiki/List_of_biological_databases" title="List of biological databases">List of biological databases</a>.</div>
<div role="note" class="hatnote relarticle mainarticle">Main article: <a href="/wiki/Biological_database" title="Biological database">Biological database</a></div>
<table class="infobox" style="width:22em;width:22em">
<caption>Gene Disease Database</caption>
<tr>
<th scope="row" style="width:33%">Classification</th>
<td><a href="/wiki/Bioinformatics" title="Bioinformatics">Bioinformatics</a></td>
</tr>
<tr>
<th scope="row" style="width:33%">Subclassification</th>
<td><a href="/wiki/Databases" title="Databases" class="mw-redirect">Databases</a></td>
</tr>
<tr>
<th scope="row" style="width:33%">Type of Databases</th>
<td><a href="/wiki/List_of_biological_databases" title="List of biological databases">Biological</a></td>
</tr>
<tr>
<th scope="row" style="width:33%">Subtype of Databases</th>
<td>Gene-Disease</td>
</tr>
<tr>
<td colspan="2" style="text-align:right">
<div class="plainlinks hlist navbar mini">
<ul>
<li class="nv-view"><a href="/w/index.php?title=Template:Gene_Disease_Databases&amp;action=edit&amp;redlink=1" class="new" title="Template:Gene Disease Databases (page does not exist)"><abbr title="View this template">v</abbr></a></li>
<li class="nv-talk"><a href="/w/index.php?title=Template_talk:Gene_Disease_Databases&amp;action=edit&amp;redlink=1" class="new" title="Template talk:Gene Disease Databases (page does not exist)"><abbr title="Discuss this template">t</abbr></a></li>
<li class="nv-edit"><a class="external text" href="//en.wikipedia.org/w/index.php?title=Template:Gene_Disease_Databases&amp;action=edit"><abbr title="Edit this template">e</abbr></a></li>
</ul>
</div>
</td>
</tr>
</table>
<p>In <a href="/wiki/Bioinformatics" title="Bioinformatics">Bioinformatics</a>, a <b>Gene Disease Database</b> is a systematized collection of data, typically structured to model aspects of reality, in a way to comprehend the underlying mechanisms of complex diseases, by understanding multiple composite interactions between <a href="/wiki/Phenotype" title="Phenotype">phenotype</a>-<a href="/wiki/Genotype" title="Genotype">genotype</a> relationships and gene-disease mechanisms.<sup id="cite_ref-A._Bauer1_1-0" class="reference"><a href="#cite_note-A._Bauer1-1"><span>[</span>1<span>]</span></a></sup> Gene Disease Databases integrates human gene-disease associations from various expert curated databases and text-mining derived associations including Mendelian, complex and environmental diseases.<sup id="cite_ref-Botstein2_2-0" class="reference"><a href="#cite_note-Botstein2-2"><span>[</span>2<span>]</span></a></sup><sup id="cite_ref-pmid18819940_3-0" class="reference"><a href="#cite_note-pmid18819940-3"><span>[</span>3<span>]</span></a></sup></p>
<p></p>
<div id="toc" class="toc">
<div id="toctitle">
<h2>Contents</h2>
</div>
<ul>
<li class="toclevel-1 tocsection-1"><a href="#Introduction"><span class="tocnumber">1</span> <span class="toctext">Introduction</span></a>
<ul>
<li class="toclevel-2 tocsection-2"><a href="#Challenges_creating_Gene_Disease_Databases"><span class="tocnumber">1.1</span> <span class="toctext">Challenges creating Gene Disease Databases</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-3"><a href="#Types_of_Databases"><span class="tocnumber">2</span> <span class="toctext">Types of Databases</span></a>
<ul>
<li class="toclevel-2 tocsection-4"><a href="#Curated_databases"><span class="tocnumber">2.1</span> <span class="toctext">Curated databases</span></a>
<ul>
<li class="toclevel-3 tocsection-5"><a href="#The_Comparative_Toxicogenomics_Database_.28CTD.29"><span class="tocnumber">2.1.1</span> <span class="toctext">The Comparative Toxicogenomics Database (CTD)</span></a></li>
<li class="toclevel-3 tocsection-6"><a href="#The_Universal_Protein_Resource_.28UNIPROT.29"><span class="tocnumber">2.1.2</span> <span class="toctext">The Universal Protein Resource (UNIPROT)</span></a></li>
</ul>
</li>
<li class="toclevel-2 tocsection-7"><a href="#Predictive_databases"><span class="tocnumber">2.2</span> <span class="toctext">Predictive databases</span></a>
<ul>
<li class="toclevel-3 tocsection-8"><a href="#The_Mouse_genome_Database_.28MGD.29"><span class="tocnumber">2.2.1</span> <span class="toctext">The Mouse genome Database (MGD)</span></a></li>
<li class="toclevel-3 tocsection-9"><a href="#The_Rat_Genome_Database_.28RGD.29"><span class="tocnumber">2.2.2</span> <span class="toctext">The Rat Genome Database (RGD)</span></a></li>
<li class="toclevel-3 tocsection-10"><a href="#The_Online_Mendelian_Inheritance_in_Man_.28OMIM.29"><span class="tocnumber">2.2.3</span> <span class="toctext">The Online Mendelian Inheritance in Man (OMIM)</span></a></li>
<li class="toclevel-3 tocsection-11"><a href="#Ensembl_SIFT_tool"><span class="tocnumber">2.2.4</span> <span class="toctext">Ensembl SIFT tool</span></a></li>
</ul>
</li>
<li class="toclevel-2 tocsection-12"><a href="#Literature_databases"><span class="tocnumber">2.3</span> <span class="toctext">Literature databases</span></a>
<ul>
<li class="toclevel-3 tocsection-13"><a href="#Genetic_Association_Database_.28GAD.29"><span class="tocnumber">2.3.1</span> <span class="toctext">Genetic Association Database (GAD)</span></a></li>
<li class="toclevel-3 tocsection-14"><a href="#The_literature-derived_human_gene-disease_network_.28LHGDN.29"><span class="tocnumber">2.3.2</span> <span class="toctext">The literature-derived human gene-disease network (LHGDN)</span></a></li>
<li class="toclevel-3 tocsection-15"><a href="#BeFree_Data"><span class="tocnumber">2.3.3</span> <span class="toctext">BeFree Data</span></a></li>
</ul>
</li>
<li class="toclevel-2 tocsection-16"><a href="#Integrative_databases"><span class="tocnumber">2.4</span> <span class="toctext">Integrative databases</span></a>
<ul>
<li class="toclevel-3 tocsection-17"><a href="#The_Gene_Disease_Associations_Database_DisGeNET"><span class="tocnumber">2.4.1</span> <span class="toctext">The Gene Disease Associations Database DisGeNET</span></a></li>
</ul>
</li>
</ul>
</li>
<li class="toclevel-1 tocsection-18"><a href="#Some_use_cases_of_Gene-Disease_Databases"><span class="tocnumber">3</span> <span class="toctext">Some use cases of Gene-Disease Databases</span></a></li>
<li class="toclevel-1 tocsection-19"><a href="#Remarks_about_the_future_in_Gene_Disease_Databases"><span class="tocnumber">4</span> <span class="toctext">Remarks about the future in Gene Disease Databases</span></a></li>
<li class="toclevel-1 tocsection-20"><a href="#See_also"><span class="tocnumber">5</span> <span class="toctext">See also</span></a></li>
<li class="toclevel-1 tocsection-21"><a href="#References"><span class="tocnumber">6</span> <span class="toctext">References</span></a></li>
</ul>
</div>
<p></p>
<h2><span class="mw-headline" id="Introduction">Introduction</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Gene_Disease_Database&amp;action=edit&amp;section=1" title="Edit section: Introduction">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>Experts in different areas of biology and bioinformatics have been trying to comprehend the molecular mechanisms of diseases to design preventive and therapeutic strategies for a long time. For some illnesses, it has become apparent that it is not enough to obtain an index of the disease-related genes but to uncover how disruptions of molecular grids in the cell give rise to disease phenotypes.<sup id="cite_ref-3_American_4-0" class="reference"><a href="#cite_note-3_American-4"><span>[</span>4<span>]</span></a></sup> Moreover, with the unprecedented wealth of information available, even obtaining such catalogues is extremely difficult.</p>
<p><a href="/wiki/Heredity" title="Heredity">Genetic</a> illnesses are caused by aberrations in genes or <a href="/wiki/Chromosomes" title="Chromosomes" class="mw-redirect">chromosomes</a>. Many genetic diseases are developed from before birth. Genetic disorders account for a significant number of the health care problems in our society. Advances in the understanding of this diseases have increased both the life span and quality of life for many of those affected by genetic disorders. Recent developments in bioinformatics and laboratory genetics have made possible the better delineation of certain malformation and mental retardation syndromes, so that their mode of inheritance can be understood. This information enables the genetic counselor to predict the risk for occurrence of a large number of genetic disorders.<sup id="cite_ref-Botstein2_2-1" class="reference"><a href="#cite_note-Botstein2-2"><span>[</span>2<span>]</span></a></sup> Most genetic counseling is done, however, only after the birth of at least one affected individual has alerted the family to their predilection for having children with a genetic disorder. The association of a single gene to a disease is rare and a genetic disease may or may not be a transmissible disorder.<sup id="cite_ref-5" class="reference"><a href="#cite_note-5"><span>[</span>5<span>]</span></a></sup> Some genetic diseases are inherited from the parent’s genes, but others are caused by new <a href="/wiki/Mutations" title="Mutations" class="mw-redirect">mutations</a> or changes to the <a href="/wiki/DNA" title="DNA">DNA</a>. In other occurrences, the same disease, for instance, some forms of <a href="/wiki/Carcinoma" title="Carcinoma">carcinoma</a> or <a href="/wiki/Cancer" title="Cancer">melanoma</a>, may stem from an inbred condition in some people, from new changes in other people, and from non-genetic causes in still other individuals.<sup id="cite_ref-6" class="reference"><a href="#cite_note-6"><span>[</span>6<span>]</span></a></sup></p>
<div class="floatleft"><a href="/wiki/File:Geneprioritizationworkflow.jpg" class="image" title="Gene prioritization workflow of human diseases: Typical lists come from linkage regions, chromosomal aberrations, association study loci, deferentially expressed gene lists or genes identified by sequencing variants. Alternatively, the complete genome can be prioritized, but substantially more false positives would then be expected."><img alt="A Gene prioritization" src="//upload.wikimedia.org/wikipedia/commons/e/e0/Geneprioritizationworkflow.jpg" width="650" height="444" data-file-width="650" data-file-height="444" /></a></div>
<p>There are more than six thousand known single-gene disorders (monogenic), which occur in about 1 out of every 200 births.<sup id="cite_ref-A._Bauer1_1-1" class="reference"><a href="#cite_note-A._Bauer1-1"><span>[</span>1<span>]</span></a></sup> As their term suggests, these diseases are caused by a mutation in one gene. By contrast, polygenic disorders are caused by several genes, regularly in combination with environmental factors.<sup id="cite_ref-7" class="reference"><a href="#cite_note-7"><span>[</span>7<span>]</span></a></sup> Examples of genetic phenotypes include <a href="/wiki/Alzheimer%27s_disease" title="Alzheimer's disease">Alzheimer's disease</a>, breast cancer, leukemia, Down syndrome, heart defects, and deafness; therefore, cataloguing to sort out all the diseases related to genes is needed.</p>
<h3><span class="mw-headline" id="Challenges_creating_Gene_Disease_Databases">Challenges creating Gene Disease Databases</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Gene_Disease_Database&amp;action=edit&amp;section=2" title="Edit section: Challenges creating Gene Disease Databases">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>At different stages of any gene disease project, molecular biologists need to choose, even after careful statistical data analysis, which genes or proteins to investigate further experimentally and which to leave out because of limited resources. Computational methods that integrate complex, heterogeneous data sets, such as expression data, sequence information, functional annotation and the biomedical literature, allow prioritizing genes for future study in a more informed way. Such methods can substantially increase the yield of downstream studies and are becoming invaluable to researchers. So one of the main concerns in biological and <a href="/wiki/Biomedical" title="Biomedical" class="mw-redirect">biomedical</a> research is to recognise the underlying mechanisms behind this intricate genetic phenotypes. Great effort has been spent on finding the genes related to diseases<sup id="cite_ref-A._Bauer2_8-0" class="reference"><a href="#cite_note-A._Bauer2-8"><span>[</span>8<span>]</span></a></sup></p>
<p>However, increasingly evidences point out that most human diseases cannot be attributed to a single gene but arise due to complex interactions among multiple genetic variants and environmental risk factors. Several databases have been developed storing associations between genes and diseases such as the Comparative Toxicogenomics Database (CTD), Online Mendelian Inheritance in Man (OMIM), the genetic Association Database (GAD) or the Disease genetic Association Database (DisGeNET). Each of these databases focuses on different aspects of the phenotype-genotype relationship, and due to the nature of the database curation process, they are not complete, but in a way they are fully complementary between each other.<sup id="cite_ref-9" class="reference"><a href="#cite_note-9"><span>[</span>9<span>]</span></a></sup></p>
<h2><span class="mw-headline" id="Types_of_Databases">Types of Databases</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Gene_Disease_Database&amp;action=edit&amp;section=3" title="Edit section: Types of Databases">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>Essentially, there are four types of databases: curated databases, predictive databases, literature databases and integrative databases<sup id="cite_ref-A._Bauer1_1-2" class="reference"><a href="#cite_note-A._Bauer1-1"><span>[</span>1<span>]</span></a></sup></p>
<h3><span class="mw-headline" id="Curated_databases">Curated databases</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Gene_Disease_Database&amp;action=edit&amp;section=4" title="Edit section: Curated databases">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>The term curated data refers to information, that may comprise the most sophisticated computational formats for structured data, scientific updates, and curated knowledge, that has been composed and prepared under the regulation of one or more experts considered to be qualified to engage in such an activity<sup id="cite_ref-Buneman_10-0" class="reference"><a href="#cite_note-Buneman-10"><span>[</span>10<span>]</span></a></sup> The implication is that the resulting database is of high quality. The contrast is with data which may have been gathered through some automated process or using particularly low or inexpert unsupported data quality and possibly untrustworthy.<sup id="cite_ref-Buneman_10-1" class="reference"><a href="#cite_note-Buneman-10"><span>[</span>10<span>]</span></a></sup> Some of the most common examples include: CTD and UNIPROT.</p>
<h4><span class="mw-headline" id="The_Comparative_Toxicogenomics_Database_.28CTD.29">The Comparative Toxicogenomics Database (CTD)</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Gene_Disease_Database&amp;action=edit&amp;section=5" title="Edit section: The Comparative Toxicogenomics Database (CTD)">edit</a><span class="mw-editsection-bracket">]</span></span></h4>
<p>The <a href="/wiki/Comparative_Toxicogenomics_Database" title="Comparative Toxicogenomics Database">Comparative Toxicogenomics Database</a>, helps to understand about the effects of environmental compounds on human health by integrating data from curated scientific literature to describe biochemical interactions with genes and proteins, and links between diseases and chemicals, and diseases and genes or proteins.<sup id="cite_ref-Murphy_11-0" class="reference"><a href="#cite_note-Murphy-11"><span>[</span>11<span>]</span></a></sup> CTD contains curated data defining cross-species chemical–gene/protein interactions and chemical– and gene–disease associations to illuminate molecular mechanisms underlying variable susceptibility and environmentally influenced diseases. These data deliver insights into complex chemical–gene and protein interaction networks. One of the main sources in this Database is curated information from OMIM.<sup id="cite_ref-Murphy_11-1" class="reference"><a href="#cite_note-Murphy-11"><span>[</span>11<span>]</span></a></sup></p>
<p>CTD is a unique resource where bioinformatics specialists read the scientific literature and manually curate four types of core data:</p>
<ul>
<li>Chemical-gene interactions</li>
<li>Chemical-disease associations</li>
<li>Gene-disease associations</li>
<li>Chemical-phenotype associations</li>
</ul>
<h4><span class="mw-headline" id="The_Universal_Protein_Resource_.28UNIPROT.29">The Universal Protein Resource (UNIPROT)</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Gene_Disease_Database&amp;action=edit&amp;section=6" title="Edit section: The Universal Protein Resource (UNIPROT)">edit</a><span class="mw-editsection-bracket">]</span></span></h4>
<p>The Universal Protein Resource (<a href="/wiki/UniProt" title="UniProt">UniProt</a>) is an inclusive resource for protein sequence and annotation data. It is a comprehensive, first-class and freely accessible database of protein sequence and functional information, that has many entries being derived from <a href="/wiki/Genome" title="Genome">genome</a> sequencing projects. It contains a large amount of information about the biological function of proteins derived from the study literature, which can hint to a direct connexion between gene-protein-disease.<sup id="cite_ref-12" class="reference"><a href="#cite_note-12"><span>[</span>12<span>]</span></a></sup></p>
<table class="infobox vevent" style="width:22em;width:300px !important;">
<caption class="summary">UniProt</caption>
<tr>
<th colspan="2" style="text-align:center;font-size:125%;font-weight:bold">
<div class="floatright"><a href="/wiki/File:UPlogo1.png" class="image"><img alt="UPlogo1.png" src="//upload.wikimedia.org/wikipedia/en/3/38/UPlogo1.png" width="320" height="146" data-file-width="320" data-file-height="146" /></a></div>
</th>
</tr>
<tr>
<th colspan="2" style="text-align:center;background-color: #ccf">Content</th>
</tr>
<tr>
<th scope="row" style="white-space: nowrap">Description</th>
<td><b>UniProt</b> is the <i>universal protein</i> resource, a central repository of <a href="/wiki/Protein" title="Protein">protein</a> data created by combining the Swiss-Prot, TrEMBL and PIR-PSD <a href="/wiki/Database" title="Database">databases</a>.</td>
</tr>
<tr>
<th scope="row" style="white-space: nowrap">Data types captured</th>
<td>Protein annotation</td>
</tr>
<tr>
<th scope="row" style="white-space: nowrap"><a href="/wiki/Organism" title="Organism">Organisms</a></th>
<td>All</td>
</tr>
<tr>
<th colspan="2" style="text-align:center;background-color: #ccf">Contact</th>
</tr>
<tr>
<th scope="row" style="white-space: nowrap"><a href="/wiki/Research_center" title="Research center">Research center</a></th>
<td><a href="/wiki/European_Bioinformatics_Institute" title="European Bioinformatics Institute">EMBL-EBI</a>, UK; <a href="/wiki/Swiss_Institute_of_Bioinformatics" title="Swiss Institute of Bioinformatics">SIB</a>, Switzerland; <a href="/wiki/Protein_Information_Resource" title="Protein Information Resource">PIR</a>, US.</td>
</tr>
<tr>
<th scope="row" style="white-space: nowrap">Primary citation</th>
<td>Ongoing and future developments at the Universal Protein Resource<sup id="cite_ref-pmid21051339_13-0" class="reference"><a href="#cite_note-pmid21051339-13"><span>[</span>13<span>]</span></a></sup></td>
</tr>
<tr>
<th colspan="2" style="text-align:center;background-color: #ccf">Access</th>
</tr>
<tr>
<th scope="row" style="white-space: nowrap"><a href="/wiki/File_format" title="File format">Data format</a></th>
<td>Custom flat file, <a href="/wiki/FASTA" title="FASTA">FASTA</a>, <a href="/wiki/GFF" title="GFF" class="mw-redirect">GFF</a>, <a href="/wiki/Resource_Description_Framework" title="Resource Description Framework">RDF</a>, <a href="/wiki/XML" title="XML">XML</a>.</td>
</tr>
<tr>
<th scope="row" style="white-space: nowrap">Website</th>
<td><span class="url"><a rel="nofollow" class="external text" href="http://www.uniprot.org">www<wbr />.uniprot<wbr />.org</a></span></td>
</tr>
<tr>
<th scope="row" style="white-space: nowrap">Download URL</th>
<td><span class="url"><a rel="nofollow" class="external text" href="http://www.uniprot.org/downloads">www<wbr />.uniprot<wbr />.org<wbr />/downloads</a></span> &amp; for downloading complete data sets <span class="url"><a rel="nofollow" class="external text" href="http://ftp.uniprot.org">ftp<wbr />.uniprot<wbr />.org</a></span></td>
</tr>
<tr>
<th scope="row" style="white-space: nowrap"><a href="/wiki/Web_service" title="Web service">Web service</a> URL</th>
<td>Yes – <a href="/wiki/Java_(programming_language)" title="Java (programming language)">JAVA</a> <a href="/wiki/API" title="API" class="mw-redirect">API</a> see info <span class="url"><a rel="nofollow" class="external text" href="http://www.ebi.ac.uk/uniprot/remotingAPI/">here</a></span> &amp; <a href="/wiki/REST" title="REST" class="mw-redirect">REST</a> see info <span class="url"><a rel="nofollow" class="external text" href="http://www.uniprot.org/faq/28">here</a></span></td>
</tr>
<tr>
<th colspan="2" style="text-align:center;background-color: #ccf">Tools</th>
</tr>
<tr>
<th scope="row" style="white-space: nowrap"><a href="/wiki/Web_application" title="Web application">Web</a></th>
<td>Advanced search, <a href="/wiki/BLAST" title="BLAST">BLAST</a>, <a href="/wiki/Clustal" title="Clustal">Clustal</a>O, bulk retrieval/download, ID mapping</td>
</tr>
<tr>
<th colspan="2" style="text-align:center;background-color: #ccf">Miscellaneous</th>
</tr>
<tr>
<th scope="row" style="white-space: nowrap"><a href="/wiki/Software_license" title="Software license">License</a></th>
<td><a href="/wiki/Creative_Commons" title="Creative Commons">Creative Commons</a> Attribution-NoDerivs</td>
</tr>
<tr>
<th scope="row" style="white-space: nowrap"><a href="/wiki/Software_versioning" title="Software versioning">Versioning</a></th>
<td>Yes</td>
</tr>
<tr>
<th scope="row" style="white-space: nowrap">Data release frequency</th>
<td>4 weeks</td>
</tr>
<tr>
<th scope="row" style="white-space: nowrap">Curation policy</th>
<td>Yes – manual and automatic. Rules for automatic annotation generated by database curators and computational algorithms.</td>
</tr>
<tr>
<th scope="row" style="white-space: nowrap">Bookmarkable entities</th>
<td>Yes – both individual protein entries and searches</td>
</tr>
</table>
<table class="infobox biography vcard" style="width:22em">
<tr>
<th colspan="2" style="text-align:center;font-size:125%;font-weight:bold"><span class="fn">The process of database compilation and curation.</span></th>
</tr>
<tr>
<td colspan="2" style="text-align:center">
<div class="center">
<div class="floatnone"><a href="/wiki/File:Databasecompilation1.png" class="image"><img alt="Databasecompilation1.png" src="//upload.wikimedia.org/wikipedia/commons/thumb/1/19/Databasecompilation1.png/600px-Databasecompilation1.png" width="600" height="352" srcset="//upload.wikimedia.org/wikipedia/commons/1/19/Databasecompilation1.png 1.5x, //upload.wikimedia.org/wikipedia/commons/1/19/Databasecompilation1.png 2x" data-file-width="899" data-file-height="527" /></a></div>
</div>
<div>The curated data may comprise a process from practical experience and literature review to web publication of the database<sup id="cite_ref-14" class="reference"><a href="#cite_note-14"><span>[</span>14<span>]</span></a></sup></div>
</td>
</tr>
</table>
<h3><span class="mw-headline" id="Predictive_databases">Predictive databases</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Gene_Disease_Database&amp;action=edit&amp;section=7" title="Edit section: Predictive databases">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>A predictive database is one based on statistical inference. One particular approach to such inference is known as predictive inference, but the prediction can be undertaken within any of the several approaches to statistical inference. Indeed, one description of biostatistics is that it provides a means of transferring knowledge about a sample of a genetic population to the whole population (genomics), and to other related genes or genomes, which the same as prediction over time is not necessarily.<sup id="cite_ref-15" class="reference"><a href="#cite_note-15"><span>[</span>15<span>]</span></a></sup> When information is transferred across time, often to specific points in time, the process is known as forecasting. Three of the main examples of databases that can be considered in this category include: The Mouse genome Database (MGD), The Rat genome Database (RGD), OMIM and the SIFT Tool from Ensembl.<sup id="cite_ref-A._Bauer1_1-3" class="reference"><a href="#cite_note-A._Bauer1-1"><span>[</span>1<span>]</span></a></sup></p>
<h4><span class="mw-headline" id="The_Mouse_genome_Database_.28MGD.29">The Mouse genome Database (MGD)</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Gene_Disease_Database&amp;action=edit&amp;section=8" title="Edit section: The Mouse genome Database (MGD)">edit</a><span class="mw-editsection-bracket">]</span></span></h4>
<p>The Mouse genome Database (MGD) is the international community resource for integrated genetic, genomic and biological data about the laboratory mouse. MGD provides full annotation of phenotypes and human disease associations for mouse models (genotypes) using terms from the Mammalian Phenotype Ontology and disease names from OMIM.<sup id="cite_ref-16" class="reference"><a href="#cite_note-16"><span>[</span>16<span>]</span></a></sup></p>
<h4><span class="mw-headline" id="The_Rat_Genome_Database_.28RGD.29">The Rat Genome Database (RGD)</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Gene_Disease_Database&amp;action=edit&amp;section=9" title="Edit section: The Rat Genome Database (RGD)">edit</a><span class="mw-editsection-bracket">]</span></span></h4>
<table class="infobox vevent" style="width:22em;width:300px !important;">
<caption class="summary">RGD</caption>
<tr>
<th colspan="2" style="text-align:center;font-size:125%;font-weight:bold"><a href="/wiki/File:RGD_Logo_June2015.jpg" class="image" title="The main logo for the Rat Genome Database as of June 2015"><img alt="The main logo for the Rat Genome Database as of June 2015" src="//upload.wikimedia.org/wikipedia/commons/thumb/0/0a/RGD_Logo_June2015.jpg/200px-RGD_Logo_June2015.jpg" width="200" height="83" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/0/0a/RGD_Logo_June2015.jpg/300px-RGD_Logo_June2015.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/0/0a/RGD_Logo_June2015.jpg/400px-RGD_Logo_June2015.jpg 2x" data-file-width="898" data-file-height="372" /></a></th>
</tr>
<tr>
<th colspan="2" style="text-align:center;background-color: #ccf">Content</th>
</tr>
<tr>
<th scope="row" style="white-space: nowrap">Description</th>
<td>The <b>Rat Genome Database</b></td>
</tr>
<tr>
<th scope="row" style="white-space: nowrap"><a href="/wiki/Organism" title="Organism">Organisms</a></th>
<td><i>Rattus norvegicus</i> (rat)</td>
</tr>
<tr>
<th colspan="2" style="text-align:center;background-color: #ccf">Contact</th>
</tr>
<tr>
<th scope="row" style="white-space: nowrap"><a href="/wiki/Research_center" title="Research center">Research center</a></th>
<td><a href="/wiki/Medical_College_of_Wisconsin" title="Medical College of Wisconsin">Medical College of Wisconsin</a></td>
</tr>
<tr>
<th scope="row" style="white-space: nowrap"><a href="/wiki/Laboratory" title="Laboratory">Laboratory</a></th>
<td><a href="/w/index.php?title=Human_Molecular_and_Genetics_Center&amp;action=edit&amp;redlink=1" class="new" title="Human Molecular and Genetics Center (page does not exist)">Human Molecular and Genetics Center</a></td>
</tr>
<tr>
<th scope="row" style="white-space: nowrap"><a href="/wiki/Author" title="Author">Authors</a></th>
<td>Mary E. Shimoyama, PhD; Howard J. Jacob, PhD</td>
</tr>
<tr>
<th scope="row" style="white-space: nowrap">Primary citation</th>
<td><a class="external mw-magiclink-pmid" rel="nofollow" href="//www.ncbi.nlm.nih.gov/pubmed/25355511?dopt=Abstract">PMID 25355511</a></td>
</tr>
<tr>
<th colspan="2" style="text-align:center;background-color: #ccf">Access</th>
</tr>
<tr>
<th scope="row" style="white-space: nowrap">Website</th>
<td><span class="url"><a rel="nofollow" class="external text" href="http://rgd.mcw.edu/">rgd<wbr />.mcw<wbr />.edu</a></span></td>
</tr>
<tr>
<th scope="row" style="white-space: nowrap">Download URL</th>
<td><a rel="nofollow" class="external text" href="ftp://rgd.mcw.edu/pub/data_release/">RGD Data Release</a></td>
</tr>
<tr>
<th colspan="2" style="text-align:center;background-color: #ccf">Tools</th>
</tr>
<tr>
<th colspan="2" style="text-align:center;background-color: #ccf">Miscellaneous</th>
</tr>
</table>
<p>The <a href="/wiki/Rat_Genome_Database" title="Rat Genome Database">Rat Genome Database</a> (RGD) began as a collaborative effort between leading research institutions involved in rat genetic and genomic research. The rat continues to be extensively used by researchers as a model organism for investigating the biology and pathophysiology of disease. In the past several years, there has been a rapid increase in rat genetic and genomic data.<sup id="cite_ref-Dwinell_17-0" class="reference"><a href="#cite_note-Dwinell-17"><span>[</span>17<span>]</span></a></sup> This explosion of information highlighted the need for a centralized database to efficiently and effectively collect, manage, and distribute a rat-centric view of this data to researchers around the world. The Rat Genome Database was created to serve as a repository of rat genetic and genomic data, as well as mapping, strain, and physiological information. It also facilitates investigators research efforts by providing tools to search, mine, and predict this data.<sup id="cite_ref-Dwinell_17-1" class="reference"><a href="#cite_note-Dwinell-17"><span>[</span>17<span>]</span></a></sup></p>
<p>Data at RGD that is useful for researchers investigating disease genes include disease annotations for rat, mouse and human genes. Annotations are manually curated from the literature, or downloaded via automated pipelines from other disease-related databases. Downloaded annotations are mapped to the same disease vocabulary used for manual annotations to provide consistency across the dataset. RGD also maintains disease-related quantitative phenotype data for the rat (PhenoMiner).<sup id="cite_ref-pmid25355511_18-0" class="reference"><a href="#cite_note-pmid25355511-18"><span>[</span>18<span>]</span></a></sup></p>
<h4><span class="mw-headline" id="The_Online_Mendelian_Inheritance_in_Man_.28OMIM.29">The Online Mendelian Inheritance in Man (<a href="/wiki/OMIM" title="OMIM" class="mw-redirect">OMIM</a>)</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Gene_Disease_Database&amp;action=edit&amp;section=10" title="Edit section: The Online Mendelian Inheritance in Man (OMIM)">edit</a><span class="mw-editsection-bracket">]</span></span></h4>
<table class="infobox vevent" style="width:22em;width:300px !important;">
<caption class="summary">The Online Mendelian Inheritance in Man</caption>
<tr>
<th colspan="2" style="text-align:center;background-color: #ccf">Content</th>
</tr>
<tr>
<th scope="row" style="white-space: nowrap">Description</th>
<td>OMIM is a compendium of human genes and genetic phenotypes.</td>
</tr>
<tr>
<th scope="row" style="white-space: nowrap"><a href="/wiki/Organism" title="Organism">Organisms</a></th>
<td>Human (<i>H. Sapiens</i>)</td>
</tr>
<tr>
<th colspan="2" style="text-align:center;background-color: #ccf">Contact</th>
</tr>
<tr>
<th scope="row" style="white-space: nowrap"><a href="/wiki/Research_center" title="Research center">Research center</a></th>
<td><a href="/wiki/National_Center_for_Biotechnology_Information" title="National Center for Biotechnology Information">NCBI</a></td>
</tr>
<tr>
<th scope="row" style="white-space: nowrap">Primary citation</th>
<td><a class="external mw-magiclink-pmid" rel="nofollow" href="//www.ncbi.nlm.nih.gov/pubmed/25398906?dopt=Abstract">PMID 25398906</a></td>
</tr>
<tr>
<th colspan="2" style="text-align:center;background-color: #ccf">Access</th>
</tr>
<tr>
<th scope="row" style="white-space: nowrap">Website</th>
<td><span class="url"><a rel="nofollow" class="external text" href="http://www.ncbi.nlm.nih.gov/omim">www<wbr />.ncbi<wbr />.nlm<wbr />.nih<wbr />.gov<wbr />/omim</a></span></td>
</tr>
<tr>
<th colspan="2" style="text-align:center;background-color: #ccf">Tools</th>
</tr>
<tr>
<th colspan="2" style="text-align:center;background-color: #ccf">Miscellaneous</th>
</tr>
</table>
<p>Supported by the NCBI, The Online Mendelian Inheritance in Man (OMIM) is a database that catalogues all the known diseases with a genetic component, and predicts their relationship to relevant genes in the human genome and provides references for further research and tools for genomic analysis of a catalogued gene.<sup id="cite_ref-Homosh_19-0" class="reference"><a href="#cite_note-Homosh-19"><span>[</span>19<span>]</span></a></sup> OMIM is a comprehensive, authoritative compendium of human genes and genetic phenotypes that is freely available and updated daily. The database has been used as a resource for predicting relevant information to inherited conditions.<sup id="cite_ref-Homosh_19-1" class="reference"><a href="#cite_note-Homosh-19"><span>[</span>19<span>]</span></a></sup></p>
<div class="thumb tleft">
<div class="thumbinner" style="width:222px;"><a href="/wiki/File:Pathwayhomogeneity1.png" class="image"><img alt="A Pathway Hogeneity" src="//upload.wikimedia.org/wikipedia/commons/thumb/1/13/Pathwayhomogeneity1.png/220px-Pathwayhomogeneity1.png" width="220" height="159" class="thumbimage" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/1/13/Pathwayhomogeneity1.png/330px-Pathwayhomogeneity1.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/1/13/Pathwayhomogeneity1.png/440px-Pathwayhomogeneity1.png 2x" data-file-width="1345" data-file-height="975" /></a>
<div class="thumbcaption">
<div class="magnify"><a href="/wiki/File:Pathwayhomogeneity1.png" class="internal" title="Enlarge"></a></div>
<b>Pathway Hogeneity vs Associated Genes</b> Showing the concept that diseases have large association with a variety of genes, a mean pathway homogeneity values of single diseases and random controls are plotted for four networks binned by the number of associated gene products per disease. <i>This graph shows how difficult is to correlate a bigger number of diseases vs concordance in 4 different databases, hence Gene Disease Databases test these relationships</i></div>
</div>
</div>
<h4><span class="mw-headline" id="Ensembl_SIFT_tool"><a href="/wiki/Ensembl" title="Ensembl">Ensembl</a> SIFT tool</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Gene_Disease_Database&amp;action=edit&amp;section=11" title="Edit section: Ensembl SIFT tool">edit</a><span class="mw-editsection-bracket">]</span></span></h4>
<table class="infobox vevent" style="width:22em;width:300px !important;">
<caption class="summary">The Ensembl genome database project.</caption>
<tr>
<th colspan="2" style="text-align:center;font-size:125%;font-weight:bold"><a href="/wiki/File:Ensembl_release58_sgcb_screenshot.png" class="image"><img alt="Ensembl release58 sgcb screenshot.png" src="//upload.wikimedia.org/wikipedia/en/thumb/e/ef/Ensembl_release58_sgcb_screenshot.png/300px-Ensembl_release58_sgcb_screenshot.png" width="300" height="156" srcset="//upload.wikimedia.org/wikipedia/en/thumb/e/ef/Ensembl_release58_sgcb_screenshot.png/450px-Ensembl_release58_sgcb_screenshot.png 1.5x, //upload.wikimedia.org/wikipedia/en/thumb/e/ef/Ensembl_release58_sgcb_screenshot.png/600px-Ensembl_release58_sgcb_screenshot.png 2x" data-file-width="844" data-file-height="439" /></a></th>
</tr>
<tr>
<th colspan="2" style="text-align:center;background-color: #ccf">Content</th>
</tr>
<tr>
<th scope="row" style="white-space: nowrap">Description</th>
<td><b>Ensembl</b></td>
</tr>
<tr>
<th colspan="2" style="text-align:center;background-color: #ccf">Contact</th>
</tr>
<tr>
<th scope="row" style="white-space: nowrap"><a href="/wiki/Research_center" title="Research center">Research center</a></th>
<td>
<div class="plainlist">
<ul>
<li><a href="/wiki/Wellcome_Trust_Sanger_Institute" title="Wellcome Trust Sanger Institute">Wellcome Trust Sanger Institute</a></li>
<li><a href="/wiki/European_Bioinformatics_Institute" title="European Bioinformatics Institute">European Bioinformatics Institute</a></li>
</ul>
</div>
</td>
</tr>
<tr>
<th scope="row" style="white-space: nowrap">Primary citation</th>
<td>Hubbard, <i>et al.</i> (2002)<sup id="cite_ref-Hubbard2002_20-0" class="reference"><a href="#cite_note-Hubbard2002-20"><span>[</span>20<span>]</span></a></sup></td>
</tr>
<tr>
<th colspan="2" style="text-align:center;background-color: #ccf">Access</th>
</tr>
<tr>
<th scope="row" style="white-space: nowrap">Website</th>
<td><span class="url"><a rel="nofollow" class="external text" href="http://www.ensembl.org/">www<wbr />.ensembl<wbr />.org</a></span></td>
</tr>
<tr>
<th colspan="2" style="text-align:center;background-color: #ccf">Tools</th>
</tr>
<tr>
<th colspan="2" style="text-align:center;background-color: #ccf">Miscellaneous</th>
</tr>
</table>
<p>This one of the largest resources available for all genomic and genetic studies, it provides a centralized resource for geneticists, molecular biologists and other researchers studying the genomes of our own species and other vertebrates and model disease organisms. Ensembl is one of several well-known genome browsers for the retrieval of genomic-disease information. Ensembl imports variation data from a variety of different sources, Ensembl predicts the effects of variants.<sup id="cite_ref-Flicek_21-0" class="reference"><a href="#cite_note-Flicek-21"><span>[</span>21<span>]</span></a></sup> For each variation that is mapped to the reference genome, each Ensembl transcript is identified that overlap the variation. Then it uses a rule-based approach to predict the effects that each allele of the variation may have on the transcript. The set of consequence terms, defined by the Sequence Ontology (SO) can be currently assigned to each combination of an allele and a transcript. Each allele of each variation may have a different effect in different transcripts. A variety of different tools are used to predict human mutations in the Ensembl database, one of the most widely used is SIFT, that predicts whether an amino acid substitution is likely to affect protein function based on sequence homology and the physic-chemical similarity between the alternate amino acids. The data provided for each amino acid substitution is a score and a qualitative prediction (either 'tolerated' or 'deleterious'). The score is the normalized probability that the amino acid change is tolerated so scores near 0 are more likely to be deleterious. The qualitative prediction is derived from this score such that substitutions with a score &lt; 0.05 are called 'deleterious' and all others are called 'tolerated'.SIFT can be applied to naturally occurring nonsynonymous polymorphisms and laboratory-induced missense mutations, that will lead to build relationships in phenotype characteristics, proteomics and genomics<sup id="cite_ref-Flicek_21-1" class="reference"><a href="#cite_note-Flicek-21"><span>[</span>21<span>]</span></a></sup></p>
<h3><span class="mw-headline" id="Literature_databases">Literature databases</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Gene_Disease_Database&amp;action=edit&amp;section=12" title="Edit section: Literature databases">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>This sort of databases summarize books, articles, book reviews, dissertations, and annotations about gene-disease databases. Some of the following are examples of this type: GAD, LGHDN and BeFree Data.</p>
<h4><span class="mw-headline" id="Genetic_Association_Database_.28GAD.29">Genetic Association Database (GAD)</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Gene_Disease_Database&amp;action=edit&amp;section=13" title="Edit section: Genetic Association Database (GAD)">edit</a><span class="mw-editsection-bracket">]</span></span></h4>
<p>The Genetic Association Database is an archive of human genetic association studies of complex diseases. GAD is primarily focused on archiving information on common complex human disease rather than rare Mendelian disorders as found in the OMIM. It includes curated summary data extracted from published papers in peer reviewed journals on candidate gene and genome Wide Association Studies (<a href="/wiki/Genome-wide_association_study" title="Genome-wide association study">GWAS</a>).<sup id="cite_ref-22" class="reference"><a href="#cite_note-22"><span>[</span>22<span>]</span></a></sup></p>
<h4><span class="mw-headline" id="The_literature-derived_human_gene-disease_network_.28LHGDN.29">The literature-derived human gene-disease network (LHGDN)</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Gene_Disease_Database&amp;action=edit&amp;section=14" title="Edit section: The literature-derived human gene-disease network (LHGDN)">edit</a><span class="mw-editsection-bracket">]</span></span></h4>
<p>The literature-derived human gene-disease network (LHGDN) is a text mining derived database with focus on extracting and classifying gene-disease associations with respect to several biomolecular conditions. It uses a machine learning based algorithm to extract semantic gene-disease relations from a textual source of interest. It is part of the Linked Life Data, of the LMU in Munchen, Germany.<sup id="cite_ref-A._Bauer1_1-4" class="reference"><a href="#cite_note-A._Bauer1-1"><span>[</span>1<span>]</span></a></sup></p>
<h4><span class="mw-headline" id="BeFree_Data">BeFree Data</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Gene_Disease_Database&amp;action=edit&amp;section=15" title="Edit section: BeFree Data">edit</a><span class="mw-editsection-bracket">]</span></span></h4>
<p>Extracts gene-disease associations from MEDLINE abstract using the BeFree system. BeFree is composed of a biomedical Named Entity Recognition (BioNER) module to detect diseases and genes and a relation extraction module based on morphosyntactic information.<sup id="cite_ref-23" class="reference"><a href="#cite_note-23"><span>[</span>23<span>]</span></a></sup></p>
<h3><span class="mw-headline" id="Integrative_databases">Integrative databases</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Gene_Disease_Database&amp;action=edit&amp;section=16" title="Edit section: Integrative databases">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>This sort of databases include Mendelian, compound and environmental diseases in an integrated gene-disease association archive and show that the concept of modularity applies for all of them They provide a functional analysis of diseases in case of important new biological insights, which might not be discovered when considering each of the gene-disease associations independently. Hence, they present a suitable framework for the study of how genetic and environmental factors, such as drugs, contribute to diseases. The best example for this sort of database is DisGeNET.<sup id="cite_ref-A._Bauer2_8-1" class="reference"><a href="#cite_note-A._Bauer2-8"><span>[</span>8<span>]</span></a></sup><sup id="cite_ref-Pinero_24-0" class="reference"><a href="#cite_note-Pinero-24"><span>[</span>24<span>]</span></a></sup></p>
<h4><span class="mw-headline" id="The_Gene_Disease_Associations_Database_DisGeNET">The Gene Disease Associations Database <a rel="nofollow" class="external text" href="http://www.disgenet.org">DisGeNET</a></span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Gene_Disease_Database&amp;action=edit&amp;section=17" title="Edit section: The Gene Disease Associations Database DisGeNET">edit</a><span class="mw-editsection-bracket">]</span></span></h4>
<table class="infobox vevent" style="width:22em;width:300px !important;">
<caption class="summary">DisGeNET</caption>
<tr>
<th colspan="2" style="text-align:center;background-color: #ccf">Content</th>
</tr>
<tr>
<th scope="row" style="white-space: nowrap">Description</th>
<td>Integrates human gene-disease associations</td>
</tr>
<tr>
<th scope="row" style="white-space: nowrap">Data types captured</th>
<td>Associations Database</td>
</tr>
<tr>
<th scope="row" style="white-space: nowrap"><a href="/wiki/Organism" title="Organism">Organisms</a></th>
<td>Human (<i>H. Sapiens</i>)</td>
</tr>
<tr>
<th colspan="2" style="text-align:center;background-color: #ccf">Contact</th>
</tr>
<tr>
<th scope="row" style="white-space: nowrap"><a href="/wiki/Research_center" title="Research center">Research center</a></th>
<td>Research Programme on Biomedical Informatics (GRIB) IMIM-UPF</td>
</tr>
<tr>
<th scope="row" style="white-space: nowrap"><a href="/wiki/Laboratory" title="Laboratory">Laboratory</a></th>
<td>Integrative Biomedical Informatics Group</td>
</tr>
<tr>
<th scope="row" style="white-space: nowrap"><a href="/wiki/Author" title="Author">Authors</a></th>
<td>Ferran Sanz and Laura I. Furlong (Pinero et al, 2015)</td>
</tr>
<tr>
<th scope="row" style="white-space: nowrap">Primary citation</th>
<td><a class="external mw-magiclink-pmid" rel="nofollow" href="//www.ncbi.nlm.nih.gov/pubmed/25877637?dopt=Abstract">PMID 25877637</a></td>
</tr>
<tr>
<th colspan="2" style="text-align:center;background-color: #ccf">Access</th>
</tr>
<tr>
<th scope="row" style="white-space: nowrap">Website</th>
<td><span class="url"><a rel="nofollow" class="external text" href="http://www.disgenet.org/">www<wbr />.disgenet<wbr />.org</a></span></td>
</tr>
<tr>
<th colspan="2" style="text-align:center;background-color: #ccf">Tools</th>
</tr>
<tr>
<th colspan="2" style="text-align:center;background-color: #ccf">Miscellaneous</th>
</tr>
<tr>
<th scope="row" style="white-space: nowrap">Data release frequency</th>
<td>annual</td>
</tr>
<tr>
<th scope="row" style="white-space: nowrap">Version</th>
<td>3</td>
</tr>
</table>
<table class="infobox biography vcard" style="width:22em">
<tr>
<th colspan="2" style="text-align:center;font-size:125%;font-weight:bold"><span class="fn">DisGeNET gene-disease association ontology</span></th>
</tr>
<tr>
<td colspan="2" style="text-align:center">
<div class="floatleft"><a href="/wiki/File:Disgenet.png" class="image"><img alt="Disgenet.png" src="//upload.wikimedia.org/wikipedia/commons/thumb/b/ba/Disgenet.png/500px-Disgenet.png" width="500" height="329" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/b/ba/Disgenet.png/750px-Disgenet.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/b/ba/Disgenet.png/1000px-Disgenet.png 2x" data-file-width="1352" data-file-height="889" /></a></div>
<div>The description of each association type in this ontology is:
<ol>
<li>Therapeutic Association: The gene/protein has a therapeutic role in the amelioration of the disease.</li>
<li>Biomarker Association: The gene/protein either plays a role in the etiology of the disease (e.g. participates in the molecular mechanism that leads to disease) or is a biomarker for a disease.</li>
<li>Genetic Variation Association: Used when a sequence variation (a mutation, a SNP) is associated to the disease phenotype, but there is still no evidence to say that the variation causes the disease. In some cases the presence of the variants increase the susceptibility to the disease. In general, the NCBI SNP identifiers are provided.</li>
<li>Altered Expression Association: Alterations in the function of the protein by means of altered expression of the gene are associated with the disease phenotype.</li>
<li>Post-translational Modification Association: Alterations in the function of the protein by means of post-translational modifications (methylation or phosphorylation of the protein) are associated with the disease phenotype.</li>
</ol>
<sup id="cite_ref-A._Bauer1_1-5" class="reference"><a href="#cite_note-A._Bauer1-1"><span>[</span>1<span>]</span></a></sup></div>
</td>
</tr>
</table>
<p><a href="/wiki/DisGeNET" title="DisGeNET">DisGeNET</a> is a comprehensive gene-disease association database that integrates associations from several sources that covers different biomedical aspects of diseases.<sup id="cite_ref-Pinero_24-1" class="reference"><a href="#cite_note-Pinero-24"><span>[</span>24<span>]</span></a></sup> In particular, it is focused on the current knowledge of human genetic diseases including Mendelian, complex and environmental diseases. To assess the concept of modularity of human diseases, this database performs a systematic study of the emergent properties of human gene-disease networks by means of network topology and functional annotation analysis.<sup id="cite_ref-A._Bauer1_1-6" class="reference"><a href="#cite_note-A._Bauer1-1"><span>[</span>1<span>]</span></a></sup> The results indicate a highly shared genetic origin of human diseases and show that for most diseases, including Mendelian, complex and environmental diseases, functional modules exist. Moreover, a core set of biological pathways is found to be associated with most human diseases. Obtaining similar results when studying <a href="/wiki/Disease_cluster" title="Disease cluster">clusters</a> of diseases, the findings in this database suggest that related diseases might arise due to dysfunction of common biological processes in the cell. The network analysis of this integrated database points out that data integration is needed to obtain a comprehensive view of the genetic landscape of human diseases and that the genetic origin of complex diseases is much more common than expected.<sup id="cite_ref-A._Bauer1_1-7" class="reference"><a href="#cite_note-A._Bauer1-1"><span>[</span>1<span>]</span></a></sup></p>
<h2><span class="mw-headline" id="Some_use_cases_of_Gene-Disease_Databases">Some use cases of Gene-Disease Databases</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Gene_Disease_Database&amp;action=edit&amp;section=18" title="Edit section: Some use cases of Gene-Disease Databases">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>Some of the most interesting cases using Gene-Disease Databases can be found in the following papers:<sup id="cite_ref-A._Bauer1_1-8" class="reference"><a href="#cite_note-A._Bauer1-1"><span>[</span>1<span>]</span></a></sup><sup id="cite_ref-A._Bauer2_8-2" class="reference"><a href="#cite_note-A._Bauer2-8"><span>[</span>8<span>]</span></a></sup></p>
<ul>
<li>A network approach to clinical intervention in neurodegenerative diseases. Santiago, JA, Potashkin JA. Trends in Molecular Medicine (2014) <a rel="nofollow" class="external text" href="http://10.1016/j.molmed.2014.10.002">10.1016/j.molmed.2014.10.002</a></li>
<li>Control of VEGF-A transcriptional programs by pausing and genomic compartmentalization. Kaikkonen MU, Niskanen H, Romanoski CE, Kansanen E, Kivelä AM, Laitalainen J, Heinz S, Benner C, Glass CK, Ylä-Herttuala S. Nucleic Acids Res. (2014) <a rel="nofollow" class="external text" href="http://10.1093/nar/gku1036">10.1093/nar/gku1036</a></li>
<li>Network medicine analysis of COPD multimorbidities. Grosdidier S, Ferrer A, Faner R, Piñero J, Roca J, Cosío B, Agustí A, Gea J, Sanz F, Furlong LI. Respir Res. (2014) <a rel="nofollow" class="external text" href="http://10.1186/s12931-014-0111-4">10.1186/s12931-014-0111-4</a></li>
<li>An R-based tool for miRNA data analysis and correlation with clinical ontologies. Cristiano F, Veltri P. Proceedings of the 5th ACM Conference on Bioinformatics, Computational Biology, and Health Informatics (2014) <a rel="nofollow" class="external text" href="http://10.1145/2649387.2660847">10.1145/2649387.2660847</a></li>
<li>Using 2-node hypergraph clustering coefficients to analyze disease-gene networks. Renick Gallagher S, Dombrower M, Goldberg DS. Proceedings of the 5th ACM Conference on Bioinformatics, Computational Biology, and Health Informatics (2014) <a rel="nofollow" class="external text" href="http://10.1145/2649387.2660817">10.1145/2649387.2660817</a></li>
<li>Organ system heterogeneity DB: a database for the visualization of phenotypes at the organ system level. Mannil D, Vogt I, Prinz J, Campillos M. Nucleic Acids Res. (2014) <a rel="nofollow" class="external text" href="http://10.1093/nar/gku948">10.1093/nar/gku948</a></li>
<li>Molecularly and clinically related drugs and diseases are enriched in phenotypically similar drug-disease pairs. Vogt I, Prinz J, Campillos M. Genome Med. (2014) <a rel="nofollow" class="external text" href="http://10.1186/s13073-014-0052-z">10.1186/s13073-014-0052-z</a></li>
<li>System-based approaches to decode the molecular links in Parkinson's disease and diabetes. Santiago, JA, Potashkin JA. Neurobiol Dis. (2014) <a rel="nofollow" class="external text" href="http://10.1016/j.nbd.2014.03.019">10.1016/j.nbd.2014.03.019</a></li>
<li>Prioritizing Disease‐Linked Variants, Genes, and Pathways with an Interactive whole Genome Analysis Pipeline. Lee IH, Lee K, Hsing M, Choe Y, Park JH, Kim SH, Bohn JM, Neu MB, Hwang KB, Green RC, Kohane IS, Kong SW. Hum Mutat. (2014) <a rel="nofollow" class="external text" href="http://10.1002/humu.22520">10.1002/humu.22520</a></li>
<li>A Computational Framework to Infer Human Disease-Associated Long Noncoding RNAs. Liu MX, Chen X, Chen G, Cui QH, Yan GY. PloS One 9.1 (2014) <a rel="nofollow" class="external text" href="http://10.1371/journal.pone.0084408">10.1371/journal.pone.0084408</a></li>
<li>Choline protects against cardiac hypertrophy induced by increased after-load. Zhao Y, Wang C, Wu J, Wang Y, Zhu W, Zhang Y, Du Z. Int J Biol Sci. (2013) <a rel="nofollow" class="external text" href="http://10.7150/ijbs.5976">10.7150/ijbs.5976</a></li>
<li>Detection of differentially methylated gene promoters in failing and nonfailing human left ventricle myocardium using computation analysis. Koczor CA, Lee EK, Torres RA, Boyd A, Vega JD, Uppal K, Yuan F, Fields EJ, Samarel AM, Lewis W. Physiol Genomics. (2013) <a rel="nofollow" class="external autonumber" href="http://10.1152/physiolgenomics.00013.2013">[1]</a></li>
<li>Global DNA methylation and transcriptional analyses of human ESC-derived cardiomyocytes. Gu Y, Liu GH, Plongthongkum N, Benner C, Yi F, Qu J, Suzuki K, Yang J, Zhang W, Li M, Montserrat N, Crespo I, Del Sol A, Esteban CR, Zhang K, Belmonte JC. Protein Cell. (2013) <a rel="nofollow" class="external autonumber" href="http://10.1007/s13238-013-0016-x">[2]</a></li>
<li>Integrated analysis of transcript-level regulation of metabolism reveals disease-relevant nodes of the human metabolic network. Galhardo M1, Sinkkonen L, Berninger P, Lin J, Sauter T, Heinäniemi M. Nucleic Acids Res. (2013) doi: <a rel="nofollow" class="external autonumber" href="http://10.1093/nar/gkt989">[3]</a></li>
<li>Charting the NF-κB Pathway Interactome Map. Tieri P1, Termanini A, Bellavista E, Salvioli S, Capri M, Franceschi C. PLoS One (2012) doi: <a rel="nofollow" class="external autonumber" href="http://10.1371/journal.pone.0032678">[4]</a></li>
</ul>
<h2><span class="mw-headline" id="Remarks_about_the_future_in_Gene_Disease_Databases">Remarks about the future in Gene Disease Databases</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Gene_Disease_Database&amp;action=edit&amp;section=19" title="Edit section: Remarks about the future in Gene Disease Databases">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div class="thumb tright">
<div class="thumbinner" style="width:222px;"><a href="/wiki/File:Relationships_in_Gene_Diseases.jpg" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/thumb/1/10/Relationships_in_Gene_Diseases.jpg/220px-Relationships_in_Gene_Diseases.jpg" width="220" height="213" class="thumbimage" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/1/10/Relationships_in_Gene_Diseases.jpg/330px-Relationships_in_Gene_Diseases.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/1/10/Relationships_in_Gene_Diseases.jpg/440px-Relationships_in_Gene_Diseases.jpg 2x" data-file-width="800" data-file-height="775" /></a>
<div class="thumbcaption">
<div class="magnify"><a href="/wiki/File:Relationships_in_Gene_Diseases.jpg" class="internal" title="Enlarge"></a></div>
<b>Relationships in Gene Diseases</b></div>
</div>
</div>
<p>The completion of the human genome has changed the way the search for disease genes is performed. In the past, the approach was to focus on one or a few genes at a time. Now, projects like the DisGeNET exemplify the efforts to systematically analyze all the gene alterations involved in a single or multiple diseases.<sup id="cite_ref-25" class="reference"><a href="#cite_note-25"><span>[</span>25<span>]</span></a></sup> The next step is to produce a complete picture of the mechanistic aspects of the diseases and the design of drugs against them. For that, a combination of two approaches will be needed: a systematic search and in-depth study of each gene. The future of the field will be defined by new techniques to integrate large bodies of data from different sources and to incorporate functional information into the analysis of large-scale data generated by bioinformatics studies.<sup id="cite_ref-A._Bauer1_1-9" class="reference"><a href="#cite_note-A._Bauer1-1"><span>[</span>1<span>]</span></a></sup></p>
<p>Bioinformatics is both a term for the body of biological gene disease studies that use computer programming as part of their methodology, as well as a reference to specific analysis pipelines that are repeatedly used, particularly in the fields of genetics and genomics.<sup id="cite_ref-A._Bauer1_1-10" class="reference"><a href="#cite_note-A._Bauer1-1"><span>[</span>1<span>]</span></a></sup> Common uses of bioinformatics include the identification of candidate genes and nucleotides, <a href="/wiki/SNPs" title="SNPs" class="mw-redirect">SNPs</a>. Often, such identification is made with the aim of better understanding the genetic basis of disease, unique adaptations, desirable properties, or differences between populations. In a less formal way, bioinformatics also tries to understand the organisational principles within nucleic acid and protein sequences.<sup id="cite_ref-A._Bauer1_1-11" class="reference"><a href="#cite_note-A._Bauer1-1"><span>[</span>1<span>]</span></a></sup></p>
<p>The response of <a href="/wiki/Bioinformatics" title="Bioinformatics">bioinformatics</a> to new experimental techniques brings a new perspective into the analysis of the experimental data, as demonstrated by the advances in the analysis of information from gene disease databases and other technologies. It is expected that this trend will continue with novel approaches to respond to new techniques, such as next-generation sequencing technologies. For instance, the availability of large numbers of individual human genomes will promote the development of computational analyses of rare variants, including the statistical mining of their relations to lifestyles, drug interactions and other factors.<sup id="cite_ref-A._Bauer1_1-12" class="reference"><a href="#cite_note-A._Bauer1-1"><span>[</span>1<span>]</span></a></sup> Biomedical research will also be driven by our ability to efficiently mine the large body of existing and continuously generated biomedical data. Text-mining techniques, in particular, when combined with other molecular data, can provide information about gene mutations and interactions and will become crucial to stay ahead of the exponential growth of data generated in biomedical research. Another field that is benefiting from the advances in mining and integration of molecular, clinical and drug analysis is pharmacogenomics. <i>In silico</i> studies of the relationships between human variations and their effect on diseases will be key to the development of personalized medicine.<sup id="cite_ref-A._Bauer2_8-3" class="reference"><a href="#cite_note-A._Bauer2-8"><span>[</span>8<span>]</span></a></sup> In summary, Gene Disease Databases have already transformed the search for disease genes and has the potential to become a crucial component of other areas of medical research.<sup id="cite_ref-A._Bauer1_1-13" class="reference"><a href="#cite_note-A._Bauer1-1"><span>[</span>1<span>]</span></a></sup></p>
<h2><span class="mw-headline" id="See_also">See also</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Gene_Disease_Database&amp;action=edit&amp;section=20" title="Edit section: See also">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div class="noprint portal tright" style="border:solid #aaa 1px;margin:0.5em 0 0.5em 1em">
<table style="background:#f9f9f9;font-size:85%;line-height:110%;max-width:175px">
<tr style="vertical-align:middle">
<td style="text-align:center"><a href="/wiki/File:Issoria_lathonia.jpg" class="image"><img alt="Portal icon" src="//upload.wikimedia.org/wikipedia/commons/thumb/2/2d/Issoria_lathonia.jpg/32px-Issoria_lathonia.jpg" width="32" height="23" class="noviewer" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/2/2d/Issoria_lathonia.jpg/48px-Issoria_lathonia.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/2/2d/Issoria_lathonia.jpg/64px-Issoria_lathonia.jpg 2x" data-file-width="629" data-file-height="445" /></a></td>
<td style="padding:0 0.2em;vertical-align:middle;font-style:italic;font-weight:bold"><a href="/wiki/Portal:Biology" title="Portal:Biology">Biology portal</a></td>
</tr>
</table>
</div>
<div class="div-col columns column-count column-count-3" style="-moz-column-count: 3; -webkit-column-count: 3; column-count: 3;">
<ul>
<li><a href="/wiki/Biodiversity_informatics" title="Biodiversity informatics">Biodiversity informatics</a></li>
<li><a href="/wiki/Bioinformatics_companies" title="Bioinformatics companies" class="mw-redirect">Bioinformatics companies</a></li>
<li><a href="/wiki/Computational_biology" title="Computational biology">Computational biology</a></li>
<li><a href="/wiki/Computational_biomodeling" title="Computational biomodeling" class="mw-redirect">Computational biomodeling</a></li>
<li><a href="/wiki/Computational_genomics" title="Computational genomics">Computational genomics</a></li>
<li><a href="/wiki/Functional_genomics" title="Functional genomics">Functional genomics</a></li>
<li><a href="/wiki/Health_informatics" title="Health informatics">Health informatics</a></li>
<li><a href="/wiki/International_Society_for_Computational_Biology" title="International Society for Computational Biology">International Society for Computational Biology</a></li>
<li><a href="/wiki/Jumping_library" title="Jumping library">Jumping library</a></li>
<li><a href="/wiki/List_of_open-source_bioinformatics_software" title="List of open-source bioinformatics software">List of open-source bioinformatics software</a></li>
<li><a href="/wiki/List_of_scientific_journals_in_bioinformatics" title="List of scientific journals in bioinformatics" class="mw-redirect">List of scientific journals in bioinformatics</a></li>
<li><a href="/wiki/Phylogenetics" title="Phylogenetics">Phylogenetics</a></li>
<li><a href="/wiki/Proteomics" title="Proteomics">Proteomics</a></li>
<li><a href="/wiki/European_Bioinformatics_Institute" title="European Bioinformatics Institute">European Bioinformatics Institute</a></li>
<li><a href="/wiki/Integrative_bioinformatics" title="Integrative bioinformatics">Integrative bioinformatics</a></li>
<li><a href="/wiki/Structural_bioinformatics" title="Structural bioinformatics">Structural bioinformatics</a></li>
<li><a href="/wiki/Human_Genome_Project" title="Human Genome Project">Human Genome Project</a></li>
<li><a href="/wiki/Bioinformatics" title="Bioinformatics">Bioinformatics</a></li>
<li><a href="/wiki/Genomics" title="Genomics">Genomics</a></li>
<li><a href="/wiki/Protein" title="Protein">Protein</a></li>
<li><a href="/wiki/Biomedicine" title="Biomedicine">Biomedicine</a></li>
<li><a href="/wiki/Pathology" title="Pathology">Pathology</a></li>
<li><a href="/wiki/Molecular_Biology" title="Molecular Biology" class="mw-redirect">Molecular Biology</a></li>
<li><a href="/wiki/List_of_biological_databases" title="List of biological databases">List of biological databases</a></li>
</ul>
</div>
<h2><span class="mw-headline" id="References">References</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Gene_Disease_Database&amp;action=edit&amp;section=21" title="Edit section: References">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div class="div-col columns column-count column-count-2" style="-moz-column-count: 2; -webkit-column-count: 2; column-count: 2;">
<div class="reflist" style="list-style-type: decimal;">
<ol class="references">
<li id="cite_note-A._Bauer1-1"><span class="mw-cite-backlink">^ <a href="#cite_ref-A._Bauer1_1-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-A._Bauer1_1-1"><sup><i><b>b</b></i></sup></a> <a href="#cite_ref-A._Bauer1_1-2"><sup><i><b>c</b></i></sup></a> <a href="#cite_ref-A._Bauer1_1-3"><sup><i><b>d</b></i></sup></a> <a href="#cite_ref-A._Bauer1_1-4"><sup><i><b>e</b></i></sup></a> <a href="#cite_ref-A._Bauer1_1-5"><sup><i><b>f</b></i></sup></a> <a href="#cite_ref-A._Bauer1_1-6"><sup><i><b>g</b></i></sup></a> <a href="#cite_ref-A._Bauer1_1-7"><sup><i><b>h</b></i></sup></a> <a href="#cite_ref-A._Bauer1_1-8"><sup><i><b>i</b></i></sup></a> <a href="#cite_ref-A._Bauer1_1-9"><sup><i><b>j</b></i></sup></a> <a href="#cite_ref-A._Bauer1_1-10"><sup><i><b>k</b></i></sup></a> <a href="#cite_ref-A._Bauer1_1-11"><sup><i><b>l</b></i></sup></a> <a href="#cite_ref-A._Bauer1_1-12"><sup><i><b>m</b></i></sup></a> <a href="#cite_ref-A._Bauer1_1-13"><sup><i><b>n</b></i></sup></a></span> <span class="reference-text">A. Bauer-Mehren, "Gene-Disease network Analysis Reveals Functional Modules in <a href="/wiki/Mendelian" title="Mendelian" class="mw-redirect">Mendelian</a>, Complex and Environmental <a href="/wiki/Diseases" title="Diseases" class="mw-redirect">diseases</a>," PLOS One, pp. 1-3, 2011.</span></li>
<li id="cite_note-Botstein2-2"><span class="mw-cite-backlink">^ <a href="#cite_ref-Botstein2_2-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-Botstein2_2-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text">R. N. Botstein D, "Discovering genotypes underlying human phenotypes: past successes for Mendelian disease, future approaches for complex disease.," Nature genetics, vol. 33, no. 1, pp. 228-237, 2003.</span></li>
<li id="cite_note-pmid18819940-3"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid18819940_3-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Wren JD, Bateman A (2008). "Databases, data tombs and dust in the wind.". <i>Bioinformatics</i> <b>24</b> (19): 2127–8. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1093%2Fbioinformatics%2Fbtn464">10.1093/bioinformatics/btn464</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/18819940">18819940</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AGene+Disease+Database&amp;rft.atitle=Databases%2C+data+tombs+and+dust+in+the+wind.&amp;rft.au=Wren+JD%2C+Bateman+A&amp;rft.date=2008&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtn464&amp;rft_id=info%3Apmid%2F18819940&amp;rft.issue=19&amp;rft.jtitle=Bioinformatics&amp;rft.pages=2127-8&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=24" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-3_American-4"><span class="mw-cite-backlink"><b><a href="#cite_ref-3_American_4-0">^</a></b></span> <span class="reference-text">American Medical Informatics Association, "American Medical Informatics Association Strategic Plan," August 2011. [Online]. Available: <a rel="nofollow" class="external free" href="http://www.amia.org/inside/stratplan/">http://www.amia.org/inside/stratplan/</a>. [Accessed 15 October 2014].</span></li>
<li id="cite_note-5"><span class="mw-cite-backlink"><b><a href="#cite_ref-5">^</a></b></span> <span class="reference-text">B. H. Oti M, "The modular nature of genetic diseases.," Clinical genetics, vol. 71, no. 1, pp. 1-11, 2007.</span></li>
<li id="cite_note-6"><span class="mw-cite-backlink"><b><a href="#cite_ref-6">^</a></b></span> <span class="reference-text">A. Davis and B. King, "The Comparative Toxicogenomics Database: update 2011," Nucl. Acids Res, vol. 39, no. 1, pp. 1067-1072, 2011.</span></li>
<li id="cite_note-7"><span class="mw-cite-backlink"><b><a href="#cite_ref-7">^</a></b></span> <span class="reference-text">A. Davis and T. Wiegers, "Text Mining Effectively Scores and Ranks the Literature for Improving Chemical-Gene-Disease Curation at the Comparative Toxicogenomics Database," PLOS One, vol. 8, no. 4, pp. 1-29, 2013.</span></li>
<li id="cite_note-A._Bauer2-8"><span class="mw-cite-backlink">^ <a href="#cite_ref-A._Bauer2_8-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-A._Bauer2_8-1"><sup><i><b>b</b></i></sup></a> <a href="#cite_ref-A._Bauer2_8-2"><sup><i><b>c</b></i></sup></a> <a href="#cite_ref-A._Bauer2_8-3"><sup><i><b>d</b></i></sup></a></span> <span class="reference-text">A. Bauer-Mehren and M. Rautscha, "DisGeNET: a Cytoscape plugin to visualize, integrate, search and analyze gene–disease networks," <i>Bioinformatics</i>, vol. 26, no. 22, pp. 2924-2926, 2010.</span></li>
<li id="cite_note-9"><span class="mw-cite-backlink"><b><a href="#cite_ref-9">^</a></b></span> <span class="reference-text">I. Vogt, "Systematic analysis of gene properties influencing organ system phenotypes in mammalian perturbations," <i>Bioinformatics</i>, vol. 30, no. 21, pp. 3093-3100, 2014.</span></li>
<li id="cite_note-Buneman-10"><span class="mw-cite-backlink">^ <a href="#cite_ref-Buneman_10-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-Buneman_10-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text">P. Buneman, "Curated Databases," Bibliometrics, vol. 978, no. 1, pp. 152-162, 2008.</span></li>
<li id="cite_note-Murphy-11"><span class="mw-cite-backlink">^ <a href="#cite_ref-Murphy_11-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-Murphy_11-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text">C. Murphy and A. Davis, "Comparative Toxicogenomics Database: a knowledgebase and discovery tool for chemical–gene–disease networks," <i>Bioinformatics</i>, vol. 37, no. 1, pp. 786-792, 2009</span></li>
<li id="cite_note-12"><span class="mw-cite-backlink"><b><a href="#cite_ref-12">^</a></b></span> <span class="reference-text">The UniProt Consortium, "The Universal Protein Resource (UniProt)," <i>Nucleic Acids Research</i>, vol. 36, no. 1, pp. 190-195, 2008</span></li>
<li id="cite_note-pmid21051339-13"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid21051339_13-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Uniprot, C. (2010). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3013648">"Ongoing and future developments at the Universal Protein Resource"</a>. <i>Nucleic Acids Research</i> <b>39</b> (Database issue): D214–D219. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1093%2Fnar%2Fgkq1020">10.1093/nar/gkq1020</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3013648">3013648</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/21051339">21051339</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AGene+Disease+Database&amp;rft.atitle=Ongoing+and+future+developments+at+the+Universal+Protein+Resource&amp;rft.aufirst=C.&amp;rft.aulast=Uniprot&amp;rft.date=2010&amp;rft.genre=article&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3013648&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3013648&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkq1020&amp;rft_id=info%3Apmid%2F21051339&amp;rft.issue=Database+issue&amp;rft.jtitle=Nucleic+Acids+Research&amp;rft.pages=D214-D219&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=39" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-14"><span class="mw-cite-backlink"><b><a href="#cite_ref-14">^</a></b></span> <span class="reference-text">K. Brown, "Online Predicted human Interaction Database," <i>Bioinformatics</i>, vol. 21, no. 9, pp. 2076-2082, 2005.</span></li>
<li id="cite_note-15"><span class="mw-cite-backlink"><b><a href="#cite_ref-15">^</a></b></span> <span class="reference-text">S. Hunter and P. Jones, "InterPro in 2011: new developments in the family and domain prediction database," <i>Nucleic Acids Research</i>, vol. 10, no. 1, pp. 12-22, 2011</span></li>
<li id="cite_note-16"><span class="mw-cite-backlink"><b><a href="#cite_ref-16">^</a></b></span> <span class="reference-text">C. Bult and J. Eppig, "The Mouse genome Database (MGD): mouse biology and model systems," <i>Nucleic Acids Research</i>, vol. 36, no. 1, pp. 724-728, 2007</span></li>
<li id="cite_note-Dwinell-17"><span class="mw-cite-backlink">^ <a href="#cite_ref-Dwinell_17-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-Dwinell_17-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text">M. Dwinell, E. Worthey and S. M, "The Rat genome Database 2009: variation, ontologies and pathways," <i>Nucleic Acids Research</i>, vol. 37, no. 1, pp. 744-749, 2009</span></li>
<li id="cite_note-pmid25355511-18"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid25355511_18-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Shimoyama M, De Pons J, Hayman GT, et al. (2015). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4383884">"The Rat Genome Database 2015: genomic, phenotypic and environmental variations and disease"</a>. <i>Nucleic Acids Res.</i> <b>43</b> (Database issue): D743–50. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1093%2Fnar%2Fgku1026">10.1093/nar/gku1026</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4383884">4383884</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/25355511">25355511</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AGene+Disease+Database&amp;rft.atitle=The+Rat+Genome+Database+2015%3A+genomic%2C+phenotypic+and+environmental+variations+and+disease&amp;rft.au=De+Pons%2C+J&amp;rft.aufirst=M&amp;rft.au=Hayman%2C+GT&amp;rft.aulast=Shimoyama&amp;rft.date=2015&amp;rft.genre=article&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4383884&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4383884&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgku1026&amp;rft_id=info%3Apmid%2F25355511&amp;rft.issue=Database+issue&amp;rft.jtitle=Nucleic+Acids+Res.&amp;rft.pages=D743-50&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=43" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-Homosh-19"><span class="mw-cite-backlink">^ <a href="#cite_ref-Homosh_19-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-Homosh_19-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text">A. Homosh, "Online Mendelian Inheritance in Man (OMIM), a knowledgebase of human genes and genetic disorders," <i>Nucleic Acids Research</i>, vol. 33, no. 1, pp. 514-517, 2005</span></li>
<li id="cite_note-Hubbard2002-20"><span class="mw-cite-backlink"><b><a href="#cite_ref-Hubbard2002_20-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Hubbard T, et al. (January 2002). <a rel="nofollow" class="external text" href="http://nar.oxfordjournals.org/content/30/1/38.long">"The Ensembl genome database project"</a>. <i>Nucleic Acid Res.</i> <b>30</b> (1): 38–41. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1093%2Fnar%2F30.1.38">10.1093/nar/30.1.38</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC99161">99161</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/11752248">11752248</a><span class="reference-accessdate">. Retrieved <span class="nowrap">11 November</span> 2014</span>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AGene+Disease+Database&amp;rft.atitle=The+Ensembl+genome+database+project&amp;rft.aufirst=T&amp;rft.aulast=Hubbard&amp;rft.date=2002-01&amp;rft.genre=article&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC99161&amp;rft_id=http%3A%2F%2Fnar.oxfordjournals.org%2Fcontent%2F30%2F1%2F38.long&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2F30.1.38&amp;rft_id=info%3Apmid%2F11752248&amp;rft.issue=1&amp;rft.jtitle=Nucleic+Acid+Res.&amp;rft.pages=38-41&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=30" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-Flicek-21"><span class="mw-cite-backlink">^ <a href="#cite_ref-Flicek_21-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-Flicek_21-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text">P. Flicek and M. Ridwan, "Ensembl 2012," <i>Nucleic Acids Research</i>, vol. 40, no. 1, pp. 84-90, 2012</span></li>
<li id="cite_note-22"><span class="mw-cite-backlink"><b><a href="#cite_ref-22">^</a></b></span> <span class="reference-text">K. Becker and K. Barnes, "The genetic Association Database," Nature genetics, vol. 36, no. 1, pp. 431-432, 2004</span></li>
<li id="cite_note-23"><span class="mw-cite-backlink"><b><a href="#cite_ref-23">^</a></b></span> <span class="reference-text"><cite class="citation journal">Bravo, A; et al. (2014). "Extraction of relations between genes and diseases from text and large-scale data analysis: implications for translational research". <i>BMC Bioinformatics</i> <b>1</b> (1): 10. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1186%2Fs12859-015-0472-9">10.1186/s12859-015-0472-9</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AGene+Disease+Database&amp;rft.atitle=Extraction+of+relations+between+genes+and+diseases+from+text+and+large-scale+data+analysis%3A+implications+for+translational+research&amp;rft.aufirst=A&amp;rft.aulast=Bravo&amp;rft.date=2014&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1186%2Fs12859-015-0472-9&amp;rft.issue=1&amp;rft.jtitle=BMC+Bioinformatics&amp;rft.pages=10&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=1" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-Pinero-24"><span class="mw-cite-backlink">^ <a href="#cite_ref-Pinero_24-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-Pinero_24-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text">Piñero, et al, "DisGeNET: a discovery platform for the dynamical exploration of human diseases and their genes" Database (Oxford) doi: 10.1093/database/bav028., 2015.</span></li>
<li id="cite_note-25"><span class="mw-cite-backlink"><b><a href="#cite_ref-25">^</a></b></span> <span class="reference-text"><cite class="citation journal">Oti, M (2006). "Predicting disease genes using protein-protein interactions". <i>J. Med. Genet</i> <b>43</b>: 691–698. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1136%2Fjmg.2006.041376">10.1136/jmg.2006.041376</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AGene+Disease+Database&amp;rft.atitle=Predicting+disease+genes+using+protein-protein+interactions&amp;rft.aufirst=M&amp;rft.aulast=Oti&amp;rft.date=2006&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1136%2Fjmg.2006.041376&amp;rft.jtitle=J.+Med.+Genet&amp;rft.pages=691-698&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=43" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
</ol>
</div>
</div>


<!-- 
NewPP limit report
Parsed by mw1020
Cached time: 20160221094412
Cache expiry: 2592000
Dynamic content: false
CPU time usage: 0.617 seconds
Real time usage: 0.714 seconds
Preprocessor visited node count: 4118/1000000
Preprocessor generated node count: 0/1500000
Post‐expand include size: 56073/2097152 bytes
Template argument size: 6718/2097152 bytes
Highest expansion depth: 12/40
Expensive parser function count: 1/500
Lua time usage: 0.248/10.000 seconds
Lua memory usage: 2.91 MB/50 MB
Number of Wikibase entities loaded: 1-->

<!-- 
Transclusion expansion time report (%,ms,calls,template)
100.00%  572.646      1 - -total
 42.95%  245.940      8 - Template:Infobox
 27.20%  155.784      2 - Template:Infobox_chef
 26.54%  152.008      2 - Template:Infobox_person
 24.24%  138.824      5 - Template:Infobox_biodatabase
 20.90%  119.655      1 - Template:Reflist
 13.57%   77.730      6 - Template:Cite_journal
  6.07%   34.773      2 - Template:Wikidata_image
  4.70%   26.888      1 - Template:For
  4.19%   23.978      1 - Template:Hatnote
-->

<!-- Saved in parser cache with key enwiki:pcache:idhash:44248347-0!*!0!!en!4!* and timestamp 20160221094411 and revision id 705536691
 -->
<noscript><img src="//en.wikipedia.org/wiki/Special:CentralAutoLogin/start?type=1x1" alt="" title="" width="1" height="1" style="border: none; position: absolute;" /></noscript></div>					<div class="printfooter">
						Retrieved from "<a dir="ltr" href="https://en.wikipedia.org/w/index.php?title=Gene_Disease_Database&amp;oldid=705536691">https://en.wikipedia.org/w/index.php?title=Gene_Disease_Database&amp;oldid=705536691</a>"					</div>
				<div id='catlinks' class='catlinks' data-mw='interface'><div id="mw-normal-catlinks" class="mw-normal-catlinks"><a href="/wiki/Help:Category" title="Help:Category">Categories</a>: <ul><li><a href="/wiki/Category:Biological_databases" title="Category:Biological databases">Biological databases</a></li><li><a href="/wiki/Category:Biology" title="Category:Biology">Biology</a></li><li><a href="/wiki/Category:Bioinformatics" title="Category:Bioinformatics">Bioinformatics</a></li></ul></div><div id="mw-hidden-catlinks" class="mw-hidden-catlinks mw-hidden-cats-hidden">Hidden categories: <ul><li><a href="/wiki/Category:Use_dmy_dates_from_November_2014" title="Category:Use dmy dates from November 2014">Use dmy dates from November 2014</a></li><li><a href="/wiki/Category:Articles_which_use_infobox_templates_with_no_data_rows" title="Category:Articles which use infobox templates with no data rows">Articles which use infobox templates with no data rows</a></li><li><a href="/wiki/Category:Articles_with_hCards" title="Category:Articles with hCards">Articles with hCards</a></li></ul></div></div>				<div class="visualClear"></div>
							</div>
		</div>
		<div id="mw-navigation">
			<h2>Navigation menu</h2>

			<div id="mw-head">
									<div id="p-personal" role="navigation" class="" aria-labelledby="p-personal-label">
						<h3 id="p-personal-label">Personal tools</h3>
						<ul>
							<li id="pt-anonuserpage">Not logged in</li><li id="pt-anontalk"><a href="/wiki/Special:MyTalk" title="Discussion about edits from this IP address [n]" accesskey="n">Talk</a></li><li id="pt-anoncontribs"><a href="/wiki/Special:MyContributions" title="A list of edits made from this IP address [y]" accesskey="y">Contributions</a></li><li id="pt-createaccount"><a href="/w/index.php?title=Special:UserLogin&amp;returnto=Gene+Disease+Database&amp;type=signup" title="You are encouraged to create an account and log in; however, it is not mandatory">Create account</a></li><li id="pt-login"><a href="/w/index.php?title=Special:UserLogin&amp;returnto=Gene+Disease+Database" title="You're encouraged to log in; however, it's not mandatory. [o]" accesskey="o">Log in</a></li>						</ul>
					</div>
									<div id="left-navigation">
										<div id="p-namespaces" role="navigation" class="vectorTabs" aria-labelledby="p-namespaces-label">
						<h3 id="p-namespaces-label">Namespaces</h3>
						<ul>
															<li  id="ca-nstab-main" class="selected"><span><a href="/wiki/Gene_Disease_Database"  title="View the content page [c]" accesskey="c">Article</a></span></li>
															<li  id="ca-talk" class="new"><span><a href="/w/index.php?title=Talk:Gene_Disease_Database&amp;action=edit&amp;redlink=1"  title="Discussion about the content page [t]" accesskey="t" rel="discussion">Talk</a></span></li>
													</ul>
					</div>
										<div id="p-variants" role="navigation" class="vectorMenu emptyPortlet" aria-labelledby="p-variants-label">
												<h3 id="p-variants-label">
							<span>Variants</span><a href="#"></a>
						</h3>

						<div class="menu">
							<ul>
															</ul>
						</div>
					</div>
									</div>
				<div id="right-navigation">
										<div id="p-views" role="navigation" class="vectorTabs" aria-labelledby="p-views-label">
						<h3 id="p-views-label">Views</h3>
						<ul>
															<li id="ca-view" class="selected"><span><a href="/wiki/Gene_Disease_Database" >Read</a></span></li>
															<li id="ca-edit"><span><a href="/w/index.php?title=Gene_Disease_Database&amp;action=edit"  title="Edit this page [e]" accesskey="e">Edit</a></span></li>
															<li id="ca-history" class="collapsible"><span><a href="/w/index.php?title=Gene_Disease_Database&amp;action=history"  title="Past revisions of this page [h]" accesskey="h">View history</a></span></li>
													</ul>
					</div>
										<div id="p-cactions" role="navigation" class="vectorMenu emptyPortlet" aria-labelledby="p-cactions-label">
						<h3 id="p-cactions-label"><span>More</span><a href="#"></a></h3>

						<div class="menu">
							<ul>
															</ul>
						</div>
					</div>
										<div id="p-search" role="search">
						<h3>
							<label for="searchInput">Search</label>
						</h3>

						<form action="/w/index.php" id="searchform">
							<div id="simpleSearch">
							<input type="search" name="search" placeholder="Search" title="Search Wikipedia [f]" accesskey="f" id="searchInput" /><input type="hidden" value="Special:Search" name="title" /><input type="submit" name="fulltext" value="Search" title="Search Wikipedia for this text" id="mw-searchButton" class="searchButton mw-fallbackSearchButton" /><input type="submit" name="go" value="Go" title="Go to a page with this exact name if it exists" id="searchButton" class="searchButton" />							</div>
						</form>
					</div>
									</div>
			</div>
			<div id="mw-panel">
				<div id="p-logo" role="banner"><a class="mw-wiki-logo" href="/wiki/Main_Page"  title="Visit the main page"></a></div>
						<div class="portal" role="navigation" id='p-navigation' aria-labelledby='p-navigation-label'>
			<h3 id='p-navigation-label'>Navigation</h3>

			<div class="body">
									<ul>
						<li id="n-mainpage-description"><a href="/wiki/Main_Page" title="Visit the main page [z]" accesskey="z">Main page</a></li><li id="n-contents"><a href="/wiki/Portal:Contents" title="Guides to browsing Wikipedia">Contents</a></li><li id="n-featuredcontent"><a href="/wiki/Portal:Featured_content" title="Featured content – the best of Wikipedia">Featured content</a></li><li id="n-currentevents"><a href="/wiki/Portal:Current_events" title="Find background information on current events">Current events</a></li><li id="n-randompage"><a href="/wiki/Special:Random" title="Load a random article [x]" accesskey="x">Random article</a></li><li id="n-sitesupport"><a href="https://donate.wikimedia.org/wiki/Special:FundraiserRedirector?utm_source=donate&amp;utm_medium=sidebar&amp;utm_campaign=C13_en.wikipedia.org&amp;uselang=en" title="Support us">Donate to Wikipedia</a></li><li id="n-shoplink"><a href="//shop.wikimedia.org" title="Visit the Wikipedia store">Wikipedia store</a></li>					</ul>
							</div>
		</div>
			<div class="portal" role="navigation" id='p-interaction' aria-labelledby='p-interaction-label'>
			<h3 id='p-interaction-label'>Interaction</h3>

			<div class="body">
									<ul>
						<li id="n-help"><a href="/wiki/Help:Contents" title="Guidance on how to use and edit Wikipedia">Help</a></li><li id="n-aboutsite"><a href="/wiki/Wikipedia:About" title="Find out about Wikipedia">About Wikipedia</a></li><li id="n-portal"><a href="/wiki/Wikipedia:Community_portal" title="About the project, what you can do, where to find things">Community portal</a></li><li id="n-recentchanges"><a href="/wiki/Special:RecentChanges" title="A list of recent changes in the wiki [r]" accesskey="r">Recent changes</a></li><li id="n-contactpage"><a href="//en.wikipedia.org/wiki/Wikipedia:Contact_us" title="How to contact Wikipedia">Contact page</a></li>					</ul>
							</div>
		</div>
			<div class="portal" role="navigation" id='p-tb' aria-labelledby='p-tb-label'>
			<h3 id='p-tb-label'>Tools</h3>

			<div class="body">
									<ul>
						<li id="t-whatlinkshere"><a href="/wiki/Special:WhatLinksHere/Gene_Disease_Database" title="List of all English Wikipedia pages containing links to this page [j]" accesskey="j">What links here</a></li><li id="t-recentchangeslinked"><a href="/wiki/Special:RecentChangesLinked/Gene_Disease_Database" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</a></li><li id="t-upload"><a href="/wiki/Wikipedia:File_Upload_Wizard" title="Upload files [u]" accesskey="u">Upload file</a></li><li id="t-specialpages"><a href="/wiki/Special:SpecialPages" title="A list of all special pages [q]" accesskey="q">Special pages</a></li><li id="t-permalink"><a href="/w/index.php?title=Gene_Disease_Database&amp;oldid=705536691" title="Permanent link to this revision of the page">Permanent link</a></li><li id="t-info"><a href="/w/index.php?title=Gene_Disease_Database&amp;action=info" title="More information about this page">Page information</a></li><li id="t-wikibase"><a href="//www.wikidata.org/wiki/Q18392279" title="Link to connected data repository item [g]" accesskey="g">Wikidata item</a></li><li id="t-cite"><a href="/w/index.php?title=Special:CiteThisPage&amp;page=Gene_Disease_Database&amp;id=705536691" title="Information on how to cite this page">Cite this page</a></li>					</ul>
							</div>
		</div>
			<div class="portal" role="navigation" id='p-coll-print_export' aria-labelledby='p-coll-print_export-label'>
			<h3 id='p-coll-print_export-label'>Print/export</h3>

			<div class="body">
									<ul>
						<li id="coll-create_a_book"><a href="/w/index.php?title=Special:Book&amp;bookcmd=book_creator&amp;referer=Gene+Disease+Database">Create a book</a></li><li id="coll-download-as-rdf2latex"><a href="/w/index.php?title=Special:Book&amp;bookcmd=render_article&amp;arttitle=Gene+Disease+Database&amp;returnto=Gene+Disease+Database&amp;oldid=705536691&amp;writer=rdf2latex">Download as PDF</a></li><li id="t-print"><a href="/w/index.php?title=Gene_Disease_Database&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</a></li>					</ul>
							</div>
		</div>
			<div class="portal" role="navigation" id='p-lang' aria-labelledby='p-lang-label'>
			<h3 id='p-lang-label'>Languages</h3>

			<div class="body">
									<ul>
						<li class="uls-p-lang-dummy"><a href="#"></a></li>					</ul>
				<div class='after-portlet after-portlet-lang'><span class="wb-langlinks-add wb-langlinks-link"><a href="//www.wikidata.org/wiki/Q18392279#sitelinks-wikipedia" title="Add interlanguage links" class="wbc-editpage">Add links</a></span></div>			</div>
		</div>
				</div>
		</div>
		<div id="footer" role="contentinfo">
							<ul id="footer-info">
											<li id="footer-info-lastmod"> This page was last modified on 18 February 2016, at 01:54.</li>
											<li id="footer-info-copyright">Text is available under the <a rel="license" href="//en.wikipedia.org/wiki/Wikipedia:Text_of_Creative_Commons_Attribution-ShareAlike_3.0_Unported_License">Creative Commons Attribution-ShareAlike License</a><a rel="license" href="//creativecommons.org/licenses/by-sa/3.0/" style="display:none;"></a>;
additional terms may apply.  By using this site, you agree to the <a href="//wikimediafoundation.org/wiki/Terms_of_Use">Terms of Use</a> and <a href="//wikimediafoundation.org/wiki/Privacy_policy">Privacy Policy</a>. Wikipedia® is a registered trademark of the <a href="//www.wikimediafoundation.org/">Wikimedia Foundation, Inc.</a>, a non-profit organization.</li>
									</ul>
							<ul id="footer-places">
											<li id="footer-places-privacy"><a href="//wikimediafoundation.org/wiki/Privacy_policy" title="wmf:Privacy policy">Privacy policy</a></li>
											<li id="footer-places-about"><a href="/wiki/Wikipedia:About" title="Wikipedia:About">About Wikipedia</a></li>
											<li id="footer-places-disclaimer"><a href="/wiki/Wikipedia:General_disclaimer" title="Wikipedia:General disclaimer">Disclaimers</a></li>
											<li id="footer-places-contact"><a href="//en.wikipedia.org/wiki/Wikipedia:Contact_us">Contact Wikipedia</a></li>
											<li id="footer-places-developers"><a href="https://www.mediawiki.org/wiki/Special:MyLanguage/How_to_contribute">Developers</a></li>
											<li id="footer-places-cookiestatement"><a href="//wikimediafoundation.org/wiki/Cookie_statement">Cookie statement</a></li>
											<li id="footer-places-mobileview"><a href="//en.m.wikipedia.org/w/index.php?title=Gene_Disease_Database&amp;mobileaction=toggle_view_mobile" class="noprint stopMobileRedirectToggle">Mobile view</a></li>
									</ul>
										<ul id="footer-icons" class="noprint">
											<li id="footer-copyrightico">
							<a href="//wikimediafoundation.org/"><img src="/static/images/wikimedia-button.png" srcset="/static/images/wikimedia-button-1.5x.png 1.5x, /static/images/wikimedia-button-2x.png 2x" width="88" height="31" alt="Wikimedia Foundation"/></a>						</li>
											<li id="footer-poweredbyico">
							<a href="//www.mediawiki.org/"><img src="/static/1.27.0-wmf.13/resources/assets/poweredby_mediawiki_88x31.png" alt="Powered by MediaWiki" srcset="/static/1.27.0-wmf.13/resources/assets/poweredby_mediawiki_132x47.png 1.5x, /static/1.27.0-wmf.13/resources/assets/poweredby_mediawiki_176x62.png 2x" width="88" height="31" /></a>						</li>
									</ul>
						<div style="clear:both"></div>
		</div>
		<script>(window.RLQ = window.RLQ || []).push(function () {
mw.loader.state({"ext.globalCssJs.site":"ready","ext.globalCssJs.user":"ready","user":"ready","user.groups":"ready"});mw.loader.load(["ext.cite.a11y","mediawiki.toc","mediawiki.action.view.postEdit","site","mediawiki.user","mediawiki.hidpi","mediawiki.page.ready","mediawiki.searchSuggest","ext.eventLogging.subscriber","ext.gadget.teahouse","ext.gadget.ReferenceTooltips","ext.gadget.DRN-wizard","ext.gadget.charinsert","ext.gadget.refToolbar","ext.gadget.switcher","ext.gadget.featured-articles-links","mmv.bootstrap.autostart","ext.visualEditor.targetLoader","ext.wikimediaEvents","ext.navigationTiming","schema.UniversalLanguageSelector","ext.uls.eventlogger","ext.uls.interlanguage"]);
} );</script><script>(window.RLQ = window.RLQ || []).push(function () {
mw.config.set({"wgBackendResponseTime":828,"wgHostname":"mw1020"}); /* @nomin */
} );</script>
	</body>
</html>
