<!DOCTYPE html>
<html lang="en" dir="ltr" class="client-nojs">
<head>
<meta charset="UTF-8" />
<title>Molecular dynamics - Wikipedia, the free encyclopedia</title>
<script>document.documentElement.className = document.documentElement.className.replace( /(^|\s)client-nojs(\s|$)/, "$1client-js$2" );</script>
<script>(window.RLQ = window.RLQ || []).push(function () {
mw.config.set({"wgCanonicalNamespace":"","wgCanonicalSpecialPageName":false,"wgNamespaceNumber":0,"wgPageName":"Molecular_dynamics","wgTitle":"Molecular dynamics","wgCurRevisionId":705263074,"wgRevisionId":705263074,"wgArticleId":198608,"wgIsArticle":true,"wgIsRedirect":false,"wgAction":"view","wgUserName":null,"wgUserGroups":["*"],"wgCategories":["CS1 French-language sources (fr)","All articles with unsourced statements","Articles with unsourced statements from April 2009","Commons category with local link same as on Wikidata","Molecular dynamics","Computational chemistry","Molecular modelling","Scientific modeling"],"wgBreakFrames":false,"wgPageContentLanguage":"en","wgPageContentModel":"wikitext","wgSeparatorTransformTable":["",""],"wgDigitTransformTable":["",""],"wgDefaultDateFormat":"dmy","wgMonthNames":["","January","February","March","April","May","June","July","August","September","October","November","December"],"wgMonthNamesShort":["","Jan","Feb","Mar","Apr","May","Jun","Jul","Aug","Sep","Oct","Nov","Dec"],"wgRelevantPageName":"Molecular_dynamics","wgRelevantArticleId":198608,"wgIsProbablyEditable":true,"wgRestrictionEdit":[],"wgRestrictionMove":[],"wgWikiEditorEnabledModules":{"toolbar":true,"dialogs":true,"preview":false,"publish":false},"wgBetaFeaturesFeatures":[],"wgMediaViewerOnClick":true,"wgMediaViewerEnabledByDefault":true,"wgVisualEditor":{"pageLanguageCode":"en","pageLanguageDir":"ltr","usePageImages":true,"usePageDescriptions":true},"wikilove-recipient":"","wikilove-anon":0,"wgGatherShouldShowTutorial":true,"wgGatherEnableSample":0,"wgGatherPageImageThumbnail":"//upload.wikimedia.org/wikipedia/commons/thumb/4/42/Cudeposition.gif/84px-Cudeposition.gif","wgRelatedArticles":null,"wgRelatedArticlesUseCirrusSearch":true,"wgRelatedArticlesOnlyUseCirrusSearch":false,"wgULSAcceptLanguageList":["en-us","en"],"wgULSCurrentAutonym":"English","wgFlaggedRevsParams":{"tags":{"status":{"levels":1,"quality":2,"pristine":3}}},"wgStableRevisionId":null,"wgCategoryTreePageCategoryOptions":"{\"mode\":0,\"hideprefix\":20,\"showcount\":true,\"namespaces\":false}","wgNoticeProject":"wikipedia","wgCentralNoticeCategoriesUsingLegacy":["Fundraising","fundraising"],"wgCentralAuthMobileDomain":false,"wgWikibaseItemId":"Q901663","wgVisualEditorToolbarScrollOffset":0}); /* @nomin */mw.loader.implement("user.options",function($,jQuery){mw.user.options.set({"variant":"en"});});mw.loader.implement("user.tokens",function ( $, jQuery ) {
mw.user.tokens.set({"editToken":"+\\","patrolToken":"+\\","watchToken":"+\\","csrfToken":"+\\"}); /* @nomin */ ;

});mw.loader.load(["mediawiki.page.startup","mediawiki.legacy.wikibits","ext.centralauth.centralautologin","ext.gadget.WatchlistBase","ext.gadget.WatchlistGreenIndicators","mmv.head","ext.visualEditor.desktopArticleTarget.init","ext.uls.init","ext.uls.interface","ext.quicksurveys.init","mw.MediaWikiPlayer.loader","mw.PopUpMediaTransform","ext.centralNotice.bannerController","skins.vector.js"]);
} );</script>
<link rel="stylesheet" href="/w/load.php?debug=false&amp;lang=en&amp;modules=ext.cite.styles%7Cext.gadget.DRN-wizard%2CReferenceTooltips%2CWatchlistBase%2CWatchlistGreenIndicators%2Ccharinsert%2Cfeatured-articles-links%2CrefToolbar%2Cswitcher%2Cteahouse%7Cext.math.styles%7Cext.tmh.thumbnail.styles%7Cext.uls.nojs%7Cext.visualEditor.desktopArticleTarget.noscript%7Cext.wikimediaBadges%7Cmediawiki.legacy.commonPrint%2Cshared%7Cmediawiki.raggett%2CsectionAnchor%7Cmediawiki.skinning.interface%7Cskins.vector.styles%7Cwikibase.client.init&amp;only=styles&amp;skin=vector" />
<meta name="ResourceLoaderDynamicStyles" content="" />
<link rel="stylesheet" href="/w/load.php?debug=false&amp;lang=en&amp;modules=site&amp;only=styles&amp;skin=vector" />
<script async="" src="/w/load.php?debug=false&amp;lang=en&amp;modules=startup&amp;only=scripts&amp;skin=vector"></script>
<meta name="generator" content="MediaWiki 1.27.0-wmf.13" />
<link rel="alternate" href="android-app://org.wikipedia/http/en.m.wikipedia.org/wiki/Molecular_dynamics" />
<link rel="alternate" type="application/x-wiki" title="Edit this page" href="/w/index.php?title=Molecular_dynamics&amp;action=edit" />
<link rel="edit" title="Edit this page" href="/w/index.php?title=Molecular_dynamics&amp;action=edit" />
<link rel="apple-touch-icon" href="/static/apple-touch/wikipedia.png" />
<link rel="shortcut icon" href="/static/favicon/wikipedia.ico" />
<link rel="search" type="application/opensearchdescription+xml" href="/w/opensearch_desc.php" title="Wikipedia (en)" />
<link rel="EditURI" type="application/rsd+xml" href="//en.wikipedia.org/w/api.php?action=rsd" />
<link rel="copyright" href="//creativecommons.org/licenses/by-sa/3.0/" />
<link rel="canonical" href="https://en.wikipedia.org/wiki/Molecular_dynamics" />
<link rel="dns-prefetch" href="//meta.wikimedia.org" />
<!--[if lt IE 7]><style type="text/css">body{behavior:url("/static/1.27.0-wmf.13/skins/Vector/csshover.min.htc")}</style><![endif]-->
</head>
<body class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Molecular_dynamics skin-vector action-view">
		<div id="mw-page-base" class="noprint"></div>
		<div id="mw-head-base" class="noprint"></div>
		<div id="content" class="mw-body" role="main">
			<a id="top"></a>

							<div id="siteNotice"><!-- CentralNotice --></div>
						<div class="mw-indicators">
</div>
			<h1 id="firstHeading" class="firstHeading" lang="en">Molecular dynamics</h1>
									<div id="bodyContent" class="mw-body-content">
									<div id="siteSub">From Wikipedia, the free encyclopedia</div>
								<div id="contentSub"></div>
												<div id="jump-to-nav" class="mw-jump">
					Jump to:					<a href="#mw-head">navigation</a>, 					<a href="#p-search">search</a>
				</div>
				<div id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><div class="thumb tright">
<div class="thumbinner" style="width:322px;"><a href="/wiki/File:Cudeposition.gif" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/4/42/Cudeposition.gif" width="320" height="380" class="thumbimage" data-file-width="320" data-file-height="380" /></a>
<div class="thumbcaption">
<div class="magnify"><a href="/wiki/File:Cudeposition.gif" class="internal" title="Enlarge"></a></div>
Example of a molecular dynamics simulation in a simple system: deposition of a single <a href="/wiki/Copper" title="Copper">Cu</a> <a href="/wiki/Atom" title="Atom">atom</a> on a Cu <a href="/wiki/Miller_index" title="Miller index">(001)</a> <a href="/wiki/Surface_science" title="Surface science">surface</a>. Each circle illustrates the position of a single atom; note that the actual atomic interactions used in current simulations are more complex than those of 2-dimensional hard spheres.</div>
</div>
</div>
<div class="thumb tright">
<div class="thumbinner" style="width:402px;"><a href="/wiki/File:Molecular_dynamics_algorithm.png" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/thumb/7/7b/Molecular_dynamics_algorithm.png/400px-Molecular_dynamics_algorithm.png" width="400" height="360" class="thumbimage" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/7/7b/Molecular_dynamics_algorithm.png/600px-Molecular_dynamics_algorithm.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/7/7b/Molecular_dynamics_algorithm.png/800px-Molecular_dynamics_algorithm.png 2x" data-file-width="1009" data-file-height="909" /></a>
<div class="thumbcaption">
<div class="magnify"><a href="/wiki/File:Molecular_dynamics_algorithm.png" class="internal" title="Enlarge"></a></div>
A simplified description of the standard molecular dynamics simulation algorithm, when a predictor-corrector-type integrator is used. The forces may come either from classical <a href="/wiki/Interatomic_potentials" title="Interatomic potentials" class="mw-redirect">interatomic potentials</a> (indicated as <img class="mwe-math-fallback-image-inline tex" alt=" F = - \nabla V(\vec r) " src="//upload.wikimedia.org/math/e/5/9/e59ce9fba248aa73bd7fa60fea695935.png" />) or quantum mechanical (indicated schematically as <img class="mwe-math-fallback-image-inline tex" alt=" F=F( \Psi(\vec r) ) " src="//upload.wikimedia.org/math/3/3/4/3348c68f8104193586f50af447b0c848.png" />) methods. Note that there are large differences between different integrators; some do not exactly have the same highest-order terms as indicated in the schematic, many use also higher-order time derivatives, and some use both the current and previous time step in variable-time step schemes.</div>
</div>
</div>
<p><b>Molecular <a href="/wiki/Dynamics_(mechanics)" title="Dynamics (mechanics)">dynamics</a></b> (<b>MD</b>) is a <a href="/wiki/Computer_simulation" title="Computer simulation">computer simulation</a> method for studying the <a href="/wiki/Motion_(physics)" title="Motion (physics)">physical movements</a> of <a href="/wiki/Atoms" title="Atoms" class="mw-redirect">atoms</a> and <a href="/wiki/Molecules" title="Molecules" class="mw-redirect">molecules</a>, and is thus a type of <a href="/wiki/N-body_simulation" title="N-body simulation">N-body simulation</a>. The atoms and molecules are allowed to interact for a fixed period of time, giving a view of the dynamical evolution of the system. In the most common version, the <a href="/wiki/Trajectories" title="Trajectories" class="mw-redirect">trajectories</a> of atoms and molecules are determined by <a href="/wiki/Numerical_integration" title="Numerical integration">numerically solving</a> <a href="/wiki/Newton%27s_laws_of_motion" title="Newton's laws of motion">Newton's equations of motion</a> for a system of interacting particles, where <a href="/wiki/Force_(physics)" title="Force (physics)" class="mw-redirect">forces</a> between the particles and their <a href="/wiki/Potential_energy" title="Potential energy">potential energies</a> are calculated using <a href="/wiki/Interatomic_potential" title="Interatomic potential">interatomic potentials</a> or <a href="/wiki/Molecular_mechanics" title="Molecular mechanics">molecular mechanics</a> <a href="/wiki/Force_field_(chemistry)" title="Force field (chemistry)">force fields</a>. The method was originally developed within the field of theoretical physics in the late 1950s<sup id="cite_ref-a.26w_1-0" class="reference"><a href="#cite_note-a.26w-1"><span>[</span>1<span>]</span></a></sup><sup id="cite_ref-a.rahman_2-0" class="reference"><a href="#cite_note-a.rahman-2"><span>[</span>2<span>]</span></a></sup> but is applied today mostly in <a href="/wiki/Chemical_physics" title="Chemical physics">chemical physics</a>, <a href="/wiki/Materials_science" title="Materials science">materials science</a> and the modelling of <a href="/wiki/Biomolecule" title="Biomolecule">biomolecules</a>.</p>
<p>Because molecular systems typically consist of a vast number of particles, it is impossible to determine the properties of such <a href="/wiki/Complex_systems" title="Complex systems">complex systems</a> analytically; MD simulation circumvents this problem by using <a href="/wiki/Numerical_analysis" title="Numerical analysis">numerical</a> methods. However, long MD simulations are mathematically <a href="/wiki/Condition_number" title="Condition number">ill-conditioned</a>, generating cumulative errors in <a href="/wiki/Numerical_integration" title="Numerical integration">numerical integration</a> that can be minimized with proper selection of algorithms and parameters, but not eliminated entirely.</p>
<p>For systems which obey the <a href="/wiki/Ergodic_hypothesis" title="Ergodic hypothesis">ergodic hypothesis</a>, the evolution of a single molecular dynamics simulation may be used to determine macroscopic <a href="/wiki/Thermodynamic" title="Thermodynamic" class="mw-redirect">thermodynamic</a> properties of the system: the time averages of an ergodic system correspond to <a href="/wiki/Microcanonical_ensemble" title="Microcanonical ensemble">microcanonical ensemble</a> averages. MD has also been termed "statistical mechanics by numbers" and "<a href="/wiki/Laplace" title="Laplace" class="mw-redirect">Laplace</a>'s vision of <a href="/wiki/Newtonian_mechanics" title="Newtonian mechanics" class="mw-redirect">Newtonian mechanics</a>" of predicting the future by animating nature's forces<sup id="cite_ref-3" class="reference"><a href="#cite_note-3"><span>[</span>3<span>]</span></a></sup><sup id="cite_ref-4" class="reference"><a href="#cite_note-4"><span>[</span>4<span>]</span></a></sup> and allowing insight into molecular motion on an atomic scale.</p>
<p></p>
<div id="toc" class="toc">
<div id="toctitle">
<h2>Contents</h2>
</div>
<ul>
<li class="toclevel-1 tocsection-1"><a href="#History"><span class="tocnumber">1</span> <span class="toctext">History</span></a></li>
<li class="toclevel-1 tocsection-2"><a href="#Areas_of_application_and_limitations"><span class="tocnumber">2</span> <span class="toctext">Areas of application and limitations</span></a></li>
<li class="toclevel-1 tocsection-3"><a href="#Design_constraints"><span class="tocnumber">3</span> <span class="toctext">Design constraints</span></a>
<ul>
<li class="toclevel-2 tocsection-4"><a href="#Microcanonical_ensemble_.28NVE.29"><span class="tocnumber">3.1</span> <span class="toctext">Microcanonical ensemble (NVE)</span></a></li>
<li class="toclevel-2 tocsection-5"><a href="#Canonical_ensemble_.28NVT.29"><span class="tocnumber">3.2</span> <span class="toctext">Canonical ensemble (NVT)</span></a></li>
<li class="toclevel-2 tocsection-6"><a href="#Isothermal.E2.80.93isobaric_.28NPT.29_ensemble"><span class="tocnumber">3.3</span> <span class="toctext">Isothermal–isobaric (NPT) ensemble</span></a></li>
<li class="toclevel-2 tocsection-7"><a href="#Generalized_ensembles"><span class="tocnumber">3.4</span> <span class="toctext">Generalized ensembles</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-8"><a href="#Potentials_in_MD_simulations"><span class="tocnumber">4</span> <span class="toctext">Potentials in MD simulations</span></a>
<ul>
<li class="toclevel-2 tocsection-9"><a href="#Empirical_potentials"><span class="tocnumber">4.1</span> <span class="toctext">Empirical potentials</span></a></li>
<li class="toclevel-2 tocsection-10"><a href="#Pair_potentials_versus_many-body_potentials"><span class="tocnumber">4.2</span> <span class="toctext">Pair potentials versus many-body potentials</span></a></li>
<li class="toclevel-2 tocsection-11"><a href="#Semi-empirical_potentials"><span class="tocnumber">4.3</span> <span class="toctext">Semi-empirical potentials</span></a></li>
<li class="toclevel-2 tocsection-12"><a href="#Polarizable_potentials"><span class="tocnumber">4.4</span> <span class="toctext">Polarizable potentials</span></a></li>
<li class="toclevel-2 tocsection-13"><a href="#Potentials_in_ab-initio_methods"><span class="tocnumber">4.5</span> <span class="toctext">Potentials in <i>ab-initio</i> methods</span></a></li>
<li class="toclevel-2 tocsection-14"><a href="#Hybrid_QM.2FMM"><span class="tocnumber">4.6</span> <span class="toctext">Hybrid QM/MM</span></a></li>
<li class="toclevel-2 tocsection-15"><a href="#Coarse-graining_and_reduced_representations"><span class="tocnumber">4.7</span> <span class="toctext">Coarse-graining and reduced representations</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-16"><a href="#Steered_molecular_dynamics_.28SMD.29"><span class="tocnumber">5</span> <span class="toctext">Steered molecular dynamics (SMD)</span></a></li>
<li class="toclevel-1 tocsection-17"><a href="#Examples_of_applications"><span class="tocnumber">6</span> <span class="toctext">Examples of applications</span></a></li>
<li class="toclevel-1 tocsection-18"><a href="#Molecular_dynamics_algorithms"><span class="tocnumber">7</span> <span class="toctext">Molecular dynamics algorithms</span></a>
<ul>
<li class="toclevel-2 tocsection-19"><a href="#Integrators"><span class="tocnumber">7.1</span> <span class="toctext">Integrators</span></a></li>
<li class="toclevel-2 tocsection-20"><a href="#Short-range_interaction_algorithms"><span class="tocnumber">7.2</span> <span class="toctext">Short-range interaction algorithms</span></a></li>
<li class="toclevel-2 tocsection-21"><a href="#Long-range_interaction_algorithms"><span class="tocnumber">7.3</span> <span class="toctext">Long-range interaction algorithms</span></a></li>
<li class="toclevel-2 tocsection-22"><a href="#Parallelization_strategies"><span class="tocnumber">7.4</span> <span class="toctext">Parallelization strategies</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-23"><a href="#Specialized_hardware_for_MD_simulations"><span class="tocnumber">8</span> <span class="toctext">Specialized hardware for MD simulations</span></a></li>
<li class="toclevel-1 tocsection-24"><a href="#Graphics_card_as_a_hardware_for_MD_simulations"><span class="tocnumber">9</span> <span class="toctext">Graphics card as a hardware for MD simulations</span></a></li>
<li class="toclevel-1 tocsection-25"><a href="#See_also"><span class="tocnumber">10</span> <span class="toctext">See also</span></a></li>
<li class="toclevel-1 tocsection-26"><a href="#References"><span class="tocnumber">11</span> <span class="toctext">References</span></a>
<ul>
<li class="toclevel-2 tocsection-27"><a href="#General_references"><span class="tocnumber">11.1</span> <span class="toctext">General references</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-28"><a href="#External_links"><span class="tocnumber">12</span> <span class="toctext">External links</span></a></li>
</ul>
</div>
<p></p>
<h2><span class="mw-headline" id="History">History</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Molecular_dynamics&amp;action=edit&amp;section=1" title="Edit section: History">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>Following the earlier successes of <a href="/wiki/Monte_Carlo_simulation" title="Monte Carlo simulation" class="mw-redirect">Monte Carlo simulations</a>, the method was developed by <a href="/wiki/Berni_Alder" title="Berni Alder">Alder</a> and Wainwright in late 50s<sup id="cite_ref-a.26w_1-1" class="reference"><a href="#cite_note-a.26w-1"><span>[</span>1<span>]</span></a></sup> and <a href="/wiki/Aneesur_Rahman" title="Aneesur Rahman">Rahman</a> (independently) in the 60s.<sup id="cite_ref-a.rahman_2-1" class="reference"><a href="#cite_note-a.rahman-2"><span>[</span>2<span>]</span></a></sup> In 1957, Alder and Wainwright used an <a href="/wiki/IBM_704" title="IBM 704">IBM 704</a> computer to simulate perfectly elastic collisions between <a href="/wiki/Hard_spheres" title="Hard spheres">hard spheres</a>.<sup id="cite_ref-a.26w_1-2" class="reference"><a href="#cite_note-a.26w-1"><span>[</span>1<span>]</span></a></sup> In 1960, Gibson et al. simulated radiation damage of solid <a href="/wiki/Copper" title="Copper">copper</a> by using a Born-Mayer type of repulsive interaction along with a cohesive surface force.<sup id="cite_ref-5" class="reference"><a href="#cite_note-5"><span>[</span>5<span>]</span></a></sup> In 1964, Rahman published landmark simulations of liquid <a href="/wiki/Argon" title="Argon">argon</a> that utilized a <a href="/wiki/Lennard-Jones_potential" title="Lennard-Jones potential">Lennard-Jones potential</a>. Calculations of system properties, such as the coefficient of <a href="/wiki/Self-diffusion" title="Self-diffusion">self-diffusion</a>, compared well with experimental data.<sup id="cite_ref-a.rahman_2-2" class="reference"><a href="#cite_note-a.rahman-2"><span>[</span>2<span>]</span></a></sup></p>
<p>Even before it became possible to simulate molecular dynamics with computers, some undertook the hard work of trying it with physical models such as macroscopic spheres. The idea was to arrange them to replicate the properties of a liquid. <a href="/wiki/J.D._Bernal" title="J.D. Bernal" class="mw-redirect">J.D. Bernal</a> said, in 1962: "... I took a number of rubber balls and stuck them together with rods of a selection of different lengths ranging from 2.75 to 4 inches. I tried to do this in the first place as casually as possible, working in my own office, being interrupted every five minutes or so and not remembering what I had done before the interruption."<sup id="cite_ref-6" class="reference"><a href="#cite_note-6"><span>[</span>6<span>]</span></a></sup></p>
<h2><span class="mw-headline" id="Areas_of_application_and_limitations">Areas of application and limitations</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Molecular_dynamics&amp;action=edit&amp;section=2" title="Edit section: Areas of application and limitations">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>Beginning in theoretical <a href="/wiki/Physics" title="Physics">physics</a>, the method of MD gained popularity in <a href="/wiki/Materials_science" title="Materials science">materials science</a> and since the 1970s also in <a href="/wiki/Biochemistry" title="Biochemistry">biochemistry</a> and <a href="/wiki/Biophysics" title="Biophysics">biophysics</a>. MD is frequently used to refine three-dimensional structures of <a href="/wiki/Protein" title="Protein">proteins</a> and other <a href="/wiki/Macromolecule" title="Macromolecule">macromolecules</a> based on experimental constraints from <a href="/wiki/X-ray_crystallography" title="X-ray crystallography">X-ray crystallography</a> or <a href="/wiki/NMR" title="NMR" class="mw-redirect">NMR</a> spectroscopy. In physics, MD is used to examine the dynamics of atomic-level phenomena that cannot be observed directly, such as thin film growth and ion-subplantation. It is also used to examine the physical properties of <a href="/wiki/Nanotechnology" title="Nanotechnology">nanotechnological</a> devices that have not been or cannot yet be created. In <a href="/wiki/Biophysics" title="Biophysics">biophysics</a> and <a href="/wiki/Structural_biology" title="Structural biology">structural biology</a>, the method is frequently applied to study the motions of biological macromolecules such as <a href="/wiki/Protein" title="Protein">proteins</a> and <a href="/wiki/Nucleic_acid" title="Nucleic acid">nucleic acids</a>, which can be useful for interpreting the results of certain biophysical experiments and for modeling interactions with other molecules, as in <a href="/wiki/Ligand_docking" title="Ligand docking" class="mw-redirect">ligand docking</a>. In principle MD can be used for <a href="/wiki/De_novo_protein_structure_prediction" title="De novo protein structure prediction">ab initio prediction</a> of <a href="/wiki/Protein_structure" title="Protein structure">protein structure</a> by simulating <a href="/wiki/Protein_folding" title="Protein folding">folding</a> of the <a href="/wiki/Polypeptide_chain" title="Polypeptide chain" class="mw-redirect">polypeptide chain</a> from <a href="/wiki/Random_coil" title="Random coil">random coil</a>.</p>
<p>The results of MD simulations can be tested through comparison to experiments that measure molecular dynamics, of which a popular method is <a href="/wiki/Nuclear_magnetic_resonance_spectroscopy" title="Nuclear magnetic resonance spectroscopy">nuclear magnetic resonance spectroscopy</a>. MD-derived structure predictions can be tested through <a href="/wiki/CASP" title="CASP">community-wide experiments</a> in protein structure prediction, although the method has historically had limited success in this area. <a href="/wiki/Michael_Levitt" title="Michael Levitt">Michael Levitt</a>, who shared the <a href="/wiki/Nobel_Prize" title="Nobel Prize">Nobel Prize</a> awarded in part for the application of MD to proteins, wrote in 1999 that <a href="/wiki/CASP" title="CASP">CASP</a> participants usually did not use the method due to "a central embarrassment of molecular mechanics, namely that energy minimization or molecular dynamics generally leads to a model that is less like the experimental structure".<sup id="cite_ref-Koehl_7-0" class="reference"><a href="#cite_note-Koehl-7"><span>[</span>7<span>]</span></a></sup> Improvements in computational resources permitting more and longer MD trajectories to be calculated, combined with modern improvements in the quality of <a href="/wiki/Force_field_(chemistry)" title="Force field (chemistry)">force field</a> parameters, have yielded some improvements in performance of both structure prediction and <a href="/wiki/Homology_model" title="Homology model" class="mw-redirect">homology model</a> refinement, without reaching the point of practical utility in these areas; most such work identifies force field parameters as a key area for further development.<sup id="cite_ref-8" class="reference"><a href="#cite_note-8"><span>[</span>8<span>]</span></a></sup><sup id="cite_ref-9" class="reference"><a href="#cite_note-9"><span>[</span>9<span>]</span></a></sup><sup id="cite_ref-10" class="reference"><a href="#cite_note-10"><span>[</span>10<span>]</span></a></sup></p>
<p>Limitations of the method are related not only to the parameter sets used, but also to the underlying <a href="/wiki/Molecular_mechanics" title="Molecular mechanics">molecular mechanics</a> <a href="/wiki/Force_field_(chemistry)" title="Force field (chemistry)">force fields</a> themselves. A single run of an MD simulation optimizes the <a href="/wiki/Potential_energy" title="Potential energy">potential energy</a>, rather than the <a href="/wiki/Gibbs_free_energy" title="Gibbs free energy">free energy</a> of the protein, meaning that all <a href="/wiki/Entropy" title="Entropy">entropic contributions</a> to <a href="/wiki/Thermodynamic_stability" title="Thermodynamic stability" class="mw-redirect">thermodynamic stability</a> of <a href="/wiki/Protein_structure" title="Protein structure">protein structure</a> are neglected. The neglected contributions include the <a href="/wiki/Conformational_entropy" title="Conformational entropy">conformational entropy</a> of the polypeptide chain (which is the main factor that destabilizes protein structure) and <a href="/wiki/Hydrophobic_interactions" title="Hydrophobic interactions" class="mw-redirect">hydrophobic interactions</a> that are known as main driving force of protein folding.<sup id="cite_ref-11" class="reference"><a href="#cite_note-11"><span>[</span>11<span>]</span></a></sup> Another important factor are intramolecular <a href="/wiki/Hydrogen_bonds" title="Hydrogen bonds" class="mw-redirect">hydrogen bonds</a>,<sup id="cite_ref-Myers_12-0" class="reference"><a href="#cite_note-Myers-12"><span>[</span>12<span>]</span></a></sup> which are not explicitly included in modern force fields, but described as Coulomb interactions of atomic point charges. This is a crude approximation because hydrogen bonds have a partially <a href="/wiki/Quantum_chemistry" title="Quantum chemistry">quantum mechanical</a> nature. Furthermore, electrostatic interactions are usually calculated using the <a href="/wiki/Dielectric_constant" title="Dielectric constant" class="mw-redirect">dielectric constant</a> of <a href="/wiki/Vacuum" title="Vacuum">vacuum</a>, although the surrounding aqueous solution has a much higher dielectric constant. Using the <a href="/wiki/Macroscopic" title="Macroscopic" class="mw-redirect">macroscopic</a> dielectric constant at short interatomic distances is questionable. Finally, van der Waals interactions in MD are usually described by <a href="/wiki/Lennard-Jones_potential" title="Lennard-Jones potential">Lennard-Jones potentials</a> based on the <a href="/wiki/Fritz_London" title="Fritz London">Fritz London</a> theory that is only applicable in vacuum. However, all types of <a href="/wiki/Van_der_Waals_force" title="Van der Waals force">van der Waals forces</a> are ultimately of electrostatic origin and therefore depend on <a href="/wiki/Dielectric_permittivity" title="Dielectric permittivity" class="mw-redirect">dielectric properties of the environment</a>.<sup id="cite_ref-Israelachvili_13-0" class="reference"><a href="#cite_note-Israelachvili-13"><span>[</span>13<span>]</span></a></sup> The direct measurement of attraction forces between different materials (as <a href="/wiki/Hamaker_constant" title="Hamaker constant">Hamaker constant</a>) shows that "the interaction between hydrocarbons across water is about 10% of that across vacuum".<sup id="cite_ref-Israelachvili_13-1" class="reference"><a href="#cite_note-Israelachvili-13"><span>[</span>13<span>]</span></a></sup> The environment-dependence of van der Waals forces is neglected in standard simulations, but can be included by developing polarizable force fields.</p>
<h2><span class="mw-headline" id="Design_constraints">Design constraints</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Molecular_dynamics&amp;action=edit&amp;section=3" title="Edit section: Design constraints">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>Design of a molecular dynamics simulation should account for the available computational power. Simulation size (n=number of particles), timestep and total time duration must be selected so that the calculation can finish within a reasonable time period. However, the simulations should be long enough to be relevant to the time scales of the natural processes being studied. To make statistically valid conclusions from the simulations, the time span simulated should match the kinetics of the natural process. Otherwise, it is analogous to making conclusions about how a human walks when only looking at less than one footstep. Most scientific publications about the dynamics of proteins and DNA use data from simulations spanning nanoseconds (10<sup>−9</sup> s) to microseconds (10<sup>−6</sup> s). To obtain these simulations, several CPU-days to CPU-years are needed. Parallel algorithms allow the load to be distributed among CPUs; an example is the spatial or force decomposition algorithm.<sup id="cite_ref-14" class="reference"><a href="#cite_note-14"><span>[</span>14<span>]</span></a></sup></p>
<p>During a classical MD simulation, the most CPU intensive task is the evaluation of the <a href="/wiki/Force_field_(chemistry)" title="Force field (chemistry)">potential</a> as a function of the particles' internal coordinates. Within that energy evaluation, the most expensive one is the non-bonded or non-covalent part. In <a href="/wiki/Big_O_notation" title="Big O notation">Big O notation</a>, common molecular dynamics simulations <a href="/wiki/Analysis_of_algorithms" title="Analysis of algorithms">scale</a> by <img class="mwe-math-fallback-image-inline tex" alt="O(n^2)" src="//upload.wikimedia.org/math/1/8/9/189317b4b935a745fcfaf95940d2b4f0.png" /> if all pair-wise <a href="/wiki/Electrostatic" title="Electrostatic" class="mw-redirect">electrostatic</a> and <a href="/wiki/Van_der_Waals_forces" title="Van der Waals forces" class="mw-redirect">van der Waals interactions</a> must be accounted for explicitly. This computational cost can be reduced by employing electrostatics methods such as <a href="/wiki/Ewald_summation" title="Ewald summation">Particle Mesh Ewald</a> ( <img class="mwe-math-fallback-image-inline tex" alt="O(n \log(n))" src="//upload.wikimedia.org/math/8/f/1/8f1bbb5f5c8a76d0d83f0cdab5eb3bf8.png" /> ), <a href="/wiki/P3M" title="P3M">P3M</a> or good spherical cutoff techniques ( <img class="mwe-math-fallback-image-inline tex" alt="O(n)" src="//upload.wikimedia.org/math/7/b/a/7ba55e7c64a9405a0b39a1107e90ca94.png" /> ).</p>
<p>Another factor that impacts total CPU time required by a simulation is the size of the integration timestep. This is the time length between evaluations of the potential. The timestep must be chosen small enough to avoid <a href="/wiki/Discretization" title="Discretization">discretization</a> errors (i.e. smaller than the fastest vibrational frequency in the system). Typical timesteps for classical MD are in the order of 1&#160;femtosecond (10<sup>−15</sup> s). This value may be extended by using algorithms such as <a href="/wiki/Constraint_algorithm" title="Constraint algorithm">SHAKE</a>, which fix the vibrations of the fastest atoms (e.g. hydrogens) into place. Multiple time scale methods have also been developed, which allow for extended times between updates of slower long-range forces.<sup id="cite_ref-Streett_15-0" class="reference"><a href="#cite_note-Streett-15"><span>[</span>15<span>]</span></a></sup><sup id="cite_ref-Tuckerman1991_16-0" class="reference"><a href="#cite_note-Tuckerman1991-16"><span>[</span>16<span>]</span></a></sup><sup id="cite_ref-Tuckerman1992_17-0" class="reference"><a href="#cite_note-Tuckerman1992-17"><span>[</span>17<span>]</span></a></sup></p>
<p>For simulating molecules in a solvent, a choice should be made between <a href="/wiki/Water_model" title="Water model">explicit solvent</a> and <a href="/wiki/Implicit_solvent" title="Implicit solvent" class="mw-redirect">implicit solvent</a>. Explicit solvent particles (such as the <a href="/wiki/TIP3P" title="TIP3P" class="mw-redirect">TIP3P</a>, SPC/E and <a href="/wiki/Flexible_SPC_water_model" title="Flexible SPC water model" class="mw-redirect">SPC-f</a> water models) must be calculated expensively by the force field, while implicit solvents use a mean-field approach. Using an explicit solvent is computationally expensive, requiring inclusion of roughly ten times more particles in the simulation. But the granularity and viscosity of explicit solvent is essential to reproduce certain properties of the solute molecules. This is especially important to reproduce <a href="/wiki/Chemical_kinetics" title="Chemical kinetics">kinetics</a>.</p>
<p>In all kinds of molecular dynamics simulations, the simulation box size must be large enough to avoid <a href="/wiki/Boundary_condition" title="Boundary condition" class="mw-redirect">boundary condition</a> artifacts. Boundary conditions are often treated by choosing fixed values at the edges (which may cause artifacts), or by employing <a href="/wiki/Periodic_boundary_conditions" title="Periodic boundary conditions">periodic boundary conditions</a> in which one side of the simulation loops back to the opposite side, mimicking a bulk phase.</p>
<h3><span class="mw-headline" id="Microcanonical_ensemble_.28NVE.29">Microcanonical ensemble (NVE)</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Molecular_dynamics&amp;action=edit&amp;section=4" title="Edit section: Microcanonical ensemble (NVE)">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>In the <b>microcanonical</b>, or <b>NVE</b> ensemble, the system is isolated from changes in moles (N), volume (V) and energy (E). It corresponds to an <a href="/wiki/Adiabatic_process" title="Adiabatic process">adiabatic process</a> with no heat exchange. A microcanonical molecular dynamics trajectory may be seen as an exchange of potential and kinetic energy, with total energy being conserved. For a system of N particles with coordinates <img class="mwe-math-fallback-image-inline tex" alt="X" src="//upload.wikimedia.org/math/0/2/1/02129bb861061d1a052c592e2dc6b383.png" /> and velocities <img class="mwe-math-fallback-image-inline tex" alt="V" src="//upload.wikimedia.org/math/5/2/0/5206560a306a2e085a437fd258eb57ce.png" />, the following pair of first order differential equations may be written in <a href="/wiki/Newton%27s_notation_for_differentiation" title="Newton's notation for differentiation" class="mw-redirect">Newton's notation</a> as</p>
<dl>
<dd><img class="mwe-math-fallback-image-inline tex" alt="F(X) = -\nabla U(X)=M\dot{V}(t)" src="//upload.wikimedia.org/math/5/3/6/536d416864aa29e806ec3abd01fecd3a.png" /></dd>
<dd><img class="mwe-math-fallback-image-inline tex" alt="V(t) = \dot{X} (t). " src="//upload.wikimedia.org/math/5/1/6/516a99cf68489b22adb6e5405ab70675.png" /></dd>
</dl>
<p>The potential energy function <img class="mwe-math-fallback-image-inline tex" alt="U(X)" src="//upload.wikimedia.org/math/b/1/d/b1decc163a355b267e647cf6a9ad59d4.png" /> of the system is a function of the particle coordinates <img class="mwe-math-fallback-image-inline tex" alt="X" src="//upload.wikimedia.org/math/0/2/1/02129bb861061d1a052c592e2dc6b383.png" />. It is referred to simply as the "potential" in physics, or the "force field" in chemistry. The first equation comes from <a href="/wiki/Newton%27s_laws_of_motion" title="Newton's laws of motion">Newton's laws</a>; the force <img class="mwe-math-fallback-image-inline tex" alt="F" src="//upload.wikimedia.org/math/8/0/0/800618943025315f869e4e1f09471012.png" /> acting on each particle in the system can be calculated as the negative gradient of <img class="mwe-math-fallback-image-inline tex" alt="U(X)" src="//upload.wikimedia.org/math/b/1/d/b1decc163a355b267e647cf6a9ad59d4.png" />.</p>
<p>For every time step, each particle's position <img class="mwe-math-fallback-image-inline tex" alt="X" src="//upload.wikimedia.org/math/0/2/1/02129bb861061d1a052c592e2dc6b383.png" /> and velocity <img class="mwe-math-fallback-image-inline tex" alt="V" src="//upload.wikimedia.org/math/5/2/0/5206560a306a2e085a437fd258eb57ce.png" /> may be integrated with a <a href="/wiki/Symplectic_integrator" title="Symplectic integrator">symplectic method</a> such as <a href="/wiki/Verlet_integration" title="Verlet integration">Verlet</a>. The time evolution of <img class="mwe-math-fallback-image-inline tex" alt="X" src="//upload.wikimedia.org/math/0/2/1/02129bb861061d1a052c592e2dc6b383.png" /> and <img class="mwe-math-fallback-image-inline tex" alt="V" src="//upload.wikimedia.org/math/5/2/0/5206560a306a2e085a437fd258eb57ce.png" /> is called a trajectory. Given the initial positions (e.g. from theoretical knowledge) and velocities (e.g. randomized Gaussian), we can calculate all future (or past) positions and velocities.</p>
<p>One frequent source of confusion is the meaning of <a href="/wiki/Temperature" title="Temperature">temperature</a> in MD. Commonly we have experience with macroscopic temperatures, which involve a huge number of particles. But temperature is a statistical quantity. If there is a large enough number of atoms, statistical temperature can be estimated from the <i>instantaneous temperature</i>, which is found by equating the kinetic energy of the system to nk<sub>B</sub>T/2 where n is the number of degrees of freedom of the system.</p>
<p>A temperature-related phenomenon arises due to the small number of atoms that are used in MD simulations. For example, consider simulating the growth of a copper film starting with a substrate containing 500 atoms and a deposition energy of 100 eV. In the real world, the 100 eV from the deposited atom would rapidly be transported through and shared among a large number of atoms (<img class="mwe-math-fallback-image-inline tex" alt="10^{10}" src="//upload.wikimedia.org/math/f/3/c/f3c75c54078aadf0d0e85e5ca4890c11.png" /> or more) with no big change in temperature. When there are only 500 atoms, however, the substrate is almost immediately vaporized by the deposition. Something similar happens in biophysical simulations. The temperature of the system in NVE is naturally raised when macromolecules such as proteins undergo exothermic conformational changes and binding.</p>
<h3><span class="mw-headline" id="Canonical_ensemble_.28NVT.29">Canonical ensemble (NVT)</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Molecular_dynamics&amp;action=edit&amp;section=5" title="Edit section: Canonical ensemble (NVT)">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>In the <a href="/wiki/Canonical_ensemble" title="Canonical ensemble">canonical ensemble</a>, amount of substance (N), volume (V) and temperature (T) are conserved. It is also sometimes called constant temperature molecular dynamics (CTMD). In NVT, the energy of endothermic and exothermic processes is exchanged with a thermostat.</p>
<p>A variety of thermostat algorithms are available to add and remove energy from the boundaries of a MD simulation in a more or less realistic way, approximating the <a href="/wiki/Canonical_ensemble" title="Canonical ensemble">canonical ensemble</a>. Popular techniques to control temperature include velocity rescaling, the <a href="/wiki/Nos%C3%A9-Hoover_thermostat" title="Nosé-Hoover thermostat" class="mw-redirect">Nosé-Hoover thermostat</a>, Nosé-Hoover chains, the <a href="/wiki/Berendsen_thermostat" title="Berendsen thermostat">Berendsen thermostat</a>, the <a href="/wiki/Andersen_thermostat" title="Andersen thermostat">Andersen thermostat</a> and <a href="/wiki/Langevin_dynamics" title="Langevin dynamics">Langevin dynamics</a>. Note that the Berendsen thermostat might introduce the <a href="/wiki/Flying_ice_cube" title="Flying ice cube">flying ice cube</a> effect, which leads to unphysical translations and rotations of the simulated system.</p>
<p>It is not trivial to obtain a <a href="/wiki/Canonical_ensemble" title="Canonical ensemble">canonical distribution</a> of conformations and velocities using these algorithms. How this depends on system size, thermostat choice, thermostat parameters, time step and integrator is the subject of many articles in the field.</p>
<h3><span class="mw-headline" id="Isothermal.E2.80.93isobaric_.28NPT.29_ensemble">Isothermal–isobaric (NPT) ensemble</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Molecular_dynamics&amp;action=edit&amp;section=6" title="Edit section: Isothermal–isobaric (NPT) ensemble">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>In the <a href="/wiki/Isothermal%E2%80%93isobaric_ensemble" title="Isothermal–isobaric ensemble">isothermal–isobaric ensemble</a>, amount of substance (N), pressure (P) and temperature (T) are conserved. In addition to a thermostat, a barostat is needed. It corresponds most closely to laboratory conditions with a flask open to ambient temperature and pressure.</p>
<p>In the simulation of biological membranes, <a href="/wiki/Isotropic" title="Isotropic" class="mw-redirect">isotropic</a> pressure control is not appropriate. For lipid bilayers, pressure control occurs under constant membrane area (NPAT) or constant surface tension "gamma" (NPγT).</p>
<h3><span class="mw-headline" id="Generalized_ensembles">Generalized ensembles</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Molecular_dynamics&amp;action=edit&amp;section=7" title="Edit section: Generalized ensembles">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>The <a href="/wiki/Replica_exchange" title="Replica exchange" class="mw-redirect">replica exchange</a> method is a generalized ensemble. It was originally created to deal with the slow dynamics of disordered spin systems. It is also called parallel tempering. The replica exchange MD (REMD) formulation<sup id="cite_ref-18" class="reference"><a href="#cite_note-18"><span>[</span>18<span>]</span></a></sup> tries to overcome the multiple-minima problem by exchanging the temperature of non-interacting replicas of the system running at several temperatures.</p>
<h2><span class="mw-headline" id="Potentials_in_MD_simulations">Potentials in MD simulations</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Molecular_dynamics&amp;action=edit&amp;section=8" title="Edit section: Potentials in MD simulations">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div role="note" class="hatnote relarticle mainarticle">Main articles: <a href="/wiki/Interatomic_potentials" title="Interatomic potentials" class="mw-redirect">Interatomic potentials</a>, <a href="/wiki/Force_field_(chemistry)" title="Force field (chemistry)">Force field</a> and <a href="/wiki/Force_field_implementation" title="Force field implementation">Force field implementation</a></div>
<p>A molecular dynamics simulation requires the definition of a <a href="/wiki/Potential_function_(disambiguation)" title="Potential function (disambiguation)" class="mw-redirect mw-disambig">potential function</a>, or a description of the terms by which the particles in the simulation will interact. In chemistry and biology this is usually referred to as a <a href="/wiki/Force_field_(chemistry)" title="Force field (chemistry)">force field</a> and in materials physics as an <a href="/wiki/Interatomic_potential" title="Interatomic potential">interatomic potential</a>. Potentials may be defined at many levels of physical accuracy; those most commonly used in chemistry are based on <a href="/wiki/Molecular_mechanics" title="Molecular mechanics">molecular mechanics</a> and embody a <a href="/wiki/Classical_mechanics" title="Classical mechanics">classical</a> treatment of particle-particle interactions that can reproduce structural and <a href="/wiki/Conformational_change" title="Conformational change">conformational changes</a> but usually cannot reproduce <a href="/wiki/Chemical_reaction" title="Chemical reaction">chemical reactions</a>.</p>
<p>The reduction from a fully quantum description to a classical potential entails two main approximations. The first one is the <a href="/wiki/Born%E2%80%93Oppenheimer_approximation" title="Born–Oppenheimer approximation">Born–Oppenheimer approximation</a>, which states that the dynamics of electrons is so fast that they can be considered to react instantaneously to the motion of their nuclei. As a consequence, they may be treated separately. The second one treats the nuclei, which are much heavier than electrons, as point particles that follow classical Newtonian dynamics. In classical molecular dynamics the effect of the electrons is approximated as a single potential energy surface, usually representing the ground state.</p>
<p>When finer levels of detail are required, potentials based on <a href="/wiki/Quantum_mechanics" title="Quantum mechanics">quantum mechanics</a> are used; some techniques attempt to create hybrid <a href="/wiki/QM/MM" title="QM/MM">classical/quantum</a> potentials where the bulk of the system is treated classically but a small region is treated as a quantum system, usually undergoing a chemical transformation.</p>
<h3><span class="mw-headline" id="Empirical_potentials">Empirical potentials</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Molecular_dynamics&amp;action=edit&amp;section=9" title="Edit section: Empirical potentials">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>Empirical potentials used in chemistry are frequently called <a href="/wiki/Force_field_(chemistry)" title="Force field (chemistry)">force fields</a>, while those used in materials physics are called <a href="/wiki/Interatomic_potentials" title="Interatomic potentials" class="mw-redirect">interatomic potentials</a>.</p>
<p>Most <a href="/wiki/Force_field_(chemistry)" title="Force field (chemistry)">force fields</a> in chemistry are empirical and consist of a summation of bonded forces associated with <a href="/wiki/Chemical_bond" title="Chemical bond">chemical bonds</a>, bond angles, and bond <a href="/wiki/Dihedral_angle" title="Dihedral angle">dihedrals</a>, and non-bonded forces associated with <a href="/wiki/Van_der_Waals_force" title="Van der Waals force">van der Waals forces</a> and <a href="/wiki/Electrostatic_charge" title="Electrostatic charge" class="mw-redirect">electrostatic charge</a>. Empirical potentials represent quantum-mechanical effects in a limited way through ad-hoc functional approximations. These potentials contain free parameters such as <a href="/wiki/Electrostatic_charge" title="Electrostatic charge" class="mw-redirect">atomic charge</a>, van der Waals parameters reflecting estimates of <a href="/wiki/Atomic_radius" title="Atomic radius">atomic radius</a>, and equilibrium <a href="/wiki/Bond_length" title="Bond length">bond length</a>, angle, and dihedral; these are obtained by fitting against detailed electronic calculations (quantum chemical simulations) or experimental physical properties such as <a href="/wiki/Young%27s_modulus" title="Young's modulus">elastic constants</a>, lattice parameters and <a href="/wiki/Spectroscopy" title="Spectroscopy">spectroscopic</a> measurements.</p>
<p>Because of the non-local nature of non-bonded interactions, they involve at least weak interactions between all particles in the system. Its calculation is normally the bottleneck in the speed of MD simulations. To lower the computational cost, <a href="/wiki/Force_field_(chemistry)" title="Force field (chemistry)">force fields</a> employ numerical approximations such as shifted cutoff radii, <a href="/wiki/Reaction_field_method" title="Reaction field method">reaction field</a> algorithms, <a href="/wiki/Particle_mesh_Ewald" title="Particle mesh Ewald" class="mw-redirect">particle mesh Ewald</a> summation, or the newer <a href="/wiki/P3M" title="P3M">Particle-Particle Particle Mesh</a> (P3M).</p>
<p>Chemistry force fields commonly employ preset bonding arrangements (an exception being <i><a href="/wiki/Ab_initio_quantum_chemistry_methods" title="Ab initio quantum chemistry methods">ab-initio</a></i> dynamics), and thus are unable to model the process of chemical bond breaking and reactions explicitly. On the other hand, many of the potentials used in physics, such as those based on the <a href="/wiki/Bond_order_potential" title="Bond order potential">bond order formalism</a> can describe several different coordinations of a system and bond breaking.<sup id="cite_ref-19" class="reference"><a href="#cite_note-19"><span>[</span>19<span>]</span></a></sup><sup id="cite_ref-20" class="reference"><a href="#cite_note-20"><span>[</span>20<span>]</span></a></sup> Examples of such potentials include the <a href="/wiki/Brenner_potential" title="Brenner potential" class="mw-redirect">Brenner potential</a><sup id="cite_ref-21" class="reference"><a href="#cite_note-21"><span>[</span>21<span>]</span></a></sup> for hydrocarbons and its further developments for the C-Si-H<sup id="cite_ref-22" class="reference"><a href="#cite_note-22"><span>[</span>22<span>]</span></a></sup> and C-O-H<sup id="cite_ref-23" class="reference"><a href="#cite_note-23"><span>[</span>23<span>]</span></a></sup> systems. The <a href="/wiki/ReaxFF" title="ReaxFF">ReaxFF</a> potential<sup id="cite_ref-24" class="reference"><a href="#cite_note-24"><span>[</span>24<span>]</span></a></sup> can be considered a fully reactive hybrid between bond order potentials and chemistry force fields.</p>
<h3><span class="mw-headline" id="Pair_potentials_versus_many-body_potentials">Pair potentials versus many-body potentials</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Molecular_dynamics&amp;action=edit&amp;section=10" title="Edit section: Pair potentials versus many-body potentials">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>The potential functions representing the non-bonded energy are formulated as a sum over interactions between the particles of the system. The simplest choice, employed in many popular <a href="/wiki/Force_field_(physics)" title="Force field (physics)">force field (physics)</a>, is the "pair potential", in which the total potential energy can be calculated from the sum of energy contributions between pairs of atoms. An example of such a pair potential is the non-bonded <a href="/wiki/Lennard%E2%80%93Jones_potential" title="Lennard–Jones potential" class="mw-redirect">Lennard–Jones potential</a> (also known as the 6–12 potential), used for calculating van der Waals forces.</p>
<dl>
<dd><img class="mwe-math-fallback-image-inline tex" alt="
U(r) = 4\varepsilon \left[ \left(\frac{\sigma}{r}\right)^{12} - \left(\frac{\sigma}{r}\right)^{6} \right]
" src="//upload.wikimedia.org/math/1/8/8/1883782f03e940f640cd936f6f68adc3.png" /></dd>
</dl>
<p>Another example is the Born (ionic) model of the ionic lattice. The first term in the next equation is <a href="/wiki/Coulomb%27s_law" title="Coulomb's law">Coulomb's law</a> for a pair of ions, the second term is the short-range repulsion explained by Pauli's exclusion principle and the final term is the dispersion interaction term. Usually, a simulation only includes the dipolar term, although sometimes the quadrupolar term is included as well.(Usually known as <a href="/wiki/Buckingham_potential" title="Buckingham potential">Buckingham potential</a> model)</p>
<dl>
<dd><img class="mwe-math-fallback-image-inline tex" alt="U_{ij}(r_{ij}) = \frac {z_i z_j}{4 \pi \epsilon_0} \frac {1}{r_{ij}} + A_l \exp \frac {-r_{ij}}{p_l} + C_l r_{ij}^{-n_l} + \cdots
" src="//upload.wikimedia.org/math/3/6/8/3681fd396994c4369c812743796b2b75.png" /></dd>
</dl>
<p>In <a href="/wiki/Many_body_problem" title="Many body problem" class="mw-redirect">many-body potentials</a>, the potential energy includes the effects of three or more particles interacting with each other .<sup id="cite_ref-ReferenceA_25-0" class="reference"><a href="#cite_note-ReferenceA-25"><span>[</span>25<span>]</span></a></sup> In simulations with pairwise potentials, global interactions in the system also exist, but they occur only through pairwise terms. In many-body potentials, the potential energy cannot be found by a sum over pairs of atoms, as these interactions are calculated explicitly as a combination of higher-order terms. In the statistical view, the dependency between the variables cannot in general be expressed using only pairwise products of the degrees of freedom. For example, the <a href="/wiki/Tersoff_potential" title="Tersoff potential" class="mw-redirect">Tersoff potential</a>,<sup id="cite_ref-26" class="reference"><a href="#cite_note-26"><span>[</span>26<span>]</span></a></sup> which was originally used to simulate <a href="/wiki/Carbon" title="Carbon">carbon</a>, <a href="/wiki/Silicon" title="Silicon">silicon</a> and <a href="/wiki/Germanium" title="Germanium">germanium</a> and has since been used for a wide range of other materials, involves a sum over groups of three atoms, with the angles between the atoms being an important factor in the potential. Other examples are the <a href="/wiki/Embedded_atom_model" title="Embedded atom model">embedded-atom method</a> (EAM),<sup id="cite_ref-27" class="reference"><a href="#cite_note-27"><span>[</span>27<span>]</span></a></sup> the EDIP,<sup id="cite_ref-ReferenceA_25-1" class="reference"><a href="#cite_note-ReferenceA-25"><span>[</span>25<span>]</span></a></sup> and the Tight-Binding Second Moment Approximation (TBSMA) potentials,<sup id="cite_ref-28" class="reference"><a href="#cite_note-28"><span>[</span>28<span>]</span></a></sup> where the electron density of states in the region of an atom is calculated from a sum of contributions from surrounding atoms, and the potential energy contribution is then a function of this sum.</p>
<h3><span class="mw-headline" id="Semi-empirical_potentials">Semi-empirical potentials</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Molecular_dynamics&amp;action=edit&amp;section=11" title="Edit section: Semi-empirical potentials">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p><a href="/wiki/Semi-empirical_quantum_chemistry_methods" title="Semi-empirical quantum chemistry methods" class="mw-redirect">Semi-empirical</a> potentials make use of the matrix representation from quantum mechanics. However, the values of the matrix elements are found through empirical formulae that estimate the degree of overlap of specific atomic orbitals. The matrix is then diagonalized to determine the occupancy of the different atomic orbitals, and empirical formulae are used once again to determine the energy contributions of the orbitals.</p>
<p>There are a wide variety of semi-empirical potentials, known as <a href="/wiki/Tight_binding_(physics)" title="Tight binding (physics)" class="mw-redirect">tight-binding</a> potentials, which vary according to the atoms being modeled.</p>
<h3><span class="mw-headline" id="Polarizable_potentials">Polarizable potentials</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Molecular_dynamics&amp;action=edit&amp;section=12" title="Edit section: Polarizable potentials">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<div role="note" class="hatnote relarticle mainarticle">Main article: <a href="/wiki/Force_field_(chemistry)" title="Force field (chemistry)">Force field</a></div>
<p>Most classical force fields implicitly include the effect of <a href="/wiki/Polarizability" title="Polarizability">polarizability</a>, e.g. by scaling up the partial charges obtained from quantum chemical calculations. These partial charges are stationary with respect to the mass of the atom. But molecular dynamics simulations can explicitly model polarizability with the introduction of induced dipoles through different methods, such as <a href="/wiki/Drude_particle" title="Drude particle">Drude particles</a> or fluctuating charges. This allows for a dynamic redistribution of charge between atoms which responds to the local chemical environment.</p>
<p>For many years, polarizable MD simulations have been touted as the next generation. For homogenous liquids such as water, increased accuracy has been achieved through the inclusion of polarizability.<sup id="cite_ref-Lamoureux3_29-0" class="reference"><a href="#cite_note-Lamoureux3-29"><span>[</span>29<span>]</span></a></sup><sup id="cite_ref-Sokhan2015_30-0" class="reference"><a href="#cite_note-Sokhan2015-30"><span>[</span>30<span>]</span></a></sup><sup id="cite_ref-Cipcigan_31-0" class="reference"><a href="#cite_note-Cipcigan-31"><span>[</span>31<span>]</span></a></sup> Some promising results have also been achieved for proteins.<sup id="cite_ref-Patel2004b_32-0" class="reference"><a href="#cite_note-Patel2004b-32"><span>[</span>32<span>]</span></a></sup> However, it is still uncertain how to best approximate polarizability in a simulation.<sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">[<i><a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed"><span title="This claim needs references to reliable sources. (April 2009)">citation needed</span></a></i>]</sup></p>
<h3><span class="mw-headline" id="Potentials_in_ab-initio_methods">Potentials in <i>ab-initio</i> methods</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Molecular_dynamics&amp;action=edit&amp;section=13" title="Edit section: Potentials in ab-initio methods">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<div role="note" class="hatnote relarticle mainarticle">Main articles: <a href="/wiki/Quantum_Chemistry" title="Quantum Chemistry" class="mw-redirect">Quantum Chemistry</a> and <a href="/wiki/List_of_quantum_chemistry_and_solid_state_physics_software" title="List of quantum chemistry and solid state physics software" class="mw-redirect">List of quantum chemistry and solid state physics software</a></div>
<p>In classical molecular dynamics, a single potential energy surface (usually the ground state) is represented in the force field. This is a consequence of the <a href="/wiki/Born%E2%80%93Oppenheimer_approximation" title="Born–Oppenheimer approximation">Born–Oppenheimer approximation</a>. In excited states, chemical reactions or when a more accurate representation is needed, electronic behavior can be obtained from first principles by using a quantum mechanical method, such as <a href="/wiki/Density_functional_theory" title="Density functional theory">density functional theory</a>. This is known as Ab Initio Molecular Dynamics (AIMD). Due to the cost of treating the electronic degrees of freedom, the computational cost of this simulations is much higher than classical molecular dynamics. This implies that AIMD is limited to smaller systems and shorter periods of time.</p>
<p><i><a href="/wiki/Ab_initio" title="Ab initio">Ab-initio</a></i> <a href="/wiki/Quantum_chemistry" title="Quantum chemistry">quantum-mechanical</a> methods may be used to calculate the <a href="/wiki/Potential_energy_surface" title="Potential energy surface">potential energy</a> of a system on the fly, as needed for conformations in a trajectory. This calculation is usually made in the close neighborhood of the <a href="/wiki/Reaction_coordinate" title="Reaction coordinate">reaction coordinate</a>. Although various approximations may be used, these are based on theoretical considerations, not on empirical fitting. <i>Ab-initio</i> calculations produce a vast amount of information that is not available from empirical methods, such as density of electronic states or other electronic properties. A significant advantage of using <i>ab-initio</i> methods is the ability to study reactions that involve breaking or formation of covalent bonds, which correspond to multiple electronic states.</p>
<h3><span class="mw-headline" id="Hybrid_QM.2FMM">Hybrid QM/MM</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Molecular_dynamics&amp;action=edit&amp;section=14" title="Edit section: Hybrid QM/MM">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<div role="note" class="hatnote relarticle mainarticle">Main article: <a href="/wiki/QM/MM" title="QM/MM">QM/MM</a></div>
<p>QM (quantum-mechanical) methods are very powerful. However, they are computationally expensive, while the MM (classical or molecular mechanics) methods are fast but suffer from several limitations (require extensive parameterization; energy estimates obtained are not very accurate; cannot be used to simulate reactions where covalent bonds are broken/formed; and are limited in their abilities for providing accurate details regarding the chemical environment). A new class of method has emerged that combines the good points of QM (accuracy) and MM (speed) calculations. These methods are known as mixed or hybrid quantum-mechanical and molecular mechanics methods (hybrid QM/MM).<sup id="cite_ref-33" class="reference"><a href="#cite_note-33"><span>[</span>33<span>]</span></a></sup></p>
<p>The most important advantage of hybrid QM/MM method is the speed. The cost of doing classical molecular dynamics (MM) in the most straightforward case scales O(n<sup>2</sup>), where n is the number of atoms in the system. This is mainly due to electrostatic interactions term (every particle interacts with every other particle). However, use of cutoff radius, periodic pair-list updates and more recently the variations of the particle-mesh Ewald's (PME) method has reduced this to between O(n) to O(n<sup>2</sup>). In other words, if a system with twice as many atoms is simulated then it would take between two to four times as much computing power. On the other hand the simplest <i>ab-initio</i> calculations typically scale O(n<sup>3</sup>) or worse (Restricted <a href="/wiki/Hartree%E2%80%93Fock" title="Hartree–Fock" class="mw-redirect">Hartree–Fock</a> calculations have been suggested to scale ~O(n<sup>2.7</sup>)). To overcome the limitation, a small part of the system is treated quantum-mechanically (typically active-site of an enzyme) and the remaining system is treated classically.</p>
<p>In more sophisticated implementations, QM/MM methods exist to treat both light nuclei susceptible to quantum effects (such as hydrogens) and electronic states. This allows generation of hydrogen wave-functions (similar to electronic wave-functions). This methodology has been useful in investigating phenomena such as hydrogen tunneling. One example where QM/MM methods have provided new discoveries is the calculation of hydride transfer in the enzyme liver <a href="/wiki/Alcohol_dehydrogenase" title="Alcohol dehydrogenase">alcohol dehydrogenase</a>. In this case, <a href="/wiki/Quantum_tunneling" title="Quantum tunneling" class="mw-redirect">tunneling</a> is important for the hydrogen, as it determines the reaction rate.<sup id="cite_ref-34" class="reference"><a href="#cite_note-34"><span>[</span>34<span>]</span></a></sup></p>
<h3><span class="mw-headline" id="Coarse-graining_and_reduced_representations">Coarse-graining and reduced representations</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Molecular_dynamics&amp;action=edit&amp;section=15" title="Edit section: Coarse-graining and reduced representations">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>At the other end of the detail scale are coarse-grained and lattice models. Instead of explicitly representing every atom of the system, one uses "pseudo-atoms" to represent groups of atoms. MD simulations on very large systems may require such large computer resources that they cannot easily be studied by traditional all-atom methods. Similarly, simulations of processes on long timescales (beyond about 1 microsecond) are prohibitively expensive, because they require so many time steps. In these cases, one can sometimes tackle the problem by using reduced representations, which are also called coarse-grained models.</p>
<p>Examples for coarse graining (CG) methods are discontinuous molecular dynamics (CG-DMD)<sup id="cite_ref-35" class="reference"><a href="#cite_note-35"><span>[</span>35<span>]</span></a></sup><sup id="cite_ref-36" class="reference"><a href="#cite_note-36"><span>[</span>36<span>]</span></a></sup> and Go-models.<sup id="cite_ref-37" class="reference"><a href="#cite_note-37"><span>[</span>37<span>]</span></a></sup> Coarse-graining is done sometimes taking larger pseudo-atoms. Such united atom approximations have been used in MD simulations of biological membranes. Implementation of such approach on systems where electrical properties are of interest can be challenging owing to the difficulty of using a proper charge distribution on the pseudo-atoms.<sup id="cite_ref-38" class="reference"><a href="#cite_note-38"><span>[</span>38<span>]</span></a></sup> The aliphatic tails of lipids are represented by a few pseudo-atoms by gathering 2 to 4 methylene groups into each pseudo-atom.</p>
<p>The parameterization of these very coarse-grained models must be done empirically, by matching the behavior of the model to appropriate experimental data or all-atom simulations. Ideally, these parameters should account for both <a href="/wiki/Enthalpy" title="Enthalpy">enthalpic</a> and <a href="/wiki/Entropy" title="Entropy">entropic</a> contributions to free energy in an implicit way. When coarse-graining is done at higher levels, the accuracy of the dynamic description may be less reliable. But very coarse-grained models have been used successfully to examine a wide range of questions in structural biology, liquid crystal organization, and polymer glasses.</p>
<p>Examples of applications of coarse-graining:</p>
<ul>
<li><a href="/wiki/Protein_folding" title="Protein folding">protein folding</a> studies are often carried out using a single (or a few) pseudo-atoms per amino acid;</li>
<li><a href="/wiki/Liquid_crystal" title="Liquid crystal">liquid crystal</a> phase transitions have been examined in confined geometries and/or during flow using the <a href="/w/index.php?title=Gay-Berne_potential&amp;action=edit&amp;redlink=1" class="new" title="Gay-Berne potential (page does not exist)">Gay-Berne potential</a>, which describes anisotropic species;</li>
<li><a href="/wiki/Polymer" title="Polymer">Polymer</a> glasses during deformation have been studied using simple harmonic or <a href="/wiki/FENE" title="FENE">FENE</a> springs to connect spheres described by the <a href="/wiki/Lennard-Jones_potential" title="Lennard-Jones potential">Lennard-Jones potential</a>;</li>
<li><a href="/wiki/Supercoiling" title="Supercoiling" class="mw-redirect">DNA supercoiling</a> has been investigated using 1–3 pseudo-atoms per basepair, and at even lower resolution;</li>
<li>Packaging of <a href="/wiki/DNA" title="DNA">double-helical DNA</a> into <a href="/wiki/Bacteriophage" title="Bacteriophage">bacteriophage</a> has been investigated with models where one pseudo-atom represents one turn (about 10 basepairs) of the double helix;</li>
<li>RNA structure in the <a href="/wiki/Ribosome" title="Ribosome">ribosome</a> and other large systems has been modeled with one pseudo-atom per nucleotide.</li>
</ul>
<p>The simplest form of coarse-graining is the "united atom" (sometimes called "extended atom") and was used in most early MD simulations of proteins, lipids and nucleic acids. For example, instead of treating all four atoms of a CH<sub>3</sub> methyl group explicitly (or all three atoms of CH<sub>2</sub> methylene group), one represents the whole group with a single pseudo-atom. This pseudo-atom must, of course, be properly parameterized so that its van der Waals interactions with other groups have the proper distance-dependence. Similar considerations apply to the bonds, angles, and torsions in which the pseudo-atom participates. In this kind of united atom representation, one typically eliminates all explicit hydrogen atoms except those that have the capability to participate in hydrogen bonds ("polar hydrogens"). An example of this is the <a href="/wiki/CHARMM" title="CHARMM">Charmm 19</a> force-field.</p>
<p>The polar hydrogens are usually retained in the model, because proper treatment of hydrogen bonds requires a reasonably accurate description of the directionality and the electrostatic interactions between the donor and acceptor groups. A hydroxyl group, for example, can be both a hydrogen bond donor and a hydrogen bond acceptor, and it would be impossible to treat this with a single OH pseudo-atom. Note that about half the atoms in a protein or nucleic acid are non-polar hydrogens, so the use of united atoms can provide a substantial savings in computer time.</p>
<h2><span class="mw-headline" id="Steered_molecular_dynamics_.28SMD.29">Steered molecular dynamics (SMD)</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Molecular_dynamics&amp;action=edit&amp;section=16" title="Edit section: Steered molecular dynamics (SMD)">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>Steered molecular dynamics (SMD) simulations, or force probe simulations, apply forces to a protein in order to manipulate its structure by pulling it along desired degrees of freedom. These experiments can be used to reveal structural changes in a protein at the atomic level. SMD is often used to simulate events such as mechanical unfolding or stretching.<sup id="cite_ref-Nienhaus_39-0" class="reference"><a href="#cite_note-Nienhaus-39"><span>[</span>39<span>]</span></a></sup></p>
<p>There are two typical protocols of SMD: one in which pulling velocity is held constant and one in which applied force is constant. Typically, part of the studied system (e.g. an atom in a protein) is restrained by a harmonic potential. Forces are then applied to specific atoms at either a constant velocity or a constant force. <a href="/wiki/Umbrella_sampling" title="Umbrella sampling">Umbrella sampling</a> is used to move the system along the desired reaction coordinate by varying, for example, the forces, distances, and angles manipulated in the simulation. Through umbrella sampling, all of the system's configurations—both high-energy and low-energy—are adequately sampled. Then, each configuration's change in free energy can be calculated as the <a href="/wiki/Potential_of_mean_force" title="Potential of mean force">potential of mean force</a>.<sup id="cite_ref-Leszczy.C5.84ski_40-0" class="reference"><a href="#cite_note-Leszczy.C5.84ski-40"><span>[</span>40<span>]</span></a></sup> A popular method of computing PMF is through the weighted histogram analysis method (WHAM), which analyzes a series of umbrella sampling simulations.<sup id="cite_ref-41" class="reference"><a href="#cite_note-41"><span>[</span>41<span>]</span></a></sup><sup id="cite_ref-42" class="reference"><a href="#cite_note-42"><span>[</span>42<span>]</span></a></sup></p>
<h2><span class="mw-headline" id="Examples_of_applications">Examples of applications</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Molecular_dynamics&amp;action=edit&amp;section=17" title="Edit section: Examples of applications">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div class="thumb tright">
<div class="thumbinner" style="width:222px;"><a href="/wiki/File:MD_rotor_250K_1ns.gif" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/thumb/b/b6/MD_rotor_250K_1ns.gif/220px-MD_rotor_250K_1ns.gif" width="220" height="230" class="thumbimage" srcset="//upload.wikimedia.org/wikipedia/commons/b/b6/MD_rotor_250K_1ns.gif 1.5x, //upload.wikimedia.org/wikipedia/commons/b/b6/MD_rotor_250K_1ns.gif 2x" data-file-width="300" data-file-height="314" /></a>
<div class="thumbcaption">
<div class="magnify"><a href="/wiki/File:MD_rotor_250K_1ns.gif" class="internal" title="Enlarge"></a></div>
Molecular dynamics simulation of a <a href="/wiki/Synthetic_molecular_motor" title="Synthetic molecular motor">synthetic molecular motor</a> composed of three molecules in a nanopore (outer diameter 6.7 nm) at 250 K.</div>
</div>
</div>
<p>Molecular dynamics is used in many fields of science.</p>
<ul>
<li>First MD simulation of a simplified biological folding process was published in 1975. Its simulation published in Nature paved the way for the vast area of modern computational protein-folding.<sup id="cite_ref-43" class="reference"><a href="#cite_note-43"><span>[</span>43<span>]</span></a></sup></li>
<li>First MD simulation of a biological process was published in 1976. Its simulation published in Nature paved the way for understanding protein motion as essential in function and not just accessory.<sup id="cite_ref-44" class="reference"><a href="#cite_note-44"><span>[</span>44<span>]</span></a></sup></li>
<li>MD is the standard method to treat <a href="/wiki/Collision_cascade" title="Collision cascade">collision cascades</a> in the heat spike regime, i.e. the effects that energetic <a href="/wiki/Neutron" title="Neutron">neutron</a> and <a href="/wiki/Ion_irradiation" title="Ion irradiation" class="mw-redirect">ion irradiation</a> have on solids and solid surfaces.<sup id="cite_ref-Ave98_45-0" class="reference"><a href="#cite_note-Ave98-45"><span>[</span>45<span>]</span></a></sup><sup id="cite_ref-Smith_46-0" class="reference"><a href="#cite_note-Smith-46"><span>[</span>46<span>]</span></a></sup></li>
<li>MD simulations were successfully applied to predict the molecular basis of the most common protein mutation N370S, causing <a href="/wiki/Gaucher_Disease" title="Gaucher Disease" class="mw-redirect">Gaucher Disease</a>.<sup id="cite_ref-47" class="reference"><a href="#cite_note-47"><span>[</span>47<span>]</span></a></sup> In a follow-up publication it was shown that these blind predictions show a surprisingly high correlation with experimental work on the same mutant, published independently at a later point.<sup id="cite_ref-48" class="reference"><a href="#cite_note-48"><span>[</span>48<span>]</span></a></sup></li>
<li>MD simulations have been used to investigate the effect of surface charges on disjoining pressure of thin water films on metal surfaces.<sup id="cite_ref-49" class="reference"><a href="#cite_note-49"><span>[</span>49<span>]</span></a></sup></li>
<li>MD simulations are used along with <a href="/wiki/Multislice" title="Multislice">multislice</a> image simulations to understand <a href="/wiki/Transmission_electron_microscope" title="Transmission electron microscope" class="mw-redirect">transmission electron microscope</a> image features<sup id="cite_ref-50" class="reference"><a href="#cite_note-50"><span>[</span>50<span>]</span></a></sup></li>
</ul>
<p>The following biophysical examples illustrate notable efforts to produce simulations of a systems of very large size (a complete virus) or very long simulation times (up to 1.112 milliseconds):</p>
<ul>
<li>MD simulation of the complete <a href="/wiki/Tobacco_mosaic_satellite_virus" title="Tobacco mosaic satellite virus" class="mw-redirect">satellite tobacco mosaic virus</a> (<b>STMV</b>) (2006, Size: 1 million atoms, Simulation time: 50 ns, program: <a href="/wiki/NAMD" title="NAMD">NAMD</a>) This virus is a small, icosahedral plant virus that worsens the symptoms of infection by Tobacco Mosaic Virus (TMV). Molecular dynamics simulations were used to probe the mechanisms of <a href="/wiki/Virus#Structure" title="Virus">viral assembly</a>. The entire STMV particle consists of 60 identical copies of a single protein that make up the viral <a href="/wiki/Capsid" title="Capsid">capsid</a> (coating), and a 1063 nucleotide single stranded RNA <a href="/wiki/Genome" title="Genome">genome</a>. One key finding is that the capsid is very unstable when there is no RNA inside. The simulation would take a single 2006 desktop computer around 35 years to complete. It was thus done in many processors in parallel with continuous communication between them.<sup id="cite_ref-51" class="reference"><a href="#cite_note-51"><span>[</span>51<span>]</span></a></sup></li>
<li>Folding simulations of the <a href="/wiki/Villin" title="Villin">Villin</a> Headpiece in all-atom detail (2006, Size: 20,000 atoms; Simulation time: 500 µs = 500,000 ns, Program: <a href="/wiki/Folding@home" title="Folding@home">Folding@home</a>) This simulation was run in 200,000 CPU's of participating personal computers around the world. These computers had the Folding@home program installed, a large-scale distributed computing effort coordinated by <a href="/wiki/Vijay_Pande" title="Vijay Pande" class="mw-redirect">Vijay Pande</a> at Stanford University. The kinetic properties of the Villin Headpiece protein were probed by using many independent, short trajectories run by CPU's without continuous real-time communication. One technique employed was the Pfold value analysis, which measures the probability of folding before unfolding of a specific starting conformation. Pfold gives information about <a href="/wiki/Phi_value_analysis" title="Phi value analysis">transition state</a> structures and an ordering of conformations along the <a href="/wiki/Protein_folding" title="Protein folding">folding pathway</a>. Each trajectory in a Pfold calculation can be relatively short, but many independent trajectories are needed.<sup id="cite_ref-52" class="reference"><a href="#cite_note-52"><span>[</span>52<span>]</span></a></sup></li>
<li>Long continuous-trajectory simulations have been performed on <a href="/wiki/Anton_(computer)" title="Anton (computer)">Anton</a>, a massively parallel supercomputer designed and built around custom <a href="/wiki/ASIC" title="ASIC" class="mw-redirect">ASIC</a>s and interconnects by <a href="/wiki/D._E._Shaw_Research" title="D. E. Shaw Research">D. E. Shaw Research</a>. The longest published result of a simulation performed using Anton is a 1.112-millisecond simulation of NTL9 at 355 K; a second, independent 1.073-millisecond simulation of this configuration was also performed (as well as numerous additional simulations of over 250 µs continuous chemical time).<sup id="cite_ref-DESRES-Science2011_53-0" class="reference"><a href="#cite_note-DESRES-Science2011-53"><span>[</span>53<span>]</span></a></sup> In "How Fast-Folding Proteins Fold", researchers Kresten Lindorff-Larsen, Stefano Piana, Ron O. Dror, and <a href="/wiki/David_E._Shaw" title="David E. Shaw">David E. Shaw</a> discuss "the results of atomic-level molecular dynamics simulations, over periods ranging between 100 μs and 1 ms, that reveal a set of common principles underlying the folding of 12 structurally diverse proteins." Examination of these diverse long trajectories, enabled by specialized, custom hardware, allow them to conclude that "In most cases, folding follows a single dominant route in which elements of the native structure appear in an order highly correlated with their propensity to form in the unfolded state."<sup id="cite_ref-DESRES-Science2011_53-1" class="reference"><a href="#cite_note-DESRES-Science2011-53"><span>[</span>53<span>]</span></a></sup> In a separate study, Anton was used to conduct a 1.013-millisecond simulation of the native-state dynamics of bovine pancreatic trypsin inhibitor (BPTI) at 300 K.<sup id="cite_ref-54" class="reference"><a href="#cite_note-54"><span>[</span>54<span>]</span></a></sup></li>
<li>These molecular simulations have been used to understand the material removal mechanisms, effects of tool geometry, temperature, and process parameters such as cutting speed and cutting forces.<sup id="cite_ref-55" class="reference"><a href="#cite_note-55"><span>[</span>55<span>]</span></a></sup> It was also used to investigate the mechanisms behind the exfoliation of few layers of graphene<sup id="cite_ref-Jaya2011_56-0" class="reference"><a href="#cite_note-Jaya2011-56"><span>[</span>56<span>]</span></a></sup><sup id="cite_ref-57" class="reference"><a href="#cite_note-57"><span>[</span>57<span>]</span></a></sup> and carbon nanoscrolls.</li>
</ul>
<h2><span class="mw-headline" id="Molecular_dynamics_algorithms">Molecular dynamics algorithms</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Molecular_dynamics&amp;action=edit&amp;section=18" title="Edit section: Molecular dynamics algorithms">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<ul>
<li><a href="/wiki/Screened_Coulomb_Potentials_Implicit_Solvent_Model" title="Screened Coulomb Potentials Implicit Solvent Model">Screened Coulomb Potentials Implicit Solvent Model</a></li>
</ul>
<h3><span class="mw-headline" id="Integrators">Integrators</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Molecular_dynamics&amp;action=edit&amp;section=19" title="Edit section: Integrators">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<ul>
<li><a href="/wiki/Symplectic_integrator" title="Symplectic integrator">Symplectic integrator</a></li>
<li><a href="/wiki/Verlet-Stoermer_integration" title="Verlet-Stoermer integration" class="mw-redirect">Verlet-Stoermer integration</a></li>
<li><a href="/wiki/Runge%E2%80%93Kutta_integration" title="Runge–Kutta integration" class="mw-redirect">Runge–Kutta integration</a></li>
<li><a href="/wiki/Beeman%27s_algorithm" title="Beeman's algorithm">Beeman's algorithm</a></li>
<li><a href="/wiki/Constraint_algorithm" title="Constraint algorithm">Constraint algorithms</a> (for constrained systems)</li>
</ul>
<h3><span class="mw-headline" id="Short-range_interaction_algorithms">Short-range interaction algorithms</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Molecular_dynamics&amp;action=edit&amp;section=20" title="Edit section: Short-range interaction algorithms">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<ul>
<li><a href="/wiki/Cell_lists" title="Cell lists">Cell lists</a></li>
<li><a href="/wiki/Verlet_list" title="Verlet list">Verlet list</a></li>
<li>Bonded interactions</li>
</ul>
<h3><span class="mw-headline" id="Long-range_interaction_algorithms">Long-range interaction algorithms</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Molecular_dynamics&amp;action=edit&amp;section=21" title="Edit section: Long-range interaction algorithms">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<ul>
<li><a href="/wiki/Ewald_summation" title="Ewald summation">Ewald summation</a></li>
<li><a href="/wiki/Particle_mesh_Ewald" title="Particle mesh Ewald" class="mw-redirect">Particle mesh Ewald</a> (PME)</li>
<li>Particle–particle particle mesh <a href="/wiki/P3M" title="P3M">P3M</a></li>
<li><a href="/wiki/Shifted_force_method" title="Shifted force method">Shifted force method</a></li>
</ul>
<h3><span class="mw-headline" id="Parallelization_strategies">Parallelization strategies</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Molecular_dynamics&amp;action=edit&amp;section=22" title="Edit section: Parallelization strategies">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<ul>
<li><a href="/wiki/Domain_decomposition_method" title="Domain decomposition method" class="mw-redirect">Domain decomposition method</a> (Distribution of system data for <a href="/wiki/Parallel_computing" title="Parallel computing">parallel computing</a>)</li>
</ul>
<h2><span class="mw-headline" id="Specialized_hardware_for_MD_simulations">Specialized hardware for MD simulations</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Molecular_dynamics&amp;action=edit&amp;section=23" title="Edit section: Specialized hardware for MD simulations">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<ul>
<li><a href="/wiki/Anton_(computer)" title="Anton (computer)">Anton</a> – A specialized, massively parallel supercomputer designed to execute MD simulations.</li>
<li><a href="/wiki/MDGRAPE" title="MDGRAPE" class="mw-redirect">MDGRAPE</a> – A special purpose system built for molecular dynamics simulations, especially protein structure prediction.</li>
</ul>
<h2><span class="mw-headline" id="Graphics_card_as_a_hardware_for_MD_simulations">Graphics card as a hardware for MD simulations</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Molecular_dynamics&amp;action=edit&amp;section=24" title="Edit section: Graphics card as a hardware for MD simulations">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<ul>
<li><a href="/wiki/Molecular_modeling_on_GPU" title="Molecular modeling on GPU" class="mw-redirect">Molecular modeling on GPU</a></li>
</ul>
<h2><span class="mw-headline" id="See_also">See also</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Molecular_dynamics&amp;action=edit&amp;section=25" title="Edit section: See also">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div class="div-col columns column-width" style="-moz-column-width: 27em; -webkit-column-width: 27em; column-width: 27em;">
<ul>
<li><a href="/wiki/Molecular_modeling" title="Molecular modeling" class="mw-redirect">Molecular modeling</a></li>
<li><a href="/wiki/Computational_chemistry" title="Computational chemistry">Computational chemistry</a></li>
<li><a href="/wiki/Force_field_(chemistry)" title="Force field (chemistry)">Force field (chemistry)</a></li>
<li><a href="/wiki/Force_field_implementation" title="Force field implementation">Force field implementation</a></li>
<li><a href="/wiki/Monte_Carlo_method" title="Monte Carlo method">Monte Carlo method</a></li>
<li><a href="/wiki/Molecular_design_software" title="Molecular design software">Molecular design software</a></li>
<li><a href="/wiki/Molecular_mechanics" title="Molecular mechanics">Molecular mechanics</a></li>
<li><a href="/wiki/Car%E2%80%93Parrinello_method" title="Car–Parrinello method">Car–Parrinello method</a></li>
<li><a href="/wiki/List_of_software_for_molecular_mechanics_modeling" title="List of software for molecular mechanics modeling">Software for molecular mechanics modeling</a></li>
<li><a href="/wiki/Quantum_chemistry" title="Quantum chemistry">Quantum chemistry</a></li>
<li><a href="/wiki/Discrete_element_method" title="Discrete element method">Discrete element method</a></li>
<li><a href="/wiki/List_of_nucleic_acid_simulation_software" title="List of nucleic acid simulation software">List of nucleic acid simulation software</a></li>
<li><a href="/wiki/Molecule_editor" title="Molecule editor">Molecule editor</a></li>
</ul>
</div>
<h2><span class="mw-headline" id="References">References</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Molecular_dynamics&amp;action=edit&amp;section=26" title="Edit section: References">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div class="reflist columns references-column-width" style="-moz-column-width: 30em; -webkit-column-width: 30em; column-width: 30em; list-style-type: decimal;">
<ol class="references">
<li id="cite_note-a.26w-1"><span class="mw-cite-backlink">^ <a href="#cite_ref-a.26w_1-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-a.26w_1-1"><sup><i><b>b</b></i></sup></a> <a href="#cite_ref-a.26w_1-2"><sup><i><b>c</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">Alder, B. J.; Wainwright, T. E. (1959). "Studies in Molecular Dynamics. I. General Method". <i>J. Chem. Phys.</i> <b>31</b> (2): 459. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="http://adsabs.harvard.edu/abs/1959JChPh..31..459A">1959JChPh..31..459A</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1063%2F1.1730376">10.1063/1.1730376</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Studies+in+Molecular+Dynamics.+I.+General+Method&amp;rft.aufirst=B.+J.&amp;rft.aulast=Alder&amp;rft.au=Wainwright%2C+T.+E.&amp;rft.date=1959&amp;rft.genre=article&amp;rft_id=info%3Abibcode%2F1959JChPh..31..459A&amp;rft_id=info%3Adoi%2F10.1063%2F1.1730376&amp;rft.issue=2&amp;rft.jtitle=J.+Chem.+Phys.&amp;rft.pages=459&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=31" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-a.rahman-2"><span class="mw-cite-backlink">^ <a href="#cite_ref-a.rahman_2-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-a.rahman_2-1"><sup><i><b>b</b></i></sup></a> <a href="#cite_ref-a.rahman_2-2"><sup><i><b>c</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal"><a href="/wiki/Aneesur_Rahman" title="Aneesur Rahman">Rahman, A.</a> (19 October 1964). "Correlations in the Motion of Atoms in Liquid Argon". <i>Physical Review</i> <b>136</b> (2A): A405–A411. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="http://adsabs.harvard.edu/abs/1964PhRv..136..405R">1964PhRv..136..405R</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1103%2FPhysRev.136.A405">10.1103/PhysRev.136.A405</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Correlations+in+the+Motion+of+Atoms+in+Liquid+Argon&amp;rft.aufirst=A.&amp;rft.aulast=Rahman&amp;rft.date=1964-10-19&amp;rft.genre=article&amp;rft_id=info%3Abibcode%2F1964PhRv..136..405R&amp;rft_id=info%3Adoi%2F10.1103%2FPhysRev.136.A405&amp;rft.issue=2A&amp;rft.jtitle=Physical+Review&amp;rft.pages=A405-A411&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=136" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-3"><span class="mw-cite-backlink"><b><a href="#cite_ref-3">^</a></b></span> <span class="reference-text"><cite class="citation book">Schlick, T. (1996). "Pursuing Laplace's Vision on Modern Computers". In J. P. Mesirov, K. Schulten and D. W. Sumners. <i>Mathematical Applications to Biomolecular Structure and Dynamics, IMA Volumes in Mathematics and Its Applications</i> <b>82</b>. New York: Springer-Verlag. pp.&#160;218–247. <a href="/wiki/International_Standard_Book_Number" title="International Standard Book Number">ISBN</a>&#160;<a href="/wiki/Special:BookSources/978-0-387-94838-6" title="Special:BookSources/978-0-387-94838-6">978-0-387-94838-6</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Pursuing+Laplace%27s+Vision+on+Modern+Computers&amp;rft.aufirst=T.&amp;rft.aulast=Schlick&amp;rft.btitle=Mathematical+Applications+to+Biomolecular+Structure+and+Dynamics%2C+IMA+Volumes+in+Mathematics+and+Its+Applications&amp;rft.date=1996&amp;rft.genre=bookitem&amp;rft.isbn=978-0-387-94838-6&amp;rft.pages=218-247&amp;rft.place=New+York&amp;rft.pub=Springer-Verlag&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-4"><span class="mw-cite-backlink"><b><a href="#cite_ref-4">^</a></b></span> <span class="reference-text"><cite class="citation book">de Laplace, P. S. (1820). <i>Oeuveres Completes de Laplace, Theorie Analytique des Probabilites</i> (in French). Paris, France: Gauthier-Villars.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.aufirst=P.+S.&amp;rft.aulast=de+Laplace&amp;rft.btitle=Oeuveres+Completes+de+Laplace%2C+Theorie+Analytique+des+Probabilites&amp;rft.date=1820&amp;rft.genre=book&amp;rft.place=Paris%2C+France&amp;rft.pub=Gauthier-Villars&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-5"><span class="mw-cite-backlink"><b><a href="#cite_ref-5">^</a></b></span> <span class="reference-text"><cite class="citation journal">Gibson, J B; Goland, A N; Milgram, M; Vineyard, G H (1960). "Dynamics of Radiation Damage". <i>Phys. Rev.</i> <b>120</b> (4): 1229–1253. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="http://adsabs.harvard.edu/abs/1960PhRv..120.1229G">1960PhRv..120.1229G</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1103%2FPhysRev.120.1229">10.1103/PhysRev.120.1229</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Dynamics+of+Radiation+Damage&amp;rft.aufirst=J+B&amp;rft.au=Goland%2C+A+N&amp;rft.aulast=Gibson&amp;rft.au=Milgram%2C+M&amp;rft.au=Vineyard%2C+G+H&amp;rft.date=1960&amp;rft.genre=article&amp;rft_id=info%3Abibcode%2F1960PhRv..120.1229G&amp;rft_id=info%3Adoi%2F10.1103%2FPhysRev.120.1229&amp;rft.issue=4&amp;rft.jtitle=Phys.+Rev.&amp;rft.pages=1229-1253&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=120" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-6"><span class="mw-cite-backlink"><b><a href="#cite_ref-6">^</a></b></span> <span class="reference-text"><cite class="citation journal">Bernal, J.D. (1964). "The Bakerian lecture, 1962: The structure of liquids". <i>Proceedings of the Royal Society</i> <b>280</b> (1382): 299–322. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="http://adsabs.harvard.edu/abs/1964RSPSA.280..299B">1964RSPSA.280..299B</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1098%2Frspa.1964.0147">10.1098/rspa.1964.0147</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=The+Bakerian+lecture%2C+1962%3A+The+structure+of+liquids&amp;rft.aufirst=J.D.&amp;rft.aulast=Bernal&amp;rft.date=1964&amp;rft.genre=article&amp;rft_id=info%3Abibcode%2F1964RSPSA.280..299B&amp;rft_id=info%3Adoi%2F10.1098%2Frspa.1964.0147&amp;rft.issue=1382&amp;rft.jtitle=Proceedings+of+the+Royal+Society&amp;rft.pages=299-322&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=280" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-Koehl-7"><span class="mw-cite-backlink"><b><a href="#cite_ref-Koehl_7-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Koehl, P.; <a href="/wiki/Michael_Levitt" title="Michael Levitt">Levitt, Michael</a> (1999). "A brighter future for protein structure prediction". <i><a href="/wiki/Nature_Structural_Biology" title="Nature Structural Biology" class="mw-redirect">Nature Structural Biology</a></i> <b>6</b>: 108–111.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=A+brighter+future+for+protein+structure+prediction&amp;rft.aufirst=P.&amp;rft.aulast=Koehl&amp;rft.au=Levitt%2C+Michael&amp;rft.date=1999&amp;rft.genre=article&amp;rft.jtitle=Nature+Structural+Biology&amp;rft.pages=108-111&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=6" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-8"><span class="mw-cite-backlink"><b><a href="#cite_ref-8">^</a></b></span> <span class="reference-text"><cite class="citation journal">Raval, A; Piana, S; Eastwood, MP; Dror, RO; Shaw, DE (August 2012). "Refinement of protein structure homology models via long, all-atom molecular dynamics simulations.". <i>Proteins</i> <b>80</b> (8): 2071–9. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1002%2Fprot.24098">10.1002/prot.24098</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/22513870">22513870</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Refinement+of+protein+structure+homology+models+via+long%2C+all-atom+molecular+dynamics+simulations.&amp;rft.au=Dror%2C+RO&amp;rft.au=Eastwood%2C+MP&amp;rft.aufirst=A&amp;rft.aulast=Raval&amp;rft.au=Piana%2C+S&amp;rft.au=Shaw%2C+DE&amp;rft.date=2012-08&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1002%2Fprot.24098&amp;rft_id=info%3Apmid%2F22513870&amp;rft.issue=8&amp;rft.jtitle=Proteins&amp;rft.pages=2071-9&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=80" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-9"><span class="mw-cite-backlink"><b><a href="#cite_ref-9">^</a></b></span> <span class="reference-text"><cite class="citation journal">Beauchamp, KA; Lin, YS; Das, R; Pande, VS (10 April 2012). "Are Protein Force Fields Getting Better? A Systematic Benchmark on 524 Diverse NMR Measurements.". <i>Journal of chemical theory and computation</i> <b>8</b> (4): 1409–1414. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1021%2Fct2007814">10.1021/ct2007814</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/22754404">22754404</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Are+Protein+Force+Fields+Getting+Better%3F+A+Systematic+Benchmark+on+524+Diverse+NMR+Measurements.&amp;rft.au=Das%2C+R&amp;rft.aufirst=KA&amp;rft.aulast=Beauchamp&amp;rft.au=Lin%2C+YS&amp;rft.au=Pande%2C+VS&amp;rft.date=2012-04-10&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1021%2Fct2007814&amp;rft_id=info%3Apmid%2F22754404&amp;rft.issue=4&amp;rft.jtitle=Journal+of+chemical+theory+and+computation&amp;rft.pages=1409-1414&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=8" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-10"><span class="mw-cite-backlink"><b><a href="#cite_ref-10">^</a></b></span> <span class="reference-text"><cite class="citation journal">Piana, S; Klepeis, JL; Shaw, DE (February 2014). "Assessing the accuracy of physical models used in protein-folding simulations: quantitative evidence from long molecular dynamics simulations". <i>Current Opinion in Structural Biology</i> <b>24</b>: 98–105. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1016%2Fj.sbi.2013.12.006">10.1016/j.sbi.2013.12.006</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/24463371">24463371</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Assessing+the+accuracy+of+physical+models+used+in+protein-folding+simulations%3A+quantitative+evidence+from+long+molecular+dynamics+simulations&amp;rft.aufirst=S&amp;rft.au=Klepeis%2C+JL&amp;rft.aulast=Piana&amp;rft.au=Shaw%2C+DE&amp;rft.date=2014-02&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1016%2Fj.sbi.2013.12.006&amp;rft_id=info%3Apmid%2F24463371&amp;rft.jtitle=Current+Opinion+in+Structural+Biology&amp;rft.pages=98-105&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=24" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-11"><span class="mw-cite-backlink"><b><a href="#cite_ref-11">^</a></b></span> <span class="reference-text">Hydrophobic interactions are mostly of entropic nature at room temperature.</span></li>
<li id="cite_note-Myers-12"><span class="mw-cite-backlink"><b><a href="#cite_ref-Myers_12-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Myers, J. K.; Pace, C. N. (1996). "Hydrogen bonding stabilizes globular proteins". <i>Biophys. J.</i> <b>71</b> (4): 2033–2039. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1016%2Fs0006-3495%2896%2979401-8">10.1016/s0006-3495(96)79401-8</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/8889177">8889177</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Hydrogen+bonding+stabilizes+globular+proteins&amp;rft.aufirst=J.+K.&amp;rft.aulast=Myers&amp;rft.au=Pace%2C+C.+N.&amp;rft.date=1996&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1016%2Fs0006-3495%2896%2979401-8&amp;rft_id=info%3Apmid%2F8889177&amp;rft.issue=4&amp;rft.jtitle=Biophys.+J.&amp;rft.pages=2033-2039&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=71" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-Israelachvili-13"><span class="mw-cite-backlink">^ <a href="#cite_ref-Israelachvili_13-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-Israelachvili_13-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><a href="/wiki/Jacob_Israelachvili" title="Jacob Israelachvili">Israelachvili, Jacob</a> (1992). <i>Intermolecular and surface forces.</i> Academic Press, San Diego.</span></li>
<li id="cite_note-14"><span class="mw-cite-backlink"><b><a href="#cite_ref-14">^</a></b></span> <span class="reference-text">Plimpton, Steve. <a rel="nofollow" class="external text" href="http://www.sandia.gov/~sjplimp/md.html">Molecular Dynamics - Parallel Algorithms</a>. sandia.gov</span></li>
<li id="cite_note-Streett-15"><span class="mw-cite-backlink"><b><a href="#cite_ref-Streett_15-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Streett WB, Tildesley DJ, Saville G; Tildesley; Saville (1978). "Multiple time-step methods in molecular dynamics". <i>Mol Phys</i> <b>35</b> (3): 639–648. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="http://adsabs.harvard.edu/abs/1978MolPh..35..639S">1978MolPh..35..639S</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1080%2F00268977800100471">10.1080/00268977800100471</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Multiple+time-step+methods+in+molecular+dynamics&amp;rft.au=Saville&amp;rft.au=Streett+WB%2C+Tildesley+DJ%2C+Saville+G&amp;rft.au=Tildesley&amp;rft.date=1978&amp;rft.genre=article&amp;rft_id=info%3Abibcode%2F1978MolPh..35..639S&amp;rft_id=info%3Adoi%2F10.1080%2F00268977800100471&amp;rft.issue=3&amp;rft.jtitle=Mol+Phys&amp;rft.pages=639-648&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=35" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-Tuckerman1991-16"><span class="mw-cite-backlink"><b><a href="#cite_ref-Tuckerman1991_16-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Tuckerman ME, Berne BJ, Martyna GJ; Berne; Martyna (1991). "Molecular dynamics algorithm for multiple time scales: systems with long range forces". <i>J Chem Phys</i> <b>94</b> (10): 6811–6815. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="http://adsabs.harvard.edu/abs/1991JChPh..94.6811T">1991JChPh..94.6811T</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1063%2F1.460259">10.1063/1.460259</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Molecular+dynamics+algorithm+for+multiple+time+scales%3A+systems+with+long+range+forces&amp;rft.au=Berne&amp;rft.au=Martyna&amp;rft.au=Tuckerman+ME%2C+Berne+BJ%2C+Martyna+GJ&amp;rft.date=1991&amp;rft.genre=article&amp;rft_id=info%3Abibcode%2F1991JChPh..94.6811T&amp;rft_id=info%3Adoi%2F10.1063%2F1.460259&amp;rft.issue=10&amp;rft.jtitle=J+Chem+Phys&amp;rft.pages=6811-6815&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=94" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-Tuckerman1992-17"><span class="mw-cite-backlink"><b><a href="#cite_ref-Tuckerman1992_17-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Tuckerman ME, Berne BJ, Martyna GJ; Berne; Martyna (1992). "Reversible multiple time scale molecular dynamics". <i>J Chem Phys</i> <b>97</b> (3): 1990–2001. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="http://adsabs.harvard.edu/abs/1992JChPh..97.1990T">1992JChPh..97.1990T</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1063%2F1.463137">10.1063/1.463137</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Reversible+multiple+time+scale+molecular+dynamics&amp;rft.au=Berne&amp;rft.au=Martyna&amp;rft.au=Tuckerman+ME%2C+Berne+BJ%2C+Martyna+GJ&amp;rft.date=1992&amp;rft.genre=article&amp;rft_id=info%3Abibcode%2F1992JChPh..97.1990T&amp;rft_id=info%3Adoi%2F10.1063%2F1.463137&amp;rft.issue=3&amp;rft.jtitle=J+Chem+Phys&amp;rft.pages=1990-2001&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=97" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-18"><span class="mw-cite-backlink"><b><a href="#cite_ref-18">^</a></b></span> <span class="reference-text"><cite class="citation journal">Sugita, Yuji; Yuko Okamoto (1999). "Replica-exchange molecular dynamics method for protein folding". <i>Chem Phys Letters</i> <b>314</b>: 141–151. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="http://adsabs.harvard.edu/abs/1999CPL...314..141S">1999CPL...314..141S</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1016%2FS0009-2614%2899%2901123-9">10.1016/S0009-2614(99)01123-9</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Replica-exchange+molecular+dynamics+method+for+protein+folding&amp;rft.aufirst=Yuji&amp;rft.aulast=Sugita&amp;rft.au=Yuko+Okamoto&amp;rft.date=1999&amp;rft.genre=article&amp;rft_id=info%3Abibcode%2F1999CPL...314..141S&amp;rft_id=info%3Adoi%2F10.1016%2FS0009-2614%2899%2901123-9&amp;rft.jtitle=Chem+Phys+Letters&amp;rft.pages=141-151&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=314" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-19"><span class="mw-cite-backlink"><b><a href="#cite_ref-19">^</a></b></span> <span class="reference-text"><cite class="citation journal">Sinnott, S. B.; Brenner, D. W. (2012). "Three decades of many-body potentials in materials research". <i>MRS Bulletin</i> <b>37</b> (5): 469–473. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1557%2Fmrs.2012.88">10.1557/mrs.2012.88</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Three+decades+of+many-body+potentials+in+materials+research&amp;rft.au=Brenner%2C+D.+W.&amp;rft.aufirst=S.+B.&amp;rft.aulast=Sinnott&amp;rft.date=2012&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1557%2Fmrs.2012.88&amp;rft.issue=5&amp;rft.jtitle=MRS+Bulletin&amp;rft.pages=469-473&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=37" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-20"><span class="mw-cite-backlink"><b><a href="#cite_ref-20">^</a></b></span> <span class="reference-text"><cite class="citation journal">Albe, K.; Nordlund, K.; Averback, R. S. (2002). "Modeling metal-semiconductor interaction: Analytical bond-order potential for platinum-carbon". <i>Phys. Rev. B</i> <b>65</b> (19): 195124. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="http://adsabs.harvard.edu/abs/2002PhRvB..65s5124A">2002PhRvB..65s5124A</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1103%2Fphysrevb.65.195124">10.1103/physrevb.65.195124</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Modeling+metal-semiconductor+interaction%3A+Analytical+bond-order++potential+for+platinum-carbon&amp;rft.au=Averback%2C+R.+S.&amp;rft.aufirst=K.&amp;rft.aulast=Albe&amp;rft.au=Nordlund%2C+K.&amp;rft.date=2002&amp;rft.genre=article&amp;rft_id=info%3Abibcode%2F2002PhRvB..65s5124A&amp;rft_id=info%3Adoi%2F10.1103%2Fphysrevb.65.195124&amp;rft.issue=19&amp;rft.jtitle=Phys.+Rev.+B&amp;rft.pages=195124&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=65" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-21"><span class="mw-cite-backlink"><b><a href="#cite_ref-21">^</a></b></span> <span class="reference-text"><cite class="citation journal">Brenner, D. W. (1990). "Empirical potential for hydrocarbons for use in simulating the chemical vapor deposition of diamond films". <i>Phys. Rev. B</i> <b>42</b> (15): 9458–9471. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="http://adsabs.harvard.edu/abs/1990PhRvB..42.9458B">1990PhRvB..42.9458B</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1103%2FPhysRevB.42.9458">10.1103/PhysRevB.42.9458</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Empirical+potential+for+hydrocarbons+for+use+in+simulating+the+chemical+vapor+deposition+of+diamond+films&amp;rft.aufirst=D.+W.&amp;rft.aulast=Brenner&amp;rft.date=1990&amp;rft.genre=article&amp;rft_id=info%3Abibcode%2F1990PhRvB..42.9458B&amp;rft_id=info%3Adoi%2F10.1103%2FPhysRevB.42.9458&amp;rft.issue=15&amp;rft.jtitle=Phys.+Rev.+B&amp;rft.pages=9458-9471&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=42" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-22"><span class="mw-cite-backlink"><b><a href="#cite_ref-22">^</a></b></span> <span class="reference-text"><cite class="citation journal">Beardmore, Keith; Smith, Roger (1996). "Empirical potentials for C-Si-H systems with application to C<sub>60</sub> interactions with Si crystal surfaces". <i>Philosophical Magazine A</i> <b>74</b> (6): 1439–1466. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1080%2F01418619608240734">10.1080/01418619608240734</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Empirical+potentials+for+C-Si-H+systems+with+application+to+C%3Csub%3E60%3C%2Fsub%3E+interactions+with+Si+crystal+surfaces&amp;rft.aufirst=Keith&amp;rft.aulast=Beardmore&amp;rft.au=Smith%2C+Roger&amp;rft.date=1996&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1080%2F01418619608240734&amp;rft.issue=6&amp;rft.jtitle=Philosophical+Magazine+A&amp;rft.pages=1439-1466&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=74" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-23"><span class="mw-cite-backlink"><b><a href="#cite_ref-23">^</a></b></span> <span class="reference-text"><cite class="citation journal">Ni, Boris; Lee, Ki-Ho; Sinnott, Susan B (2004). "A reactive empirical bond order (rebo) potential for hydrocarbon oxygen interactions". <i>Journal of Physics: Condensed Matter</i> <b>16</b> (41): 7261–7275. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1088%2F0953-8984%2F16%2F41%2F008">10.1088/0953-8984/16/41/008</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=A+reactive+empirical+bond+order+%28rebo%29+potential+for+hydrocarbon+oxygen+interactions&amp;rft.aufirst=Boris&amp;rft.aulast=Ni&amp;rft.au=Lee%2C+Ki-Ho&amp;rft.au=Sinnott%2C+Susan+B&amp;rft.date=2004&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1088%2F0953-8984%2F16%2F41%2F008&amp;rft.issue=41&amp;rft.jtitle=Journal+of+Physics%3A+Condensed+Matter&amp;rft.pages=7261-7275&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=16" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-24"><span class="mw-cite-backlink"><b><a href="#cite_ref-24">^</a></b></span> <span class="reference-text"><cite class="citation journal">van Duin, A.; Siddharth Dasgupta, François Lorant and William A. Goddard III; Lorant, Francois; Goddard, William A. (2001). "ReaxFF: A Reactive Force Field for Hydrocarbons". <i>J. Phys. Chem. A</i> <b>105</b> (41): 9398. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1021%2Fjp004368u">10.1021/jp004368u</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=ReaxFF%3A++A+Reactive+Force+Field+for+Hydrocarbons&amp;rft.aufirst=A.&amp;rft.au=Goddard%2C+William+A.&amp;rft.aulast=van+Duin&amp;rft.au=Lorant%2C+Francois&amp;rft.au=Siddharth+Dasgupta%2C+Fran%C3%A7ois+Lorant+and+William+A.+Goddard+III&amp;rft.date=2001&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1021%2Fjp004368u&amp;rft.issue=41&amp;rft.jtitle=J.+Phys.+Chem.+A&amp;rft.pages=9398&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=105" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-ReferenceA-25"><span class="mw-cite-backlink">^ <a href="#cite_ref-ReferenceA_25-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-ReferenceA_25-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">Justo, J. F.; Bazant, M. Z.; Kaxiras, E.; Bulatov, V. V.; Yip, S. (1998). "Interatomic potential for silicon defects and disordered phases". <i>Phys. Rev. B</i> <b>58</b>: 2539–2550. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1103%2FPhysRevB.58.2539">10.1103/PhysRevB.58.2539</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Interatomic+potential+for+silicon+defects+and+disordered+phases&amp;rft.au=Bazant%2C+M.+Z.&amp;rft.au=Bulatov%2C+V.+V.&amp;rft.aufirst=J.+F.&amp;rft.au=Kaxiras%2C+E.&amp;rft.aulast=Justo&amp;rft.au=Yip%2C+S.&amp;rft.date=1998&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1103%2FPhysRevB.58.2539&amp;rft.jtitle=Phys.+Rev.+B&amp;rft.pages=2539-2550&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=58" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-26"><span class="mw-cite-backlink"><b><a href="#cite_ref-26">^</a></b></span> <span class="reference-text"><cite class="citation journal">Tersoff, J. (1989). "Modeling solid-state chemistry: Interatomic potentials for multicomponent systems". <i>Phys. Rev. B</i> <b>39</b> (8): 5566–5568. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="http://adsabs.harvard.edu/abs/1989PhRvB..39.5566T">1989PhRvB..39.5566T</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1103%2FPhysRevB.39.5566">10.1103/PhysRevB.39.5566</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Modeling+solid-state+chemistry%3A+Interatomic+potentials+for+multicomponent+systems&amp;rft.aufirst=J.&amp;rft.aulast=Tersoff&amp;rft.date=1989&amp;rft.genre=article&amp;rft_id=info%3Abibcode%2F1989PhRvB..39.5566T&amp;rft_id=info%3Adoi%2F10.1103%2FPhysRevB.39.5566&amp;rft.issue=8&amp;rft.jtitle=Phys.+Rev.+B&amp;rft.pages=5566-5568&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=39" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-27"><span class="mw-cite-backlink"><b><a href="#cite_ref-27">^</a></b></span> <span class="reference-text"><cite class="citation journal">Daw, M. S.; S. M. Foiles and M. I. Baskes (1993). "The embedded-atom method: a review of theory and applications". <i>Mat. Sci. And Engr. Rep.</i> <b>9</b> (7–8): 251–310. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1016%2F0920-2307%2893%2990001-U">10.1016/0920-2307(93)90001-U</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=The+embedded-atom+method%3A+a+review+of+theory+and+applications&amp;rft.aufirst=M.+S.&amp;rft.aulast=Daw&amp;rft.au=S.+M.+Foiles+and+M.+I.+Baskes&amp;rft.date=1993&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1016%2F0920-2307%2893%2990001-U&amp;rft.issue=7%938&amp;rft.jtitle=Mat.+Sci.+And+Engr.+Rep.&amp;rft.pages=251-310&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=9" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-28"><span class="mw-cite-backlink"><b><a href="#cite_ref-28">^</a></b></span> <span class="reference-text"><cite class="citation journal">Cleri, F.; V. Rosato (1993). "Tight-binding potentials for transition metals and alloys". <i>Phys. Rev. B</i> <b>48</b>: 22–33. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="http://adsabs.harvard.edu/abs/1993PhRvB..48...22C">1993PhRvB..48...22C</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1103%2FPhysRevB.48.22">10.1103/PhysRevB.48.22</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Tight-binding+potentials+for+transition+metals+and+alloys&amp;rft.aufirst=F.&amp;rft.aulast=Cleri&amp;rft.au=V.+Rosato&amp;rft.date=1993&amp;rft.genre=article&amp;rft_id=info%3Abibcode%2F1993PhRvB..48...22C&amp;rft_id=info%3Adoi%2F10.1103%2FPhysRevB.48.22&amp;rft.jtitle=Phys.+Rev.+B&amp;rft.pages=22-33&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=48" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-Lamoureux3-29"><span class="mw-cite-backlink"><b><a href="#cite_ref-Lamoureux3_29-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Lamoureux G, Harder E, Vorobyov IV, Roux B, MacKerell AD; Harder; Vorobyov; Roux; MacKerell (2006). "A polarizable model of water for molecular dynamics simulations of biomolecules". <i>Chem Phys Lett</i> <b>418</b>: 245–249. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="http://adsabs.harvard.edu/abs/2006CPL...418..245L">2006CPL...418..245L</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1016%2Fj.cplett.2005.10.135">10.1016/j.cplett.2005.10.135</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=A+polarizable+model+of+water+for+molecular+dynamics+simulations+of+biomolecules&amp;rft.au=Harder&amp;rft.au=Lamoureux+G%2C+Harder+E%2C+Vorobyov+IV%2C+Roux+B%2C+MacKerell+AD&amp;rft.au=MacKerell&amp;rft.au=Roux&amp;rft.au=Vorobyov&amp;rft.date=2006&amp;rft.genre=article&amp;rft_id=info%3Abibcode%2F2006CPL...418..245L&amp;rft_id=info%3Adoi%2F10.1016%2Fj.cplett.2005.10.135&amp;rft.jtitle=Chem+Phys+Lett&amp;rft.pages=245-249&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=418" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-Sokhan2015-30"><span class="mw-cite-backlink"><b><a href="#cite_ref-Sokhan2015_30-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Sokhan VP, Jones AP, Cipcigan FS, Crain J, Martyna GJ (2015). "Signature properties of water: Their molecular electronic origins". <i>Proceedings of the National Academy of Sciences</i> <b>112</b>: 6341–6346. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1073%2Fpnas.1418982112">10.1073/pnas.1418982112</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Signature+properties+of+water%3A+Their+molecular+electronic+origins&amp;rft.au=Sokhan+VP%2C+Jones+AP%2C+Cipcigan+FS%2C+Crain+J%2C+Martyna+GJ&amp;rft.date=2015&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1073%2Fpnas.1418982112&amp;rft.jtitle=Proceedings+of+the+National+Academy+of+Sciences&amp;rft.pages=6341-6346&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=112" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-Cipcigan-31"><span class="mw-cite-backlink"><b><a href="#cite_ref-Cipcigan_31-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Cipcigan FS, Sokhan VP, Jones AP, Crain J, Martyna GJ (2015). "Hydrogen bonding and molecular orientation at the liquid–vapour interface of water". <i>Physical Chemistry Chemical Physics</i> <b>17</b>: 8660–8669. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1039%2FC4CP05506C">10.1039/C4CP05506C</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Hydrogen+bonding+and+molecular+orientation+at+the+liquid%93vapour+interface+of+water&amp;rft.au=Cipcigan+FS%2C+Sokhan+VP%2C+Jones+AP%2C+Crain+J%2C+Martyna+GJ&amp;rft.date=2015&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1039%2FC4CP05506C&amp;rft.jtitle=Physical+Chemistry+Chemical+Physics&amp;rft.pages=8660-8669&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=17" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-Patel2004b-32"><span class="mw-cite-backlink"><b><a href="#cite_ref-Patel2004b_32-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Patel, S.; MacKerell, Jr. AD; Brooks III, Charles L (2004). "CHARMM fluctuating charge force field for proteins: II protein/solvent properties from molecular dynamics simulations using a nonadditive electrostatic model". <i>J Comput Chem</i> <b>25</b> (12): 1504–1514. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1002%2Fjcc.20077">10.1002/jcc.20077</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/15224394">15224394</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=CHARMM+fluctuating+charge+force+field+for+proteins%3A+II+protein%2Fsolvent+properties+from+molecular+dynamics+simulations+using+a+nonadditive+electrostatic+model&amp;rft.au=Patel%2C+S.%3B+MacKerell%2C+Jr.+AD%3B+Brooks+III%2C+Charles+L&amp;rft.date=2004&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1002%2Fjcc.20077&amp;rft_id=info%3Apmid%2F15224394&amp;rft.issue=12&amp;rft.jtitle=J+Comput+Chem&amp;rft.pages=1504-1514&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=25" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-33"><span class="mw-cite-backlink"><b><a href="#cite_ref-33">^</a></b></span> <span class="reference-text">The methodology for such techniques was introduced by Warshel and coworkers. In the recent years have been pioneered by several groups including: <a href="/wiki/Arieh_Warshel" title="Arieh Warshel">Arieh Warshel</a> (<a href="/wiki/University_of_Southern_California" title="University of Southern California">University of Southern California</a>), Weitao Yang (<a href="/wiki/Duke_University" title="Duke University">Duke University</a>), Sharon Hammes-Schiffer (<a href="/wiki/The_Pennsylvania_State_University" title="The Pennsylvania State University" class="mw-redirect">The Pennsylvania State University</a>), Donald Truhlar and Jiali Gao (<a href="/wiki/University_of_Minnesota" title="University of Minnesota">University of Minnesota</a>) and Kenneth Merz (<a href="/wiki/University_of_Florida" title="University of Florida">University of Florida</a>).</span></li>
<li id="cite_note-34"><span class="mw-cite-backlink"><b><a href="#cite_ref-34">^</a></b></span> <span class="reference-text"><cite class="citation journal">Billeter, SR; SP Webb; PK Agarwal; T Iordanov; S Hammes-Schiffer (2001). "Hydride Transfer in Liver Alcohol Dehydrogenase: Quantum Dynamics, Kinetic Isotope Effects, and Role of Enzyme Motion". <i>J Am Chem Soc</i> <b>123</b> (45): 11262–11272. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1021%2Fja011384b">10.1021/ja011384b</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/11697969">11697969</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Hydride+Transfer+in+Liver+Alcohol+Dehydrogenase%3A+Quantum+Dynamics%2C+Kinetic+Isotope+Effects%2C+and+Role+of+Enzyme+Motion&amp;rft.aufirst=SR&amp;rft.aulast=Billeter&amp;rft.au=PK+Agarwal&amp;rft.au=S+Hammes-Schiffer&amp;rft.au=SP+Webb&amp;rft.au=T+Iordanov&amp;rft.date=2001&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1021%2Fja011384b&amp;rft_id=info%3Apmid%2F11697969&amp;rft.issue=45&amp;rft.jtitle=J+Am+Chem+Soc&amp;rft.pages=11262-11272&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=123" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-35"><span class="mw-cite-backlink"><b><a href="#cite_ref-35">^</a></b></span> <span class="reference-text"><cite class="citation journal">Smith, A; CK Hall (2001). "Alpha-Helix Formation: Discontinuous Molecular Dynamics on an Intermediate-Resolution Protein Model". <i>Proteins</i> <b>44</b> (3): 344–360. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1002%2Fprot.1100">10.1002/prot.1100</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/11455608">11455608</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Alpha-Helix+Formation%3A+Discontinuous+Molecular+Dynamics+on+an+Intermediate-Resolution+Protein+Model&amp;rft.au=CK+Hall&amp;rft.aufirst=A&amp;rft.aulast=Smith&amp;rft.date=2001&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1002%2Fprot.1100&amp;rft_id=info%3Apmid%2F11455608&amp;rft.issue=3&amp;rft.jtitle=Proteins&amp;rft.pages=344-360&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=44" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-36"><span class="mw-cite-backlink"><b><a href="#cite_ref-36">^</a></b></span> <span class="reference-text"><cite class="citation journal">Ding, F; JM Borreguero; SV Buldyrey; HE Stanley; NV Dokholyan (2003). "Mechanism for the alpha-helix to beta-hairpin transition". <i>J Am Chem Soc</i> <b>53</b> (2): 220–228. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1002%2Fprot.10468">10.1002/prot.10468</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/14517973">14517973</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Mechanism+for+the+alpha-helix+to+beta-hairpin+transition&amp;rft.aufirst=F&amp;rft.au=HE+Stanley&amp;rft.au=JM+Borreguero&amp;rft.aulast=Ding&amp;rft.au=NV+Dokholyan&amp;rft.au=SV+Buldyrey&amp;rft.date=2003&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1002%2Fprot.10468&amp;rft_id=info%3Apmid%2F14517973&amp;rft.issue=2&amp;rft.jtitle=J+Am+Chem+Soc&amp;rft.pages=220-228&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=53" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-37"><span class="mw-cite-backlink"><b><a href="#cite_ref-37">^</a></b></span> <span class="reference-text"><cite class="citation journal">Paci, E; M Vendruscolo; M Karplus (2002). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1302383">"Validity of Go Models: Comparison with a Solvent-Shielded Empirical Energy Decomposition"</a>. <i>Biophys J</i> <b>83</b> (6): 3032–3038. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="http://adsabs.harvard.edu/abs/2002BpJ....83.3032P">2002BpJ....83.3032P</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1016%2FS0006-3495%2802%2975308-3">10.1016/S0006-3495(02)75308-3</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1302383">1302383</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/12496075">12496075</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Validity+of+Go+Models%3A+Comparison+with+a+Solvent-Shielded+Empirical+Energy+Decomposition&amp;rft.aufirst=E&amp;rft.aulast=Paci&amp;rft.au=M+Karplus&amp;rft.au=M+Vendruscolo&amp;rft.date=2002&amp;rft.genre=article&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1302383&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1302383&amp;rft_id=info%3Abibcode%2F2002BpJ....83.3032P&amp;rft_id=info%3Adoi%2F10.1016%2FS0006-3495%2802%2975308-3&amp;rft_id=info%3Apmid%2F12496075&amp;rft.issue=6&amp;rft.jtitle=Biophys+J&amp;rft.pages=3032-3038&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=83" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-38"><span class="mw-cite-backlink"><b><a href="#cite_ref-38">^</a></b></span> <span class="reference-text"><cite class="citation journal">Chakrabarty, A; T Cagin (2010). "Coarse grain modeling of polyimide copolymers". <i>Polymer</i> <b>51</b> (12): 2786–2794. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1016%2Fj.polymer.2010.03.060">10.1016/j.polymer.2010.03.060</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Coarse+grain+modeling+of+polyimide+copolymers&amp;rft.aufirst=A&amp;rft.aulast=Chakrabarty&amp;rft.au=T+Cagin&amp;rft.date=2010&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1016%2Fj.polymer.2010.03.060&amp;rft.issue=12&amp;rft.jtitle=Polymer&amp;rft.pages=2786-2794&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=51" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-Nienhaus-39"><span class="mw-cite-backlink"><b><a href="#cite_ref-Nienhaus_39-0">^</a></b></span> <span class="reference-text"><cite class="citation book">Nienhaus, Gerd Ulrich (2005). <i>Protein-ligand interactions: methods and applications</i>. pp.&#160;54–56. <a href="/wiki/International_Standard_Book_Number" title="International Standard Book Number">ISBN</a>&#160;<a href="/wiki/Special:BookSources/978-1-61737-525-5" title="Special:BookSources/978-1-61737-525-5">978-1-61737-525-5</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.au=Nienhaus%2C+Gerd+Ulrich&amp;rft.btitle=Protein-ligand+interactions%3A+methods+and+applications&amp;rft.date=2005&amp;rft.genre=book&amp;rft.isbn=978-1-61737-525-5&amp;rft.pages=54-56&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-Leszczy.C5.84ski-40"><span class="mw-cite-backlink"><b><a href="#cite_ref-Leszczy.C5.84ski_40-0">^</a></b></span> <span class="reference-text"><cite class="citation book">Leszczyński, Jerzy (2005). <i>Computational chemistry: reviews of current trends, Volume 9</i>. pp.&#160;54–56. <a href="/wiki/International_Standard_Book_Number" title="International Standard Book Number">ISBN</a>&#160;<a href="/wiki/Special:BookSources/978-981-256-742-0" title="Special:BookSources/978-981-256-742-0">978-981-256-742-0</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.au=Leszczy%C5%84ski%2C+Jerzy&amp;rft.btitle=Computational+chemistry%3A+reviews+of+current+trends%2C+Volume+9&amp;rft.date=2005&amp;rft.genre=book&amp;rft.isbn=978-981-256-742-0&amp;rft.pages=54-56&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-41"><span class="mw-cite-backlink"><b><a href="#cite_ref-41">^</a></b></span> <span class="reference-text"><cite class="citation journal">Kumar, Shankar; Rosenberg, John M.; Bouzida, Djamal; Swendsen, Robert H.; Kollman, Peter A. (30 September 1992). "The weighted histogram analysis method for free-energy calculations on biomolecules. I. The method". <i>Journal of Computational Chemistry</i> <b>13</b> (8): 1011–1021. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1002%2Fjcc.540130812">10.1002/jcc.540130812</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=The+weighted+histogram+analysis+method+for+free-energy+calculations+on+biomolecules.+I.+The+method&amp;rft.au=Bouzida%2C+Djamal&amp;rft.aufirst=Shankar&amp;rft.au=Kollman%2C+Peter+A.&amp;rft.aulast=Kumar&amp;rft.au=Rosenberg%2C+John+M.&amp;rft.au=Swendsen%2C+Robert+H.&amp;rft.date=1992-09-30&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1002%2Fjcc.540130812&amp;rft.issue=8&amp;rft.jtitle=Journal+of+Computational+Chemistry&amp;rft.pages=1011-1021&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=13" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-42"><span class="mw-cite-backlink"><b><a href="#cite_ref-42">^</a></b></span> <span class="reference-text"><cite class="citation journal">Bartels, Christian (1 December 2000). "Analyzing biased Monte Carlo and molecular dynamics simulations". <i>Chemical Physics Letters</i> <b>331</b> (5–6): 446–454. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="http://adsabs.harvard.edu/abs/2000CPL...331..446B">2000CPL...331..446B</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1016%2FS0009-2614%2800%2901215-X">10.1016/S0009-2614(00)01215-X</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Analyzing+biased+Monte+Carlo+and+molecular+dynamics+simulations&amp;rft.aufirst=Christian&amp;rft.aulast=Bartels&amp;rft.date=2000-12-01&amp;rft.genre=article&amp;rft_id=info%3Abibcode%2F2000CPL...331..446B&amp;rft_id=info%3Adoi%2F10.1016%2FS0009-2614%2800%2901215-X&amp;rft.issue=5%936&amp;rft.jtitle=Chemical+Physics+Letters&amp;rft.pages=446-454&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=331" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-43"><span class="mw-cite-backlink"><b><a href="#cite_ref-43">^</a></b></span> <span class="reference-text"><cite class="citation journal">Levitt, M; A Warshel (1975). "Computer Simulations of Protein Folding". <i>Nature</i> <b>253</b> (5494): 694–8. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="http://adsabs.harvard.edu/abs/1975Natur.253..694L">1975Natur.253..694L</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1038%2F253694a0">10.1038/253694a0</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/1167625">1167625</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Computer+Simulations+of+Protein+Folding&amp;rft.au=A+Warshel&amp;rft.aufirst=M&amp;rft.aulast=Levitt&amp;rft.date=1975&amp;rft.genre=article&amp;rft_id=info%3Abibcode%2F1975Natur.253..694L&amp;rft_id=info%3Adoi%2F10.1038%2F253694a0&amp;rft_id=info%3Apmid%2F1167625&amp;rft.issue=5494&amp;rft.jtitle=Nature&amp;rft.pages=694-8&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=253" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-44"><span class="mw-cite-backlink"><b><a href="#cite_ref-44">^</a></b></span> <span class="reference-text"><cite class="citation journal">Warshel, A (1976). "Bicycle-pedal Model for the First Step in the Vision Process". <i>Nature</i> <b>260</b> (5553): 679–683. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="http://adsabs.harvard.edu/abs/1976Natur.260..694B">1976Natur.260..694B</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1038%2F260679a0">10.1038/260679a0</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/1264239">1264239</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Bicycle-pedal+Model+for+the+First+Step+in+the+Vision+Process&amp;rft.aufirst=A&amp;rft.aulast=Warshel&amp;rft.date=1976&amp;rft.genre=article&amp;rft_id=info%3Abibcode%2F1976Natur.260..694B&amp;rft_id=info%3Adoi%2F10.1038%2F260679a0&amp;rft_id=info%3Apmid%2F1264239&amp;rft.issue=5553&amp;rft.jtitle=Nature&amp;rft.pages=679-683&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=260" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-Ave98-45"><span class="mw-cite-backlink"><b><a href="#cite_ref-Ave98_45-0">^</a></b></span> <span class="reference-text"><cite class="citation book">Averback, R. S.; Diaz de la Rubia, T. (1998). "Displacement damage in irradiated metals and semiconductors". In H. Ehrenfest and F. Spaepen. <i>Solid State Physics</i> <b>51</b>. New York: Academic Press. pp.&#160;281–402.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Displacement+damage+in+irradiated+metals+and+semiconductors&amp;rft.au=Diaz+de+la+Rubia%2C+T.&amp;rft.aufirst=R.+S.&amp;rft.aulast=Averback&amp;rft.btitle=Solid+State+Physics&amp;rft.date=1998&amp;rft.genre=bookitem&amp;rft.pages=281-402&amp;rft.place=New+York&amp;rft.pub=Academic+Press&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-Smith-46"><span class="mw-cite-backlink"><b><a href="#cite_ref-Smith_46-0">^</a></b></span> <span class="reference-text"><cite class="citation book">Smith, R., ed. (1997). <i>Atomic &amp; ion collisions in solids and at surfaces: theory, simulation and applications</i>. Cambridge, UK: Cambridge University Press.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.btitle=Atomic+%26+ion+collisions+in+solids+and+at+surfaces%3A+theory%2C+simulation+and+applications&amp;rft.date=1997&amp;rft.genre=book&amp;rft.place=Cambridge%2C+UK&amp;rft.pub=Cambridge+University+Press&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-47"><span class="mw-cite-backlink"><b><a href="#cite_ref-47">^</a></b></span> <span class="reference-text"><cite class="citation journal">Offman, MN; M Krol; I Silman; JL Sussman; AH Futerman (2010). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3009936">"Molecular basis of reduced glucosylceramidase activity in the most common Gaucher disease mutant, N370S"</a>. <i>J. Biol. Chem.</i> <b>285</b> (53): 42105–42114. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1074%2Fjbc.M110.172098">10.1074/jbc.M110.172098</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3009936">3009936</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/20980259">20980259</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Molecular+basis+of+reduced+glucosylceramidase+activity+in+the+most+common+Gaucher+disease+mutant%2C+N370S&amp;rft.au=AH+Futerman&amp;rft.aufirst=MN&amp;rft.au=I+Silman&amp;rft.au=JL+Sussman&amp;rft.aulast=Offman&amp;rft.au=M+Krol&amp;rft.date=2010&amp;rft.genre=article&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3009936&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3009936&amp;rft_id=info%3Adoi%2F10.1074%2Fjbc.M110.172098&amp;rft_id=info%3Apmid%2F20980259&amp;rft.issue=53&amp;rft.jtitle=J.+Biol.+Chem.&amp;rft.pages=42105-42114&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=285" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-48"><span class="mw-cite-backlink"><b><a href="#cite_ref-48">^</a></b></span> <span class="reference-text"><cite class="citation journal">Offman, MN; M Krol; B Rost; I Silman; JL Sussman; AH Futerman (2011). "Comparison of a molecular dynamics model with the X-ray structure of the N370S acid-beta-glucosidase mutant that causes Gaucher disease". <i>Protein Eng. Des. Sel.</i> <b>24</b> (10): 773–775. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1093%2Fprotein%2Fgzr032">10.1093/protein/gzr032</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/21724649">21724649</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Comparison+of+a+molecular+dynamics+model+with+the+X-ray+structure+of+the+N370S+acid-beta-glucosidase+mutant+that+causes+Gaucher+disease&amp;rft.au=AH+Futerman&amp;rft.au=B+Rost&amp;rft.aufirst=MN&amp;rft.au=I+Silman&amp;rft.au=JL+Sussman&amp;rft.aulast=Offman&amp;rft.au=M+Krol&amp;rft.date=2011&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1093%2Fprotein%2Fgzr032&amp;rft_id=info%3Apmid%2F21724649&amp;rft.issue=10&amp;rft.jtitle=Protein+Eng.+Des.+Sel.&amp;rft.pages=773-775&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=24" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-49"><span class="mw-cite-backlink"><b><a href="#cite_ref-49">^</a></b></span> <span class="reference-text"><cite class="citation journal">Hu, Han; Sun, Ying (2013). "Molecular dynamics simulations of disjoining pressure effect in ultra-thin water film on a metal surface". <i>Appl. Phys. Lett.</i> <b>14</b> (26): 263110. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="http://adsabs.harvard.edu/abs/2013ApPhL.103z3110H">2013ApPhL.103z3110H</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1063%2F1.4858469">10.1063/1.4858469</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Molecular+dynamics+simulations+of+disjoining+pressure+effect+in+ultra-thin+water+film+on+a+metal+surface&amp;rft.aufirst=Han&amp;rft.aulast=Hu&amp;rft.au=Sun%2C+Ying&amp;rft.date=2013&amp;rft.genre=article&amp;rft_id=info%3Abibcode%2F2013ApPhL.103z3110H&amp;rft_id=info%3Adoi%2F10.1063%2F1.4858469&amp;rft.issue=26&amp;rft.jtitle=Appl.+Phys.+Lett.&amp;rft.pages=263110&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=14" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-50"><span class="mw-cite-backlink"><b><a href="#cite_ref-50">^</a></b></span> <span class="reference-text"><cite class="citation journal">Welch, D. A.; Mehdi, B. L.; Hatchell, H. J.; Faller, R.; Evans, J. E.; Browning, N. D. (2015). "Using molecular dynamics to quantify the electrical double layer and examine the potential for its direct observation in the in-situ TEM". <i>Advanced Structural and Chemical Imaging</i> <b>1</b>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1186%2Fs40679-014-0002-2">10.1186/s40679-014-0002-2</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Using+molecular+dynamics+to+quantify+the+electrical+double+layer+and+examine+the+potential+for+its+direct+observation+in+the+in-situ+TEM&amp;rft.au=Browning%2C+N.+D.&amp;rft.au=Evans%2C+J.+E.&amp;rft.au=Faller%2C+R.&amp;rft.aufirst=D.+A.&amp;rft.au=Hatchell%2C+H.+J.&amp;rft.aulast=Welch&amp;rft.au=Mehdi%2C+B.+L.&amp;rft.date=2015&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1186%2Fs40679-014-0002-2&amp;rft.jtitle=Advanced+Structural+and+Chemical+Imaging&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=1" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-51"><span class="mw-cite-backlink"><b><a href="#cite_ref-51">^</a></b></span> <span class="reference-text"><cite class="citation web">Freddolino P, Arkhipov A, Larson SB, McPherson A, Schulten K. <a rel="nofollow" class="external text" href="http://www.ks.uiuc.edu/Research/STMV/">"Molecular dynamics simulation of the Satellite Tobacco Mosaic Virus (STMV)"</a>. <i>Theoretical and Computational Biophysics Group</i>. University of Illinois at Urbana Champaign.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Molecular+dynamics+simulation+of+the+Satellite+Tobacco+Mosaic+Virus+%28STMV%29&amp;rft.au=Freddolino+P%2C+Arkhipov+A%2C+Larson+SB%2C+McPherson+A%2C+Schulten+K&amp;rft.genre=unknown&amp;rft_id=http%3A%2F%2Fwww.ks.uiuc.edu%2FResearch%2FSTMV%2F&amp;rft.jtitle=Theoretical+and+Computational+Biophysics+Group&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-52"><span class="mw-cite-backlink"><b><a href="#cite_ref-52">^</a></b></span> <span class="reference-text"><a rel="nofollow" class="external text" href="http://folding.stanford.edu/">The Folding@Home Project</a> and <a rel="nofollow" class="external text" href="http://folding.stanford.edu/papers.html">recent papers</a> published using trajectories from it. Vijay Pande Group. Stanford University</span></li>
<li id="cite_note-DESRES-Science2011-53"><span class="mw-cite-backlink">^ <a href="#cite_ref-DESRES-Science2011_53-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-DESRES-Science2011_53-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">Lindorff-Larsen, Kresten; Piana, Stefano; Dror, Ron O.; Shaw, David E. (2011). "How Fast-Folding Proteins Fold". <i>Science</i> <b>334</b> (6055): 517–520. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="http://adsabs.harvard.edu/abs/2011Sci...334..517L">2011Sci...334..517L</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1126%2Fscience.1208351">10.1126/science.1208351</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/22034434">22034434</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=How+Fast-Folding+Proteins+Fold&amp;rft.au=Dror%2C+Ron+O.&amp;rft.aufirst=Kresten&amp;rft.aulast=Lindorff-Larsen&amp;rft.au=Piana%2C+Stefano&amp;rft.au=Shaw%2C+David+E.&amp;rft.date=2011&amp;rft.genre=article&amp;rft_id=info%3Abibcode%2F2011Sci...334..517L&amp;rft_id=info%3Adoi%2F10.1126%2Fscience.1208351&amp;rft_id=info%3Apmid%2F22034434&amp;rft.issue=6055&amp;rft.jtitle=Science&amp;rft.pages=517-520&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=334" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-54"><span class="mw-cite-backlink"><b><a href="#cite_ref-54">^</a></b></span> <span class="reference-text"><cite class="citation journal">Shaw, David E.; Maragakis, Paul; Lindorff-Larsen, Kresten; Piana, Stefano; Dror, Ron O.; Eastwood, Michael P.; Bank, Joseph A.; Jumper, John M.; Salmon, John K.; et al. (2010). "Atomic-Level Characterization of the Structural Dynamics of Proteins". <i>Science</i> <b>330</b> (6002): 341–346. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="http://adsabs.harvard.edu/abs/2010Sci...330..341S">2010Sci...330..341S</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1126%2Fscience.1187409">10.1126/science.1187409</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/20947758">20947758</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Atomic-Level+Characterization+of+the+Structural+Dynamics+of+Proteins&amp;rft.au=Bank%2C+Joseph+A.&amp;rft.au=Dror%2C+Ron+O.&amp;rft.au=Eastwood%2C+Michael+P.&amp;rft.aufirst=David+E.&amp;rft.au=Jumper%2C+John+M.&amp;rft.aulast=Shaw&amp;rft.au=Lindorff-Larsen%2C+Kresten&amp;rft.au=Maragakis%2C+Paul&amp;rft.au=Piana%2C+Stefano&amp;rft.au=Salmon%2C+John+K.&amp;rft.date=2010&amp;rft.genre=article&amp;rft_id=info%3Abibcode%2F2010Sci...330..341S&amp;rft_id=info%3Adoi%2F10.1126%2Fscience.1187409&amp;rft_id=info%3Apmid%2F20947758&amp;rft.issue=6002&amp;rft.jtitle=Science&amp;rft.pages=341-346&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=330" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-55"><span class="mw-cite-backlink"><b><a href="#cite_ref-55">^</a></b></span> <span class="reference-text"><cite class="citation journal">Goel S, Luo; Reuben R L (2012). "Molecular dynamics simulation model for the quantitative assessment of tool wear during single point diamond turning of cubic silicon carbide". <i>Comput. Mater. Sci</i> <b>51</b>: 402–408. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1016%2Fj.commatsci.2011.07.052">10.1016/j.commatsci.2011.07.052</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Molecular+dynamics+simulation+model+for+the+quantitative+assessment+of+tool+wear+during+single+point+diamond+turning+of+cubic+silicon+carbide&amp;rft.au=Goel+S%2C+Luo&amp;rft.au=Reuben+R+L&amp;rft.date=2012&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1016%2Fj.commatsci.2011.07.052&amp;rft.jtitle=Comput.+Mater.+Sci&amp;rft.pages=402-408&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=51" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-Jaya2011-56"><span class="mw-cite-backlink"><b><a href="#cite_ref-Jaya2011_56-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Jayasena, Buddhika; Subbiah Sathyan (2011). <a rel="nofollow" class="external text" href="http://www.nanoscalereslett.com/content/6/1/95">"A novel mechanical cleavage method for synthesizing few-layer graphenes"</a>. <i>Nanoscale Research Letters</i> <b>6</b> (1): 95. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="http://adsabs.harvard.edu/abs/2011NRL.....6...95J">2011NRL.....6...95J</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1186%2F1556-276X-6-95">10.1186/1556-276X-6-95</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3212245">3212245</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/21711598">21711598</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=A+novel+mechanical+cleavage+method+for+synthesizing+few-layer+graphenes&amp;rft.aufirst=Buddhika&amp;rft.aulast=Jayasena&amp;rft.au=Subbiah+Sathyan&amp;rft.date=2011&amp;rft.genre=article&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3212245&amp;rft_id=http%3A%2F%2Fwww.nanoscalereslett.com%2Fcontent%2F6%2F1%2F95&amp;rft_id=info%3Abibcode%2F2011NRL.....6...95J&amp;rft_id=info%3Adoi%2F10.1186%2F1556-276X-6-95&amp;rft_id=info%3Apmid%2F21711598&amp;rft.issue=1&amp;rft.jtitle=Nanoscale+Research+Letters&amp;rft.pages=95&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=6" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-57"><span class="mw-cite-backlink"><b><a href="#cite_ref-57">^</a></b></span> <span class="reference-text"><cite class="citation journal">Jayasena, B; Reddy C.D; Subbiah S (2013). <a rel="nofollow" class="external text" href="http://iopscience.iop.org/0957-4484/24/20/205301/">"Separation, folding and shearing of graphene layers during wedge-based mechanical exfoliation"</a>. <i>Nanotechnology.</i> <b>24</b> (20): 205301. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="http://adsabs.harvard.edu/abs/2013Nanot..24t5301J">2013Nanot..24t5301J</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1088%2F0957-4484%2F24%2F20%2F205301">10.1088/0957-4484/24/20/205301</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/23598423">23598423</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Separation%2C+folding+and+shearing+of+graphene+layers+during+wedge-based+mechanical+exfoliation&amp;rft.aufirst=B&amp;rft.aulast=Jayasena&amp;rft.au=Reddy+C.D&amp;rft.au=Subbiah+S&amp;rft.date=2013&amp;rft.genre=article&amp;rft_id=http%3A%2F%2Fiopscience.iop.org%2F0957-4484%2F24%2F20%2F205301%2F&amp;rft_id=info%3Abibcode%2F2013Nanot..24t5301J&amp;rft_id=info%3Adoi%2F10.1088%2F0957-4484%2F24%2F20%2F205301&amp;rft_id=info%3Apmid%2F23598423&amp;rft.issue=20&amp;rft.jtitle=Nanotechnology.&amp;rft.pages=205301&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=24" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
</ol>
</div>
<h3><span class="mw-headline" id="General_references">General references</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Molecular_dynamics&amp;action=edit&amp;section=27" title="Edit section: General references">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<div class="refbegin" style="">
<ul>
<li>M. P. Allen, D. J. Tildesley (1989) <i>Computer simulation of liquids</i>. Oxford University Press. <a href="/wiki/Special:BookSources/0198556454" class="internal mw-magiclink-isbn">ISBN 0-19-855645-4</a>.</li>
<li>J. A. McCammon, S. C. Harvey (1987) <i>Dynamics of Proteins and Nucleic Acids</i>. Cambridge University Press. <a href="/wiki/Special:BookSources/0521307503" class="internal mw-magiclink-isbn">ISBN 0-521-30750-3</a> (hardback).</li>
<li>D. C. Rapaport (1996) <i>The Art of Molecular Dynamics Simulation</i>. <a href="/wiki/Special:BookSources/0521445612" class="internal mw-magiclink-isbn">ISBN 0-521-44561-2</a>.</li>
<li><cite class="citation book"><a href="/wiki/Michael_Griebel" title="Michael Griebel">M. Griebel</a>; S. Knapek; G. Zumbusch (2007). <i>Numerical Simulation in Molecular Dynamics</i>. Berlin, Heidelberg: Springer. <a href="/wiki/International_Standard_Book_Number" title="International Standard Book Number">ISBN</a>&#160;<a href="/wiki/Special:BookSources/978-3-540-68094-9" title="Special:BookSources/978-3-540-68094-9">978-3-540-68094-9</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.au=G.+Zumbusch&amp;rft.au=M.+Griebel&amp;rft.au=S.+Knapek&amp;rft.btitle=Numerical+Simulation+in+Molecular+Dynamics&amp;rft.date=2007&amp;rft.genre=book&amp;rft.isbn=978-3-540-68094-9&amp;rft.place=Berlin%2C+Heidelberg&amp;rft.pub=Springer&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook" class="Z3988"><span style="display:none;">&#160;</span></span></li>
<li><cite class="citation book"><a href="/wiki/Daan_Frenkel" title="Daan Frenkel">Frenkel, Daan</a>; Smit, Berend (2002) [2001]. <i>Understanding Molecular Simulation&#160;: from algorithms to applications</i>. San Diego: Academic Press. <a href="/wiki/International_Standard_Book_Number" title="International Standard Book Number">ISBN</a>&#160;<a href="/wiki/Special:BookSources/0-12-267351-4" title="Special:BookSources/0-12-267351-4">0-12-267351-4</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.aufirst=Daan&amp;rft.aulast=Frenkel&amp;rft.au=Smit%2C+Berend&amp;rft.btitle=Understanding+Molecular+Simulation+%3A+from+algorithms+to+applications&amp;rft.date=2002&amp;rft.genre=book&amp;rft.isbn=0-12-267351-4&amp;rft.place=San+Diego&amp;rft.pub=Academic+Press&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook" class="Z3988"><span style="display:none;">&#160;</span></span></li>
<li>J. M. Haile (2001) <i>Molecular Dynamics Simulation: Elementary Methods</i>. <a href="/wiki/Special:BookSources/047118439X" class="internal mw-magiclink-isbn">ISBN 0-471-18439-X</a></li>
<li>R. J. Sadus, <i>Molecular Simulation of Fluids: Theory, Algorithms and Object-Orientation</i>, 2002, <a href="/wiki/Special:BookSources/0444510826" class="internal mw-magiclink-isbn">ISBN 0-444-51082-6</a></li>
<li>Oren M. Becker, Alexander D. Mackerell, Jr., Benoît Roux, Masakatsu Watanabe (2001) <i>Computational Biochemistry and Biophysics</i>. Marcel Dekker. <a href="/wiki/Special:BookSources/082470455X" class="internal mw-magiclink-isbn">ISBN 0-8247-0455-X</a>.</li>
<li>Andrew Leach (2001) <i>Molecular Modelling: Principles and Applications</i>. (2nd Edition) Prentice Hall. <a href="/wiki/Special:BookSources/9780582382107" class="internal mw-magiclink-isbn">ISBN 978-0-582-38210-7</a>.</li>
<li><a href="/wiki/Tamar_Schlick" title="Tamar Schlick">Tamar Schlick</a> (2002) <i>Molecular Modeling and Simulation</i>. Springer. <a href="/wiki/Special:BookSources/038795404X" class="internal mw-magiclink-isbn">ISBN 0-387-95404-X</a>.</li>
<li><a href="/wiki/William_G_Hoover" title="William G Hoover" class="mw-redirect">William Graham Hoover</a> (1991) <i>Computational Statistical Mechanics</i>, Elsevier, <a href="/wiki/Special:BookSources/0444881921" class="internal mw-magiclink-isbn">ISBN 0-444-88192-1</a>.</li>
<li>D. J. Evans and G. P. Morriss (2008) <i>Statistical Mechanics of Nonequilibrium Liquids</i>, Second Edition, Cambridge University Press, <a href="/wiki/Special:BookSources/9780521857918" class="internal mw-magiclink-isbn">ISBN 978-0-521-85791-8</a>.</li>
<li><cite class="citation journal">Bou-Rabee, Nawaf (2014). <a rel="nofollow" class="external text" href="http://www.mdpi.com/1099-4300/16/1/138">"Time Integrators for Molecular Dynamics"</a>. <i>Entropy</i> (MDPI) <b>16</b> (1): 138–162. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="http://adsabs.harvard.edu/abs/2013Entrp..16..138B">2013Entrp..16..138B</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.3390%2Fe16010138">10.3390/e16010138</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMolecular+dynamics&amp;rft.atitle=Time+Integrators+for+Molecular+Dynamics&amp;rft.au=Bou-Rabee%2C+Nawaf&amp;rft.date=2014&amp;rft.genre=article&amp;rft_id=http%3A%2F%2Fwww.mdpi.com%2F1099-4300%2F16%2F1%2F138&amp;rft_id=info%3Abibcode%2F2013Entrp..16..138B&amp;rft_id=info%3Adoi%2F10.3390%2Fe16010138&amp;rft.issue=1&amp;rft.jtitle=Entropy&amp;rft.pages=138-162&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=16" class="Z3988"><span style="display:none;">&#160;</span></span></li>
</ul>
</div>
<h2><span class="mw-headline" id="External_links">External links</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Molecular_dynamics&amp;action=edit&amp;section=28" title="Edit section: External links">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<table class="mbox-small plainlinks sistersitebox" style="border:1px solid #aaa;background-color:#f9f9f9">
<tr>
<td class="mbox-image"><img alt="" src="//upload.wikimedia.org/wikipedia/en/thumb/4/4a/Commons-logo.svg/30px-Commons-logo.svg.png" width="30" height="40" srcset="//upload.wikimedia.org/wikipedia/en/thumb/4/4a/Commons-logo.svg/45px-Commons-logo.svg.png 1.5x, //upload.wikimedia.org/wikipedia/en/thumb/4/4a/Commons-logo.svg/59px-Commons-logo.svg.png 2x" data-file-width="1024" data-file-height="1376" /></td>
<td class="mbox-text plainlist">Wikimedia Commons has media related to <i><b><a href="//commons.wikimedia.org/wiki/Category:Molecular_dynamics" class="extiw" title="commons:Category:Molecular dynamics">Molecular dynamics</a></b></i>.</td>
</tr>
</table>
<ul>
<li><a rel="nofollow" class="external text" href="http://gpugrid.net/">The GPUGRID.net Project</a> (<a href="/wiki/GPUGRID.net" title="GPUGRID.net">GPUGRID.net</a>)</li>
<li><a rel="nofollow" class="external text" href="http://researchweb.watson.ibm.com/bluegene/">The Blue Gene Project</a> (<a href="/wiki/IBM" title="IBM">IBM</a>)JawBreakers.org</li>
<li><a rel="nofollow" class="external text" href="http://deshawresearch.com/">D. E. Shaw Research</a> (<a href="/wiki/D._E._Shaw_Research" title="D. E. Shaw Research">D. E. Shaw Research</a>)</li>
<li><a rel="nofollow" class="external text" href="http://www.tandf.co.uk/journals/titles/00268976.asp">Molecular Physics</a></li>
<li><a rel="nofollow" class="external text" href="http://nanohub.org/resources/5838">Online course on (MSE 597G) An Introduction to Molecular Dynamics</a> by <a href="/wiki/Alejandro_Strachan" title="Alejandro Strachan">Alejandro Strachan</a></li>
<li><a rel="nofollow" class="external text" href="http://nanohub.org/resources/7570">Lecture Notes on <i>Short Course on Molecular Dynamics Simulation</i> Ashlie Martini (2009)</a></li>
<li><a rel="nofollow" class="external text" href="http://www.sklogwiki.org/SklogWiki/index.php/Materials_modelling_and_computer_simulation_codes">Materials modelling and computer simulation codes</a></li>
<li><a rel="nofollow" class="external text" href="http://www.layruoru.com/dokuwiki/doku.php/">A few tips on molecular dynamics</a></li>
<li><a rel="nofollow" class="external text" href="http://dissertations.ub.rug.nl/faculties/science/1996/h.bekker/">Molecular dynamics simulation methods revised</a></li>
<li><a rel="nofollow" class="external text" href="http://www.youtube.com/watch?v=x8Atqz5YvzQ">Movie of MD simulation of water (Youtube)</a></li>
</ul>


<!-- 
NewPP limit report
Parsed by mw1177
Cached time: 20160221113541
Cache expiry: 2592000
Dynamic content: false
CPU time usage: 0.639 seconds
Real time usage: 0.818 seconds
Preprocessor visited node count: 3905/1000000
Preprocessor generated node count: 0/1500000
Post‐expand include size: 108959/2097152 bytes
Template argument size: 794/2097152 bytes
Highest expansion depth: 11/40
Expensive parser function count: 1/500
Lua time usage: 0.285/10.000 seconds
Lua memory usage: 3.66 MB/50 MB
Number of Wikibase entities loaded: 1-->

<!-- 
Transclusion expansion time report (%,ms,calls,template)
100.00%  576.532      1 - -total
 66.44%  383.027      1 - Template:Reflist
 43.49%  250.741     46 - Template:Cite_journal
  7.34%   42.294      8 - Template:Cite_book
  6.76%   38.995      1 - Template:Citation_needed
  5.85%   33.733      4 - Template:Main
  5.70%   32.857      1 - Template:Fix
  5.60%   32.296      1 - Template:Commons_category
  5.07%   29.239      2 - Template:Category_handler
  1.97%   11.374      1 - Template:Commons
-->

<!-- Saved in parser cache with key enwiki:pcache:idhash:198608-0!*!0!!en!4!*!math=0 and timestamp 20160221113540 and revision id 705263074
 -->
<noscript><img src="//en.wikipedia.org/wiki/Special:CentralAutoLogin/start?type=1x1" alt="" title="" width="1" height="1" style="border: none; position: absolute;" /></noscript></div>					<div class="printfooter">
						Retrieved from "<a dir="ltr" href="https://en.wikipedia.org/w/index.php?title=Molecular_dynamics&amp;oldid=705263074">https://en.wikipedia.org/w/index.php?title=Molecular_dynamics&amp;oldid=705263074</a>"					</div>
				<div id='catlinks' class='catlinks' data-mw='interface'><div id="mw-normal-catlinks" class="mw-normal-catlinks"><a href="/wiki/Help:Category" title="Help:Category">Categories</a>: <ul><li><a href="/wiki/Category:Molecular_dynamics" title="Category:Molecular dynamics">Molecular dynamics</a></li><li><a href="/wiki/Category:Computational_chemistry" title="Category:Computational chemistry">Computational chemistry</a></li><li><a href="/wiki/Category:Molecular_modelling" title="Category:Molecular modelling">Molecular modelling</a></li><li><a href="/wiki/Category:Scientific_modeling" title="Category:Scientific modeling">Scientific modeling</a></li></ul></div><div id="mw-hidden-catlinks" class="mw-hidden-catlinks mw-hidden-cats-hidden">Hidden categories: <ul><li><a href="/wiki/Category:CS1_French-language_sources_(fr)" title="Category:CS1 French-language sources (fr)">CS1 French-language sources (fr)</a></li><li><a href="/wiki/Category:All_articles_with_unsourced_statements" title="Category:All articles with unsourced statements">All articles with unsourced statements</a></li><li><a href="/wiki/Category:Articles_with_unsourced_statements_from_April_2009" title="Category:Articles with unsourced statements from April 2009">Articles with unsourced statements from April 2009</a></li><li><a href="/wiki/Category:Commons_category_with_local_link_same_as_on_Wikidata" title="Category:Commons category with local link same as on Wikidata">Commons category with local link same as on Wikidata</a></li></ul></div></div>				<div class="visualClear"></div>
							</div>
		</div>
		<div id="mw-navigation">
			<h2>Navigation menu</h2>

			<div id="mw-head">
									<div id="p-personal" role="navigation" class="" aria-labelledby="p-personal-label">
						<h3 id="p-personal-label">Personal tools</h3>
						<ul>
							<li id="pt-anonuserpage">Not logged in</li><li id="pt-anontalk"><a href="/wiki/Special:MyTalk" title="Discussion about edits from this IP address [n]" accesskey="n">Talk</a></li><li id="pt-anoncontribs"><a href="/wiki/Special:MyContributions" title="A list of edits made from this IP address [y]" accesskey="y">Contributions</a></li><li id="pt-createaccount"><a href="/w/index.php?title=Special:UserLogin&amp;returnto=Molecular+dynamics&amp;type=signup" title="You are encouraged to create an account and log in; however, it is not mandatory">Create account</a></li><li id="pt-login"><a href="/w/index.php?title=Special:UserLogin&amp;returnto=Molecular+dynamics" title="You're encouraged to log in; however, it's not mandatory. [o]" accesskey="o">Log in</a></li>						</ul>
					</div>
									<div id="left-navigation">
										<div id="p-namespaces" role="navigation" class="vectorTabs" aria-labelledby="p-namespaces-label">
						<h3 id="p-namespaces-label">Namespaces</h3>
						<ul>
															<li  id="ca-nstab-main" class="selected"><span><a href="/wiki/Molecular_dynamics"  title="View the content page [c]" accesskey="c">Article</a></span></li>
															<li  id="ca-talk"><span><a href="/wiki/Talk:Molecular_dynamics"  title="Discussion about the content page [t]" accesskey="t" rel="discussion">Talk</a></span></li>
													</ul>
					</div>
										<div id="p-variants" role="navigation" class="vectorMenu emptyPortlet" aria-labelledby="p-variants-label">
												<h3 id="p-variants-label">
							<span>Variants</span><a href="#"></a>
						</h3>

						<div class="menu">
							<ul>
															</ul>
						</div>
					</div>
									</div>
				<div id="right-navigation">
										<div id="p-views" role="navigation" class="vectorTabs" aria-labelledby="p-views-label">
						<h3 id="p-views-label">Views</h3>
						<ul>
															<li id="ca-view" class="selected"><span><a href="/wiki/Molecular_dynamics" >Read</a></span></li>
															<li id="ca-edit"><span><a href="/w/index.php?title=Molecular_dynamics&amp;action=edit"  title="Edit this page [e]" accesskey="e">Edit</a></span></li>
															<li id="ca-history" class="collapsible"><span><a href="/w/index.php?title=Molecular_dynamics&amp;action=history"  title="Past revisions of this page [h]" accesskey="h">View history</a></span></li>
													</ul>
					</div>
										<div id="p-cactions" role="navigation" class="vectorMenu emptyPortlet" aria-labelledby="p-cactions-label">
						<h3 id="p-cactions-label"><span>More</span><a href="#"></a></h3>

						<div class="menu">
							<ul>
															</ul>
						</div>
					</div>
										<div id="p-search" role="search">
						<h3>
							<label for="searchInput">Search</label>
						</h3>

						<form action="/w/index.php" id="searchform">
							<div id="simpleSearch">
							<input type="search" name="search" placeholder="Search" title="Search Wikipedia [f]" accesskey="f" id="searchInput" /><input type="hidden" value="Special:Search" name="title" /><input type="submit" name="fulltext" value="Search" title="Search Wikipedia for this text" id="mw-searchButton" class="searchButton mw-fallbackSearchButton" /><input type="submit" name="go" value="Go" title="Go to a page with this exact name if it exists" id="searchButton" class="searchButton" />							</div>
						</form>
					</div>
									</div>
			</div>
			<div id="mw-panel">
				<div id="p-logo" role="banner"><a class="mw-wiki-logo" href="/wiki/Main_Page"  title="Visit the main page"></a></div>
						<div class="portal" role="navigation" id='p-navigation' aria-labelledby='p-navigation-label'>
			<h3 id='p-navigation-label'>Navigation</h3>

			<div class="body">
									<ul>
						<li id="n-mainpage-description"><a href="/wiki/Main_Page" title="Visit the main page [z]" accesskey="z">Main page</a></li><li id="n-contents"><a href="/wiki/Portal:Contents" title="Guides to browsing Wikipedia">Contents</a></li><li id="n-featuredcontent"><a href="/wiki/Portal:Featured_content" title="Featured content – the best of Wikipedia">Featured content</a></li><li id="n-currentevents"><a href="/wiki/Portal:Current_events" title="Find background information on current events">Current events</a></li><li id="n-randompage"><a href="/wiki/Special:Random" title="Load a random article [x]" accesskey="x">Random article</a></li><li id="n-sitesupport"><a href="https://donate.wikimedia.org/wiki/Special:FundraiserRedirector?utm_source=donate&amp;utm_medium=sidebar&amp;utm_campaign=C13_en.wikipedia.org&amp;uselang=en" title="Support us">Donate to Wikipedia</a></li><li id="n-shoplink"><a href="//shop.wikimedia.org" title="Visit the Wikipedia store">Wikipedia store</a></li>					</ul>
							</div>
		</div>
			<div class="portal" role="navigation" id='p-interaction' aria-labelledby='p-interaction-label'>
			<h3 id='p-interaction-label'>Interaction</h3>

			<div class="body">
									<ul>
						<li id="n-help"><a href="/wiki/Help:Contents" title="Guidance on how to use and edit Wikipedia">Help</a></li><li id="n-aboutsite"><a href="/wiki/Wikipedia:About" title="Find out about Wikipedia">About Wikipedia</a></li><li id="n-portal"><a href="/wiki/Wikipedia:Community_portal" title="About the project, what you can do, where to find things">Community portal</a></li><li id="n-recentchanges"><a href="/wiki/Special:RecentChanges" title="A list of recent changes in the wiki [r]" accesskey="r">Recent changes</a></li><li id="n-contactpage"><a href="//en.wikipedia.org/wiki/Wikipedia:Contact_us" title="How to contact Wikipedia">Contact page</a></li>					</ul>
							</div>
		</div>
			<div class="portal" role="navigation" id='p-tb' aria-labelledby='p-tb-label'>
			<h3 id='p-tb-label'>Tools</h3>

			<div class="body">
									<ul>
						<li id="t-whatlinkshere"><a href="/wiki/Special:WhatLinksHere/Molecular_dynamics" title="List of all English Wikipedia pages containing links to this page [j]" accesskey="j">What links here</a></li><li id="t-recentchangeslinked"><a href="/wiki/Special:RecentChangesLinked/Molecular_dynamics" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</a></li><li id="t-upload"><a href="/wiki/Wikipedia:File_Upload_Wizard" title="Upload files [u]" accesskey="u">Upload file</a></li><li id="t-specialpages"><a href="/wiki/Special:SpecialPages" title="A list of all special pages [q]" accesskey="q">Special pages</a></li><li id="t-permalink"><a href="/w/index.php?title=Molecular_dynamics&amp;oldid=705263074" title="Permanent link to this revision of the page">Permanent link</a></li><li id="t-info"><a href="/w/index.php?title=Molecular_dynamics&amp;action=info" title="More information about this page">Page information</a></li><li id="t-wikibase"><a href="//www.wikidata.org/wiki/Q901663" title="Link to connected data repository item [g]" accesskey="g">Wikidata item</a></li><li id="t-cite"><a href="/w/index.php?title=Special:CiteThisPage&amp;page=Molecular_dynamics&amp;id=705263074" title="Information on how to cite this page">Cite this page</a></li>					</ul>
							</div>
		</div>
			<div class="portal" role="navigation" id='p-coll-print_export' aria-labelledby='p-coll-print_export-label'>
			<h3 id='p-coll-print_export-label'>Print/export</h3>

			<div class="body">
									<ul>
						<li id="coll-create_a_book"><a href="/w/index.php?title=Special:Book&amp;bookcmd=book_creator&amp;referer=Molecular+dynamics">Create a book</a></li><li id="coll-download-as-rdf2latex"><a href="/w/index.php?title=Special:Book&amp;bookcmd=render_article&amp;arttitle=Molecular+dynamics&amp;returnto=Molecular+dynamics&amp;oldid=705263074&amp;writer=rdf2latex">Download as PDF</a></li><li id="t-print"><a href="/w/index.php?title=Molecular_dynamics&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</a></li>					</ul>
							</div>
		</div>
			<div class="portal" role="navigation" id='p-wikibase-otherprojects' aria-labelledby='p-wikibase-otherprojects-label'>
			<h3 id='p-wikibase-otherprojects-label'>Other projects</h3>

			<div class="body">
									<ul>
						<li class="wb-otherproject-link wb-otherproject-commons"><a href="https://commons.wikimedia.org/wiki/Category:Molecular_dynamics" hreflang="en">Wikimedia Commons</a></li><li class="wb-otherproject-link wb-otherproject-wikiquote"><a href="https://en.wikiquote.org/wiki/Molecular_dynamics" hreflang="en">Wikiquote</a></li>					</ul>
							</div>
		</div>
			<div class="portal" role="navigation" id='p-lang' aria-labelledby='p-lang-label'>
			<h3 id='p-lang-label'>Languages</h3>

			<div class="body">
									<ul>
						<li class="interlanguage-link interwiki-ca"><a href="//ca.wikipedia.org/wiki/Din%C3%A0mica_molecular" title="Dinàmica molecular – Catalan" lang="ca" hreflang="ca">Català</a></li><li class="interlanguage-link interwiki-de"><a href="//de.wikipedia.org/wiki/Molekulardynamik-Simulation" title="Molekulardynamik-Simulation – German" lang="de" hreflang="de">Deutsch</a></li><li class="interlanguage-link interwiki-et"><a href="//et.wikipedia.org/wiki/Molekulaard%C3%BCnaamika" title="Molekulaardünaamika – Estonian" lang="et" hreflang="et">Eesti</a></li><li class="interlanguage-link interwiki-es"><a href="//es.wikipedia.org/wiki/Din%C3%A1mica_molecular" title="Dinámica molecular – Spanish" lang="es" hreflang="es">Español</a></li><li class="interlanguage-link interwiki-fa"><a href="//fa.wikipedia.org/wiki/%D8%AF%DB%8C%D9%86%D8%A7%D9%85%DB%8C%DA%A9_%D9%85%D9%84%DA%A9%D9%88%D9%84%DB%8C" title="دینامیک ملکولی – Persian" lang="fa" hreflang="fa">فارسی</a></li><li class="interlanguage-link interwiki-fr"><a href="//fr.wikipedia.org/wiki/Dynamique_mol%C3%A9culaire" title="Dynamique moléculaire – French" lang="fr" hreflang="fr">Français</a></li><li class="interlanguage-link interwiki-ko"><a href="//ko.wikipedia.org/wiki/%EB%B6%84%EC%9E%90%EB%8F%99%EC%97%AD%ED%95%99" title="분자동역학 – Korean" lang="ko" hreflang="ko">한국어</a></li><li class="interlanguage-link interwiki-it"><a href="//it.wikipedia.org/wiki/Dinamica_molecolare" title="Dinamica molecolare – Italian" lang="it" hreflang="it">Italiano</a></li><li class="interlanguage-link interwiki-nl"><a href="//nl.wikipedia.org/wiki/Moleculaire_dynamica" title="Moleculaire dynamica – Dutch" lang="nl" hreflang="nl">Nederlands</a></li><li class="interlanguage-link interwiki-ja"><a href="//ja.wikipedia.org/wiki/%E5%88%86%E5%AD%90%E5%8B%95%E5%8A%9B%E5%AD%A6%E6%B3%95" title="分子動力学法 – Japanese" lang="ja" hreflang="ja">日本語</a></li><li class="interlanguage-link interwiki-no"><a href="//no.wikipedia.org/wiki/Molekyl%C3%A6rdynamikk" title="Molekylærdynamikk – Norwegian" lang="no" hreflang="no">Norsk bokmål</a></li><li class="interlanguage-link interwiki-pl"><a href="//pl.wikipedia.org/wiki/Dynamika_molekularna" title="Dynamika molekularna – Polish" lang="pl" hreflang="pl">Polski</a></li><li class="interlanguage-link interwiki-pt"><a href="//pt.wikipedia.org/wiki/Din%C3%A2mica_molecular" title="Dinâmica molecular – Portuguese" lang="pt" hreflang="pt">Português</a></li><li class="interlanguage-link interwiki-ru"><a href="//ru.wikipedia.org/wiki/%D0%9C%D0%B5%D1%82%D0%BE%D0%B4_%D0%BA%D0%BB%D0%B0%D1%81%D1%81%D0%B8%D1%87%D0%B5%D1%81%D0%BA%D0%BE%D0%B9_%D0%BC%D0%BE%D0%BB%D0%B5%D0%BA%D1%83%D0%BB%D1%8F%D1%80%D0%BD%D0%BE%D0%B9_%D0%B4%D0%B8%D0%BD%D0%B0%D0%BC%D0%B8%D0%BA%D0%B8" title="Метод классической молекулярной динамики – Russian" lang="ru" hreflang="ru">Русский</a></li><li class="interlanguage-link interwiki-sr"><a href="//sr.wikipedia.org/wiki/%D0%9C%D0%BE%D0%BB%D0%B5%D0%BA%D1%83%D0%BB%D1%81%D0%BA%D0%B0_%D0%B4%D0%B8%D0%BD%D0%B0%D0%BC%D0%B8%D0%BA%D0%B0" title="Молекулска динамика – Serbian" lang="sr" hreflang="sr">Српски / srpski</a></li><li class="interlanguage-link interwiki-sh"><a href="//sh.wikipedia.org/wiki/Molekulska_dinamika" title="Molekulska dinamika – Serbo-Croatian" lang="sh" hreflang="sh">Srpskohrvatski / српскохрватски</a></li><li class="interlanguage-link interwiki-sv"><a href="//sv.wikipedia.org/wiki/Molekyldynamik" title="Molekyldynamik – Swedish" lang="sv" hreflang="sv">Svenska</a></li><li class="interlanguage-link interwiki-tr"><a href="//tr.wikipedia.org/wiki/Molek%C3%BCler_dinamik" title="Moleküler dinamik – Turkish" lang="tr" hreflang="tr">Türkçe</a></li><li class="interlanguage-link interwiki-uk"><a href="//uk.wikipedia.org/wiki/%D0%9C%D0%BE%D0%BB%D0%B5%D0%BA%D1%83%D0%BB%D1%8F%D1%80%D0%BD%D0%B0_%D0%B4%D0%B8%D0%BD%D0%B0%D0%BC%D1%96%D0%BA%D0%B0" title="Молекулярна динаміка – Ukrainian" lang="uk" hreflang="uk">Українська</a></li><li class="interlanguage-link interwiki-zh"><a href="//zh.wikipedia.org/wiki/%E5%88%86%E5%AD%90%E5%8A%A8%E5%8A%9B%E5%AD%A6" title="分子动力学 – Chinese" lang="zh" hreflang="zh">中文</a></li><li class="uls-p-lang-dummy"><a href="#"></a></li>					</ul>
				<div class='after-portlet after-portlet-lang'><span class="wb-langlinks-edit wb-langlinks-link"><a href="//www.wikidata.org/wiki/Q901663#sitelinks-wikipedia" title="Edit interlanguage links" class="wbc-editpage">Edit links</a></span></div>			</div>
		</div>
				</div>
		</div>
		<div id="footer" role="contentinfo">
							<ul id="footer-info">
											<li id="footer-info-lastmod"> This page was last modified on 16 February 2016, at 13:33.</li>
											<li id="footer-info-copyright">Text is available under the <a rel="license" href="//en.wikipedia.org/wiki/Wikipedia:Text_of_Creative_Commons_Attribution-ShareAlike_3.0_Unported_License">Creative Commons Attribution-ShareAlike License</a><a rel="license" href="//creativecommons.org/licenses/by-sa/3.0/" style="display:none;"></a>;
additional terms may apply.  By using this site, you agree to the <a href="//wikimediafoundation.org/wiki/Terms_of_Use">Terms of Use</a> and <a href="//wikimediafoundation.org/wiki/Privacy_policy">Privacy Policy</a>. Wikipedia® is a registered trademark of the <a href="//www.wikimediafoundation.org/">Wikimedia Foundation, Inc.</a>, a non-profit organization.</li>
									</ul>
							<ul id="footer-places">
											<li id="footer-places-privacy"><a href="//wikimediafoundation.org/wiki/Privacy_policy" title="wmf:Privacy policy">Privacy policy</a></li>
											<li id="footer-places-about"><a href="/wiki/Wikipedia:About" title="Wikipedia:About">About Wikipedia</a></li>
											<li id="footer-places-disclaimer"><a href="/wiki/Wikipedia:General_disclaimer" title="Wikipedia:General disclaimer">Disclaimers</a></li>
											<li id="footer-places-contact"><a href="//en.wikipedia.org/wiki/Wikipedia:Contact_us">Contact Wikipedia</a></li>
											<li id="footer-places-developers"><a href="https://www.mediawiki.org/wiki/Special:MyLanguage/How_to_contribute">Developers</a></li>
											<li id="footer-places-cookiestatement"><a href="//wikimediafoundation.org/wiki/Cookie_statement">Cookie statement</a></li>
											<li id="footer-places-mobileview"><a href="//en.m.wikipedia.org/w/index.php?title=Molecular_dynamics&amp;mobileaction=toggle_view_mobile" class="noprint stopMobileRedirectToggle">Mobile view</a></li>
									</ul>
										<ul id="footer-icons" class="noprint">
											<li id="footer-copyrightico">
							<a href="//wikimediafoundation.org/"><img src="/static/images/wikimedia-button.png" srcset="/static/images/wikimedia-button-1.5x.png 1.5x, /static/images/wikimedia-button-2x.png 2x" width="88" height="31" alt="Wikimedia Foundation"/></a>						</li>
											<li id="footer-poweredbyico">
							<a href="//www.mediawiki.org/"><img src="/static/1.27.0-wmf.13/resources/assets/poweredby_mediawiki_88x31.png" alt="Powered by MediaWiki" srcset="/static/1.27.0-wmf.13/resources/assets/poweredby_mediawiki_132x47.png 1.5x, /static/1.27.0-wmf.13/resources/assets/poweredby_mediawiki_176x62.png 2x" width="88" height="31" /></a>						</li>
									</ul>
						<div style="clear:both"></div>
		</div>
		<script>(window.RLQ = window.RLQ || []).push(function () {
mw.loader.state({"ext.globalCssJs.site":"ready","ext.globalCssJs.user":"ready","user":"ready","user.groups":"ready"});mw.loader.load(["ext.cite.a11y","mediawiki.toc","mediawiki.action.view.postEdit","site","mediawiki.user","mediawiki.hidpi","mediawiki.page.ready","mediawiki.searchSuggest","ext.eventLogging.subscriber","ext.gadget.teahouse","ext.gadget.ReferenceTooltips","ext.gadget.DRN-wizard","ext.gadget.charinsert","ext.gadget.refToolbar","ext.gadget.switcher","ext.gadget.featured-articles-links","mmv.bootstrap.autostart","ext.visualEditor.targetLoader","ext.wikimediaEvents","ext.navigationTiming","schema.UniversalLanguageSelector","ext.uls.eventlogger","ext.uls.interlanguage"]);
} );</script><script>(window.RLQ = window.RLQ || []).push(function () {
mw.config.set({"wgBackendResponseTime":98,"wgHostname":"mw1172"}); /* @nomin */
} );</script>
	</body>
</html>
