<!DOCTYPE html>
<html lang="en" dir="ltr" class="client-nojs">
<head>
<meta charset="UTF-8" />
<title>Proteomics - Wikipedia, the free encyclopedia</title>
<script>document.documentElement.className = document.documentElement.className.replace( /(^|\s)client-nojs(\s|$)/, "$1client-js$2" );</script>
<script>(window.RLQ = window.RLQ || []).push(function () {
mw.config.set({"wgCanonicalNamespace":"","wgCanonicalSpecialPageName":false,"wgNamespaceNumber":0,"wgPageName":"Proteomics","wgTitle":"Proteomics","wgCurRevisionId":705701793,"wgRevisionId":705701793,"wgArticleId":55172,"wgIsArticle":true,"wgIsRedirect":false,"wgAction":"view","wgUserName":null,"wgUserGroups":["*"],"wgCategories":["All articles lacking reliable references","Articles lacking reliable references from November 2013","Pages with DOIs inactive since 2015","All articles with unsourced statements","Articles with unsourced statements from October 2013","Articles with unsourced statements from October 2014","Articles with unsourced statements from January 2012","CS1 errors: missing author or editor","Articles with DMOZ links","Proteomics","Bioinformatics","Genomics","Proteins"],"wgBreakFrames":false,"wgPageContentLanguage":"en","wgPageContentModel":"wikitext","wgSeparatorTransformTable":["",""],"wgDigitTransformTable":["",""],"wgDefaultDateFormat":"dmy","wgMonthNames":["","January","February","March","April","May","June","July","August","September","October","November","December"],"wgMonthNamesShort":["","Jan","Feb","Mar","Apr","May","Jun","Jul","Aug","Sep","Oct","Nov","Dec"],"wgRelevantPageName":"Proteomics","wgRelevantArticleId":55172,"wgIsProbablyEditable":true,"wgRestrictionEdit":[],"wgRestrictionMove":[],"wgWikiEditorEnabledModules":{"toolbar":true,"dialogs":true,"preview":false,"publish":false},"wgBetaFeaturesFeatures":[],"wgMediaViewerOnClick":true,"wgMediaViewerEnabledByDefault":true,"wgVisualEditor":{"pageLanguageCode":"en","pageLanguageDir":"ltr","usePageImages":true,"usePageDescriptions":true},"wikilove-recipient":"","wikilove-anon":0,"wgGatherShouldShowTutorial":true,"wgGatherEnableSample":0,"wgGatherPageImageThumbnail":"//upload.wikimedia.org/wikipedia/commons/thumb/5/5b/Protein_pattern_analyzer.jpg/100px-Protein_pattern_analyzer.jpg","wgRelatedArticles":null,"wgRelatedArticlesUseCirrusSearch":true,"wgRelatedArticlesOnlyUseCirrusSearch":false,"wgULSAcceptLanguageList":["en-us","en"],"wgULSCurrentAutonym":"English","wgFlaggedRevsParams":{"tags":{"status":{"levels":1,"quality":2,"pristine":3}}},"wgStableRevisionId":null,"wgCategoryTreePageCategoryOptions":"{\"mode\":0,\"hideprefix\":20,\"showcount\":true,\"namespaces\":false}","wgNoticeProject":"wikipedia","wgCentralNoticeCategoriesUsingLegacy":["Fundraising","fundraising"],"wgCentralAuthMobileDomain":false,"wgWikibaseItemId":"Q471857","wgVisualEditorToolbarScrollOffset":0}); /* @nomin */mw.loader.implement("user.options",function($,jQuery){mw.user.options.set({"variant":"en"});});mw.loader.implement("user.tokens",function ( $, jQuery ) {
mw.user.tokens.set({"editToken":"+\\","patrolToken":"+\\","watchToken":"+\\","csrfToken":"+\\"}); /* @nomin */ ;

});mw.loader.load(["mediawiki.page.startup","mediawiki.legacy.wikibits","ext.centralauth.centralautologin","ext.gadget.WatchlistBase","ext.gadget.WatchlistGreenIndicators","mmv.head","ext.visualEditor.desktopArticleTarget.init","ext.uls.init","ext.uls.interface","ext.quicksurveys.init","mw.MediaWikiPlayer.loader","mw.PopUpMediaTransform","ext.centralNotice.bannerController","skins.vector.js"]);
} );</script>
<link rel="stylesheet" href="/w/load.php?debug=false&amp;lang=en&amp;modules=ext.cite.styles%7Cext.gadget.DRN-wizard%2CReferenceTooltips%2CWatchlistBase%2CWatchlistGreenIndicators%2Ccharinsert%2Cfeatured-articles-links%2CrefToolbar%2Cswitcher%2Cteahouse%7Cext.tmh.thumbnail.styles%7Cext.uls.nojs%7Cext.visualEditor.desktopArticleTarget.noscript%7Cext.wikimediaBadges%7Cmediawiki.legacy.commonPrint%2Cshared%7Cmediawiki.raggett%2CsectionAnchor%7Cmediawiki.skinning.interface%7Cskins.vector.styles%7Cwikibase.client.init&amp;only=styles&amp;skin=vector" />
<meta name="ResourceLoaderDynamicStyles" content="" />
<link rel="stylesheet" href="/w/load.php?debug=false&amp;lang=en&amp;modules=site&amp;only=styles&amp;skin=vector" />
<script async="" src="/w/load.php?debug=false&amp;lang=en&amp;modules=startup&amp;only=scripts&amp;skin=vector"></script>
<meta name="generator" content="MediaWiki 1.27.0-wmf.13" />
<link rel="alternate" href="android-app://org.wikipedia/http/en.m.wikipedia.org/wiki/Proteomics" />
<link rel="alternate" type="application/x-wiki" title="Edit this page" href="/w/index.php?title=Proteomics&amp;action=edit" />
<link rel="edit" title="Edit this page" href="/w/index.php?title=Proteomics&amp;action=edit" />
<link rel="apple-touch-icon" href="/static/apple-touch/wikipedia.png" />
<link rel="shortcut icon" href="/static/favicon/wikipedia.ico" />
<link rel="search" type="application/opensearchdescription+xml" href="/w/opensearch_desc.php" title="Wikipedia (en)" />
<link rel="EditURI" type="application/rsd+xml" href="//en.wikipedia.org/w/api.php?action=rsd" />
<link rel="copyright" href="//creativecommons.org/licenses/by-sa/3.0/" />
<link rel="canonical" href="https://en.wikipedia.org/wiki/Proteomics" />
<link rel="dns-prefetch" href="//meta.wikimedia.org" />
<!--[if lt IE 7]><style type="text/css">body{behavior:url("/static/1.27.0-wmf.13/skins/Vector/csshover.min.htc")}</style><![endif]-->
</head>
<body class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Proteomics skin-vector action-view">
		<div id="mw-page-base" class="noprint"></div>
		<div id="mw-head-base" class="noprint"></div>
		<div id="content" class="mw-body" role="main">
			<a id="top"></a>

							<div id="siteNotice"><!-- CentralNotice --></div>
						<div class="mw-indicators">
</div>
			<h1 id="firstHeading" class="firstHeading" lang="en">Proteomics</h1>
									<div id="bodyContent" class="mw-body-content">
									<div id="siteSub">From Wikipedia, the free encyclopedia</div>
								<div id="contentSub"></div>
												<div id="jump-to-nav" class="mw-jump">
					Jump to:					<a href="#mw-head">navigation</a>, 					<a href="#p-search">search</a>
				</div>
				<div id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><div role="note" class="hatnote">For the journal Proteomics, see <a href="/wiki/Proteomics_(journal)" title="Proteomics (journal)">Proteomics (journal)</a>.</div>
<div class="thumb tright">
<div class="thumbinner" style="width:302px;"><a href="/wiki/File:Protein_pattern_analyzer.jpg" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/thumb/5/5b/Protein_pattern_analyzer.jpg/300px-Protein_pattern_analyzer.jpg" width="300" height="225" class="thumbimage" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/5/5b/Protein_pattern_analyzer.jpg/450px-Protein_pattern_analyzer.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/5/5b/Protein_pattern_analyzer.jpg/600px-Protein_pattern_analyzer.jpg 2x" data-file-width="1024" data-file-height="768" /></a>
<div class="thumbcaption">
<div class="magnify"><a href="/wiki/File:Protein_pattern_analyzer.jpg" class="internal" title="Enlarge"></a></div>
Robotic preparation of <a href="/wiki/MALDI" title="MALDI" class="mw-redirect">MALDI</a> <a href="/wiki/Mass_spectrometry" title="Mass spectrometry">mass spectrometry</a> samples on a sample carrier.</div>
</div>
</div>
<p><b>Proteomics</b> is the large-scale study of <a href="/wiki/Protein" title="Protein">proteins</a>, particularly their <a href="/wiki/Protein_structure" title="Protein structure">structures</a> and <a href="/wiki/Functional_genomics" title="Functional genomics">functions</a>.<sup id="cite_ref-pmid9740045_1-0" class="reference"><a href="#cite_note-pmid9740045-1"><span>[</span>1<span>]</span></a></sup><sup id="cite_ref-pmid10189717_2-0" class="reference"><a href="#cite_note-pmid10189717-2"><span>[</span>2<span>]</span></a></sup> Proteins are vital parts of living organisms, as they are the main components of the physiological <a href="/wiki/Metabolic_pathways" title="Metabolic pathways" class="mw-redirect">metabolic pathways</a> of <a href="/wiki/Biological_cell" title="Biological cell" class="mw-redirect">cells</a>. The term <i>proteomics</i> was first coined in 1997<sup id="cite_ref-3" class="reference"><a href="#cite_note-3"><span>[</span>3<span>]</span></a></sup> to make an analogy with <a href="/wiki/Genomics" title="Genomics">genomics</a>, the study of the <a href="/wiki/Genome" title="Genome">genome</a>. The word <i>proteome</i> is a <a href="/wiki/Portmanteau" title="Portmanteau">portmanteau</a> of <i>prote</i>in and gen<i>ome</i>, and was coined by <a href="/wiki/Marc_Wilkins_(geneticist)" title="Marc Wilkins (geneticist)">Marc Wilkins</a> in 1994 while working on the concept as a PhD student.<sup id="cite_ref-wilkins1996_4-0" class="reference"><a href="#cite_note-wilkins1996-4"><span>[</span>4<span>]</span></a></sup><sup id="cite_ref-5" class="reference"><a href="#cite_note-5"><span>[</span>5<span>]</span></a></sup></p>
<p>The <a href="/wiki/Proteome" title="Proteome">proteome</a> is the entire set of proteins,<sup id="cite_ref-wilkins1996_4-1" class="reference"><a href="#cite_note-wilkins1996-4"><span>[</span>4<span>]</span></a></sup> produced or modified by an organism or system. This varies with time and distinct requirements, or stresses, that a cell or organism undergoes. Proteomics is an interdisciplinary domain formed on the basis of the research and development of the <a href="/wiki/Human_Genome_Project" title="Human Genome Project">Human Genome Project</a>;<sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">[<i><a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed"><span title="This claim needs references to reliable sources. (October 2013)">citation needed</span></a></i>]</sup> it is also emerging scientific research and exploration of proteomes from the overall level of intracellular protein composition, structure, and its own unique activity patterns. It is an important component of functional genomics.</p>
<p>While <i>proteomics</i> generally refers to the large-scale experimental analysis of proteins, it is often specifically used for protein purification and mass spectrometry.</p>
<p></p>
<div id="toc" class="toc">
<div id="toctitle">
<h2>Contents</h2>
</div>
<ul>
<li class="toclevel-1 tocsection-1"><a href="#Complexity_of_the_problem"><span class="tocnumber">1</span> <span class="toctext">Complexity of the problem</span></a>
<ul>
<li class="toclevel-2 tocsection-2"><a href="#Post-translational_modifications"><span class="tocnumber">1.1</span> <span class="toctext">Post-translational modifications</span></a>
<ul>
<li class="toclevel-3 tocsection-3"><a href="#Phosphorylation"><span class="tocnumber">1.1.1</span> <span class="toctext">Phosphorylation</span></a></li>
<li class="toclevel-3 tocsection-4"><a href="#Ubiquitination"><span class="tocnumber">1.1.2</span> <span class="toctext">Ubiquitination</span></a></li>
<li class="toclevel-3 tocsection-5"><a href="#Additional_modifications"><span class="tocnumber">1.1.3</span> <span class="toctext">Additional modifications</span></a></li>
</ul>
</li>
<li class="toclevel-2 tocsection-6"><a href="#Distinct_proteins_are_made_under_distinct_settings"><span class="tocnumber">1.2</span> <span class="toctext">Distinct proteins are made under distinct settings</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-7"><a href="#Limitations_of_genomics_and_proteomics_studies"><span class="tocnumber">2</span> <span class="toctext">Limitations of genomics and proteomics studies</span></a></li>
<li class="toclevel-1 tocsection-8"><a href="#Methods_of_studying_proteins"><span class="tocnumber">3</span> <span class="toctext">Methods of studying proteins</span></a>
<ul>
<li class="toclevel-2 tocsection-9"><a href="#Protein_detection_with_antibodies_.28immunoassays.29"><span class="tocnumber">3.1</span> <span class="toctext">Protein detection with antibodies (immunoassays)</span></a></li>
<li class="toclevel-2 tocsection-10"><a href="#Antibody-free_protein_detection"><span class="tocnumber">3.2</span> <span class="toctext">Antibody-free protein detection</span></a>
<ul>
<li class="toclevel-3 tocsection-11"><a href="#Detection_methods"><span class="tocnumber">3.2.1</span> <span class="toctext">Detection methods</span></a></li>
<li class="toclevel-3 tocsection-12"><a href="#Separation_methods"><span class="tocnumber">3.2.2</span> <span class="toctext">Separation methods</span></a></li>
</ul>
</li>
<li class="toclevel-2 tocsection-13"><a href="#Hybrid_technologies"><span class="tocnumber">3.3</span> <span class="toctext">Hybrid technologies</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-14"><a href="#Establishing_protein.E2.80.93protein_interactions"><span class="tocnumber">4</span> <span class="toctext">Establishing protein–protein interactions</span></a></li>
<li class="toclevel-1 tocsection-15"><a href="#Practical_applications_of_proteomics"><span class="tocnumber">5</span> <span class="toctext">Practical applications of proteomics</span></a>
<ul>
<li class="toclevel-2 tocsection-16"><a href="#Biomarkers"><span class="tocnumber">5.1</span> <span class="toctext">Biomarkers</span></a></li>
<li class="toclevel-2 tocsection-17"><a href="#Proteogenomics"><span class="tocnumber">5.2</span> <span class="toctext">Proteogenomics</span></a></li>
<li class="toclevel-2 tocsection-18"><a href="#Current_research_methodologies"><span class="tocnumber">5.3</span> <span class="toctext">Current research methodologies</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-19"><a href="#Structural_proteomics"><span class="tocnumber">6</span> <span class="toctext">Structural proteomics</span></a></li>
<li class="toclevel-1 tocsection-20"><a href="#Expression_proteomics"><span class="tocnumber">7</span> <span class="toctext">Expression proteomics</span></a></li>
<li class="toclevel-1 tocsection-21"><a href="#Interaction_proteomics"><span class="tocnumber">8</span> <span class="toctext">Interaction proteomics</span></a></li>
<li class="toclevel-1 tocsection-22"><a href="#Current_proteomic_technologies"><span class="tocnumber">9</span> <span class="toctext">Current proteomic technologies</span></a>
<ul>
<li class="toclevel-2 tocsection-23"><a href="#Mass_spectrometry_and_protein_profiling"><span class="tocnumber">9.1</span> <span class="toctext">Mass spectrometry and protein profiling</span></a></li>
<li class="toclevel-2 tocsection-24"><a href="#Protein_chips"><span class="tocnumber">9.2</span> <span class="toctext">Protein chips</span></a></li>
<li class="toclevel-2 tocsection-25"><a href="#Reverse-phased_protein_microarrays"><span class="tocnumber">9.3</span> <span class="toctext">Reverse-phased protein microarrays</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-26"><a href="#Bioinformatics_for_proteomics_.28proteome_informatics.29"><span class="tocnumber">10</span> <span class="toctext">Bioinformatics for proteomics (proteome informatics)</span></a>
<ul>
<li class="toclevel-2 tocsection-27"><a href="#Protein_identification"><span class="tocnumber">10.1</span> <span class="toctext">Protein identification</span></a></li>
<li class="toclevel-2 tocsection-28"><a href="#Protein_structure"><span class="tocnumber">10.2</span> <span class="toctext">Protein structure</span></a></li>
<li class="toclevel-2 tocsection-29"><a href="#Post-translational_modifications_2"><span class="tocnumber">10.3</span> <span class="toctext">Post-translational modifications</span></a></li>
<li class="toclevel-2 tocsection-30"><a href="#Computational_methods_in_studying_protein_biomarkers"><span class="tocnumber">10.4</span> <span class="toctext">Computational methods in studying protein biomarkers</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-31"><a href="#Emerging_trends_in_proteomics"><span class="tocnumber">11</span> <span class="toctext">Emerging trends in proteomics</span></a>
<ul>
<li class="toclevel-2 tocsection-32"><a href="#Human_plasma_proteome"><span class="tocnumber">11.1</span> <span class="toctext">Human plasma proteome</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-33"><a href="#See_also"><span class="tocnumber">12</span> <span class="toctext">See also</span></a>
<ul>
<li class="toclevel-2 tocsection-34"><a href="#Protein_databases"><span class="tocnumber">12.1</span> <span class="toctext">Protein databases</span></a></li>
<li class="toclevel-2 tocsection-35"><a href="#Research_centers"><span class="tocnumber">12.2</span> <span class="toctext">Research centers</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-36"><a href="#References"><span class="tocnumber">13</span> <span class="toctext">References</span></a></li>
<li class="toclevel-1 tocsection-37"><a href="#Bibliography"><span class="tocnumber">14</span> <span class="toctext">Bibliography</span></a></li>
<li class="toclevel-1 tocsection-38"><a href="#External_links"><span class="tocnumber">15</span> <span class="toctext">External links</span></a></li>
</ul>
</div>
<p></p>
<h2><span class="mw-headline" id="Complexity_of_the_problem">Complexity of the problem</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=1" title="Edit section: Complexity of the problem">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>After genomics and <a href="/wiki/Transcriptome" title="Transcriptome">transcriptomics</a>, proteomics is the next step in the study of biological systems. It is more complicated than genomics because an organism's genome is more or less constant, whereas the proteome differs from cell to cell and from time to time. Distinct genes are expressed in different cell types, which means that even the basic set of proteins that are produced in a cell needs to be identified.</p>
<p>In the past this phenomenon was done by RNA analysis, but it was found not to correlate with protein content.<sup id="cite_ref-6" class="reference"><a href="#cite_note-6"><span>[</span>6<span>]</span></a></sup><sup id="cite_ref-7" class="reference"><a href="#cite_note-7"><span>[</span>7<span>]</span></a></sup> It is now known that mRNA is not always translated into protein,<sup id="cite_ref-8" class="reference"><a href="#cite_note-8"><span>[</span>8<span>]</span></a></sup> and the amount of protein produced for a given amount of mRNA depends on the gene it is transcribed from and on the current physiological state of the cell. Proteomics confirms the presence of the protein and provides a direct measure of the quantity present.</p>
<h3><span class="mw-headline" id="Post-translational_modifications">Post-translational modifications</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=2" title="Edit section: Post-translational modifications">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>Not only does the translation from mRNA cause differences, but many proteins are also subjected to a wide variety of chemical modifications after translation. Many of these post-translational modifications are critical to the protein's function.</p>
<h4><span class="mw-headline" id="Phosphorylation">Phosphorylation</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=3" title="Edit section: Phosphorylation">edit</a><span class="mw-editsection-bracket">]</span></span></h4>
<p>One such modification is <a href="/wiki/Phosphorylation" title="Phosphorylation">phosphorylation</a>, which happens to many <a href="/wiki/Enzymes" title="Enzymes" class="mw-redirect">enzymes</a> and structural proteins in the process of <a href="/wiki/Cell_signaling" title="Cell signaling">cell signaling</a>. The addition of a phosphate to particular amino acids—most commonly <a href="/wiki/Serine" title="Serine">serine</a> and <a href="/wiki/Threonine" title="Threonine">threonine</a><sup id="cite_ref-9" class="reference"><a href="#cite_note-9"><span>[</span>9<span>]</span></a></sup> mediated by serine/threonine <a href="/wiki/Kinase" title="Kinase">kinases</a>, or more rarely <a href="/wiki/Tyrosine" title="Tyrosine">tyrosine</a> mediated by tyrosine <a href="/wiki/Kinases" title="Kinases" class="mw-redirect">kinases</a>—causes a protein to become a target for binding or interacting with a distinct set of other proteins that recognize the phosphorylated domain.</p>
<p>Because protein phosphorylation is one of the most-studied protein modifications, many "proteomic" efforts are geared to determining the set of phosphorylated proteins in a particular cell or tissue-type under particular circumstances. This alerts the scientist to the signaling pathways that may be active in that instance.</p>
<h4><span class="mw-headline" id="Ubiquitination">Ubiquitination</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=4" title="Edit section: Ubiquitination">edit</a><span class="mw-editsection-bracket">]</span></span></h4>
<p><a href="/wiki/Ubiquitin" title="Ubiquitin">Ubiquitin</a> is a small protein that can be affixed to certain protein substrates by enzymes called <a href="/wiki/E3_ubiquitin_ligase" title="E3 ubiquitin ligase" class="mw-redirect">E3 ubiquitin ligases</a>. Determining which proteins are poly-ubiquitinated helps understand how protein pathways are regulated. This is, therefore, an additional legitimate "proteomic" study. Similarly, once a researcher determines which substrates are ubiquitinated by each ligase, determining the set of ligases expressed in a particular cell type is helpful.</p>
<h4><span class="mw-headline" id="Additional_modifications">Additional modifications</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=5" title="Edit section: Additional modifications">edit</a><span class="mw-editsection-bracket">]</span></span></h4>
<p>In addition to <a href="/wiki/Phosphorylation" title="Phosphorylation">phosphorylation</a> and <a href="/wiki/Ubiquitination" title="Ubiquitination" class="mw-redirect">ubiquitination</a>, proteins can be subjected to (among others) <a href="/wiki/Methylation" title="Methylation">methylation</a>, <a href="/wiki/Acetylation" title="Acetylation">acetylation</a>, <a href="/wiki/Glycosylation" title="Glycosylation">glycosylation</a>, <a href="/wiki/Oxidation" title="Oxidation" class="mw-redirect">oxidation</a> and <a href="/wiki/Nitrosylation" title="Nitrosylation">nitrosylation</a>. Some proteins undergo all these modifications, often in time-dependent combinations. This illustrates the potential complexity of studying protein structure and function.</p>
<h3><span class="mw-headline" id="Distinct_proteins_are_made_under_distinct_settings">Distinct proteins are made under distinct settings</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=6" title="Edit section: Distinct proteins are made under distinct settings">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>Even studying a particular cell type, that cell may make different sets of proteins at different times, or under different conditions. Furthermore, as mentioned, any one protein can undergo a wide range of post-translational modifications.</p>
<p>Therefore, a "proteomics" study can become complex, even if the topic of the study is restricted. In more ambitious settings, such as when a biomarker for a tumor is sought&#160;– when the proteomics scientist is obliged to study blood serum samples from multiple cancer patients.<sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">[<i><a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed"><span title="This claim needs references to reliable sources. (October 2014)">citation needed</span></a></i>]</sup></p>
<h2><span class="mw-headline" id="Limitations_of_genomics_and_proteomics_studies">Limitations of genomics and proteomics studies</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=7" title="Edit section: Limitations of genomics and proteomics studies">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>Proteomics gives a different level of understanding than genomics for many reasons:</p>
<ul>
<li>the level of transcription of a gene gives only a rough estimate of its <i>level of translation</i> into a protein.<sup id="cite_ref-Gygi1999_10-0" class="reference"><a href="#cite_note-Gygi1999-10"><span>[</span>10<span>]</span></a></sup> An <a href="/wiki/MRNA" title="MRNA" class="mw-redirect">mRNA</a> produced in abundance may be degraded rapidly or translated inefficiently, resulting in a small amount of protein.</li>
<li>as mentioned above many proteins experience <i><a href="/wiki/Post-translational_modification" title="Post-translational modification">post-translational modifications</a></i> that profoundly affect their activities; for example some proteins are not active until they become phosphorylated. Methods such as <a href="/wiki/Phosphoproteomics" title="Phosphoproteomics">phosphoproteomics</a> and <a href="/wiki/Glycoproteomics" title="Glycoproteomics">glycoproteomics</a> are used to study post-translational modifications.</li>
<li>many transcripts give rise to more than one protein, through <a href="/wiki/Alternative_splicing" title="Alternative splicing">alternative splicing</a> or alternative post-translational modifications.</li>
<li>many proteins form complexes with other proteins or RNA molecules, and only function in the presence of these other molecules.</li>
<li>protein degradation rate plays an important role in protein content.<sup id="cite_ref-11" class="reference"><a href="#cite_note-11"><span>[</span>11<span>]</span></a></sup></li>
</ul>
<p><i>Reproducibility</i>. Proteomics experiments conducted in one laboratory are not easily reproduced in another. For instance, Peng et al.<sup id="cite_ref-Peng2003_12-0" class="reference"><a href="#cite_note-Peng2003-12"><span>[</span>12<span>]</span></a></sup> have identified 1504 yeast proteins in a proteomics experiment of which only 858 were found in a similar previous study.<sup id="cite_ref-Washburn2001_13-0" class="reference"><a href="#cite_note-Washburn2001-13"><span>[</span>13<span>]</span></a></sup> Further, the previous study identified 607 proteins that were not found by Peng et al. This translates to a reproducibility of 57% (Peng vs. Washburn) to 59% (Washburn vs. Peng).</p>
<h2><span class="mw-headline" id="Methods_of_studying_proteins">Methods of studying proteins</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=8" title="Edit section: Methods of studying proteins">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>In proteomics, there are multiple methods to study proteins. Generally, proteins can either be detected using <a href="/wiki/Antibody" title="Antibody">antibodies (immunoassays)</a> or using <a href="/wiki/Mass_spectrometry" title="Mass spectrometry">mass spectrometry</a>. If a complex biological sample is analyzed, then biochemical separation has to be used before the detection step as there are too many analytes in the sample to perform accurate detection and quantification.</p>
<h3><span class="mw-headline" id="Protein_detection_with_antibodies_.28immunoassays.29">Protein detection with antibodies (immunoassays)</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=9" title="Edit section: Protein detection with antibodies (immunoassays)">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p><a href="/wiki/Antibodies" title="Antibodies" class="mw-redirect">Antibodies</a> to particular proteins or to their modified forms have been used in <a href="/wiki/Biochemistry" title="Biochemistry">biochemistry</a> and <a href="/wiki/Cell_biology" title="Cell biology">cell biology</a> studies. These are among the most common tools used by molecular biologists today. There are several specific techniques and protocols that use antibodies for protein detection. The <a href="/wiki/Enzyme-linked_immunosorbent_assay" title="Enzyme-linked immunosorbent assay" class="mw-redirect">enzyme-linked immunosorbent assay</a> (ELISA) has been used for decades to detect and quantitatively measure proteins in samples. The <a href="/wiki/Western_blot" title="Western blot">Western blot</a> can be used for detection and quantification of individual proteins, where in an initial step a complex protein mixture is separated using <a href="/wiki/SDS-PAGE" title="SDS-PAGE" class="mw-redirect">SDS-PAGE</a> and then the protein of interested identified using an antibody.</p>
<p>Modified proteins can be studied by developing an <a href="/wiki/Antibody" title="Antibody">antibody</a> specific to that modification. For example, there are antibodies that only recognize certain proteins when they are tyrosine-<a href="/wiki/Phosphorylated" title="Phosphorylated" class="mw-redirect">phosphorylated</a>, known as phospho-specific antibodies. Also, there are antibodies specific to other modifications. These can be used to determine the set of proteins that have undergone the modification of interest.</p>
<h3><span class="mw-headline" id="Antibody-free_protein_detection">Antibody-free protein detection</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=10" title="Edit section: Antibody-free protein detection">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>While protein detection with antibodies are still very common in molecular biology, also other methods have been developed that do not rely on an antibody. These methods offer various advantages, for instance they are often able to determine the sequence of a protein or peptide, they may have higher throughput than antibody-based and they sometimes can identify and quantify proteins for which no antibody exists.</p>
<h4><span class="mw-headline" id="Detection_methods">Detection methods</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=11" title="Edit section: Detection methods">edit</a><span class="mw-editsection-bracket">]</span></span></h4>
<p>One of the earliest method for protein analysis has been <a href="/wiki/Edman_degradation" title="Edman degradation">Edman degradation</a> (introduced in 1967) where a single <a href="/wiki/Peptide" title="Peptide">peptide</a> is subjected to multiple steps of chemical degradation to resolve its sequence. These methods have mostly been supplanted by technologies that offer higher throughput.</p>
<p>More recent methods use <a href="/wiki/Mass_spectrometry" title="Mass spectrometry">mass spectrometry</a>-based techniques, a development that was made possible by the discovery of "soft ionization" methods such as <a href="/wiki/Matrix-assisted_laser_desorption/ionization" title="Matrix-assisted laser desorption/ionization">matrix-assisted laser desorption/ionization (MALDI)</a> and <a href="/wiki/Electrospray_ionization" title="Electrospray ionization">electrospray ionization (ESI)</a> developed in the 1980s. These methods gave rise to the <a href="/wiki/Top-down_proteomics" title="Top-down proteomics">top-down</a> and the <a href="/wiki/Bottom-up_proteomics" title="Bottom-up proteomics">bottom-up</a> proteomics workflows where often additional separation is performed before analysis (see below).</p>
<h4><span class="mw-headline" id="Separation_methods">Separation methods</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=12" title="Edit section: Separation methods">edit</a><span class="mw-editsection-bracket">]</span></span></h4>
<p>For the analysis of complex biological samples, a reduction of sample complexity is required. This can be performed off-line by <a href="/wiki/SDS-PAGE" title="SDS-PAGE" class="mw-redirect">one-dimensional</a> or <a href="/wiki/Two-dimensional_gel_electrophoresis" title="Two-dimensional gel electrophoresis">two dimensional</a> separation. More recently, on-line methods have been developed where individual peptides (in bottom-up proteomics approaches) are separated using <a href="/wiki/Reversed-phase_chromatography" title="Reversed-phase chromatography">Reversed-phase chromatography</a> and then directly ionized using <a href="/wiki/Electrospray_ionization" title="Electrospray ionization">ESI</a>; the direct coupling of separation and analysis explains the term "on-line" analysis.</p>
<h3><span class="mw-headline" id="Hybrid_technologies">Hybrid technologies</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=13" title="Edit section: Hybrid technologies">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>There are several hybrid technologies that use antibody-based purification of individual analytes and then perform mass spectrometric analysis for identification and quantification. Examples of these methods are the <a href="/wiki/Mass_spectrometric_immunoassay" title="Mass spectrometric immunoassay">MSIA (mass spectrometric immunoassay)</a> developed by Randall Nelson in 1995<sup id="cite_ref-NelsonKrone1995_14-0" class="reference"><a href="#cite_note-NelsonKrone1995-14"><span>[</span>14<span>]</span></a></sup> and the SISCAPA (Stable Isotope Standard Capture with Anti-Peptide Antibodies) method, introduced by Leigh Anderson in 2004.<sup id="cite_ref-15" class="reference"><a href="#cite_note-15"><span>[</span>15<span>]</span></a></sup></p>
<h2><span class="mw-headline" id="Establishing_protein.E2.80.93protein_interactions">Establishing protein–protein interactions</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=14" title="Edit section: Establishing protein–protein interactions">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>Most proteins function in collaboration with other proteins, and one goal of proteomics is to identify which proteins interact. This is especially useful in determining potential partners in <a href="/wiki/Cell_signaling" title="Cell signaling">cell signaling</a> cascades.</p>
<p>Several methods are available to probe protein–protein interactions. The traditional method is yeast <a href="/wiki/Two-hybrid_screening" title="Two-hybrid screening">two-hybrid analysis</a>. New methods include <a href="/wiki/Surface_plasmon_resonance" title="Surface plasmon resonance">surface plasmon resonance</a> (SPR),<sup id="cite_ref-16" class="reference"><a href="#cite_note-16"><span>[</span>16<span>]</span></a></sup><sup id="cite_ref-17" class="reference"><a href="#cite_note-17"><span>[</span>17<span>]</span></a></sup> <a href="/wiki/Protein_microarray" title="Protein microarray">protein microarrays</a>, <a href="/wiki/Immunoaffinity_chromatography" title="Immunoaffinity chromatography" class="mw-redirect">immunoaffinity chromatography</a> followed by <a href="/wiki/Mass_spectrometry" title="Mass spectrometry">mass spectrometry</a>, <a href="/wiki/Dual_polarisation_interferometry" title="Dual polarisation interferometry" class="mw-redirect">dual polarisation interferometry</a>, <a href="/wiki/Microscale_Thermophoresis" title="Microscale Thermophoresis" class="mw-redirect">Microscale Thermophoresis</a> and experimental methods such as <a href="/wiki/Phage_display" title="Phage display">phage display</a> and computational methods.</p>
<h2><span class="mw-headline" id="Practical_applications_of_proteomics">Practical applications of proteomics</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=15" title="Edit section: Practical applications of proteomics">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>One major development to come from the study of human genes and proteins has been the identification of potential new drugs for the treatment of disease. This relies on genome and proteome information to identify proteins associated with a disease, which computer software can then use as targets for new drugs. For example, if a certain protein is implicated in a disease, its 3D structure provides the information to design drugs to interfere with the action of the protein. A molecule that fits the active site of an enzyme, but cannot be released by the enzyme, inactivates the enzyme. This is the basis of new drug-discovery tools, which aim to find new drugs to inactivate proteins involved in disease. As genetic differences among individuals are found, researchers expect to use these techniques to develop personalized drugs that are more effective for the individual.<sup id="cite_ref-Vaidyanathan12_18-0" class="reference"><a href="#cite_note-Vaidyanathan12-18"><span>[</span>18<span>]</span></a></sup></p>
<p>Proteomics is also used to reveal complex plant-insect interactions that help identify candidate genes involved in the defensive response of plants to herbivory.<sup id="cite_ref-19" class="reference"><a href="#cite_note-19"><span>[</span>19<span>]</span></a></sup><sup id="cite_ref-20" class="reference"><a href="#cite_note-20"><span>[</span>20<span>]</span></a></sup><sup id="cite_ref-21" class="reference"><a href="#cite_note-21"><span>[</span>21<span>]</span></a></sup></p>
<h3><span class="mw-headline" id="Biomarkers">Biomarkers</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=16" title="Edit section: Biomarkers">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<div role="note" class="hatnote relarticle mainarticle">Main article: <a href="/wiki/Biomarker" title="Biomarker">Biomarker</a></div>
<p>The <a href="/wiki/National_Institutes_of_Health" title="National Institutes of Health">National Institutes of Health</a> has defined a biomarker as “a characteristic that is objectively measured and evaluated as an indicator of normal biological processes, pathogenic processes, or pharmacologic responses to a therapeutic intervention.”<sup id="cite_ref-22" class="reference"><a href="#cite_note-22"><span>[</span>22<span>]</span></a></sup><sup id="cite_ref-23" class="reference"><a href="#cite_note-23"><span>[</span>23<span>]</span></a></sup></p>
<p>Understanding the proteome, the structure and function of each protein and the complexities of protein–protein interactions is critical for developing the most effective diagnostic techniques and disease treatments in the future. For example, proteomics is highly useful in identification of candidate biomarkers (proteins in body fluids that are of value for diagnosis), identification of the bacterial antigens that are targeted by the immune response, and identification of possible immunohistochemistry markers of infectious or neoplastic diseases.<sup id="cite_ref-Ceciliani2014_24-0" class="reference"><a href="#cite_note-Ceciliani2014-24"><span>[</span>24<span>]</span></a></sup></p>
<p>An interesting use of proteomics is using specific protein biomarkers to diagnose disease. A number of techniques allow to test for proteins produced during a particular disease, which helps to diagnose the disease quickly. Techniques include <a href="/wiki/Western_blot" title="Western blot">western blot</a>, <a href="/wiki/Immunohistochemical_staining" title="Immunohistochemical staining" class="mw-redirect">immunohistochemical staining</a>, <a href="/wiki/Enzyme_linked_immunosorbent_assay" title="Enzyme linked immunosorbent assay" class="mw-redirect">enzyme linked immunosorbent assay</a> (ELISA) or <a href="/wiki/Mass_spectrometry" title="Mass spectrometry">mass spectrometry</a>.<sup id="cite_ref-Klopfleisch1_25-0" class="reference"><a href="#cite_note-Klopfleisch1-25"><span>[</span>25<span>]</span></a></sup><sup id="cite_ref-26" class="reference"><a href="#cite_note-26"><span>[</span>26<span>]</span></a></sup> <a href="/wiki/Secretomics" title="Secretomics">Secretomics</a>, a subfield of proteomics that studies <a href="/wiki/Secretory_protein" title="Secretory protein">secreted proteins</a> and secretion pathways using proteomic approaches, has recently emerged as an important tool for the discovery of biomarkers of disease.<sup id="cite_ref-pmid17425459_27-0" class="reference"><a href="#cite_note-pmid17425459-27"><span>[</span>27<span>]</span></a></sup></p>
<h3><span class="mw-headline" id="Proteogenomics">Proteogenomics</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=17" title="Edit section: Proteogenomics">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>In what is now commonly referred to as <a href="/wiki/Proteogenomics" title="Proteogenomics">proteogenomics</a>, proteomic technologies such as <a href="/wiki/Mass_spectrometry" title="Mass spectrometry">mass spectrometry</a> are used for improving gene annotations. Parallel analysis of the genome and the proteome facilitates discovery of post-translational modifications and proteolytic events,<sup id="cite_ref-Gupta07_28-0" class="reference"><a href="#cite_note-Gupta07-28"><span>[</span>28<span>]</span></a></sup> especially when comparing multiple species (comparative proteogenomics).<sup id="cite_ref-Gupta08_29-0" class="reference"><a href="#cite_note-Gupta08-29"><span>[</span>29<span>]</span></a></sup></p>
<h3><span class="mw-headline" id="Current_research_methodologies">Current research methodologies</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=18" title="Edit section: Current research methodologies">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>Fluorescence two-dimensional differential gel electrophoresis (2-D DIGE)<sup id="cite_ref-Tonge01_30-0" class="reference"><a href="#cite_note-Tonge01-30"><span>[</span>30<span>]</span></a></sup> can be used to quantify variation in the 2-D DIGE process and establish statistically valid thresholds for assigning quantitative changes between samples.<sup id="cite_ref-Tonge01_30-1" class="reference"><a href="#cite_note-Tonge01-30"><span>[</span>30<span>]</span></a></sup></p>
<p>Comparative proteomic analysis can reveal the role of proteins in complex biological systems, including reproduction. For example, treatment with the insecticide triazophos causes an increase in the content of brown planthopper (<i>Nilaparvata lugens</i> (Stål)) male accessory gland proteins (Acps) that can be transferred to females via mating, causing an increase in fecundity (i.e. birth rate) of females.<sup id="cite_ref-31" class="reference"><a href="#cite_note-31"><span>[</span>31<span>]</span></a></sup> To identify changes in the types of accessory gland proteins (Acps) and reproductive proteins that mated female planthoppers received from male planthoppers, researchers conducted a comparative proteomic analysis of mated <i>N. lugens</i> females.<sup id="cite_ref-Ge_et_al_2011_32-0" class="reference"><a href="#cite_note-Ge_et_al_2011-32"><span>[</span>32<span>]</span></a></sup> The results indicated that these proteins participate in the reproductive process of <i>N. lugens</i> adult females and males.<sup id="cite_ref-Ge_et_al_2011_32-1" class="reference"><a href="#cite_note-Ge_et_al_2011-32"><span>[</span>32<span>]</span></a></sup></p>
<p>Proteome analysis of <i>Arabidopsis peroxisomes</i><sup id="cite_ref-Reumann11_33-0" class="reference"><a href="#cite_note-Reumann11-33"><span>[</span>33<span>]</span></a></sup> has been established as the major unbiased approach for identifying new peroxisomal proteins on a large scale.<sup id="cite_ref-Reumann11_33-1" class="reference"><a href="#cite_note-Reumann11-33"><span>[</span>33<span>]</span></a></sup></p>
<p>There are many approaches to characterizing the human proteome, which is estimated to contain between 20,000 and 25,000 non-redundant proteins. The number of unique protein species will likely increase by between 50,000 and 500,000 due to RNA splicing and proteolysis events, and when post-translational modification are also considered, the total number of unique human proteins is estimated to range in the low millions.<sup id="cite_ref-34" class="reference"><a href="#cite_note-34"><span>[</span>34<span>]</span></a></sup><sup id="cite_ref-35" class="reference"><a href="#cite_note-35"><span>[</span>35<span>]</span></a></sup></p>
<p>In addition, the first promising attempts to decipher the proteome of animal tumors have recently been reported.<sup id="cite_ref-Klopfleisch1_25-1" class="reference"><a href="#cite_note-Klopfleisch1-25"><span>[</span>25<span>]</span></a></sup> This method used as a functional method in <i><a href="/wiki/Macrobrachium_rosenbergii" title="Macrobrachium rosenbergii">Macrobrachium rosenbergii</a></i> protein profiling.<sup id="cite_ref-FOOTNOTEAlinejad2015_36-0" class="reference"><a href="#cite_note-FOOTNOTEAlinejad2015-36"><span>[</span>36<span>]</span></a></sup></p>
<h2><span class="mw-headline" id="Structural_proteomics">Structural proteomics</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=19" title="Edit section: Structural proteomics">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>Structural proteomics includes the analysis of protein structures at large-scale. It compares protein structures and helps identify functions of newly discovered genes. The structural analysis also helps to understand that where drugs bind to proteins and also show where proteins interact with each other. This understanding is achieved using different technologies such as X-ray crystallography and NMR spectroscopy.<sup id="cite_ref-What_is_Proteomics.3F_37-0" class="reference"><a href="#cite_note-What_is_Proteomics.3F-37"><span>[</span>37<span>]</span></a></sup></p>
<h2><span class="mw-headline" id="Expression_proteomics">Expression proteomics</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=20" title="Edit section: Expression proteomics">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>Expression proteomics includes the analysis of <a href="/wiki/Protein_expression_(biotechnology)" title="Protein expression (biotechnology)" class="mw-redirect">protein expression</a> at larger scale. It helps identify main proteins in a particular sample, and those proteins differentially expressed in related samples—such as diseased vs. healthy tissue. If a protein is found only in a diseased sample then it can be a useful drug target or diagnostic marker. Proteins with same or similar expression profiles may also be functionally related. There are technologies such as 2D-PAGE and mass spectrometry that are used in expression proteomics.<sup id="cite_ref-What_is_Proteomics.3F_37-1" class="reference"><a href="#cite_note-What_is_Proteomics.3F-37"><span>[</span>37<span>]</span></a></sup></p>
<h2><span class="mw-headline" id="Interaction_proteomics">Interaction proteomics</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=21" title="Edit section: Interaction proteomics">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>Interaction proteomics is the analysis of protein interactions at larger scale. The characterization of protein-protein interactions are useful to determine the protein functions and it also explains the way proteins assemble in bigger complexes. Technologies such as affinity purification, mass spectrometry, and the yeast two-hybrid system are particularly useful in interaction proteomics.<sup id="cite_ref-What_is_Proteomics.3F_37-2" class="reference"><a href="#cite_note-What_is_Proteomics.3F-37"><span>[</span>37<span>]</span></a></sup></p>
<p>Proteome analysis techniques are not simple and straightforward as those used in transcriptomics. The benefit of proteomics, however, is that it deals with the real functional molecules of the cells. It is known that strong gene expression results in an abundant mRNA but it does not necessarily mean that the corresponding protein is also abundant. In proteomics things are not so simple as one gene does not always produce the same protein. The genes usually consist of a series of sub structures, which are called exons. These sub structures can be joined in a variety of ways, which helps to give momentum to a whole series of very similar but different proteins. Further increasing complications, once proteins are made, they are ornamented with different other chemicals. These chemicals can be phosphate, sugars or fats. The effect of the decorations is severe on the function of protein; for example phosphate normally behaves as an on-off switch and sugars usually tell the proteins where to go and attach in the cell. Therefore, it was comparatively very simple and easy to sequence the human genome as there are only 46 molecules and they are made up of 4 building blocks or letters (A, C, G, T) whereas proteins have 20 building blocks, each of which can be customized or ornamented after the protein is built. Hence, proteomics have to deal with ca. 30,000 genes that can be arranged to give some 800,000 proteins that can be modified and decorated with over 300 different chemicals. Additionally, proteomics also describe the nature of proteins, where they are being produced in a particular cell type and at a specific time, the way they are modified in the cell, the location where they are modified and also what they are in contact with. Finally, the most difficult thing is to determine the function of the protein.<sup id="cite_ref-What_is_Proteomics.3F_37-3" class="reference"><a href="#cite_note-What_is_Proteomics.3F-37"><span>[</span>37<span>]</span></a></sup></p>
<h2><span class="mw-headline" id="Current_proteomic_technologies">Current proteomic technologies</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=22" title="Edit section: Current proteomic technologies">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>Proteomics has steadily gained momentum over the past decade with the evolution of several approaches. Few of these are new and others build on traditional methods. Mass spectrometry-based methods and micro arrays are the most common technologies for large-scale study of proteins.</p>
<h3><span class="mw-headline" id="Mass_spectrometry_and_protein_profiling">Mass spectrometry and protein profiling</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=23" title="Edit section: Mass spectrometry and protein profiling">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<div role="note" class="hatnote relarticle mainarticle">Main article: <a href="/wiki/Mass_spectrometry" title="Mass spectrometry">Mass spectrometry</a></div>
<p>There are two mass spectrometry-based methods currently used for protein profiling. The more established and widespread method uses high resolution, two-dimensional electrophoresis to separate proteins from different samples in parallel, followed by selection and staining of differentially expressed proteins to be identified by mass spectrometry. Despite the advances in 2DE and its maturity, it has its limits as well. The central concern is the inability to resolve all the proteins within a sample, given their dramatic range in expression level and differing properties.<sup id="cite_ref-Weston_.26_Hood_2004_38-0" class="reference"><a href="#cite_note-Weston_.26_Hood_2004-38"><span>[</span>38<span>]</span></a></sup></p>
<p>The second quantitative approach uses stable isotope tags to differentially label proteins from two different complex mixtures. Here, the proteins within a complex mixture are labeled first isotopically, and then digested to yield labeled peptides. The labeled mixtures are then combined, the peptides separated by multidimensional liquid chromatography and analyzed by tandem mass spectrometry. Isotope coded affinity tag (ICAT) reagents are the widely used isotope tags. In this method, the cysteine residues of proteins get covalently attached to the ICAT reagent, thereby reducing the complexity of the mixtures omitting the non-cysteine residues.</p>
<p>Quantitative proteomics using stable isotopic tagging is an increasingly useful tool in modern development. Firstly, chemical reactions have been used to introduce tags into specific sites or proteins for the purpose of probing specific protein functionalities. The isolation of phosphorylated peptides has been achieved using isotopic labeling and selective chemistries to capture the fraction of protein among the complex mixture. Secondly, the ICAT technology was used to differentiate between partially purified or purified macromolecular complexes such as large RNA polymerase II pre-initiation complex and the proteins complexed with yeast transcription factor. Thirdly, ICAT labeling was recently combined with chromatin isolation to identify and quantify chromatin-associated proteins. Finally ICAT reagents are useful for proteomic profiling of cellular organelles and specific cellular fractions.<sup id="cite_ref-Weston_.26_Hood_2004_38-1" class="reference"><a href="#cite_note-Weston_.26_Hood_2004-38"><span>[</span>38<span>]</span></a></sup></p>
<p>Another quantitative approach is the Accurate Mass and Time (AMT) tag approach developed by <a href="/wiki/Richard_D._Smith" title="Richard D. Smith">Richard D. Smith</a> and coworkers at <a href="/wiki/Pacific_Northwest_National_Laboratory" title="Pacific Northwest National Laboratory">Pacific Northwest National Laboratory</a>. In this approach, increased throughput and sensitivity is achieved by avoiding the needed for tandem mass spectrometry, and making use of precisely determined separation time information and highly accurate mass determinations for peptide and protein identifications.</p>
<h3><span class="mw-headline" id="Protein_chips">Protein chips</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=24" title="Edit section: Protein chips">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>Balancing the use of mass spectrometers in proteomics and in medicine is the use of protein micro arrays. The aim behind protein micro arrays is to print thousands of protein detecting features for the interrogation of biological samples. Antibody arrays are an example in which a host of different antibodies are arrayed to detect their respective antigens from a sample of human blood. Another approach is the arraying of multiple protein types for the study of properties like protein-DNA, protein-protein and protein-ligand interactions. Ideally, the functional proteomic arrays would contain the entire complement of the proteins of a given organism. The first version of such arrays consisted of 5000 purified proteins from yeast deposited onto glass microscopic slides. Despite the success of first chip, it was a greater challenge for protein arrays to be implemented. Proteins are inherently much more difficult to work with than DNA. They have a broad dynamic range, are less stable than DNA and their structure is difficult to preserve on glass slides, though they are essential for most assays. The global ICAT technology has striking advantages over protein chip technologies.<sup id="cite_ref-Weston_.26_Hood_2004_38-2" class="reference"><a href="#cite_note-Weston_.26_Hood_2004-38"><span>[</span>38<span>]</span></a></sup></p>
<h3><span class="mw-headline" id="Reverse-phased_protein_microarrays">Reverse-phased protein microarrays</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=25" title="Edit section: Reverse-phased protein microarrays">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>This is a promising and newer microarray application for the diagnosis, study and treatment of complex diseases such as cancer. The technology merges laser capture microdissection (LCM) with micro array technology, to produce reverse phase protein microarrays. In this type of microarrays, the whole collection of protein themselves are immobilized with the intent of capturing various stages of disease within an individual patient. When used with LCM, reverse phase arrays can monitor the fluctuating state of proteome among different cell population within a small area of human tissue. This is useful for profiling the status of cellular signaling molecules, among a cross section of tissue that includes both normal and cancerous cells. This approach is useful in monitoring the status of key factors in normal prostate epithelium and invasive prostate cancer tissues. LCM then dissects these tissue and protein lysates were arrayed onto nitrocellulose slides, which were probed with specific antibodies. This method can track all kinds of molecular events and can compare diseased and healthy tissues within the same patient enabling the development of treatment strategies and diagnosis. The ability to acquire proteomics snapshots of neighboring cell populations, using reverse phase microarrays in conjunction with LCM has a number of applications beyond the study of tumors. The approach can provide insights into normal physiology and pathology of all the tissues and is invaluable for characterizing developmental processes and anomalies.<sup id="cite_ref-Weston_.26_Hood_2004_38-3" class="reference"><a href="#cite_note-Weston_.26_Hood_2004-38"><span>[</span>38<span>]</span></a></sup></p>
<h2><span class="mw-headline" id="Bioinformatics_for_proteomics_.28proteome_informatics.29">Bioinformatics for proteomics (proteome informatics)</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=26" title="Edit section: Bioinformatics for proteomics (proteome informatics)">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>There is a large amount of proteomics data being collected with the help of high throughput technologies such as mass spectrometry and microarray. It would often take weeks or months to analyze the data and perform comparisons by hand. For this reason, biologists and chemists are collaborating with computer scientists and mathematicians to create programs and pipeline to computationally analyze the protein data. Using <a href="/wiki/Bioinformatics" title="Bioinformatics">bioinformatics</a> techniques, researchers are capable of faster analysis and data storage. A good place to find lists of current programs and databases is on the <a href="/wiki/ExPASy" title="ExPASy">ExPASy</a> bioinformatics resource portal &lt;<a rel="nofollow" class="external free" href="http://www.expasy.org/proteomics">http://www.expasy.org/proteomics</a>&gt;. The applications of bioinformatics-based proteomics includes medicine, disease diagnosis, biomarker identification, and many more.</p>
<h3><span class="mw-headline" id="Protein_identification">Protein identification</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=27" title="Edit section: Protein identification">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>Mass spectrometry and microarray produce peptide fragmentation information but do not give identification of specific proteins present in the original sample. Due to the lack of specific protein identification, past researchers were forced to decipher the peptide fragments themselves. However, there are currently programs available for protein identification. These programs take the peptide sequences output from mass spectrometry and microarray and return information about matching or similar proteins. This is done through algorithms implemented by the program which perform alignments with proteins from known databases such as UniProt <sup id="cite_ref-39" class="reference"><a href="#cite_note-39"><span>[</span>39<span>]</span></a></sup> and PROSITE <sup id="cite_ref-40" class="reference"><a href="#cite_note-40"><span>[</span>40<span>]</span></a></sup> to predict what proteins are in the sample with a degree of certainty.</p>
<h3><span class="mw-headline" id="Protein_structure">Protein structure</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=28" title="Edit section: Protein structure">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<div role="note" class="hatnote relarticle mainarticle">Main article: <a href="/wiki/Protein_structure_prediction" title="Protein structure prediction">Protein structure prediction</a></div>
<p>The <a href="/wiki/Biomolecular_structure" title="Biomolecular structure">biomolecular structure</a> forms the 3D configuration of the protein. Understanding the protein's structure aids in identification of the protein's interactions and function. It used to be that the 3D structure of proteins could only be determined using <a href="/wiki/X-ray_crystallography" title="X-ray crystallography">X-ray crystallography</a> and <a href="/wiki/NMR_spectroscopy" title="NMR spectroscopy" class="mw-redirect">NMR spectroscopy</a>. Now, through bioinformatics, there are computer programs that can predict and model the structure of proteins. These programs use the chemical properties of amino acids and structural properties of known proteins to predict the 3D model of sample proteins. This also allows scientists to take a look at protein interactions on a larger scale. In addition, biomedical engineers are developing methods to factor in the flexibility of protein structures to make comparisons and predictions.<sup id="cite_ref-41" class="reference"><a href="#cite_note-41"><span>[</span>41<span>]</span></a></sup></p>
<h3><span class="mw-headline" id="Post-translational_modifications_2">Post-translational modifications</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=29" title="Edit section: Post-translational modifications">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>Unfortunately, most programs available for protein analysis are not written for proteins that have undergone <a href="/wiki/Post-translational_modifications" title="Post-translational modifications" class="mw-redirect">post-translational modifications</a>.<sup id="cite_ref-42" class="reference"><a href="#cite_note-42"><span>[</span>42<span>]</span></a></sup> Some programs will accept post-translational modifications to aid in protein identification but then ignore the modification during further protein analysis. It is important to account for these modifications since they can affect the protein's structure. In turn, computational analysis of post-translational modifications has gained the attention of the scientific community. The current post-translational modification programs are only predictive.<sup id="cite_ref-43" class="reference"><a href="#cite_note-43"><span>[</span>43<span>]</span></a></sup> Chemists, biologists and computer scientists are working together to create and introduce new pipelines that allow for analysis of post-translational modifications that have been experimentally identified for their effect on the protein's structure and function.</p>
<h3><span class="mw-headline" id="Computational_methods_in_studying_protein_biomarkers">Computational methods in studying protein biomarkers</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=30" title="Edit section: Computational methods in studying protein biomarkers">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>One example of the use of bioinformatics and the use of computational methods is the study of protein biomarkers. Computational predictive models<sup id="cite_ref-44" class="reference"><a href="#cite_note-44"><span>[</span>44<span>]</span></a></sup> have shown that extensive and diverse feto-maternal protein trafficking occurs during pregnancy and can be readily detected non-invasively in maternal whole blood. This computational approach circumvented a major limitation, the abundance of maternal proteins interfering with the detection of <a href="/wiki/Fetal_protein" title="Fetal protein">fetal proteins</a>, to fetal proteomic analysis of maternal blood. Computational models can use fetal gene transcripts previously identified in maternal <a href="/wiki/Whole_blood" title="Whole blood">whole blood</a> to create a comprehensive proteomic network of the term <a href="/wiki/Neonate" title="Neonate" class="mw-redirect">neonate</a>. Such work shows that the fetal proteins detected in pregnant woman’s blood originate from a diverse group of tissues and organs from the developing fetus. The proteomic networks contain many <a href="/wiki/Biomarker_(medicine)" title="Biomarker (medicine)">biomarkers</a> that are proxies for development and illustrate the potential clinical application of this technology as a way to monitor normal and abnormal fetal development.</p>
<p>An information theoretic framework has also been introduced for <a href="/wiki/Biomarker_(medicine)" title="Biomarker (medicine)">biomarker</a> discovery, integrating biofluid and tissue information.<sup id="cite_ref-45" class="reference"><a href="#cite_note-45"><span>[</span>45<span>]</span></a></sup> This new approach takes advantage of functional synergy between certain biofluids and tissues with the potential for clinically significant findings not possible if tissues and biofluids were considered individually. By conceptualizing tissue-biofluid as information channels, significant biofluid proxies can be identified and then used for guided development of clinical diagnostics. Candidate biomarkers are then predicted based on information transfer criteria across the tissue-biofluid channels. Significant biofluid-tissue relationships can be used to prioritize clinical validation of biomarkers.<sup class="noprint Inline-Template Template-Fact" style="white-space:nowrap;">[<i><a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed"><span title="This claim needs references to reliable sources. (January 2012)">citation needed</span></a></i>]</sup></p>
<h2><span class="mw-headline" id="Emerging_trends_in_proteomics">Emerging trends in proteomics</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=31" title="Edit section: Emerging trends in proteomics">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>A number of emerging concepts have the potential to improve current features of proteomics. Obtaining absolute quantification of proteins and monitoring post-translational modifications are the two tasks that impact the understanding of protein function in healthy and diseased cells. Advances in quantitative proteomics would clearly enable more in-depth analysis of cellular systems. For many cellular events, the protein concentrations do not change; rather, their function is modulated by post-transitional modifications (PTM). Methods of monitoring PTM are an underdeveloped area in proteomics. Selecting a particular subset of protein for analysis substantially reduces protein complexity, making it advantageous for diagnostic purposes where blood is the starting material. Another important aspect of proteomics, yet not addressed, is that proteomics methods should focus on studying proteins in the context of the environment. The increasing use of chemical cross linkers, introduced into living cells to fix protein-protein, protein-DNA and other interactions, may ameliorate this problem partially. The challenge is to identify suitable methods of preserving relevant interactions. Another goal for studying protein is to develop more sophisticated methods to image proteins and other molecules in living cells and real time.<sup id="cite_ref-Weston_.26_Hood_2004_38-4" class="reference"><a href="#cite_note-Weston_.26_Hood_2004-38"><span>[</span>38<span>]</span></a></sup></p>
<h3><span class="mw-headline" id="Human_plasma_proteome">Human plasma proteome</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=32" title="Edit section: Human plasma proteome">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<p>Characterizing the human plasma proteome has become a major goal in the proteomics arena. The plasma proteome is without doubt the most complex proteome in the human body. It contains immunoglobulin, cytokines, protein hormones, and secreted proteins indicative of infection on top of resident, hemostatic proteins. It also contains tissue leakage proteins due to the blood circulation through different tissues in the body. The blood thus contains information on the physiological state of all tissues and, combined with its accessibility, makes the blood proteome invaluable for medical purposes. Even with the recent advancements in proteomics, characterizing the proteome of blood plasma is a daunting challenge.</p>
<p>Temporal and spatial dynamics further complicate the study of human plasma proteome. The turnover of some proteins is quite faster than others and the protein content of an artery may substantially vary from that of a vein. All these differences make even the simplest proteomic task of cataloging the proteome seem out of reach. To tackle this problem, priorities need to be established. Capturing the most meaningful subset of proteins among the entire proteome to generate a diagnostic tool is one such priority. Secondly, since cancer is associated with enhanced glycosylation of proteins, methods that focus on this part of proteins will also be useful. Again: multiparameter analysis best reveals a pathological state. As these technologies improve, the disease profiles should be continually related to respective gene expression changes.<sup id="cite_ref-Weston_.26_Hood_2004_38-5" class="reference"><a href="#cite_note-Weston_.26_Hood_2004-38"><span>[</span>38<span>]</span></a></sup></p>
<h2><span class="mw-headline" id="See_also">See also</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=33" title="Edit section: See also">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div class="noprint portal tright" style="border:solid #aaa 1px;margin:0.5em 0 0.5em 1em">
<table style="background:#f9f9f9;font-size:85%;line-height:110%;max-width:175px">
<tr style="vertical-align:middle">
<td style="text-align:center"><a href="/wiki/File:Issoria_lathonia.jpg" class="image"><img alt="Portal icon" src="//upload.wikimedia.org/wikipedia/commons/thumb/2/2d/Issoria_lathonia.jpg/32px-Issoria_lathonia.jpg" width="32" height="23" class="noviewer" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/2/2d/Issoria_lathonia.jpg/48px-Issoria_lathonia.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/2/2d/Issoria_lathonia.jpg/64px-Issoria_lathonia.jpg 2x" data-file-width="629" data-file-height="445" /></a></td>
<td style="padding:0 0.2em;vertical-align:middle;font-style:italic;font-weight:bold"><a href="/wiki/Portal:Biology" title="Portal:Biology">Biology portal</a></td>
</tr>
<tr style="vertical-align:middle">
<td style="text-align:center"><a href="/wiki/File:Esculaap4.svg" class="image"><img alt="Portal icon" src="//upload.wikimedia.org/wikipedia/commons/thumb/6/66/Esculaap4.svg/18px-Esculaap4.svg.png" width="18" height="28" class="noviewer" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/6/66/Esculaap4.svg/28px-Esculaap4.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/6/66/Esculaap4.svg/37px-Esculaap4.svg.png 2x" data-file-width="200" data-file-height="300" /></a></td>
<td style="padding:0 0.2em;vertical-align:middle;font-style:italic;font-weight:bold"><a href="/wiki/Portal:Medicine" title="Portal:Medicine">Medicine portal</a></td>
</tr>
<tr style="vertical-align:middle">
<td style="text-align:center"><a href="/wiki/File:TPI1_structure.png" class="image"><img alt="Portal icon" src="//upload.wikimedia.org/wikipedia/commons/thumb/1/1c/TPI1_structure.png/32px-TPI1_structure.png" width="32" height="20" class="noviewer" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/1/1c/TPI1_structure.png/48px-TPI1_structure.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/1/1c/TPI1_structure.png/64px-TPI1_structure.png 2x" data-file-width="1452" data-file-height="914" /></a></td>
<td style="padding:0 0.2em;vertical-align:middle;font-style:italic;font-weight:bold"><a href="/wiki/Portal:Molecular_and_cellular_biology" title="Portal:Molecular and cellular biology" class="mw-redirect">Molecular and cellular biology portal</a></td>
</tr>
</table>
</div>
<div class="div-col columns column-width" style="-moz-column-width: 20em; -webkit-column-width: 20em; column-width: 20em;">
<ul>
<li><a href="/wiki/Activity_based_proteomics" title="Activity based proteomics" class="mw-redirect">Activity based proteomics</a></li>
<li><a href="/wiki/Bioinformatics" title="Bioinformatics">Bioinformatics</a></li>
<li><a href="/wiki/Bottom-up_proteomics" title="Bottom-up proteomics">Bottom-up proteomics</a></li>
<li><a href="/wiki/Cytomics" title="Cytomics">Cytomics</a></li>
<li><a href="/wiki/Functional_genomics" title="Functional genomics">Functional genomics</a></li>
<li><a href="/wiki/Genomics" title="Genomics">Genomics</a></li>
<li><a href="/wiki/Heat_stabilization" title="Heat stabilization">Heat stabilization</a></li>
<li><a href="/wiki/Human_proteome_project" title="Human proteome project">Human proteome project</a></li>
<li><a href="/wiki/Immunomics" title="Immunomics">Immunomics</a></li>
<li><a href="/wiki/Immunoproteomics" title="Immunoproteomics">Immunoproteomics</a></li>
<li><a href="/wiki/Lipidomics" title="Lipidomics">Lipidomics</a></li>
<li><a href="/wiki/List_of_biological_databases" title="List of biological databases">List of biological databases</a></li>
<li><a href="/wiki/List_of_omics_topics_in_biology" title="List of omics topics in biology">List of omics topics in biology</a></li>
<li><a href="/wiki/Metabolomics" title="Metabolomics">Metabolomics</a></li>
<li><a href="/wiki/PEGylation" title="PEGylation">PEGylation</a></li>
<li><a href="/wiki/Phosphoproteomics" title="Phosphoproteomics">Phosphoproteomics</a></li>
<li><a href="/wiki/Proteogenomics" title="Proteogenomics">Proteogenomics</a></li>
<li><a href="/wiki/Proteomic_chemistry" title="Proteomic chemistry" class="mw-redirect">Proteomic chemistry</a></li>
<li><a href="/wiki/Secretomics" title="Secretomics">Secretomics</a></li>
<li><a href="/wiki/Shotgun_proteomics" title="Shotgun proteomics">Shotgun proteomics</a></li>
<li><a href="/wiki/Top-down_proteomics" title="Top-down proteomics">Top-down proteomics</a></li>
<li><a href="/wiki/Systems_biology" title="Systems biology">Systems biology</a></li>
<li><a href="/wiki/Transcriptomics" title="Transcriptomics" class="mw-redirect">Transcriptomics</a></li>
<li><a href="/wiki/Yeast_two-hybrid_system" title="Yeast two-hybrid system" class="mw-redirect">Yeast two-hybrid system</a></li>
</ul>
</div>
<h3><span class="mw-headline" id="Protein_databases">Protein databases</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=34" title="Edit section: Protein databases">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<div class="div-col columns column-width" style="-moz-column-width: 30em; -webkit-column-width: 30em; column-width: 30em;">
<ul>
<li><a href="/wiki/Human_Protein_Atlas" title="Human Protein Atlas">Human Protein Atlas</a></li>
<li><a href="/wiki/Human_Protein_Reference_Database" title="Human Protein Reference Database">Human Protein Reference Database</a></li>
<li><a href="/wiki/National_Center_for_Biotechnology_Information" title="National Center for Biotechnology Information">National Center for Biotechnology Information</a> (NCBI)</li>
<li><a href="/wiki/Protein_Data_Bank" title="Protein Data Bank">Protein Data Bank</a> (PDB)</li>
<li><a href="/wiki/Protein_Information_Resource" title="Protein Information Resource">Protein Information Resource</a> (PIR)</li>
<li><a href="/wiki/Proteomics_Identifications_Database" title="Proteomics Identifications Database">Proteomics Identifications Database</a> (PRIDE)</li>
<li><a href="/wiki/Proteopedia" title="Proteopedia">Proteopedia</a>—The collaborative, 3D encyclopedia of proteins and other molecules</li>
<li><a href="/wiki/Swiss-Prot" title="Swiss-Prot" class="mw-redirect">Swiss-Prot</a></li>
<li><a href="/wiki/UniProt" title="UniProt">UniProt</a></li>
</ul>
</div>
<h3><span class="mw-headline" id="Research_centers">Research centers</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=35" title="Edit section: Research centers">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<ul>
<li><a href="/wiki/European_Bioinformatics_Institute" title="European Bioinformatics Institute">European Bioinformatics Institute</a></li>
<li><a href="/wiki/Netherlands_Proteomics_Centre" title="Netherlands Proteomics Centre">Netherlands Proteomics Centre</a> (NPC)</li>
</ul>
<h2><span class="mw-headline" id="References">References</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=36" title="Edit section: References">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div class="reflist columns references-column-count references-column-count-2" style="-moz-column-count: 2; -webkit-column-count: 2; column-count: 2; list-style-type: decimal;">
<ol class="references">
<li id="cite_note-pmid9740045-1"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid9740045_1-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Anderson NL, Anderson NG; Anderson (1998). "Proteome and proteomics: new technologies, new concepts, and new words". <i>Electrophoresis</i> <b>19</b> (11): 1853–61. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1002%2Felps.1150191103">10.1002/elps.1150191103</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/9740045">9740045</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Proteome+and+proteomics%3A+new+technologies%2C+new+concepts%2C+and+new+words&amp;rft.au=Anderson&amp;rft.au=Anderson+NL%2C+Anderson+NG&amp;rft.date=1998&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1002%2Felps.1150191103&amp;rft_id=info%3Apmid%2F9740045&amp;rft.issue=11&amp;rft.jtitle=Electrophoresis&amp;rft.pages=1853-61&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=19" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-pmid10189717-2"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid10189717_2-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Blackstock WP, Weir MP; Weir (1999). "Proteomics: quantitative and physical mapping of cellular proteins". <i>Trends Biotechnol.</i> <b>17</b> (3): 121–7. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1016%2FS0167-7799%2898%2901245-1">10.1016/S0167-7799(98)01245-1</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/10189717">10189717</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Proteomics%3A+quantitative+and+physical+mapping+of+cellular+proteins&amp;rft.au=Blackstock+WP%2C+Weir+MP&amp;rft.au=Weir&amp;rft.date=1999&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1016%2FS0167-7799%2898%2901245-1&amp;rft_id=info%3Apmid%2F10189717&amp;rft.issue=3&amp;rft.jtitle=Trends+Biotechnol.&amp;rft.pages=121-7&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=17" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-3"><span class="mw-cite-backlink"><b><a href="#cite_ref-3">^</a></b></span> <span class="reference-text"><cite class="citation journal">P. James (1997). "Protein identification in the post-genome era: the rapid rise of proteomics". <i>Quarterly reviews of biophysics</i> <b>30</b> (4): 279–331. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1017%2FS0033583597003399">10.1017/S0033583597003399</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/9634650">9634650</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Protein+identification+in+the+post-genome+era%3A+the+rapid+rise+of+proteomics&amp;rft.au=P.+James&amp;rft.date=1997&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1017%2FS0033583597003399&amp;rft_id=info%3Apmid%2F9634650&amp;rft.issue=4&amp;rft.jtitle=Quarterly+reviews+of+biophysics&amp;rft.pages=279-331&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=30" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-wilkins1996-4"><span class="mw-cite-backlink">^ <a href="#cite_ref-wilkins1996_4-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-wilkins1996_4-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">Marc R. Wilkins, Christian Pasquali, Ron D. Appel, Keli Ou, Olivier Golaz, Jean-Charles Sanchez, Jun X. Yan, Andrew. A. Gooley, Graham Hughes, Ian Humphery-Smith, Keith L. Williams &amp; Denis F. Hochstrasser; Pasquali; Appel; Ou; Golaz; Sanchez; Yan; Gooley; Hughes; Humphery-Smith; Williams; Hochstrasser (1996). "From Proteins to Proteomes: Large Scale Protein Identification by Two-Dimensional Electrophoresis and Arnino Acid Analysis". <i>Nature Biotechnology</i> <b>14</b> (1): 61–65. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1038%2Fnbt0196-61">10.1038/nbt0196-61</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/9636313">9636313</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=From+Proteins+to+Proteomes%3A+Large+Scale+Protein+Identification+by+Two-Dimensional+Electrophoresis+and+Arnino+Acid+Analysis&amp;rft.au=Appel&amp;rft.au=Golaz&amp;rft.au=Gooley&amp;rft.au=Hochstrasser&amp;rft.au=Hughes&amp;rft.au=Humphery-Smith&amp;rft.au=Marc+R.+Wilkins%2C+Christian+Pasquali%2C+Ron+D.+Appel%2C+Keli+Ou%2C+Olivier+Golaz%2C+Jean-Charles+Sanchez%2C+Jun+X.+Yan%2C+Andrew.+A.+Gooley%2C+Graham+Hughes%2C+Ian+Humphery-Smith%2C+Keith+L.+Williams+%26+Denis+F.+Hochstrasser&amp;rft.au=Ou&amp;rft.au=Pasquali&amp;rft.au=Sanchez&amp;rft.au=Williams&amp;rft.au=Yan&amp;rft.date=1996&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1038%2Fnbt0196-61&amp;rft_id=info%3Apmid%2F9636313&amp;rft.issue=1&amp;rft.jtitle=Nature+Biotechnology&amp;rft.pages=61-65&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=14" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-5"><span class="mw-cite-backlink"><b><a href="#cite_ref-5">^</a></b></span> <span class="reference-text"><a rel="nofollow" class="external text" href="http://www.babs.unsw.edu.au/directory.php?personnelID=12">UNSW Staff Bio: Professor Marc Wilkins</a> <a rel="nofollow" class="external text" href="https://web.archive.org/web/20110822151059/http://www.babs.unsw.edu.au/directory.php?personnelID=12">Archived</a> August 22, 2011 at the <a href="/wiki/Wayback_Machine" title="Wayback Machine">Wayback Machine</a></span></li>
<li id="cite_note-6"><span class="mw-cite-backlink"><b><a href="#cite_ref-6">^</a></b></span> <span class="reference-text"><cite class="citation journal">Simon Rogers, Mark Girolami, Walter Kolch, Katrina M. Waters, Tao Liu, Brian Thrall and H. Steven Wiley (2008). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4141638">"Investigating the correspondence between transcriptomic and proteomic expression profiles using coupled cluster models"</a>. <i>Bioinformatics</i> <b>24</b> (24): 2894–2900. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1093%2Fbioinformatics%2Fbtn553">10.1093/bioinformatics/btn553</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC4141638">4141638</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/18974169">18974169</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Investigating+the+correspondence+between+transcriptomic+and+proteomic+expression+profiles+using+coupled+cluster+models&amp;rft.au=Simon+Rogers%2C+Mark+Girolami%2C+Walter+Kolch%2C+Katrina+M.+Waters%2C+Tao+Liu%2C+Brian+Thrall+and+H.+Steven+Wiley&amp;rft.date=2008&amp;rft.genre=article&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4141638&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4141638&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtn553&amp;rft_id=info%3Apmid%2F18974169&amp;rft.issue=24&amp;rft.jtitle=Bioinformatics&amp;rft.pages=2894-2900&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=24" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-7"><span class="mw-cite-backlink"><b><a href="#cite_ref-7">^</a></b></span> <span class="reference-text"><cite class="citation journal">Vikas Dhingraa, Mukta Gupta, Tracy Andacht and Zhen F. Fu (2005). "New frontiers in proteomics research: A perspective". <i>International Journal of Pharmaceutics</i> <b>299</b> (1–2): 1–18. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1016%2Fj.ijpharm.2005.04.010">10.1016/j.ijpharm.2005.04.010</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/15979831">15979831</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=New+frontiers+in+proteomics+research%3A+A+perspective&amp;rft.au=Vikas+Dhingraa%2C+Mukta+Gupta%2C+Tracy+Andacht+and+Zhen+F.+Fu&amp;rft.date=2005&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1016%2Fj.ijpharm.2005.04.010&amp;rft_id=info%3Apmid%2F15979831&amp;rft.issue=1%932&amp;rft.jtitle=International+Journal+of+Pharmaceutics&amp;rft.pages=1-18&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=299" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-8"><span class="mw-cite-backlink"><b><a href="#cite_ref-8">^</a></b></span> <span class="reference-text"><cite class="citation web">Buckingham, Steven (May 2003). <a rel="nofollow" class="external text" href="http://www.nature.com/horizon/rna/background/micrornas.html">"The major world of microRNAs"</a><span class="reference-accessdate">. Retrieved <span class="nowrap">2009-01-14</span></span>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.aufirst=Steven&amp;rft.aulast=Buckingham&amp;rft.btitle=The+major+world+of+microRNAs&amp;rft.date=2003-05&amp;rft.genre=unknown&amp;rft_id=http%3A%2F%2Fwww.nature.com%2Fhorizon%2Frna%2Fbackground%2Fmicrornas.html&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-9"><span class="mw-cite-backlink"><b><a href="#cite_ref-9">^</a></b></span> <span class="reference-text"><cite class="citation journal">Olsen JV, Blagoev B, Gnad F, Macek B, Kumar C, Mortensen P, Mann M; Blagoev; Gnad; Macek; Kumar; Mortensen; Mann (2006). "Global, in vivo, and site-specific phosphorylation dynamics in signaling networks". <i>Cell</i> <b>127</b> (3): 635–648. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1016%2Fj.cell.2006.09.026">10.1016/j.cell.2006.09.026</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/17081983">17081983</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Global%2C+in+vivo%2C+and+site-specific+phosphorylation+dynamics+in+signaling+networks&amp;rft.au=Blagoev&amp;rft.au=Gnad&amp;rft.au=Kumar&amp;rft.au=Macek&amp;rft.au=Mann&amp;rft.au=Mortensen&amp;rft.au=Olsen+JV%2C+Blagoev+B%2C+Gnad+F%2C+Macek+B%2C+Kumar+C%2C+Mortensen+P%2C+Mann+M&amp;rft.date=2006&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1016%2Fj.cell.2006.09.026&amp;rft_id=info%3Apmid%2F17081983&amp;rft.issue=3&amp;rft.jtitle=Cell&amp;rft.pages=635-648&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=127" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-Gygi1999-10"><span class="mw-cite-backlink"><b><a href="#cite_ref-Gygi1999_10-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Gygi, S. P.; Rochon, Y.; Franza, B. R.; Aebersold, R. (1999). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC83965">"Correlation between protein and mRNA abundance in yeast"</a>. <i>Molecular and Cellular Biology</i> <b>19</b> (3): 1720–1730. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC83965">83965</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/10022859">10022859</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Correlation+between+protein+and+mRNA+abundance+in+yeast&amp;rft.au=Aebersold%2C+R.&amp;rft.aufirst=S.+P.&amp;rft.au=Franza%2C+B.+R.&amp;rft.aulast=Gygi&amp;rft.au=Rochon%2C+Y.&amp;rft.date=1999&amp;rft.genre=article&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC83965&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC83965&amp;rft_id=info%3Apmid%2F10022859&amp;rft.issue=3&amp;rft.jtitle=Molecular+and+Cellular+Biology&amp;rft.pages=1720-1730&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=19" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-11"><span class="mw-cite-backlink"><b><a href="#cite_ref-11">^</a></b></span> <span class="reference-text"><cite class="citation journal">Archana Belle, Amos Tanay, Ledion Bitincka, Ron Shamir and Erin K. O’Shea (2006). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1550773">"Quantification of protein half-lives in the budding yeast proteome"</a>. <i>PNAS</i> <b>103</b> (35): 13004–13009. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="http://adsabs.harvard.edu/abs/2006PNAS..10313004B">2006PNAS..10313004B</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1073%2Fpnas.0605420103">10.1073/pnas.0605420103</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1550773">1550773</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/16916930">16916930</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Quantification+of+protein+half-lives+in+the+budding+yeast+proteome&amp;rft.au=Archana+Belle%2C+Amos+Tanay%2C+Ledion+Bitincka%2C+Ron+Shamir+and+Erin+K.+O%99Shea&amp;rft.date=2006&amp;rft.genre=article&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1550773&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1550773&amp;rft_id=info%3Abibcode%2F2006PNAS..10313004B&amp;rft_id=info%3Adoi%2F10.1073%2Fpnas.0605420103&amp;rft_id=info%3Apmid%2F16916930&amp;rft.issue=35&amp;rft.jtitle=PNAS&amp;rft.pages=13004-13009&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=103" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-Peng2003-12"><span class="mw-cite-backlink"><b><a href="#cite_ref-Peng2003_12-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Peng, J.; Elias, J. E.; Thoreen, C. C.; Licklider, L. J.; Gygi, S. P. (2003). "Evaluation of multidimensional chromatography coupled with tandem mass spectrometry (LC/LC-MS/MS) for large-scale protein analysis: The yeast proteome". <i>Journal of Proteome Research</i> <b>2</b> (1): 43–50. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1021%2Fpr025556v">10.1021/pr025556v</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/12643542">12643542</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Evaluation+of+multidimensional+chromatography+coupled+with+tandem+mass+spectrometry+%28LC%2FLC-MS%2FMS%29+for+large-scale+protein+analysis%3A+The+yeast+proteome&amp;rft.au=Elias%2C+J.+E.&amp;rft.aufirst=J.&amp;rft.au=Gygi%2C+S.+P.&amp;rft.aulast=Peng&amp;rft.au=Licklider%2C+L.+J.&amp;rft.au=Thoreen%2C+C.+C.&amp;rft.date=2003&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1021%2Fpr025556v&amp;rft_id=info%3Apmid%2F12643542&amp;rft.issue=1&amp;rft.jtitle=Journal+of+Proteome+Research&amp;rft.pages=43-50&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=2" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-Washburn2001-13"><span class="mw-cite-backlink"><b><a href="#cite_ref-Washburn2001_13-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Washburn, M. P.; Wolters, D.; Yates, J. R. (2001). "Large-scale analysis of the yeast proteome by multidimensional protein identification technology". <i>Nature Biotechnology</i> <b>19</b> (3): 242–247. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1038%2F85686">10.1038/85686</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/11231557">11231557</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Large-scale+analysis+of+the+yeast+proteome+by+multidimensional+protein+identification+technology&amp;rft.aufirst=M.+P.&amp;rft.aulast=Washburn&amp;rft.au=Wolters%2C+D.&amp;rft.au=Yates%2C+J.+R.&amp;rft.date=2001&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1038%2F85686&amp;rft_id=info%3Apmid%2F11231557&amp;rft.issue=3&amp;rft.jtitle=Nature+Biotechnology&amp;rft.pages=242-247&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=19" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-NelsonKrone1995-14"><span class="mw-cite-backlink"><b><a href="#cite_ref-NelsonKrone1995_14-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Nelson, Randall W.; Krone, Jennifer R.; Bieber, Allan L.; Williams, Peter. (1995). "Mass Spectrometric Immunoassay". <i>Analytical Chemistry</i> <b>67</b> (7): 1153–1158. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1021%2Fac00103a003">10.1021/ac00103a003</a>. <a href="/wiki/International_Standard_Serial_Number" title="International Standard Serial Number">ISSN</a>&#160;<a rel="nofollow" class="external text" href="//www.worldcat.org/issn/0003-2700">0003-2700</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Mass+Spectrometric+Immunoassay&amp;rft.au=Bieber%2C+Allan+L.&amp;rft.aufirst=Randall+W.&amp;rft.au=Krone%2C+Jennifer+R.&amp;rft.aulast=Nelson&amp;rft.au=Williams%2C+Peter.&amp;rft.date=1995&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1021%2Fac00103a003&amp;rft.issn=0003-2700&amp;rft.issue=7&amp;rft.jtitle=Analytical+Chemistry&amp;rft.pages=1153-1158&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=67" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-15"><span class="mw-cite-backlink"><b><a href="#cite_ref-15">^</a></b></span> <span class="reference-text"><a rel="nofollow" class="external free" href="https://www.broadinstitute.org/scientific-community/science/platforms/proteomics/siscapa">https://www.broadinstitute.org/scientific-community/science/platforms/proteomics/siscapa</a></span></li>
<li id="cite_note-16"><span class="mw-cite-backlink"><b><a href="#cite_ref-16">^</a></b></span> <span class="reference-text"><cite class="citation journal">de Mol, Nico J. (2012). <a rel="nofollow" class="external text" href="http://link.springer.com/protocol/10.1007%2F978-1-61779-349-3_4">"Surface Plasmon Resonance for Proteomics"</a>. <i>Methods Mol Biol</i>. Methods in Molecular Biology <b>800</b>: 33–53. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1007%2F978-1-61779-349-3_4">10.1007/978-1-61779-349-3_4</a>. <a href="/wiki/International_Standard_Book_Number" title="International Standard Book Number">ISBN</a>&#160;<a href="/wiki/Special:BookSources/978-1-61779-348-6" title="Special:BookSources/978-1-61779-348-6">978-1-61779-348-6</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/21964781">21964781</a><span class="reference-accessdate">. Retrieved <span class="nowrap">2014-11-19</span></span>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Surface+Plasmon+Resonance+for+Proteomics&amp;rft.aufirst=Nico+J.&amp;rft.aulast=de+Mol&amp;rft.date=2012&amp;rft.genre=article&amp;rft_id=http%3A%2F%2Flink.springer.com%2Fprotocol%2F10.1007%252F978-1-61779-349-3_4&amp;rft_id=info%3Adoi%2F10.1007%2F978-1-61779-349-3_4&amp;rft_id=info%3Apmid%2F21964781&amp;rft.isbn=978-1-61779-348-6&amp;rft.jtitle=Methods+Mol+Biol.&amp;rft.pages=33-53&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=800" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-17"><span class="mw-cite-backlink"><b><a href="#cite_ref-17">^</a></b></span> <span class="reference-text"><cite class="citation journal">Visser, Natasja FC; Heck, Albert JR (2008). <a rel="nofollow" class="external text" href="http://informahealthcare.com/doi/abs/10.1586/14789450.5.3.425">"Surface plasmon resonance mass spectrometry in proteomics"</a>. <i>Expert Rev Proteomics</i> <b>5</b> (3): 425–33. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1586%2F14789450.5.3.425">10.1586/14789450.5.3.425</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/18532910">18532910</a><span class="reference-accessdate">. Retrieved <span class="nowrap">2014-11-19</span></span>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Surface+plasmon+resonance+mass+spectrometry+in+proteomics&amp;rft.aufirst=Natasja+FC&amp;rft.au=Heck%2C+Albert+JR&amp;rft.aulast=Visser&amp;rft.date=2008&amp;rft.genre=article&amp;rft_id=http%3A%2F%2Finformahealthcare.com%2Fdoi%2Fabs%2F10.1586%2F14789450.5.3.425&amp;rft_id=info%3Adoi%2F10.1586%2F14789450.5.3.425&amp;rft_id=info%3Apmid%2F18532910&amp;rft.issue=3&amp;rft.jtitle=Expert+Rev+Proteomics&amp;rft.pages=425-33&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=5" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-Vaidyanathan12-18"><span class="mw-cite-backlink"><b><a href="#cite_ref-Vaidyanathan12_18-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Vaidyanathan G (March 2012). "Redefining clinical trials: the age of personalized medicine". <i>Cell</i> <b>148</b> (6): 1079–80. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1016%2Fj.cell.2012.02.041">10.1016/j.cell.2012.02.041</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/22424218">22424218</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Redefining+clinical+trials%3A+the+age+of+personalized+medicine&amp;rft.au=Vaidyanathan+G&amp;rft.date=2012-03&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1016%2Fj.cell.2012.02.041&amp;rft_id=info%3Apmid%2F22424218&amp;rft.issue=6&amp;rft.jtitle=Cell&amp;rft.pages=1079-80&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=148" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-19"><span class="mw-cite-backlink"><b><a href="#cite_ref-19">^</a></b></span> <span class="reference-text"><cite class="citation journal">Rakwal, Randeep; Komatsu, Setsuko (2000). "Role of jasmonate in the rice (Oryza sativa L.) self-defense mechanism using proteome analysis". <i>Electrophoresis</i> <b>21</b> (12): 2492–500. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1002%2F1522-2683%2820000701%2921%3A12%3C2492%3A%3AAID-ELPS2492%3E3.0.CO%3B2-2">10.1002/1522-2683(20000701)21:12&lt;2492::AID-ELPS2492&gt;3.0.CO;2-2</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/10939463">10939463</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Role+of+jasmonate+in+the+rice+%28Oryza+sativa+L.%29+self-defense+mechanism+using+proteome+analysis&amp;rft.aufirst=Randeep&amp;rft.au=Komatsu%2C+Setsuko&amp;rft.aulast=Rakwal&amp;rft.date=2000&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1002%2F1522-2683%2820000701%2921%3A12%3C2492%3A%3AAID-ELPS2492%3E3.0.CO%3B2-2&amp;rft_id=info%3Apmid%2F10939463&amp;rft.issue=12&amp;rft.jtitle=Electrophoresis&amp;rft.pages=2492-500&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=21" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-20"><span class="mw-cite-backlink"><b><a href="#cite_ref-20">^</a></b></span> <span class="reference-text"><cite class="citation journal">Wu, Jianqiang; Baldwin, Ian T. (2010). "New Insights into Plant Responses to the Attack from Insect Herbivores". <i>Annual Review of Genetics</i> <b>44</b>: 1–24. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1146%2Fannurev-genet-102209-163500">10.1146/annurev-genet-102209-163500</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/20649414">20649414</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=New+Insights+into+Plant+Responses+to+the+Attack+from+Insect+Herbivores&amp;rft.au=Baldwin%2C+Ian+T.&amp;rft.aufirst=Jianqiang&amp;rft.aulast=Wu&amp;rft.date=2010&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1146%2Fannurev-genet-102209-163500&amp;rft_id=info%3Apmid%2F20649414&amp;rft.jtitle=Annual+Review+of+Genetics&amp;rft.pages=1-24&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=44" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-21"><span class="mw-cite-backlink"><b><a href="#cite_ref-21">^</a></b></span> <span class="reference-text"><cite class="citation journal">Sangha J.S., Chen Y.H., Kaur Jatinder, Khan Wajahatullah, Abduljaleel Zainularifeen, Alanazi Mohammed S., Mills Aaron, Adalla Candida B., Bennett John; et al. (2013). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3588078">"Proteome Analysis of Rice (Oryza sativa L.) Mutants Reveals Differentially Induced Proteins during Brown Planthopper (Nilaparvata lugens) Infestation"</a>. <i>Int. J. Mo Sci</i> <b>14</b> (2): 3921–3945. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.3390%2Fijms14023921">10.3390/ijms14023921</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3588078">3588078</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/23434671">23434671</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Proteome+Analysis+of+Rice+%28Oryza+sativa+L.%29+Mutants+Reveals+Differentially+Induced+Proteins+during+Brown+Planthopper+%28Nilaparvata+lugens%29+Infestation&amp;rft.au=Sangha+J.S.%2C+Chen+Y.H.%2C+Kaur+Jatinder%2C+Khan+Wajahatullah%2C+Abduljaleel+Zainularifeen%2C+Alanazi+Mohammed+S.%2C+Mills+Aaron%2C+Adalla+Candida+B.%2C+Bennett+John&amp;rft.date=2013&amp;rft.genre=article&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3588078&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3588078&amp;rft_id=info%3Adoi%2F10.3390%2Fijms14023921&amp;rft_id=info%3Apmid%2F23434671&amp;rft.issue=2&amp;rft.jtitle=Int.+J.+Mo+Sci&amp;rft.pages=3921-3945&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=14" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-22"><span class="mw-cite-backlink"><b><a href="#cite_ref-22">^</a></b></span> <span class="reference-text"><cite class="citation journal">Strimbu, Kyle; Tavel, Jorge A (2010). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3078627">"What are biomarkers?"</a>. <i>Current Opinion in HIV and AIDS</i> <b>5</b> (6): 463–6. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1097%2FCOH.0b013e32833ed177">10.1097/COH.0b013e32833ed177</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3078627">3078627</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/20978388">20978388</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=What+are+biomarkers%3F&amp;rft.aufirst=Kyle&amp;rft.aulast=Strimbu&amp;rft.au=Tavel%2C+Jorge+A&amp;rft.date=2010&amp;rft.genre=article&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3078627&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3078627&amp;rft_id=info%3Adoi%2F10.1097%2FCOH.0b013e32833ed177&amp;rft_id=info%3Apmid%2F20978388&amp;rft.issue=6&amp;rft.jtitle=Current+Opinion+in+HIV+and+AIDS&amp;rft.pages=463-6&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=5" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-23"><span class="mw-cite-backlink"><b><a href="#cite_ref-23">^</a></b></span> <span class="reference-text"><cite class="citation journal">Biomarkers Definitions Working Group (2001). "Biomarkers and surrogate endpoints: preferred definitions and conceptual framework". <i>Clinical Pharmacology &amp; Therapeutics</i> <b>69</b> (3): 89–95. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1067%2Fmcp.2001.113989">10.1067/mcp.2001.113989</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/11240971">11240971</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Biomarkers+and+surrogate+endpoints%3A+preferred+definitions+and+conceptual+framework&amp;rft.au=Biomarkers+Definitions+Working+Group&amp;rft.date=2001&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1067%2Fmcp.2001.113989&amp;rft_id=info%3Apmid%2F11240971&amp;rft.issue=3&amp;rft.jtitle=Clinical+Pharmacology+%26+Therapeutics&amp;rft.pages=89-95&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=69" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-Ceciliani2014-24"><span class="mw-cite-backlink"><b><a href="#cite_ref-Ceciliani2014_24-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Ceciliani, F; Eckersall D; Burchmore R; Lecchi C (March 2014). <a rel="nofollow" class="external text" href="http://vet.sagepub.com/content/51/2/351.long">"Proteomics in veterinary medicine: applications and trends in disease pathogenesis and diagnostics"</a>. <i>Veterinary Pathology</i> <b>51</b> (2): 351–362. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1177%2F0300985813502819">10.1177/0300985813502819</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/24045891">24045891</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Proteomics+in+veterinary+medicine%3A+applications+and+trends+in+disease+pathogenesis+and+diagnostics&amp;rft.au=Burchmore+R&amp;rft.au=Eckersall+D&amp;rft.aufirst=F&amp;rft.aulast=Ceciliani&amp;rft.au=Lecchi+C&amp;rft.date=2014-03&amp;rft.genre=article&amp;rft_id=http%3A%2F%2Fvet.sagepub.com%2Fcontent%2F51%2F2%2F351.long&amp;rft_id=info%3Adoi%2F10.1177%2F0300985813502819&amp;rft_id=info%3Apmid%2F24045891&amp;rft.issue=2&amp;rft.jtitle=Veterinary+Pathology&amp;rft.pages=351-362&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=51" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-Klopfleisch1-25"><span class="mw-cite-backlink">^ <a href="#cite_ref-Klopfleisch1_25-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-Klopfleisch1_25-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">Klopfleisch R, Klose P, Weise C, Bondzio A, Multhaup G, Einspanier R, Gruber AD.; Klose; Weise; Bondzio; Multhaup; Einspanier; Gruber (2010). "Proteome of metastatic canine mammary carcinomas: similarities to and differences from human breast cancer". <i>J Proteome Res</i> <b>9</b> (12): 6380–91. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1021%2Fpr100671c">10.1021/pr100671c</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/20932060">20932060</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Proteome+of+metastatic+canine+mammary+carcinomas%3A+similarities+to+and+differences+from+human+breast+cancer&amp;rft.au=Bondzio&amp;rft.au=Einspanier&amp;rft.au=Gruber&amp;rft.au=Klopfleisch+R%2C+Klose+P%2C+Weise+C%2C+Bondzio+A%2C+Multhaup+G%2C+Einspanier+R%2C+Gruber+AD.&amp;rft.au=Klose&amp;rft.au=Multhaup&amp;rft.au=Weise&amp;rft.date=2010&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1021%2Fpr100671c&amp;rft_id=info%3Apmid%2F20932060&amp;rft.issue=12&amp;rft.jtitle=J+Proteome+Res&amp;rft.pages=6380-91&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=9" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-26"><span class="mw-cite-backlink"><b><a href="#cite_ref-26">^</a></b></span> <span class="reference-text"><cite class="citation journal">Klopfleisch R, Gruber AD; Gruber (2009). "Increased expression of BRCA2 and RAD51 in lymph node metastases of canine mammary adenocarcinomas". <i>Veterinary Pathology</i> <b>46</b> (3): 416–22. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1354%2Fvp.08-VP-0212-K-FL">10.1354/vp.08-VP-0212-K-FL</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/19176491">19176491</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Increased+expression+of+BRCA2+and+RAD51+in+lymph+node+metastases+of+canine+mammary+adenocarcinomas&amp;rft.au=Gruber&amp;rft.au=Klopfleisch+R%2C+Gruber+AD&amp;rft.date=2009&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1354%2Fvp.08-VP-0212-K-FL&amp;rft_id=info%3Apmid%2F19176491&amp;rft.issue=3&amp;rft.jtitle=Veterinary+Pathology&amp;rft.pages=416-22&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=46" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-pmid17425459-27"><span class="mw-cite-backlink"><b><a href="#cite_ref-pmid17425459_27-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Hathout, Yetrib (2007). "Approaches to the study of the cell secretome". <i>Expert Review of Proteomics</i> <b>4</b> (2): 239–48. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1586%2F14789450.4.2.239">10.1586/14789450.4.2.239</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/17425459">17425459</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Approaches+to+the+study+of+the+cell+secretome&amp;rft.aufirst=Yetrib&amp;rft.aulast=Hathout&amp;rft.date=2007&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1586%2F14789450.4.2.239&amp;rft_id=info%3Apmid%2F17425459&amp;rft.issue=2&amp;rft.jtitle=Expert+Review+of+Proteomics&amp;rft.pages=239-48&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=4" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-Gupta07-28"><span class="mw-cite-backlink"><b><a href="#cite_ref-Gupta07_28-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Gupta N, Tanner S, Jaitly N; et al. (September 2007). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1950905">"Whole proteome analysis of post-translational modifications: applications of mass-spectrometry for proteogenomic annotation"</a>. <i>Genome Res.</i> <b>17</b> (9): 1362–77. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1101%2Fgr.6427907">10.1101/gr.6427907</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1950905">1950905</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/17690205">17690205</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Whole+proteome+analysis+of+post-translational+modifications%3A+applications+of+mass-spectrometry+for+proteogenomic+annotation&amp;rft.au=Gupta+N%2C+Tanner+S%2C+Jaitly+N&amp;rft.date=2007-09&amp;rft.genre=article&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1950905&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1950905&amp;rft_id=info%3Adoi%2F10.1101%2Fgr.6427907&amp;rft_id=info%3Apmid%2F17690205&amp;rft.issue=9&amp;rft.jtitle=Genome+Res.&amp;rft.pages=1362-77&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=17" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-Gupta08-29"><span class="mw-cite-backlink"><b><a href="#cite_ref-Gupta08_29-0">^</a></b></span> <span class="reference-text"><cite class="citation journal">Gupta N, Benhamida J, Bhargava V; et al. (July 2008). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2493402">"Comparative proteogenomics: combining mass spectrometry and comparative genomics to analyze multiple genomes"</a>. <i>Genome Res.</i> <b>18</b> (7): 1133–42. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1101%2Fgr.074344.107">10.1101/gr.074344.107</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2493402">2493402</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/18426904">18426904</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Comparative+proteogenomics%3A+combining+mass+spectrometry+and+comparative+genomics+to+analyze+multiple+genomes&amp;rft.au=Gupta+N%2C+Benhamida+J%2C+Bhargava+V&amp;rft.date=2008-07&amp;rft.genre=article&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2493402&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2493402&amp;rft_id=info%3Adoi%2F10.1101%2Fgr.074344.107&amp;rft_id=info%3Apmid%2F18426904&amp;rft.issue=7&amp;rft.jtitle=Genome+Res.&amp;rft.pages=1133-42&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=18" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-Tonge01-30"><span class="mw-cite-backlink">^ <a href="#cite_ref-Tonge01_30-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-Tonge01_30-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">Tonge R, Shaw J, Middleton B; et al. (March 2001). "Validation and development of fluorescence two-dimensional differential gel electrophoresis proteomics technology". <i>Proteomics</i> <b>1</b> (3): 377–96. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1002%2F1615-9861%28200103%291%3A3%3C377%3A%3AAID-PROT377%3E3.0.CO%3B2-6">10.1002/1615-9861(200103)1:3&lt;377::AID-PROT377&gt;3.0.CO;2-6</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/11680884">11680884</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Validation+and+development+of+fluorescence+two-dimensional+differential+gel+electrophoresis+proteomics+technology&amp;rft.au=Tonge+R%2C+Shaw+J%2C+Middleton+B&amp;rft.date=2001-03&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1002%2F1615-9861%28200103%291%3A3%3C377%3A%3AAID-PROT377%3E3.0.CO%3B2-6&amp;rft_id=info%3Apmid%2F11680884&amp;rft.issue=3&amp;rft.jtitle=Proteomics&amp;rft.pages=377-96&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=1" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-31"><span class="mw-cite-backlink"><b><a href="#cite_ref-31">^</a></b></span> <span class="reference-text"><cite class="citation journal">Li-Ping Wang, Jun Shen, Lin-Quan Ge, Jin-Cai Wu, Guo-Qin Yang, Gary C. Jahn; Shen; Ge; Wu; Yang; Jahn (November 2010). "Insecticide-induced increase in the protein content of male accessory glands and its effect on the fecundity of females in the brown planthopper, <i>Nilaparvata lugens</i> Stål (Hemiptera: Delphacidae)". <i>Crop Protection</i> <b>29</b> (11): 1280–5. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1016%2Fj.cropro.2010.07.009">10.1016/j.cropro.2010.07.009</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Insecticide-induced+increase+in+the+protein+content+of+male+accessory+glands+and+its+effect+on+the+fecundity+of+females+in+the+brown+planthopper%2C+Nilaparvata+lugens+St%C3%A5l+%28Hemiptera%3A+Delphacidae%29&amp;rft.au=Ge&amp;rft.au=Jahn&amp;rft.au=Li-Ping+Wang%2C+Jun+Shen%2C+Lin-Quan+Ge%2C+Jin-Cai+Wu%2C+Guo-Qin+Yang%2C+Gary+C.+Jahn&amp;rft.au=Shen&amp;rft.au=Wu&amp;rft.au=Yang&amp;rft.date=2010-11&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1016%2Fj.cropro.2010.07.009&amp;rft.issue=11&amp;rft.jtitle=Crop+Protection&amp;rft.pages=1280-5&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=29" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-Ge_et_al_2011-32"><span class="mw-cite-backlink">^ <a href="#cite_ref-Ge_et_al_2011_32-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-Ge_et_al_2011_32-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">Ge, Lin-Quan; Cheng, Yao; Wu, Jin-Cai; Jahn, Gary C. (2011). "Proteomic Analysis of Insecticide Triazophos-Induced Mating-Responsive Proteins ofNilaparvata lugensStål (Hemiptera: Delphacidae)". <i>Journal of Proteome Research</i> <b>10</b> (10): 4597–612. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1021%2Fpr200414g">10.1021/pr200414g</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/21800909">21800909</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Proteomic+Analysis+of+Insecticide+Triazophos-Induced+Mating-Responsive+Proteins+ofNilaparvata+lugensSt%C3%A5l+%28Hemiptera%3A+Delphacidae%29&amp;rft.au=Cheng%2C+Yao&amp;rft.aufirst=Lin-Quan&amp;rft.au=Jahn%2C+Gary+C.&amp;rft.aulast=Ge&amp;rft.au=Wu%2C+Jin-Cai&amp;rft.date=2011&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1021%2Fpr200414g&amp;rft_id=info%3Apmid%2F21800909&amp;rft.issue=10&amp;rft.jtitle=Journal+of+Proteome+Research&amp;rft.pages=4597-612&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=10" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-Reumann11-33"><span class="mw-cite-backlink">^ <a href="#cite_ref-Reumann11_33-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-Reumann11_33-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">Reumann S (May 2011). "Toward a definition of the complete proteome of plant peroxisomes: Where experimental proteomics must be complemented by bioinformatics". <i>Proteomics</i> <b>11</b> (9): 1764–79. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1002%2Fpmic.201000681">10.1002/pmic.201000681</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/21472859">21472859</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Toward+a+definition+of+the+complete+proteome+of+plant+peroxisomes%3A+Where+experimental+proteomics+must+be+complemented+by+bioinformatics&amp;rft.au=Reumann+S&amp;rft.date=2011-05&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1002%2Fpmic.201000681&amp;rft_id=info%3Apmid%2F21472859&amp;rft.issue=9&amp;rft.jtitle=Proteomics&amp;rft.pages=1764-79&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=11" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-34"><span class="mw-cite-backlink"><b><a href="#cite_ref-34">^</a></b></span> <span class="reference-text"><cite class="citation journal">Uhlen M, Ponten F; Ponten (April 2005). "Antibody-based proteomics for human tissue profiling". <i>Mol. Cell Proteomics</i> <b>4</b> (4): 384–93. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1074%2Fmcp.R500009-MCP200">10.1074/mcp.R500009-MCP200</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/15695805">15695805</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Antibody-based+proteomics+for+human+tissue+profiling&amp;rft.au=Ponten&amp;rft.au=Uhlen+M%2C+Ponten+F&amp;rft.date=2005-04&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1074%2Fmcp.R500009-MCP200&amp;rft_id=info%3Apmid%2F15695805&amp;rft.issue=4&amp;rft.jtitle=Mol.+Cell+Proteomics&amp;rft.pages=384-93&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=4" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-35"><span class="mw-cite-backlink"><b><a href="#cite_ref-35">^</a></b></span> <span class="reference-text"><cite class="citation journal">Ole Nørregaard Jensen (2004). "Modification-specific proteomics: characterization of post-translational modifications by mass spectrometry". <i>Current Opinion in Chemical Biology</i> <b>8</b> (1): 33–41. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1016%2Fj.cbpa.2003.12.009">10.1016/j.cbpa.2003.12.009</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/15036154">15036154</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Modification-specific+proteomics%3A+characterization+of+post-translational+modifications+by+mass+spectrometry&amp;rft.au=Ole+N%C3%B8rregaard+Jensen&amp;rft.date=2004&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1016%2Fj.cbpa.2003.12.009&amp;rft_id=info%3Apmid%2F15036154&amp;rft.issue=1&amp;rft.jtitle=Current+Opinion+in+Chemical+Biology&amp;rft.pages=33-41&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=8" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-FOOTNOTEAlinejad2015-36"><span class="mw-cite-backlink"><b><a href="#cite_ref-FOOTNOTEAlinejad2015_36-0">^</a></b></span> <span class="reference-text"><a href="#CITEREFAlinejad2015">Alinejad 2015</a>.</span></li>
<li id="cite_note-What_is_Proteomics.3F-37"><span class="mw-cite-backlink">^ <a href="#cite_ref-What_is_Proteomics.3F_37-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-What_is_Proteomics.3F_37-1"><sup><i><b>b</b></i></sup></a> <a href="#cite_ref-What_is_Proteomics.3F_37-2"><sup><i><b>c</b></i></sup></a> <a href="#cite_ref-What_is_Proteomics.3F_37-3"><sup><i><b>d</b></i></sup></a></span> <span class="reference-text"><cite class="citation web"><a rel="nofollow" class="external text" href="http://www.proteomic.org/html/proteomics_.html">"What is Proteomics?"</a>. ProteoConsult.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.btitle=What+is+Proteomics%3F&amp;rft.genre=unknown&amp;rft_id=http%3A%2F%2Fwww.proteomic.org%2Fhtml%2Fproteomics_.html&amp;rft.pub=ProteoConsult&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook" class="Z3988"><span style="display:none;">&#160;</span></span><sup class="noprint Inline-Template" style="white-space:nowrap;">[<i><a href="/wiki/Wikipedia:Identifying_reliable_sources_(medicine)" title="Wikipedia:Identifying reliable sources (medicine)"><span title="Material near this tag may use an unreliable or less reliable medical source. (November 2013)">unreliable medical source?</span></a></i>]</sup></span></li>
<li id="cite_note-Weston_.26_Hood_2004-38"><span class="mw-cite-backlink">^ <a href="#cite_ref-Weston_.26_Hood_2004_38-0"><sup><i><b>a</b></i></sup></a> <a href="#cite_ref-Weston_.26_Hood_2004_38-1"><sup><i><b>b</b></i></sup></a> <a href="#cite_ref-Weston_.26_Hood_2004_38-2"><sup><i><b>c</b></i></sup></a> <a href="#cite_ref-Weston_.26_Hood_2004_38-3"><sup><i><b>d</b></i></sup></a> <a href="#cite_ref-Weston_.26_Hood_2004_38-4"><sup><i><b>e</b></i></sup></a> <a href="#cite_ref-Weston_.26_Hood_2004_38-5"><sup><i><b>f</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">Weston, Andrea D.; Hood, Leroy (2004). "Systems Biology, Proteomics, and the Future of Health Care: Toward Predictive, Preventative, and Personalized Medicine". <i>Journal of Proteome Research</i> <b>3</b> (2): 179–96. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1021%2Fpr0499693">10.1021/pr0499693</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/15113093">15113093</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Systems+Biology%2C+Proteomics%2C+and+the+Future+of+Health+Care%3A++Toward+Predictive%2C+Preventative%2C+and+Personalized+Medicine&amp;rft.aufirst=Andrea+D.&amp;rft.au=Hood%2C+Leroy&amp;rft.aulast=Weston&amp;rft.date=2004&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1021%2Fpr0499693&amp;rft_id=info%3Apmid%2F15113093&amp;rft.issue=2&amp;rft.jtitle=Journal+of+Proteome+Research&amp;rft.pages=179-96&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=3" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-39"><span class="mw-cite-backlink"><b><a href="#cite_ref-39">^</a></b></span> <span class="reference-text"><a rel="nofollow" class="external text" href="http://www.uniprot.org/">Uniprot</a></span></li>
<li id="cite_note-40"><span class="mw-cite-backlink"><b><a href="#cite_ref-40">^</a></b></span> <span class="reference-text"><a rel="nofollow" class="external text" href="http://prosite.expasy.org/">PROSITE</a></span></li>
<li id="cite_note-41"><span class="mw-cite-backlink"><b><a href="#cite_ref-41">^</a></b></span> <span class="reference-text"><cite class="citation journal">Wang H, Chu C, Wang W, Pai T; Chu; Wang; Pai (April 2014). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3992163">"A local average distance descriptor for flexible protein structure comparison"</a>. <i>BMC Bioinformatics</i> <b>15</b> (95): 1471–2105. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:10.1186/prca.200900109 (inactive 2015-02-01). <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3992163">3992163</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/24694083">24694083</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=A+local+average+distance+descriptor+for+flexible+protein+structure+comparison&amp;rft.au=Chu&amp;rft.au=Pai&amp;rft.au=Wang&amp;rft.au=Wang+H%2C+Chu+C%2C+Wang+W%2C+Pai+T&amp;rft.date=2014-04&amp;rft.genre=article&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3992163&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3992163&amp;rft_id=info%3Adoi%2F10.1186%2Fprca.200900109&amp;rft_id=info%3Apmid%2F24694083&amp;rft.issue=95&amp;rft.jtitle=BMC+Bioinformatics&amp;rft.pages=1471-2105&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=15" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-42"><span class="mw-cite-backlink"><b><a href="#cite_ref-42">^</a></b></span> <span class="reference-text"><cite class="citation journal">Petrov D, Margreitter C, Gandits M, Ostenbrink C, Zagrovic B; Margreitter; Grandits; Oostenbrink; Zagrovic (July 2013). "A systematic framework for molecular dynamics simulations of protein post-translational modifications". <i>PLoS Computational Biology</i> <b>9</b> (7): e1003154. <a href="/wiki/Bibcode" title="Bibcode">Bibcode</a>:<a rel="nofollow" class="external text" href="http://adsabs.harvard.edu/abs/2013PLSCB...9E3154P">2013PLSCB...9E3154P</a>. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1371%2Fjournal.pcbi.1003154">10.1371/journal.pcbi.1003154</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/23874192">23874192</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=A+systematic+framework+for+molecular+dynamics+simulations+of+protein+post-translational+modifications&amp;rft.au=Grandits&amp;rft.au=Margreitter&amp;rft.au=Oostenbrink&amp;rft.au=Petrov+D%2C+Margreitter+C%2C+Gandits+M%2C+Ostenbrink+C%2C+Zagrovic+B&amp;rft.au=Zagrovic&amp;rft.date=2013-07&amp;rft.genre=article&amp;rft_id=info%3Abibcode%2F2013PLSCB...9E3154P&amp;rft_id=info%3Adoi%2F10.1371%2Fjournal.pcbi.1003154&amp;rft_id=info%3Apmid%2F23874192&amp;rft.issue=7&amp;rft.jtitle=PLoS+Computational+Biology&amp;rft.pages=e1003154&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=9" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-43"><span class="mw-cite-backlink"><b><a href="#cite_ref-43">^</a></b></span> <span class="reference-text"><cite class="citation journal">Margreitter C, Petro D, Zagrovic B; Petrov; Zagrovic (May 2013). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3692090">"Vienna-PTM web server: a toolkit for MD simulations of portein post-translational modifications"</a>. <i>Nucl. Acids Res.</i> <b>41</b> (Web Server issue): W422–6. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1093%2Fnar%2Fgkt416">10.1093/nar/gkt416</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC3692090">3692090</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/23703210">23703210</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Vienna-PTM+web+server%3A+a+toolkit+for+MD+simulations+of+portein+post-translational+modifications&amp;rft.au=Margreitter+C%2C+Petro+D%2C+Zagrovic+B&amp;rft.au=Petrov&amp;rft.au=Zagrovic&amp;rft.date=2013-05&amp;rft.genre=article&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3692090&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3692090&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkt416&amp;rft_id=info%3Apmid%2F23703210&amp;rft.issue=Web+Server+issue&amp;rft.jtitle=Nucl.+Acids+Res.&amp;rft.pages=W422-6&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=41" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-44"><span class="mw-cite-backlink"><b><a href="#cite_ref-44">^</a></b></span> <span class="reference-text"><cite class="citation journal">Maron JL, Alterovitz G, Ramoni M, Johnson KL, Bianchi DW; Alterovitz; Ramoni; Johnson; Bianchi (December 2009). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2825712">"High-throughput discovery and characterization of fetal protein trafficking in the blood of pregnant women"</a>. <i>Proteomics: Clinical Applications</i> <b>3</b> (12): 1389–96. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1002%2Fprca.200900109">10.1002/prca.200900109</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2825712">2825712</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/20186258">20186258</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=High-throughput+discovery+and+characterization+of+fetal+protein+trafficking+in+the+blood+of+pregnant+women&amp;rft.au=Alterovitz&amp;rft.au=Bianchi&amp;rft.au=Johnson&amp;rft.au=Maron+JL%2C+Alterovitz+G%2C+Ramoni+M%2C+Johnson+KL%2C+Bianchi+DW&amp;rft.au=Ramoni&amp;rft.date=2009-12&amp;rft.genre=article&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2825712&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2825712&amp;rft_id=info%3Adoi%2F10.1002%2Fprca.200900109&amp;rft_id=info%3Apmid%2F20186258&amp;rft.issue=12&amp;rft.jtitle=Proteomics%3A+Clinical+Applications&amp;rft.pages=1389-96&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=3" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
<li id="cite_note-45"><span class="mw-cite-backlink"><b><a href="#cite_ref-45">^</a></b></span> <span class="reference-text"><cite class="citation journal">Alterovitz G, Xiang M, Liu J, Chang A, Ramoni MF; Xiang; Liu; Chang; Ramoni (2008). <a rel="nofollow" class="external text" href="http://psb.stanford.edu/psb-online/proceedings/psb08/abstracts/2008_p231.html">"System-wide peripheral biomarker discovery using information theory"</a>. <i>Pacific Symposium on Biocomputing</i>: 231–42. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1142%2F9789812776136_0024">10.1142/9789812776136_0024</a>. <a href="/wiki/International_Standard_Book_Number" title="International Standard Book Number">ISBN</a>&#160;<a href="/wiki/Special:BookSources/9789812776082" title="Special:BookSources/9789812776082">9789812776082</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/18229689">18229689</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=System-wide+peripheral+biomarker+discovery+using+information+theory&amp;rft.au=Alterovitz+G%2C+Xiang+M%2C+Liu+J%2C+Chang+A%2C+Ramoni+MF&amp;rft.au=Chang&amp;rft.au=Liu&amp;rft.au=Ramoni&amp;rft.au=Xiang&amp;rft.date=2008&amp;rft.genre=article&amp;rft_id=http%3A%2F%2Fpsb.stanford.edu%2Fpsb-online%2Fproceedings%2Fpsb08%2Fabstracts%2F2008_p231.html&amp;rft_id=info%3Adoi%2F10.1142%2F9789812776136_0024&amp;rft_id=info%3Apmid%2F18229689&amp;rft.isbn=9789812776082&amp;rft.jtitle=Pacific+Symposium+on+Biocomputing&amp;rft.pages=231-42&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&#160;</span></span></span></li>
</ol>
</div>
<h2><span class="mw-headline" id="Bibliography">Bibliography</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=37" title="Edit section: Bibliography">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div class="refbegin columns references-column-count references-column-count-2" style="-moz-column-count: 2; -webkit-column-count: 2; column-count: 2;">
<ul>
<li>Ceciliani F, Eckersall D, Burchmore R, Lecchi C. <a rel="nofollow" class="external text" href="http://vet.sagepub.com/content/51/2/351.long">Proteomics in veterinary medicine: applications and trends in disease pathogenesis and diagnostics</a>. Vet Pathol. 2014 Mar;51(2):351-62.</li>
<li>Belhajjame, K. et al. <a rel="nofollow" class="external text" href="http://www.allhands.org.uk/2005/proceedings/papers/525.pdf">Proteome Data Integration: Characteristics and Challenges</a>. Proceedings of the UK e-Science All Hands Meeting, <a href="/wiki/Special:BookSources/1904425534" class="internal mw-magiclink-isbn">ISBN 1-904425-53-4</a>, September 2005, Nottingham, UK.</li>
<li><cite class="citation book">Twyman RM (2004). <i>Principles Of Proteomics (Advanced Text Series)</i>. Oxford, UK: BIOS Scientific Publishers. <a href="/wiki/International_Standard_Book_Number" title="International Standard Book Number">ISBN</a>&#160;<a href="/wiki/Special:BookSources/1-85996-273-4" title="Special:BookSources/1-85996-273-4">1-85996-273-4</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.au=Twyman+RM&amp;rft.btitle=Principles+Of+Proteomics+%28Advanced+Text+Series%29&amp;rft.date=2004&amp;rft.genre=book&amp;rft.isbn=1-85996-273-4&amp;rft.place=Oxford%2C+UK&amp;rft.pub=BIOS+Scientific+Publishers&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook" class="Z3988"><span style="display:none;">&#160;</span></span> (covers almost all branches of proteomics)</li>
<li><cite class="citation book">Naven T, Westermeier R (2002). <i>Proteomics in Practice: A Laboratory Manual of Proteome Analysis</i>. Weinheim: Wiley-VCH. <a href="/wiki/International_Standard_Book_Number" title="International Standard Book Number">ISBN</a>&#160;<a href="/wiki/Special:BookSources/3-527-30354-5" title="Special:BookSources/3-527-30354-5">3-527-30354-5</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.au=Naven+T%2C+Westermeier+R&amp;rft.btitle=Proteomics+in+Practice%3A+A+Laboratory+Manual+of+Proteome+Analysis&amp;rft.date=2002&amp;rft.genre=book&amp;rft.isbn=3-527-30354-5&amp;rft.place=Weinheim&amp;rft.pub=Wiley-VCH&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook" class="Z3988"><span style="display:none;">&#160;</span></span> (focused on 2D-gels, good on detail)</li>
<li><cite class="citation book">Liebler DC (2002). <i>Introduction to proteomics: tools for the new biology</i>. Totowa, NJ: Humana Press. <a href="/wiki/International_Standard_Book_Number" title="International Standard Book Number">ISBN</a>&#160;<a href="/wiki/Special:BookSources/0-89603-992-7" title="Special:BookSources/0-89603-992-7">0-89603-992-7</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.au=Liebler+DC&amp;rft.btitle=Introduction+to+proteomics%3A+tools+for+the+new+biology&amp;rft.date=2002&amp;rft.genre=book&amp;rft.isbn=0-89603-992-7&amp;rft.place=Totowa%2C+NJ&amp;rft.pub=Humana+Press&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook" class="Z3988"><span style="display:none;">&#160;</span></span> <a href="/wiki/Special:BookSources/0585418799" class="internal mw-magiclink-isbn">ISBN 0-585-41879-9</a> (electronic, on Netlibrary?), <a href="/wiki/Special:BookSources/0896039919" class="internal mw-magiclink-isbn">ISBN 0-89603-991-9</a> hbk</li>
<li><cite class="citation book">Wilkins MR, Williams KL, Appel RD, Hochstrasser DF (1997). <i>Proteome Research: New Frontiers in Functional Genomics (Principles and Practice)</i>. Berlin: Springer. <a href="/wiki/International_Standard_Book_Number" title="International Standard Book Number">ISBN</a>&#160;<a href="/wiki/Special:BookSources/3-540-62753-7" title="Special:BookSources/3-540-62753-7">3-540-62753-7</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.au=Wilkins+MR%2C+Williams+KL%2C+Appel+RD%2C+Hochstrasser+DF&amp;rft.btitle=Proteome+Research%3A+New+Frontiers+in+Functional+Genomics+%28Principles+and+Practice%29&amp;rft.date=1997&amp;rft.genre=book&amp;rft.isbn=3-540-62753-7&amp;rft.place=Berlin&amp;rft.pub=Springer&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook" class="Z3988"><span style="display:none;">&#160;</span></span></li>
<li><cite class="citation journal">Arora PS, Yamagiwa H, Srivastava A, Bolander ME, Sarkar G; Yamagiwa; Srivastava; Bolander; Sarkar (2005). "Comparative evaluation of two two-dimensional gel electrophoresis image analysis software applications using synovial fluids from patients with joint disease". <i>J Orthop Sci</i> <b>10</b> (2): 160–6. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1007%2Fs00776-004-0878-0">10.1007/s00776-004-0878-0</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/15815863">15815863</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Comparative+evaluation+of+two+two-dimensional+gel+electrophoresis+image+analysis+software+applications+using+synovial+fluids+from+patients+with+joint+disease&amp;rft.au=Arora+PS%2C+Yamagiwa+H%2C+Srivastava+A%2C+Bolander+ME%2C+Sarkar+G&amp;rft.au=Bolander&amp;rft.au=Sarkar&amp;rft.au=Srivastava&amp;rft.au=Yamagiwa&amp;rft.date=2005&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1007%2Fs00776-004-0878-0&amp;rft_id=info%3Apmid%2F15815863&amp;rft.issue=2&amp;rft.jtitle=J+Orthop+Sci&amp;rft.pages=160-6&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=10" class="Z3988"><span style="display:none;">&#160;</span></span></li>
<li>Rediscovering Biology Online Textbook. Unit 2 Proteins and Proteomics. 1997–2006.</li>
<li><cite class="citation book">Weaver RF (2005). <i>Molecular biology</i> (3rd ed.). New York: McGraw-Hill. pp.&#160;840–9. <a href="/wiki/International_Standard_Book_Number" title="International Standard Book Number">ISBN</a>&#160;<a href="/wiki/Special:BookSources/0-07-284611-9" title="Special:BookSources/0-07-284611-9">0-07-284611-9</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.au=Weaver+RF&amp;rft.btitle=Molecular+biology&amp;rft.date=2005&amp;rft.edition=3rd&amp;rft.genre=book&amp;rft.isbn=0-07-284611-9&amp;rft.pages=840-9&amp;rft.place=New+York&amp;rft.pub=McGraw-Hill&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook" class="Z3988"><span style="display:none;">&#160;</span></span></li>
<li><cite class="citation book">Reece J, Campbell N (2002). <i>Biology</i> (6th ed.). San Francisco: Benjamin Cummings. pp.&#160;392–3. <a href="/wiki/International_Standard_Book_Number" title="International Standard Book Number">ISBN</a>&#160;<a href="/wiki/Special:BookSources/0-8053-6624-5" title="Special:BookSources/0-8053-6624-5">0-8053-6624-5</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.au=Reece+J%2C+Campbell+N&amp;rft.btitle=Biology&amp;rft.date=2002&amp;rft.edition=6th&amp;rft.genre=book&amp;rft.isbn=0-8053-6624-5&amp;rft.pages=392-3&amp;rft.place=San+Francisco&amp;rft.pub=Benjamin+Cummings&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook" class="Z3988"><span style="display:none;">&#160;</span></span></li>
<li><cite class="citation journal">Hye A, Lynham S, Thambisetty M, et al. (November 2006). "Proteome-based plasma biomarkers for Alzheimer's disease". <i>Brain</i> <b>129</b> (Pt 11): 3042–50. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1093%2Fbrain%2Fawl279">10.1093/brain/awl279</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/17071923">17071923</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Proteome-based+plasma+biomarkers+for+Alzheimer%27s+disease&amp;rft.au=Byers%2C+H.+L.&amp;rft.au=Campbell%2C+J.&amp;rft.au=Causevic%2C+M.&amp;rft.au=Hooper%2C+C.&amp;rft.au=Hye+A&amp;rft.au=Lynham+S&amp;rft.au=Rijsdijk%2C+F.&amp;rft.au=Tabrizi%2C+S.+J.&amp;rft.au=Thambisetty+M&amp;rft.date=2006-11&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1093%2Fbrain%2Fawl279&amp;rft_id=info%3Apmid%2F17071923&amp;rft.issue=Pt+11&amp;rft.jtitle=Brain&amp;rft.pages=3042-50&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=129" class="Z3988"><span style="display:none;">&#160;</span></span> <span style="font-size:100%" class="error citation-comment"><code style="color:inherit; border:inherit; padding:inherit;">|first10=</code> missing <code style="color:inherit; border:inherit; padding:inherit;">|last10=</code> in Authors list (<a href="/wiki/Help:CS1_errors#first_missing_last" title="Help:CS1 errors">help</a>);</span> <span style="font-size:100%" class="error citation-comment"><code style="color:inherit; border:inherit; padding:inherit;">|first11=</code> missing <code style="color:inherit; border:inherit; padding:inherit;">|last11=</code> in Authors list (<a href="/wiki/Help:CS1_errors#first_missing_last" title="Help:CS1 errors">help</a>)</span></li>
<li><cite class="citation journal">Perroud B, Lee J, Valkova N, et al. (2006). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1665458">"Pathway analysis of kidney cancer using proteomics and metabolic profiling"</a>. <i>Mol Cancer</i> <b>5</b>: 64. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1186%2F1476-4598-5-64">10.1186/1476-4598-5-64</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1665458">1665458</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/17123452">17123452</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Pathway+analysis+of+kidney+cancer+using+proteomics+and+metabolic+profiling&amp;rft.au=Dhirapong%2C+Amy&amp;rft.au=Fiehn%2C+Oliver&amp;rft.au=K%C3%BCltz%2C+Dietmar&amp;rft.au=Lee+J&amp;rft.au=Lin%2C+Pei-Yin&amp;rft.au=Perroud+B&amp;rft.au=Valkova+N&amp;rft.au=Weiss%2C+Robert+H&amp;rft.date=2006&amp;rft.genre=article&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1665458&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1665458&amp;rft_id=info%3Adoi%2F10.1186%2F1476-4598-5-64&amp;rft_id=info%3Apmid%2F17123452&amp;rft.jtitle=Mol+Cancer&amp;rft.pages=64&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=5" class="Z3988"><span style="display:none;">&#160;</span></span></li>
<li><cite class="citation journal">Yohannes E, Chang J, Christ GJ, Davies KP, Chance MR; Chang; Christ; Davies; Chance (July 2008). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2493381">"Proteomics analysis identifies molecular targets related to diabetes mellitus-associated bladder dysfunction"</a>. <i>Mol. Cell Proteomics</i> <b>7</b> (7): 1270–85. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1074%2Fmcp.M700563-MCP200">10.1074/mcp.M700563-MCP200</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC2493381">2493381</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/18337374">18337374</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Proteomics+analysis+identifies+molecular+targets+related+to+diabetes+mellitus-associated+bladder+dysfunction&amp;rft.au=Chance&amp;rft.au=Chang&amp;rft.au=Christ&amp;rft.au=Davies&amp;rft.au=Yohannes+E%2C+Chang+J%2C+Christ+GJ%2C+Davies+KP%2C+Chance+MR&amp;rft.date=2008-07&amp;rft.genre=article&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2493381&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2493381&amp;rft_id=info%3Adoi%2F10.1074%2Fmcp.M700563-MCP200&amp;rft_id=info%3Apmid%2F18337374&amp;rft.issue=7&amp;rft.jtitle=Mol.+Cell+Proteomics&amp;rft.pages=1270-85&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=7" class="Z3988"><span style="display:none;">&#160;</span></span></li>
<li><cite class="citation journal">Macaulay IC, Carr P, Gusnanto A, Ouwehand WH, Fitzgerald D, Watkins NA; Carr; Gusnanto; Ouwehand; Fitzgerald; Watkins (December 2005). <a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1297260">"Platelet genomics and proteomics in human health and disease"</a>. <i>J Clin Invest.</i> <b>115</b> (12): 3370–7. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1172%2FJCI26885">10.1172/JCI26885</a>. <a href="/wiki/PubMed_Central" title="PubMed Central">PMC</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pmc/articles/PMC1297260">1297260</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/16322782">16322782</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Platelet+genomics+and+proteomics+in+human+health+and+disease&amp;rft.au=Carr&amp;rft.au=Fitzgerald&amp;rft.au=Gusnanto&amp;rft.au=Macaulay+IC%2C+Carr+P%2C+Gusnanto+A%2C+Ouwehand+WH%2C+Fitzgerald+D%2C+Watkins+NA&amp;rft.au=Ouwehand&amp;rft.au=Watkins&amp;rft.date=2005-12&amp;rft.genre=article&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1297260&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1297260&amp;rft_id=info%3Adoi%2F10.1172%2FJCI26885&amp;rft_id=info%3Apmid%2F16322782&amp;rft.issue=12&amp;rft.jtitle=J+Clin+Invest.&amp;rft.pages=3370-7&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=115" class="Z3988"><span style="display:none;">&#160;</span></span></li>
<li><cite class="citation journal">Rogers MA, Clarke P, Noble J, et al. (15 October 2003). <a rel="nofollow" class="external text" href="http://cancerres.aacrjournals.org/cgi/pmidlookup?view=long&amp;pmid=14583499">"Proteomic profiling of urinary proteins in renal cancer by surface enhanced laser desorption ionization and neural-network analysis: identification of key issues affecting potential clinical utility"</a>. <i>Cancer Res.</i> <b>63</b> (20): 6971–83. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/14583499">14583499</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Proteomic+profiling+of+urinary+proteins+in+renal+cancer+by+surface+enhanced+laser+desorption+ionization+and+neural-network+analysis%3A+identification+of+key+issues+affecting+potential+clinical+utility&amp;rft.au=Banks%2C+RE&amp;rft.au=Clarke+P&amp;rft.au=Munro%2C+NP&amp;rft.au=Noble+J&amp;rft.au=Paul%2C+A&amp;rft.au=Rogers+MA&amp;rft.au=Selby%2C+PJ&amp;rft.date=2003-10-15&amp;rft.genre=article&amp;rft_id=http%3A%2F%2Fcancerres.aacrjournals.org%2Fcgi%2Fpmidlookup%3Fview%3Dlong%26pmid%3D14583499&amp;rft_id=info%3Apmid%2F14583499&amp;rft.issue=20&amp;rft.jtitle=Cancer+Res.&amp;rft.pages=6971-83&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=63" class="Z3988"><span style="display:none;">&#160;</span></span></li>
<li><cite class="citation journal">Vasan RS (May 2006). "Biomarkers of cardiovascular disease: molecular basis and practical considerations". <i>Circulation</i> <b>113</b> (19): 2335–62. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1161%2FCIRCULATIONAHA.104.482570">10.1161/CIRCULATIONAHA.104.482570</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/16702488">16702488</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Biomarkers+of+cardiovascular+disease%3A+molecular+basis+and+practical+considerations&amp;rft.au=Vasan+RS&amp;rft.date=2006-05&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1161%2FCIRCULATIONAHA.104.482570&amp;rft_id=info%3Apmid%2F16702488&amp;rft.issue=19&amp;rft.jtitle=Circulation&amp;rft.pages=2335-62&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=113" class="Z3988"><span style="display:none;">&#160;</span></span></li>
<li><a rel="nofollow" class="external text" href="http://medlib.med.utah.edu/WebPath/TUTORIAL/MYOCARD/MYOCARD.html">"Myocardial Infarction"</a>. (Retrieved 29 November 2006)</li>
<li>Introduction to Antibodies&#160;– <a rel="nofollow" class="external text" href="http://www.chemicon.com/resource/ANT101/a2C.asp">Enzyme-Linked Immunosorbent Assay (ELISA)</a>. (Retrieved 29 November 2006)</li>
<li><cite class="citation journal">Decramer, Stephane; Wittke, Stefan; Mischak, Harald; Zürbig, Petra; Walden, Michael; Bouissou, François; Bascands, Jean-Loup; Schanstra, Joost P (2006). "Predicting the clinical outcome of congenital unilateral ureteropelvic junction obstruction in newborn by urinary proteome analysis". <i>Nature Medicine</i> <b>12</b> (4): 398–400. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1038%2Fnm1384">10.1038/nm1384</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/16550189">16550189</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Predicting+the+clinical+outcome+of+congenital+unilateral+ureteropelvic+junction+obstruction+in+newborn+by+urinary+proteome+analysis&amp;rft.au=Bascands%2C+Jean-Loup&amp;rft.au=Bouissou%2C+Fran%C3%A7ois&amp;rft.aufirst=Stephane&amp;rft.aulast=Decramer&amp;rft.au=Mischak%2C+Harald&amp;rft.au=Schanstra%2C+Joost+P&amp;rft.au=Walden%2C+Michael&amp;rft.au=Wittke%2C+Stefan&amp;rft.au=Z%C3%BCrbig%2C+Petra&amp;rft.date=2006&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1038%2Fnm1384&amp;rft_id=info%3Apmid%2F16550189&amp;rft.issue=4&amp;rft.jtitle=Nature+Medicine&amp;rft.pages=398-400&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=12" class="Z3988"><span style="display:none;">&#160;</span></span></li>
<li><cite class="citation journal">Mayer U (January 2008). "Protein Information Crawler (PIC): extensive spidering of multiple protein information resources for large protein sets". <i>Proteomics</i> <b>8</b> (1): 42–4. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1002%2Fpmic.200700865">10.1002/pmic.200700865</a>. <a href="/wiki/PubMed_Identifier" title="PubMed Identifier" class="mw-redirect">PMID</a>&#160;<a rel="nofollow" class="external text" href="//www.ncbi.nlm.nih.gov/pubmed/18095364">18095364</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Protein+Information+Crawler+%28PIC%29%3A+extensive+spidering+of+multiple+protein+information+resources+for+large+protein+sets&amp;rft.au=Mayer+U&amp;rft.date=2008-01&amp;rft.genre=article&amp;rft_id=info%3Adoi%2F10.1002%2Fpmic.200700865&amp;rft_id=info%3Apmid%2F18095364&amp;rft.issue=1&amp;rft.jtitle=Proteomics&amp;rft.pages=42-4&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=8" class="Z3988"><span style="display:none;">&#160;</span></span></li>
<li>Jörg von Hagen, VCH-Wiley 2008 <i>Proteomics Sample Preparation. <a href="/wiki/Special:BookSources/9783527317967" class="internal mw-magiclink-isbn">ISBN 978-3-527-31796-7</a></i></li>
<li><cite id="CITEREFAlinejad2015" class="citation journal">Alinejad, Tahereh (September 2015). <a rel="nofollow" class="external text" href="http://www.sciencedirect.com/science/article/pii/S2214540015000274">"Proteomic analysis of differentially expressed protein in hemocytes of wild giant freshwater prawn Macrobrachium rosenbergii infected with infectious hypodermal and hematopoietic necrosis virus (IHHNV)"</a>. <i>meta gene</i> <b>5</b>: Pages 55–67. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="//dx.doi.org/10.1016%2Fj.mgene.2015.05.004">10.1016/j.mgene.2015.05.004</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AProteomics&amp;rft.atitle=Proteomic+analysis+of+differentially+expressed+protein+in+hemocytes+of+wild+giant+freshwater+prawn+Macrobrachium+rosenbergii+infected+with+infectious+hypodermal+and+hematopoietic+necrosis+virus+%28IHHNV%29&amp;rft.aufirst=Tahereh&amp;rft.aulast=Alinejad&amp;rft.date=2015-09&amp;rft.genre=article&amp;rft_id=http%3A%2F%2Fwww.sciencedirect.com%2Fscience%2Farticle%2Fpii%2FS2214540015000274&amp;rft_id=info%3Adoi%2F10.1016%2Fj.mgene.2015.05.004&amp;rft.jtitle=meta+gene&amp;rft.pages=Pages+55-67&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.volume=5" class="Z3988"><span style="display:none;">&#160;</span></span></li>
</ul>
</div>
<h2><span class="mw-headline" id="External_links">External links</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=38" title="Edit section: External links">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<table class="mbox-small plainlinks sistersitebox" style="border:1px solid #aaa;background-color:#f9f9f9">
<tr>
<td class="mbox-image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/thumb/f/f8/Wiktionary-logo-en.svg/37px-Wiktionary-logo-en.svg.png" width="37" height="40" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/f/f8/Wiktionary-logo-en.svg/55px-Wiktionary-logo-en.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/f/f8/Wiktionary-logo-en.svg/73px-Wiktionary-logo-en.svg.png 2x" data-file-width="1000" data-file-height="1089" /></td>
<td class="mbox-text plainlist">Look up <i><b><a href="//en.wiktionary.org/wiki/Special:Search/proteomics" class="extiw" title="wiktionary:Special:Search/proteomics">proteomics</a></b></i> in Wiktionary, the free dictionary.</td>
</tr>
</table>
<table class="mbox-small plainlinks sistersitebox" style="border:1px solid #aaa;background-color:#f9f9f9">
<tr>
<td class="mbox-image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/thumb/d/df/Wikibooks-logo-en-noslogan.svg/40px-Wikibooks-logo-en-noslogan.svg.png" width="40" height="40" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/d/df/Wikibooks-logo-en-noslogan.svg/60px-Wikibooks-logo-en-noslogan.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/d/df/Wikibooks-logo-en-noslogan.svg/80px-Wikibooks-logo-en-noslogan.svg.png 2x" data-file-width="400" data-file-height="400" /></td>
<td class="mbox-text plainlist">Wikibooks has more on the topic of: <i><b><a href="//en.wikibooks.org/wiki/Special:Search/Proteomics" class="extiw" title="wikibooks:Special:Search/Proteomics">Proteomics</a></b></i></td>
</tr>
</table>
<table class="mbox-small plainlinks sistersitebox" style="border:1px solid #aaa;background-color:#f9f9f9">
<tr>
<td class="mbox-image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/thumb/9/91/Wikiversity-logo.svg/40px-Wikiversity-logo.svg.png" width="40" height="32" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/9/91/Wikiversity-logo.svg/60px-Wikiversity-logo.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/9/91/Wikiversity-logo.svg/80px-Wikiversity-logo.svg.png 2x" data-file-width="1000" data-file-height="800" /></td>
<td class="mbox-text plainlist">At <a href="/wiki/Wikiversity" title="Wikiversity">Wikiversity</a>, you can learn more and teach others about <b>Proteomics</b> at the <a href="//en.wikiversity.org/wiki/Topic:Proteomics" class="extiw" title="v:Topic:Proteomics">Department of Proteomics</a></td>
</tr>
</table>
<ul>
<li><a rel="nofollow" class="external text" href="https://www.dmoz.org//Science/Biology/Biochemistry_and_Molecular_Biology/Biomolecules/Proteins_and_Enzymes/Proteomics">Proteomics</a> at <a href="/wiki/DMOZ" title="DMOZ">DMOZ</a></li>
</ul>
<table class="navbox" style="border-spacing:0">
<tr>
<td style="padding:2px">
<table class="nowraplinks collapsible autocollapse navbox-inner" style="border-spacing:0;background:transparent;color:inherit">
<tr>
<th scope="col" class="navbox-title" colspan="2">
<div class="plainlinks hlist navbar mini">
<ul>
<li class="nv-view"><a href="/wiki/Template:Genomics" title="Template:Genomics"><abbr title="View this template" style=";;background:none transparent;border:none;">v</abbr></a></li>
<li class="nv-talk"><a href="/wiki/Template_talk:Genomics" title="Template talk:Genomics"><abbr title="Discuss this template" style=";;background:none transparent;border:none;">t</abbr></a></li>
<li class="nv-edit"><a class="external text" href="//en.wikipedia.org/w/index.php?title=Template:Genomics&amp;action=edit"><abbr title="Edit this template" style=";;background:none transparent;border:none;">e</abbr></a></li>
</ul>
</div>
<div style="font-size:114%"><a href="/wiki/Omics" title="Omics">Omics</a></div>
</th>
</tr>
<tr style="height:2px">
<td colspan="2"></td>
</tr>
<tr>
<th scope="row" class="navbox-group"><a href="/wiki/Genomics" title="Genomics">Genomics</a></th>
<td class="navbox-list navbox-odd hlist" style="text-align:left;border-left-width:2px;border-left-style:solid;width:100%;padding:0px">
<div style="padding:0em 0.25em">
<ul>
<li><a href="/wiki/Cognitive_genomics" title="Cognitive genomics">Cognitive genomics</a></li>
<li><a href="/wiki/Computational_genomics" title="Computational genomics">Computational genomics</a></li>
<li><a href="/wiki/Comparative_genomics" title="Comparative genomics">Comparative genomics</a></li>
<li><a href="/wiki/Functional_genomics" title="Functional genomics">Functional genomics</a></li>
<li><a href="/wiki/Genome_project" title="Genome project">Genome project</a>
<ul>
<li><a href="/wiki/Human_Genome_Project" title="Human Genome Project">Human Genome Project</a></li>
</ul>
</li>
<li><a href="/wiki/Metagenomics" title="Metagenomics">Metagenomics</a></li>
<li><a href="/wiki/Personal_genomics" title="Personal genomics">Personal genomics</a></li>
<li><a href="/wiki/Social_genomics" title="Social genomics">Social genomics</a></li>
<li><a href="/wiki/Structural_genomics" title="Structural genomics">Structural genomics</a></li>
</ul>
</div>
</td>
</tr>
<tr style="height:2px">
<td colspan="2"></td>
</tr>
<tr>
<th scope="row" class="navbox-group"><a href="/wiki/Bioinformatics" title="Bioinformatics">Bioinformatics</a></th>
<td class="navbox-list navbox-even hlist" style="text-align:left;border-left-width:2px;border-left-style:solid;width:100%;padding:0px">
<div style="padding:0em 0.25em">
<ul>
<li><a href="/wiki/Biochip" title="Biochip">Biochip</a></li>
<li><a href="/wiki/Cheminformatics" title="Cheminformatics">Cheminformatics</a></li>
<li><a href="/wiki/Chemogenomics" title="Chemogenomics">Chemogenomics</a></li>
<li><a href="/wiki/Connectomics" title="Connectomics">Connectomics</a></li>
<li><a href="/wiki/Glycomics" title="Glycomics">Glycomics</a></li>
<li><a href="/wiki/Immunomics" title="Immunomics">Immunomics</a></li>
<li><a href="/wiki/Lipidomics" title="Lipidomics">Lipidomics</a></li>
<li><a href="/wiki/Metabolomics" title="Metabolomics">Metabolomics</a></li>
<li><a href="/wiki/Microbiome" title="Microbiome" class="mw-redirect">Microbiomics</a></li>
<li><a href="/wiki/Nutrigenomics" title="Nutrigenomics">Nutrigenomics</a></li>
<li><a href="/wiki/Paleopolyploidy" title="Paleopolyploidy">Paleopolyploidy</a></li>
<li><a href="/wiki/Pharmacogenetics" title="Pharmacogenetics">Pharmacogenetics</a></li>
<li><a href="/wiki/Pharmacogenomics" title="Pharmacogenomics">Pharmacogenomics</a></li>
<li><a href="/wiki/Systems_biology" title="Systems biology">Systems biology</a></li>
<li><a href="/wiki/Toxicogenomics" title="Toxicogenomics">Toxicogenomics</a></li>
<li><a href="/wiki/Transcriptomics" title="Transcriptomics" class="mw-redirect">Transcriptomics</a></li>
</ul>
</div>
</td>
</tr>
<tr style="height:2px">
<td colspan="2"></td>
</tr>
<tr>
<th scope="row" class="navbox-group"><a href="/wiki/Structural_biology" title="Structural biology">Structural biology</a></th>
<td class="navbox-list navbox-odd hlist" style="text-align:left;border-left-width:2px;border-left-style:solid;width:100%;padding:0px">
<div style="padding:0em 0.25em">
<ul>
<li><strong class="selflink">Proteomics</strong>
<ul>
<li><a href="/wiki/Human_Proteome_Project" title="Human Proteome Project" class="mw-redirect">Human Proteome Project</a></li>
</ul>
</li>
<li><a href="/w/index.php?title=Call-map_proteomics&amp;action=edit&amp;redlink=1" class="new" title="Call-map proteomics (page does not exist)">call-map proteomics</a></li>
<li><a href="/wiki/Structure-based_drug_design" title="Structure-based drug design" class="mw-redirect">structure-based drug design</a></li>
<li><a href="/w/index.php?title=Expression_proteomics&amp;action=edit&amp;redlink=1" class="new" title="Expression proteomics (page does not exist)">expression proteomics</a></li>
</ul>
</div>
</td>
</tr>
<tr style="height:2px">
<td colspan="2"></td>
</tr>
<tr>
<th scope="row" class="navbox-group">Research tools</th>
<td class="navbox-list navbox-even hlist" style="text-align:left;border-left-width:2px;border-left-style:solid;width:100%;padding:0px">
<div style="padding:0em 0.25em">
<ul>
<li><a href="/wiki/2-D_electrophoresis" title="2-D electrophoresis" class="mw-redirect">2-D electrophoresis</a></li>
<li><a href="/wiki/Mass_spectrometer" title="Mass spectrometer" class="mw-redirect">mass spectrometer</a></li>
<li><a href="/wiki/Electrospray_ionization" title="Electrospray ionization">electrospray ionization</a></li>
<li><a href="/wiki/Matrix-assisted_laser_desorption_ionization" title="Matrix-assisted laser desorption ionization" class="mw-redirect">matrix-assisted laser desorption ionization</a></li>
<li><a href="/w/index.php?title=Matrix-assisted_laser_desorption_ionization-time_of_flight_mass_spectrometer&amp;action=edit&amp;redlink=1" class="new" title="Matrix-assisted laser desorption ionization-time of flight mass spectrometer (page does not exist)">matrix-assisted laser desorption ionization-time of flight mass spectrometer</a></li>
<li><a href="/w/index.php?title=Spotted_array-based_tools&amp;action=edit&amp;redlink=1" class="new" title="Spotted array-based tools (page does not exist)">spotted array-based tools</a></li>
<li><a href="/w/index.php?title=Microfluidic-based_tools&amp;action=edit&amp;redlink=1" class="new" title="Microfluidic-based tools (page does not exist)">microfluidic-based tools</a></li>
<li><a href="/w/index.php?title=Isotope_affinity_tags&amp;action=edit&amp;redlink=1" class="new" title="Isotope affinity tags (page does not exist)">isotope affinity tags</a></li>
<li><a href="/w/index.php?title=Molecular_scanner&amp;action=edit&amp;redlink=1" class="new" title="Molecular scanner (page does not exist)">molecular scanner</a></li>
</ul>
</div>
</td>
</tr>
<tr style="height:2px">
<td colspan="2"></td>
</tr>
<tr>
<th scope="row" class="navbox-group">Organizations</th>
<td class="navbox-list navbox-odd hlist" style="text-align:left;border-left-width:2px;border-left-style:solid;width:100%;padding:0px">
<div style="padding:0em 0.25em">
<ul>
<li><a href="/wiki/National_Institutes_of_Health" title="National Institutes of Health">National Institutes of Health</a> (USA)</li>
<li><a href="/wiki/DNA_Data_Bank_of_Japan" title="DNA Data Bank of Japan">DNA Data Bank of Japan</a>(JP)</li>
<li><a href="/wiki/European_Molecular_Biology_Laboratory" title="European Molecular Biology Laboratory">European Molecular Biology Laboratory</a>(EU)</li>
<li><a href="/wiki/Sanger_Centre" title="Sanger Centre" class="mw-redirect">Sanger Centre</a>(EN)</li>
</ul>
</div>
</td>
</tr>
</table>
</td>
</tr>
</table>
<table class="navbox" style="border-spacing:0">
<tr>
<td style="padding:2px">
<table class="nowraplinks hlist navbox-inner" style="border-spacing:0;background:transparent;color:inherit">
<tr>
<th scope="row" class="navbox-group"><a href="/wiki/Help:Authority_control" title="Help:Authority control">Authority control</a></th>
<td class="navbox-list navbox-odd" style="text-align:left;border-left-width:2px;border-left-style:solid;width:100%;padding:0px">
<div style="padding:0em 0.25em">
<ul>
<li><a href="/wiki/National_Diet_Library" title="National Diet Library">NDL</a>: <span class="uid"><a rel="nofollow" class="external text" href="http://id.ndl.go.jp/auth/ndlna/00933684">00933684</a></span></li>
</ul>
</div>
</td>
</tr>
</table>
</td>
</tr>
</table>


<!-- 
NewPP limit report
Parsed by mw1075
Cached time: 20160220200553
Cache expiry: 2592000
Dynamic content: false
CPU time usage: 1.008 seconds
Real time usage: 1.112 seconds
Preprocessor visited node count: 4356/1000000
Preprocessor generated node count: 0/1500000
Post‐expand include size: 149125/2097152 bytes
Template argument size: 2736/2097152 bytes
Highest expansion depth: 12/40
Expensive parser function count: 4/500
Lua time usage: 0.493/10.000 seconds
Lua memory usage: 4.61 MB/50 MB
Number of Wikibase entities loaded: 1-->

<!-- 
Transclusion expansion time report (%,ms,calls,template)
100.00%  923.402      1 - -total
 48.78%  450.419      1 - Template:Reflist
 42.97%  396.749     48 - Template:Cite_journal
  7.12%   65.727      3 - Template:Citation_needed
  6.30%   58.169      4 - Template:Fix
  5.50%   50.794      7 - Template:Category_handler
  3.87%   35.702      1 - Template:Authority_control
  3.67%   33.927      6 - Template:Cite_book
  3.19%   29.411      1 - Template:For
  2.86%   26.399      1 - Template:Hatnote
-->

<!-- Saved in parser cache with key enwiki:pcache:idhash:55172-0!*!0!!en!4!* and timestamp 20160220200552 and revision id 705701793
 -->
<noscript><img src="//en.wikipedia.org/wiki/Special:CentralAutoLogin/start?type=1x1" alt="" title="" width="1" height="1" style="border: none; position: absolute;" /></noscript></div>					<div class="printfooter">
						Retrieved from "<a dir="ltr" href="https://en.wikipedia.org/w/index.php?title=Proteomics&amp;oldid=705701793">https://en.wikipedia.org/w/index.php?title=Proteomics&amp;oldid=705701793</a>"					</div>
				<div id='catlinks' class='catlinks' data-mw='interface'><div id="mw-normal-catlinks" class="mw-normal-catlinks"><a href="/wiki/Help:Category" title="Help:Category">Categories</a>: <ul><li><a href="/wiki/Category:Proteomics" title="Category:Proteomics">Proteomics</a></li><li><a href="/wiki/Category:Bioinformatics" title="Category:Bioinformatics">Bioinformatics</a></li><li><a href="/wiki/Category:Genomics" title="Category:Genomics">Genomics</a></li><li><a href="/wiki/Category:Proteins" title="Category:Proteins">Proteins</a></li></ul></div><div id="mw-hidden-catlinks" class="mw-hidden-catlinks mw-hidden-cats-hidden">Hidden categories: <ul><li><a href="/wiki/Category:All_articles_lacking_reliable_references" title="Category:All articles lacking reliable references">All articles lacking reliable references</a></li><li><a href="/wiki/Category:Articles_lacking_reliable_references_from_November_2013" title="Category:Articles lacking reliable references from November 2013">Articles lacking reliable references from November 2013</a></li><li><a href="/wiki/Category:Pages_with_DOIs_inactive_since_2015" title="Category:Pages with DOIs inactive since 2015">Pages with DOIs inactive since 2015</a></li><li><a href="/wiki/Category:All_articles_with_unsourced_statements" title="Category:All articles with unsourced statements">All articles with unsourced statements</a></li><li><a href="/wiki/Category:Articles_with_unsourced_statements_from_October_2013" title="Category:Articles with unsourced statements from October 2013">Articles with unsourced statements from October 2013</a></li><li><a href="/wiki/Category:Articles_with_unsourced_statements_from_October_2014" title="Category:Articles with unsourced statements from October 2014">Articles with unsourced statements from October 2014</a></li><li><a href="/wiki/Category:Articles_with_unsourced_statements_from_January_2012" title="Category:Articles with unsourced statements from January 2012">Articles with unsourced statements from January 2012</a></li><li><a href="/wiki/Category:CS1_errors:_missing_author_or_editor" title="Category:CS1 errors: missing author or editor">CS1 errors: missing author or editor</a></li><li><a href="/wiki/Category:Articles_with_DMOZ_links" title="Category:Articles with DMOZ links">Articles with DMOZ links</a></li></ul></div></div>				<div class="visualClear"></div>
							</div>
		</div>
		<div id="mw-navigation">
			<h2>Navigation menu</h2>

			<div id="mw-head">
									<div id="p-personal" role="navigation" class="" aria-labelledby="p-personal-label">
						<h3 id="p-personal-label">Personal tools</h3>
						<ul>
							<li id="pt-anonuserpage">Not logged in</li><li id="pt-anontalk"><a href="/wiki/Special:MyTalk" title="Discussion about edits from this IP address [n]" accesskey="n">Talk</a></li><li id="pt-anoncontribs"><a href="/wiki/Special:MyContributions" title="A list of edits made from this IP address [y]" accesskey="y">Contributions</a></li><li id="pt-createaccount"><a href="/w/index.php?title=Special:UserLogin&amp;returnto=Proteomics&amp;type=signup" title="You are encouraged to create an account and log in; however, it is not mandatory">Create account</a></li><li id="pt-login"><a href="/w/index.php?title=Special:UserLogin&amp;returnto=Proteomics" title="You're encouraged to log in; however, it's not mandatory. [o]" accesskey="o">Log in</a></li>						</ul>
					</div>
									<div id="left-navigation">
										<div id="p-namespaces" role="navigation" class="vectorTabs" aria-labelledby="p-namespaces-label">
						<h3 id="p-namespaces-label">Namespaces</h3>
						<ul>
															<li  id="ca-nstab-main" class="selected"><span><a href="/wiki/Proteomics"  title="View the content page [c]" accesskey="c">Article</a></span></li>
															<li  id="ca-talk"><span><a href="/wiki/Talk:Proteomics"  title="Discussion about the content page [t]" accesskey="t" rel="discussion">Talk</a></span></li>
													</ul>
					</div>
										<div id="p-variants" role="navigation" class="vectorMenu emptyPortlet" aria-labelledby="p-variants-label">
												<h3 id="p-variants-label">
							<span>Variants</span><a href="#"></a>
						</h3>

						<div class="menu">
							<ul>
															</ul>
						</div>
					</div>
									</div>
				<div id="right-navigation">
										<div id="p-views" role="navigation" class="vectorTabs" aria-labelledby="p-views-label">
						<h3 id="p-views-label">Views</h3>
						<ul>
															<li id="ca-view" class="selected"><span><a href="/wiki/Proteomics" >Read</a></span></li>
															<li id="ca-edit"><span><a href="/w/index.php?title=Proteomics&amp;action=edit"  title="Edit this page [e]" accesskey="e">Edit</a></span></li>
															<li id="ca-history" class="collapsible"><span><a href="/w/index.php?title=Proteomics&amp;action=history"  title="Past revisions of this page [h]" accesskey="h">View history</a></span></li>
													</ul>
					</div>
										<div id="p-cactions" role="navigation" class="vectorMenu emptyPortlet" aria-labelledby="p-cactions-label">
						<h3 id="p-cactions-label"><span>More</span><a href="#"></a></h3>

						<div class="menu">
							<ul>
															</ul>
						</div>
					</div>
										<div id="p-search" role="search">
						<h3>
							<label for="searchInput">Search</label>
						</h3>

						<form action="/w/index.php" id="searchform">
							<div id="simpleSearch">
							<input type="search" name="search" placeholder="Search" title="Search Wikipedia [f]" accesskey="f" id="searchInput" /><input type="hidden" value="Special:Search" name="title" /><input type="submit" name="fulltext" value="Search" title="Search Wikipedia for this text" id="mw-searchButton" class="searchButton mw-fallbackSearchButton" /><input type="submit" name="go" value="Go" title="Go to a page with this exact name if it exists" id="searchButton" class="searchButton" />							</div>
						</form>
					</div>
									</div>
			</div>
			<div id="mw-panel">
				<div id="p-logo" role="banner"><a class="mw-wiki-logo" href="/wiki/Main_Page"  title="Visit the main page"></a></div>
						<div class="portal" role="navigation" id='p-navigation' aria-labelledby='p-navigation-label'>
			<h3 id='p-navigation-label'>Navigation</h3>

			<div class="body">
									<ul>
						<li id="n-mainpage-description"><a href="/wiki/Main_Page" title="Visit the main page [z]" accesskey="z">Main page</a></li><li id="n-contents"><a href="/wiki/Portal:Contents" title="Guides to browsing Wikipedia">Contents</a></li><li id="n-featuredcontent"><a href="/wiki/Portal:Featured_content" title="Featured content – the best of Wikipedia">Featured content</a></li><li id="n-currentevents"><a href="/wiki/Portal:Current_events" title="Find background information on current events">Current events</a></li><li id="n-randompage"><a href="/wiki/Special:Random" title="Load a random article [x]" accesskey="x">Random article</a></li><li id="n-sitesupport"><a href="https://donate.wikimedia.org/wiki/Special:FundraiserRedirector?utm_source=donate&amp;utm_medium=sidebar&amp;utm_campaign=C13_en.wikipedia.org&amp;uselang=en" title="Support us">Donate to Wikipedia</a></li><li id="n-shoplink"><a href="//shop.wikimedia.org" title="Visit the Wikipedia store">Wikipedia store</a></li>					</ul>
							</div>
		</div>
			<div class="portal" role="navigation" id='p-interaction' aria-labelledby='p-interaction-label'>
			<h3 id='p-interaction-label'>Interaction</h3>

			<div class="body">
									<ul>
						<li id="n-help"><a href="/wiki/Help:Contents" title="Guidance on how to use and edit Wikipedia">Help</a></li><li id="n-aboutsite"><a href="/wiki/Wikipedia:About" title="Find out about Wikipedia">About Wikipedia</a></li><li id="n-portal"><a href="/wiki/Wikipedia:Community_portal" title="About the project, what you can do, where to find things">Community portal</a></li><li id="n-recentchanges"><a href="/wiki/Special:RecentChanges" title="A list of recent changes in the wiki [r]" accesskey="r">Recent changes</a></li><li id="n-contactpage"><a href="//en.wikipedia.org/wiki/Wikipedia:Contact_us" title="How to contact Wikipedia">Contact page</a></li>					</ul>
							</div>
		</div>
			<div class="portal" role="navigation" id='p-tb' aria-labelledby='p-tb-label'>
			<h3 id='p-tb-label'>Tools</h3>

			<div class="body">
									<ul>
						<li id="t-whatlinkshere"><a href="/wiki/Special:WhatLinksHere/Proteomics" title="List of all English Wikipedia pages containing links to this page [j]" accesskey="j">What links here</a></li><li id="t-recentchangeslinked"><a href="/wiki/Special:RecentChangesLinked/Proteomics" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</a></li><li id="t-upload"><a href="/wiki/Wikipedia:File_Upload_Wizard" title="Upload files [u]" accesskey="u">Upload file</a></li><li id="t-specialpages"><a href="/wiki/Special:SpecialPages" title="A list of all special pages [q]" accesskey="q">Special pages</a></li><li id="t-permalink"><a href="/w/index.php?title=Proteomics&amp;oldid=705701793" title="Permanent link to this revision of the page">Permanent link</a></li><li id="t-info"><a href="/w/index.php?title=Proteomics&amp;action=info" title="More information about this page">Page information</a></li><li id="t-wikibase"><a href="//www.wikidata.org/wiki/Q471857" title="Link to connected data repository item [g]" accesskey="g">Wikidata item</a></li><li id="t-cite"><a href="/w/index.php?title=Special:CiteThisPage&amp;page=Proteomics&amp;id=705701793" title="Information on how to cite this page">Cite this page</a></li>					</ul>
							</div>
		</div>
			<div class="portal" role="navigation" id='p-coll-print_export' aria-labelledby='p-coll-print_export-label'>
			<h3 id='p-coll-print_export-label'>Print/export</h3>

			<div class="body">
									<ul>
						<li id="coll-create_a_book"><a href="/w/index.php?title=Special:Book&amp;bookcmd=book_creator&amp;referer=Proteomics">Create a book</a></li><li id="coll-download-as-rdf2latex"><a href="/w/index.php?title=Special:Book&amp;bookcmd=render_article&amp;arttitle=Proteomics&amp;returnto=Proteomics&amp;oldid=705701793&amp;writer=rdf2latex">Download as PDF</a></li><li id="t-print"><a href="/w/index.php?title=Proteomics&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</a></li>					</ul>
							</div>
		</div>
			<div class="portal" role="navigation" id='p-wikibase-otherprojects' aria-labelledby='p-wikibase-otherprojects-label'>
			<h3 id='p-wikibase-otherprojects-label'>Other projects</h3>

			<div class="body">
									<ul>
						<li class="wb-otherproject-link wb-otherproject-commons"><a href="https://commons.wikimedia.org/wiki/Category:Proteomics" hreflang="en">Wikimedia Commons</a></li><li class="wb-otherproject-link wb-otherproject-wikibooks"><a href="https://en.wikibooks.org/wiki/Proteomics" hreflang="en">Wikibooks</a></li>					</ul>
							</div>
		</div>
			<div class="portal" role="navigation" id='p-lang' aria-labelledby='p-lang-label'>
			<h3 id='p-lang-label'>Languages</h3>

			<div class="body">
									<ul>
						<li class="interlanguage-link interwiki-ar"><a href="//ar.wikipedia.org/wiki/%D8%A8%D8%B1%D9%88%D8%AA%D9%8A%D9%88%D9%85%D9%8A%D8%A7%D8%AA" title="بروتيوميات – Arabic" lang="ar" hreflang="ar">العربية</a></li><li class="interlanguage-link interwiki-bn"><a href="//bn.wikipedia.org/wiki/%E0%A6%AA%E0%A7%8D%E0%A6%B0%E0%A7%8B%E0%A6%9F%E0%A6%BF%E0%A6%AF%E0%A6%BC%E0%A7%8B%E0%A6%AE%E0%A6%BF%E0%A6%95%E0%A7%8D%E2%80%8C%E0%A6%B8" title="প্রোটিয়োমিক্‌স – Bengali" lang="bn" hreflang="bn">বাংলা</a></li><li class="interlanguage-link interwiki-bs"><a href="//bs.wikipedia.org/wiki/Proteomika" title="Proteomika – Bosnian" lang="bs" hreflang="bs">Bosanski</a></li><li class="interlanguage-link interwiki-ca"><a href="//ca.wikipedia.org/wiki/Prote%C3%B2mica" title="Proteòmica – Catalan" lang="ca" hreflang="ca">Català</a></li><li class="interlanguage-link interwiki-cs"><a href="//cs.wikipedia.org/wiki/Proteomika" title="Proteomika – Czech" lang="cs" hreflang="cs">Čeština</a></li><li class="interlanguage-link interwiki-de"><a href="//de.wikipedia.org/wiki/Proteomik" title="Proteomik – German" lang="de" hreflang="de">Deutsch</a></li><li class="interlanguage-link interwiki-es"><a href="//es.wikipedia.org/wiki/Prote%C3%B3mica" title="Proteómica – Spanish" lang="es" hreflang="es">Español</a></li><li class="interlanguage-link interwiki-eo"><a href="//eo.wikipedia.org/wiki/Proteomiko" title="Proteomiko – Esperanto" lang="eo" hreflang="eo">Esperanto</a></li><li class="interlanguage-link interwiki-fa"><a href="//fa.wikipedia.org/wiki/%D9%BE%D8%B1%D9%88%D8%AA%D8%A6%D9%88%D9%85%DB%8C%DA%A9" title="پروتئومیک – Persian" lang="fa" hreflang="fa">فارسی</a></li><li class="interlanguage-link interwiki-fr"><a href="//fr.wikipedia.org/wiki/Prot%C3%A9omique" title="Protéomique – French" lang="fr" hreflang="fr">Français</a></li><li class="interlanguage-link interwiki-gl"><a href="//gl.wikipedia.org/wiki/Prote%C3%B3mica" title="Proteómica – Galician" lang="gl" hreflang="gl">Galego</a></li><li class="interlanguage-link interwiki-ko"><a href="//ko.wikipedia.org/wiki/%EB%8B%A8%EB%B0%B1%EC%B2%B4%ED%95%99" title="단백체학 – Korean" lang="ko" hreflang="ko">한국어</a></li><li class="interlanguage-link interwiki-hi"><a href="//hi.wikipedia.org/wiki/%E0%A4%AA%E0%A5%8D%E0%A4%B0%E0%A5%8B%E0%A4%9F%E0%A5%80%E0%A4%A8%E0%A5%8B%E0%A4%AE%E0%A4%BF%E0%A4%95%E0%A5%8D%E0%A4%B8" title="प्रोटीनोमिक्स – Hindi" lang="hi" hreflang="hi">हिन्दी</a></li><li class="interlanguage-link interwiki-hr"><a href="//hr.wikipedia.org/wiki/Proteomika" title="Proteomika – Croatian" lang="hr" hreflang="hr">Hrvatski</a></li><li class="interlanguage-link interwiki-id"><a href="//id.wikipedia.org/wiki/Proteomika" title="Proteomika – Indonesian" lang="id" hreflang="id">Bahasa Indonesia</a></li><li class="interlanguage-link interwiki-it"><a href="//it.wikipedia.org/wiki/Proteomica" title="Proteomica – Italian" lang="it" hreflang="it">Italiano</a></li><li class="interlanguage-link interwiki-he"><a href="//he.wikipedia.org/wiki/%D7%A4%D7%A8%D7%95%D7%98%D7%90%D7%95%D7%9E%D7%99%D7%A7%D7%94" title="פרוטאומיקה – Hebrew" lang="he" hreflang="he">עברית</a></li><li class="interlanguage-link interwiki-jv"><a href="//jv.wikipedia.org/wiki/Proteomika" title="Proteomika – Javanese" lang="jv" hreflang="jv">Basa Jawa</a></li><li class="interlanguage-link interwiki-lt"><a href="//lt.wikipedia.org/wiki/Proteomika" title="Proteomika – Lithuanian" lang="lt" hreflang="lt">Lietuvių</a></li><li class="interlanguage-link interwiki-nl"><a href="//nl.wikipedia.org/wiki/Proteomica" title="Proteomica – Dutch" lang="nl" hreflang="nl">Nederlands</a></li><li class="interlanguage-link interwiki-ja"><a href="//ja.wikipedia.org/wiki/%E3%83%97%E3%83%AD%E3%83%86%E3%82%AA%E3%83%BC%E3%83%A0%E8%A7%A3%E6%9E%90" title="プロテオーム解析 – Japanese" lang="ja" hreflang="ja">日本語</a></li><li class="interlanguage-link interwiki-no"><a href="//no.wikipedia.org/wiki/Proteomikk" title="Proteomikk – Norwegian" lang="no" hreflang="no">Norsk bokmål</a></li><li class="interlanguage-link interwiki-pl"><a href="//pl.wikipedia.org/wiki/Proteomika" title="Proteomika – Polish" lang="pl" hreflang="pl">Polski</a></li><li class="interlanguage-link interwiki-pt"><a href="//pt.wikipedia.org/wiki/Prote%C3%B4mica" title="Proteômica – Portuguese" lang="pt" hreflang="pt">Português</a></li><li class="interlanguage-link interwiki-ru"><a href="//ru.wikipedia.org/wiki/%D0%9F%D1%80%D0%BE%D1%82%D0%B5%D0%BE%D0%BC%D0%B8%D0%BA%D0%B0" title="Протеомика – Russian" lang="ru" hreflang="ru">Русский</a></li><li class="interlanguage-link interwiki-ckb"><a href="//ckb.wikipedia.org/wiki/%D9%BE%D8%B1%DB%86%D8%AA%DB%8C%DB%86%D9%85%DB%8C%DA%A9" title="پرۆتیۆمیک – Central Kurdish" lang="ckb" hreflang="ckb">کوردیی ناوەندی</a></li><li class="interlanguage-link interwiki-sr"><a href="//sr.wikipedia.org/wiki/Proteomika" title="Proteomika – Serbian" lang="sr" hreflang="sr">Српски / srpski</a></li><li class="interlanguage-link interwiki-sh"><a href="//sh.wikipedia.org/wiki/Proteomika" title="Proteomika – Serbo-Croatian" lang="sh" hreflang="sh">Srpskohrvatski / српскохрватски</a></li><li class="interlanguage-link interwiki-fi"><a href="//fi.wikipedia.org/wiki/Proteomiikka" title="Proteomiikka – Finnish" lang="fi" hreflang="fi">Suomi</a></li><li class="interlanguage-link interwiki-sv"><a href="//sv.wikipedia.org/wiki/Proteomik" title="Proteomik – Swedish" lang="sv" hreflang="sv">Svenska</a></li><li class="interlanguage-link interwiki-uk"><a href="//uk.wikipedia.org/wiki/%D0%9F%D1%80%D0%BE%D1%82%D0%B5%D0%BE%D0%BC%D1%96%D0%BA%D0%B0" title="Протеоміка – Ukrainian" lang="uk" hreflang="uk">Українська</a></li><li class="interlanguage-link interwiki-zh"><a href="//zh.wikipedia.org/wiki/%E8%9B%8B%E7%99%BD%E8%B4%A8%E7%BB%84%E5%AD%A6" title="蛋白质组学 – Chinese" lang="zh" hreflang="zh">中文</a></li><li class="uls-p-lang-dummy"><a href="#"></a></li>					</ul>
				<div class='after-portlet after-portlet-lang'><span class="wb-langlinks-edit wb-langlinks-link"><a href="//www.wikidata.org/wiki/Q471857#sitelinks-wikipedia" title="Edit interlanguage links" class="wbc-editpage">Edit links</a></span></div>			</div>
		</div>
				</div>
		</div>
		<div id="footer" role="contentinfo">
							<ul id="footer-info">
											<li id="footer-info-lastmod"> This page was last modified on 19 February 2016, at 01:43.</li>
											<li id="footer-info-copyright">Text is available under the <a rel="license" href="//en.wikipedia.org/wiki/Wikipedia:Text_of_Creative_Commons_Attribution-ShareAlike_3.0_Unported_License">Creative Commons Attribution-ShareAlike License</a><a rel="license" href="//creativecommons.org/licenses/by-sa/3.0/" style="display:none;"></a>;
additional terms may apply.  By using this site, you agree to the <a href="//wikimediafoundation.org/wiki/Terms_of_Use">Terms of Use</a> and <a href="//wikimediafoundation.org/wiki/Privacy_policy">Privacy Policy</a>. Wikipedia® is a registered trademark of the <a href="//www.wikimediafoundation.org/">Wikimedia Foundation, Inc.</a>, a non-profit organization.</li>
									</ul>
							<ul id="footer-places">
											<li id="footer-places-privacy"><a href="//wikimediafoundation.org/wiki/Privacy_policy" title="wmf:Privacy policy">Privacy policy</a></li>
											<li id="footer-places-about"><a href="/wiki/Wikipedia:About" title="Wikipedia:About">About Wikipedia</a></li>
											<li id="footer-places-disclaimer"><a href="/wiki/Wikipedia:General_disclaimer" title="Wikipedia:General disclaimer">Disclaimers</a></li>
											<li id="footer-places-contact"><a href="//en.wikipedia.org/wiki/Wikipedia:Contact_us">Contact Wikipedia</a></li>
											<li id="footer-places-developers"><a href="https://www.mediawiki.org/wiki/Special:MyLanguage/How_to_contribute">Developers</a></li>
											<li id="footer-places-cookiestatement"><a href="//wikimediafoundation.org/wiki/Cookie_statement">Cookie statement</a></li>
											<li id="footer-places-mobileview"><a href="//en.m.wikipedia.org/w/index.php?title=Proteomics&amp;mobileaction=toggle_view_mobile" class="noprint stopMobileRedirectToggle">Mobile view</a></li>
									</ul>
										<ul id="footer-icons" class="noprint">
											<li id="footer-copyrightico">
							<a href="//wikimediafoundation.org/"><img src="/static/images/wikimedia-button.png" srcset="/static/images/wikimedia-button-1.5x.png 1.5x, /static/images/wikimedia-button-2x.png 2x" width="88" height="31" alt="Wikimedia Foundation"/></a>						</li>
											<li id="footer-poweredbyico">
							<a href="//www.mediawiki.org/"><img src="/static/1.27.0-wmf.13/resources/assets/poweredby_mediawiki_88x31.png" alt="Powered by MediaWiki" srcset="/static/1.27.0-wmf.13/resources/assets/poweredby_mediawiki_132x47.png 1.5x, /static/1.27.0-wmf.13/resources/assets/poweredby_mediawiki_176x62.png 2x" width="88" height="31" /></a>						</li>
									</ul>
						<div style="clear:both"></div>
		</div>
		<script>(window.RLQ = window.RLQ || []).push(function () {
mw.loader.state({"ext.globalCssJs.site":"ready","ext.globalCssJs.user":"ready","user":"ready","user.groups":"ready"});mw.loader.load(["ext.cite.a11y","mediawiki.toc","mediawiki.action.view.postEdit","site","mediawiki.user","mediawiki.hidpi","mediawiki.page.ready","mediawiki.searchSuggest","ext.eventLogging.subscriber","ext.gadget.teahouse","ext.gadget.ReferenceTooltips","ext.gadget.DRN-wizard","ext.gadget.charinsert","ext.gadget.refToolbar","ext.gadget.switcher","ext.gadget.featured-articles-links","mmv.bootstrap.autostart","ext.visualEditor.targetLoader","ext.wikimediaEvents","ext.navigationTiming","schema.UniversalLanguageSelector","ext.uls.eventlogger","ext.uls.interlanguage"]);
} );</script><script>(window.RLQ = window.RLQ || []).push(function () {
mw.config.set({"wgBackendResponseTime":164,"wgHostname":"mw1090"}); /* @nomin */
} );</script>
	</body>
</html>
